<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter IV</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd/favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd/doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd/" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd/doks-theme/assets/images/prismatic.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter IV</h2>
								
								
									<h1 class="hero-subheader__title">In Blissful Ignorance of Dire Straits: The Physiological Adaptability of a Simple Genetic Circuit 
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_03" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_05" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
						<div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
							<div class="align-container" data-mh>
								<div class="align-inner">
									<div class="hero-subheader__author">
										<p class="hero-subheader__author-title">
												Summary
												<i class="icon icon--chevron-down"></i>
										</p><!-- /.hero-subheader__author-title -->
										<p>Cells adapt to changing environmental conditions by repressing or activating gene expression from enormous fractions of their genome, drastically changing the molecular composition of the cell. This requires the concerted adaptation of transcription factors to the environmental signals, leading to binding or releasing of their cognate sequences. Here, we dissect a well characterized genetic circuit in a number of physiological states, make predictions of the response, and measure how the copy number of a regulator and its gene target are affected. We find the parameters defining the regulators behavior are remarkably robust to changes in the nutrient availability, but are  susceptible to temperature changes. We quantitatively explore these two effects and discuss how they challenge current models of transcriptional regulation.
</p>
									</div><!-- /.hero-subheader__author -->
								</div><!-- /.align-inner -->
							</div><!-- /.align-container -->
						</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_03" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter III
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_05" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<strong>Published as …</strong>
<hr/>
<p>A version of this chapter is currently under review. A preprint is released as Chure, G; Kaczmarek, Z. A.; Phillips, R. <em>The Physiological Adaptability of a Simple Genetic Circuit.</em> bioRxiv 2019. G.C. and R.P. designed experiments and developed theoretical models. G.C. and Z.A.K. collected and analyzed data. G.C. and R.P. wrote the paper.</p>
<h2 class="unnumbered" id="sec:growth_abstract" class="unnumbered">Abstract</h2>
<p>The intimate relationship between the environment and cellular growth rate has remained a major topic of inquiry in bacterial physiology for over a century. Now, as it becomes possible to understand how the growth rate dictates the wholesale reorganization of the intracellular molecular composition, we can interrogate the biophysical principles underlying this adaptive response. Regulation of gene expression drives this adaptation, with changes in growth rate tied to the activation or repression of genes covering enormous swaths of the genome. Here, we dissect how physiological perturbations alter the expression of a circuit which has been extensively characterized in a single physiological state. Given a complete thermodynamic model, we map changes in physiology directly to the biophysical parameters which define the expression. Controlling the growth rate via modulating the available carbon source or growth temperature, we measure the level of gene expression from a LacI-regulated promoter where the LacI copy number is directly measured in each condition, permitting parameter-free prediction of the expression level. The transcriptional output of this circuit is remarkably robust, with expression of the repressor being largely insensitive to the growth rate. The predicted gene expression quantitatively captures the observations under different carbon conditions, indicating that the biophysical parameters are indifferent to the physiology. Interestingly, temperature controls the expression level in ways that are inconsistent with the prediction, revealing temperature-dependent effects that challenge current models. This work exposes the strengths and weaknesses of thermodynamic models in fluctuating environments, posing novel challenges and utility in studying physiological adaptation.</p>
<h2 id="introduction">Introduction</h2>
<p>Cellular physiology is inextricably tied to the extracellular environment. Fluctuations in nutrient availability and variations in temperature, for example, can drastically modulate the cell’s growth rate, which is often used as a measure of the evolutionary fitness <span class="citation" data-cites="schaecter1958">(<span class="citeproc-not-found" data-reference-id="schaecter1958"><strong>???</strong></span>)</span>. In response to such environmental insults, cells have evolved myriad clever mechanisms by which they can adapt to their changing surroundings, many of which involve restructuring their proteome such that critical processes (i.e. protein translation) are allocated the necessary resources. Recent work exploring this level of adaptation using mass spectrometry, ribosomal profiling, and RNA sequencing have revealed that various classes of genes (termed “sectors”) are tuned such that the protein mass fraction of the translational machinery is prioritized over the metabolic and catabolic machinery in nutrient replete environments <span class="citation" data-cites="scott2014 klumpp2014 hui2015 schmidt2016 li2014a">(Scott et al. <a href="#ref-scott2014" role="doc-biblioref">2014</a>; Klumpp and Hwa <a href="#ref-klumpp2014" role="doc-biblioref">2014</a>; Hui et al. <a href="#ref-hui2015" role="doc-biblioref">2015</a>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Li et al. <a href="#ref-li2014a" role="doc-biblioref">2014</a>)</span>. This drastic reorganization is mediated by the regulation of gene expression, relying on the concerted action of myriad transcription factors. Notably, each gene in isolation is regulated by only one or a few components <span class="citation" data-cites="gama-castro2016">(Gama-Castro et al. <a href="#ref-gama-castro2016" role="doc-biblioref">2016</a>)</span>. The most common regulatory architecture in <em>Escherichia coli</em> is the simple repression motif in which a transcriptional repressor binds to a single site in the promoter region, occluding binding of an RNA polymerase <span class="citation" data-cites="rydenfelt2014 phillips2019">(Rydenfelt et al. <a href="#ref-rydenfelt2014" role="doc-biblioref">2014</a>; Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span>. The simple activation architecture, in which the simultaneous binding of an activator and an RNA polymerase amplifies gene expression, is another common mode of regulation. Combinatorial regulation such as dual repression, dual activation, or combined activation and repression can also be found throughout the genome, albeit with lower frequency<span class="citation" data-cites="phillips2019">(Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span>. The ubiquity of the simple repression and simple activation motifs illustrate that, for many genes, the complex systems-level response to a physiological perturbation boils down the binding and unbinding of a single regulator to its cognate binding sites.</p>
<p>Despite our knowledge of these modes of regulation, there remains a large disconnect between concrete, physical models of their behavior and experimental validation. The simple repression motif is perhaps the most thoroughly explored theoretically and experimentally <span class="citation" data-cites="phillips2019">(Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span> where equilibrium thermodynamic <span class="citation" data-cites="garcia2011 garcia2012 brewster2014 razo-mejia2018 barnes2019">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Garcia et al. <a href="#ref-garcia2012" role="doc-biblioref">2012</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Barnes et al. <a href="#ref-barnes2019" role="doc-biblioref">2019</a>)</span> and kinetic <span class="citation" data-cites="jones2014 ko1991 kepler2001 michel2010">(Jones, Brewster, and Phillips <a href="#ref-jones2014" role="doc-biblioref">2014</a>; Ko <a href="#ref-ko1991" role="doc-biblioref">1991</a>; Kepler and Elston <a href="#ref-kepler2001" role="doc-biblioref">2001</a>; Michel <a href="#ref-michel2010" role="doc-biblioref">2010</a>)</span> models have been shown to accurately predict the level of gene expression in a variety of contexts. While these experiments involved variations of repressor copy number, operator sequence, concentration of an external inducer, and amino acid substitutions, none have explored how the physiological state of the cell as governed by external factors influences gene expression. This is arguably one of the most critical variables one can experimentally tune to understand the role of these regulatory architectures play in cellular physiology writ large.</p>
<p>In this work, we interrogate the adaptability of a simple genetic circuit to various physiological stressors, namely carbon source quality and growth temperature. Following the aforementioned thermodynamic models, we build upon this theory-experiment dialogue by using environmental conditions as an experimentally tunable variable and determine their influence on various biophysical parameters. Specifically, we use physiological stressors to tune the growth rate. One mechanism by which we modulate the growth rate is by exchanging glucose in the growth medium for the poorer carbon sources glycerol and acetate, which decrease the growth rate by a factor of <span class="math inline">≈</span> 1.5 and <span class="math inline">≈</span> 4 compared to glucose, respectively. We hypothesize that different carbon sources should, if anything, only modulate the repressor copy number seeing as the relationship between growth rate and total protein content has been rigorously quantified <span class="citation" data-cites="schaecter1958 schmidt2016 li2014a jun2018">(<span class="citeproc-not-found" data-reference-id="schaecter1958"><strong>???</strong></span>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Li et al. <a href="#ref-li2014a" role="doc-biblioref">2014</a>; Jun et al. <a href="#ref-jun2018" role="doc-biblioref">2018</a>)</span>. Using single-cell time-lapse fluorescence microscopy, we directly measure the copy number of the repressor in each condition. Under a simple hypothesis, all other parameters should be unperturbed, and we can thus rely on previously determined values to make parameter-free predictions of the fold-change in gene expression.</p>
<p>Despite the decrease in growth rate, both the fold-change in gene expression and the repressor copy number remains largely unaffected. We confirm this is the case by examining how the effective free energy of the system changes between carbon sources, a method we have used previously to elucidate parametric changes due to mutations within a transcription factor <span class="citation" data-cites="chure2019">(Chure et al. <a href="#ref-chure2019" role="doc-biblioref">2019</a>)</span> and has been extensively discussed in Chapter 3. This illustrates that the energetic parameters defining the fraction of active repressors and their affinity for the DNA are ignorant of the carbon-dependent physiological states of the cell. Thus, in this context, the values of the biophysical parameters determined in one condition can be used to draw predictions in others.</p>
<p>We then examine how variations in temperature influence the transcriptional output. Unlike in the case of carbon source variation, temperature dependence is explicit in our model: the repressor-DNA binding energy and the energetic difference between the active and inactive states of the repressor are scaled to the thermal energy of the system at 37<span class="math inline"><em></em><sup>∘</sup></span> C. This is defined via the Boltzmann distribution which states that the probability of a state <span class="math inline"><em>p</em><sub><em>s</em><em>t</em><em>a</em><em>t</em><em>e</em></sub></span> is related to the energy of that state <span class="math inline"><em>ε</em><sub><em>s</em><em>t</em><em>a</em><em>t</em><em>e</em></sub></span> as</p>
<p><span id="eq:boltz"><br /><span class="math display"><em>p</em><sub><em>s</em><em>t</em><em>a</em><em>t</em><em>e</em></sub> ∝ <em>e</em><sup> − <em>ε</em><sub><em>s</em><em>t</em><em>a</em><em>t</em><em>e</em></sub>/<em>k</em><sub><em>B</em></sub><em>T</em></sup>,   (1)</span><br /></span></p>
<p>where <span class="math inline"><em>k</em><sub><em>B</em></sub></span> is the Boltzmann constant and <span class="math inline"><em>T</em></span> is the temperature of the system. Given knowledge of <span class="math inline"><em>T</em></span> for a particular experiment, we can easily draw predictions of the fold-change in gene expression. However, we find the fold-change in gene expression is inconsistent with this simple model, revealing an incomplete description of the energetics. We then examine how entropic effects neglected in the initial estimation of the energetic parameters may play an important role; a hypothesis that is supported when we examine the change in the effective free energy.</p>
<p>The results presented here are, to our knowledge, the first attempts to systematically characterize the growth-dependent effects on biophysical parameters in thermodynamic models of transcription. While some parameters of our model are affected by changing the growth rate, they change in ways that are expected or fall close within our <em>a priori</em> predictions, suggesting that such modeling can still be powerful in understanding how adaptive processes influence physiology at the level of molecular interactions.</p>
<h2 id="results">Results</h2>
<h3 id="thermodynamic-model">Thermodynamic model</h3>
<p>This chapter builds off the theoretical details presented in Chapters 2-3 of this thesis. Here, we once again consider the simple repression motif in which expression of a target gene is regulated by the action of a single allosteric repressor. The key measurable quantity of the following work is the fold-change in expression of the regulated gene. Thermodynamic models described previously <span class="citation" data-cites="garcia2011 razo-mejia2018 phillips2019">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span> and in Chapters 2-3 of this work result in a succinct input-output function to quantitatively describe the fold-change in gene expression and is of the form</p>
<p><span id="eq:foldchange_simple"><br /><span class="math display">$$
\text{fold-change} = \left(1 + p_{act}(c)\frac{R}{N_{NS}}e^{-\Delta\varepsilon_R
/ k_BT}\right)^{-1},
\qquad(2)$$</span><br /></span></p>
<p>where <span class="math inline"><em>R</em></span> is the total number of allosteric repressors per cell, <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of nonspecific binding sites for the repressor, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> is the repressor-DNA binding energy, and <span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span> is the thermal energy of the system. The prefactor <span class="math inline"><em>p</em><sub><em>a</em><em>c</em><em>t</em></sub>(<em>c</em>)</span> defines the probability of the repressor being in the active state at a given concentration of inducer <span class="math inline"><em>c</em></span>. In the absence of inducer, <span class="math inline"><em>p</em><sub><em>a</em><em>c</em><em>t</em></sub>(<em>c</em> = 0)</span> can be written as</p>
<p><span id="eq:growth_pact"><br /><span class="math display"><em>p</em><sub><em>a</em><em>c</em><em>t</em></sub>(<em>c</em> = 0) = (1+<em>e</em><sup> − <em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub>/<em>k</em><sub><em>B</em></sub><em>T</em></sup>)<sup> − 1</sup>,   (3)</span><br /></span></p>
<p>where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the energy difference between the active and inactive states. Conditioned on only a handful of experimentally accessible parameters, this model has been verified using the well-characterized LacI repressor of <em>Escherichia coli</em> where parameters such as the repressor copy number and DNA binding affinity <span class="citation" data-cites="garcia2011">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>, copy number of the regulated promoter <span class="citation" data-cites="brewster2014">(Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>, and the concentration of an extracellular inducer <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>, Chapter 2)</span> can be tuned over orders of magnitude. Chapter 3 and the associated publication <span class="citation" data-cites="chure2019">(Chure et al. <a href="#ref-chure2019" role="doc-biblioref">2019</a>)</span> illustrated that this model permits the mapping of mutations within the repressor protein directly to biophysical parameters in a manner that permits accurate prediction of double mutant phenotypes . All of these applications, however, have been performed in a single physiological state where cells are grown in a glucose-supplemented minimal medium held at 37<span class="math inline"><em></em><sup>∘</sup></span> C with aeration. In this work, we challenge this model by changing the environmental conditions away from this gold-standard condition, perturbing the physiological state of the cell.</p>
<h3 id="experimental-setup">Experimental Setup</h3>
<p>Seminal studies from the burgeoning years of bacterial physiology have demonstrated a strong dependence of the total cellular protein content on the growth rate <span class="citation" data-cites="schaechter1958">(Schaechter, Maaløe, and Kjeldgaard <a href="#ref-schaechter1958" role="doc-biblioref">1958</a>)</span>, a relationship which has been rigorously quantified in recent years using mass spectrometry <span class="citation" data-cites="schmidt2016">(Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>)</span> and ribosomal profiling <span class="citation" data-cites="li2014a">(Li et al. <a href="#ref-li2014a" role="doc-biblioref">2014</a>)</span>. Their combined results illustrate that modulation of the growth rate, either through controlling the available carbon source or the temperature of the growth medium, significantly alters the physiological state of the cell, triggering the reallocation of resources to prioritize expression of ribosome-associated genes. Eq. 2 has no explicit dependence on the available carbon source but does depend on the temperature through the energetic parameters <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> which are defined relative to the thermal energy, <span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span>. With this parametric knowledge, we are able to draw quantitative predictions of the fold-change in gene expression in these physiologically distinct states.</p>
<p>We modulated growth of <em>Escherichia coli</em> by varying either the quality of the available carbon source (differing ATP yield per C atom) or the temperature of the growth medium Fig. 1 (A). All experiments were performed in a defined M9 minimal medium supplemented with one of three carbon sources – glucose, glycerol, or acetate – at concentrations such that the total number of carbon atoms available to the culture remained the same. These carbon sources have been shown to drastically alter growth rate and gene expression profiles , indicating changes in the proteomic composition and distinct physiological states. These carbon sources yield an approximate four-fold modulation of the growth rate with doubling times ranging from <span class="math inline">≈</span> 220 minutes to <span class="math inline">≈</span> 65 minutes in an acetate or glucose supplemented medium, respectively Fig. 1 (B) and (C). While the growth temperature was varied over 10<span class="math inline"><em></em><sup>∘</sup></span> C, both 32<span class="math inline"><em></em><sup>∘</sup></span> and 42<span class="math inline"><em></em><sup>∘</sup></span> C result in approximately the same doubling time of <span class="math inline">≈</span> 90 min, which is 1.5 times slower than the optimal temperature of 37<span class="math inline"><em></em><sup>∘</sup></span> C [Fig. 1 (B) and (C)].</p>
<figure>
<img src="ch4_fig1.png" id="fig:growth_control" data-short-caption="Controlling physiological state of *E. coli* via growth rate modulation by environmental factors." alt="" /><figcaption>Figure 1: <strong>Control of physiological state via growth rate through environmental factors.</strong> (A) Bacterial growth can be controlled by varying the available carbon source (top panel) or temperature (bottom panel). (B) Bulk bacterial growth curves under all conditions illustrated in (A). The y-axis is the optical density measurements at 600 nm relative to the initial value. Interval between points is <span class="math inline">≈</span> 6 min. Points and errors represent the mean and standard deviation of three to eight biological replicates. (C) Inferred maximum growth rate of each biological replicate for each condition. Points represent the doubling time computed from the maximum growth rate. Error bars correspond to the standard deviation of the inferred growth rate. Where not visible, error bars are smaller than the marker.</figcaption>
</figure>
<p>The growth rate dependence of the proteome composition suggests that changing physiological conditions could change the total repressor copy number of the cell. As shown in our previous work , it can be difficult to differentiate between a change in repressor copy number <span class="math inline"><em>R</em></span> and the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> as there are many combinations of parameter values that yield the same fold-change. To combat this degeneracy, we used a genetically engineered strain of <em>E. coli</em> in which the expression of the repressor copy number and its regulated gene product (YFP) can be simultaneously measured. This strain, used previously to interrogate the transcription factor titration effect <span class="citation" data-cites="brewster2014">(Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>, is diagrammed in Fig. 2 (A). A dimeric form of the LacI repressor N-terminally tagged with an mCherry fluorophore is itself regulated through the action of the TetR repressor whose level of activity can be modulated through the addition of the allosteric effector anhydrous tetracycline (ATC). This dual repression genetic circuit allows for the expression of the LacI repressor to be tuned over several orders of magnitude. This is demonstrated in Fig. Fig. 2 (B) where a titration of ATC in the growth medium results in a steady increase in the expression of the LacI-mCherry gene product (red lines and points) which in turn represses expression of the YFP reporter (yellow lines and points).</p>
<p>While the mCherry fluorescence is proportional to the repressor copy number, it is not a direct measurement as the fluorescence of a single LacI-mCherry dimer is unknown <em>a priori</em>. Using video microscopy, we measure the partitioning statistics of the fluorescence intensity into two sibling cells after division Fig. 2 (C). This method, described in detail in the Materials &amp; Methods as well as in <span class="citation" data-cites="rosenfeld2002">Rosenfeld, Elowitz, and Alon (<a href="#ref-rosenfeld2002" role="doc-biblioref">2002</a>)</span>, <span class="citation" data-cites="rosenfeld2005">Rosenfeld et al. (<a href="#ref-rosenfeld2005" role="doc-biblioref">2005</a>)</span>, and <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> reveals a linear relationship between the variance in intensity between two sibling cells and the intensity of the parent cell, the slope of which is equal to the brightness of a single LacI repressor. Since this measurement is performed simultaneously with measurement of the expression of the YFP reporter, this calibration factor was determined for each unique experimental replicate. We direct the reader to the supplemental Chapter 8 for a more thorough discussion of this inference.</p>
<figure>
<img src="ch4_fig2.png" id="fig:dilution_circuit" data-short-caption="Control and quantification of repressor copy number through the binomial partitioning method." alt="" /><figcaption>Figure 2: <strong>Control and quantification of repressor copy number. (A) The dual repression expression system.</strong> The inducible repressor TetR (purple blob) is expressed from a low-copy-number plasmid in the cell and represses expression of the LacI-mCherry repressor by binding to its cognate operator (tetO). In the presence of anhydrous tetracycline (ATC, green sphere), the inactive state of TetR becomes energetically favorable, permitting expression of the LacI-mCherry construct (red). This in turn binds to the lacO operator sequence repressing the expression of the reporter Yellow Fluorescent Protein (YFP, yellow lightbulb). (B) An ATC titration curve showing anticorrelated YFP (yellow) and mCherry (red) intensities. Reported values are scaled to the maximum mean fluorescence for each channel. Points and errors correspond to the mean and standard error of eight biological replicates. (C) Determination of a fluorescence calibration factor. After cessation of LacI-mCherry expression, cells are allowed to divide, partitioning the fluorescently tagged LacI repressors into the two daughter cells (left panel). The total intensity of the parent cell is equivalent to the summed intensities of the daughters. The squared fluctuations in intensity of the two sibling cells is linearly related to the parent cell with a slope <span class="math inline"><em>α</em></span>, which is the fluorescence signal measured per partitioned repressor (right panel). Black points represent single divisions and red points are the means of 50 division events. Line corresponds to linear fit to the black points with a slope of <span class="math inline"><em>α</em> = 740 ± 40</span> a.u. per LacI.</figcaption>
</figure>
<h2 class="unnumbered" id="references" class="unnumbered">References</h2>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-barnes2019">
<p>Barnes, Stephanie L., Nathan M. Belliveau, William T. Ireland, Justin B. Kinney, and Rob Phillips. 2019. “Mapping DNA Sequence to Transcription Factor Binding Energy in Vivo.” Edited by Gary D. Stormo. <em>PLOS Computational Biology</em> 15 (2): e1006226. <a href="https://doi.org/10.1371/journal.pcbi.1006226">https://doi.org/10.1371/journal.pcbi.1006226</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-chure2019">
<p>Chure, Griffin, Manuel Razo-Mejia, Nathan M Belliveau, Tal Einav, Zofii A Kaczmarek, Stephanie L Barnes, and Rob Phillips. 2019. “Predictive Shifts in Free Energy Couple Mutations to Their Phenotypic Consequences.” <em>Proceedings of the National Academy of Sciences</em> 116 (37). <a href="https://doi.org/10.1073/pnas.1907869116">https://doi.org/10.1073/pnas.1907869116</a>.</p>
</div>
<div id="ref-gama-castro2016">
<p>Gama-Castro, Socorro, Heladia Salgado, Alberto Santos-Zavaleta, Daniela Ledezma-Tejeida, Luis Muñiz-Rascado, Jair Santiago García-Sotelo, Kevin Alquicira-Hernández, et al. 2016. “RegulonDB Version 9.0: High-Level Integration of Gene Regulation, Coexpression, Motif Clustering and Beyond.” <em>Nucleic Acids Research</em> 44 (D1): D133–D143. <a href="https://doi.org/10.1093/nar/gkv1156">https://doi.org/10.1093/nar/gkv1156</a>.</p>
</div>
<div id="ref-garcia2012">
<p>Garcia, Hernan G., Alvaro Sanchez, James Q. Boedicker, Melisa Osborne, Jeff Gelles, Jane Kondev, and Rob Phillips. 2012. “Operator Sequence Alters Gene Expression Independently of Transcription Factor Occupancy in Bacteria.” <em>Cell Reports</em> 2 (1): 150–61. <a href="https://doi.org/10.1016/j.celrep.2012.06.004">https://doi.org/10.1016/j.celrep.2012.06.004</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-hui2015">
<p>Hui, Sheng, Josh M. Silverman, Stephen S. Chen, David W. Erickson, Markus Basan, Jilong Wang, Terence Hwa, and James R. Williamson. 2015. “Quantitative Proteomic Analysis Reveals a Simple Strategy of Global Resource Allocation in Bacteria.” <em>Molecular Systems Biology</em> 11 (2). <a href="https://doi.org/10.15252/msb.20145697">https://doi.org/10.15252/msb.20145697</a>.</p>
</div>
<div id="ref-jones2014">
<p>Jones, Daniel L., Robert C. Brewster, and Rob Phillips. 2014. “Promoter Architecture Dictates Cell-to-Cell Variability in Gene Expression.” <em>Science</em> 346 (6216): 1533–6. <a href="https://doi.org/10.1126/science.1255301">https://doi.org/10.1126/science.1255301</a>.</p>
</div>
<div id="ref-jun2018">
<p>Jun, Suckjoon, Fangwei Si, Rami Pugatch, and Matthew Scott. 2018. “Fundamental Principles in Bacterial PhysiologyHistory, Recent Progress, and the Future with Focus on Cell Size Control: A Review.” <em>Reports on Progress in Physics</em> 81 (5): 056601. <a href="https://doi.org/10.1088/1361-6633/aaa628">https://doi.org/10.1088/1361-6633/aaa628</a>.</p>
</div>
<div id="ref-kepler2001">
<p>Kepler, Thomas B., and Timothy C. Elston. 2001. “Stochasticity in Transcriptional Regulation: Origins, Consequences, and Mathematical Representations.” <em>Biophysical Journal</em> 81 (6): 3116–36. <a href="https://doi.org/10.1016/S0006-3495(01)75949-8">https://doi.org/10.1016/S0006-3495(01)75949-8</a>.</p>
</div>
<div id="ref-klumpp2014">
<p>Klumpp, Stefan, and Terence Hwa. 2014. “Bacterial Growth: Global Effects on Gene Expression, Growth Feedback and Proteome Partition.” <em>Current Opinion in Biotechnology</em> 28 (August): 96–102. <a href="https://doi.org/10.1016/j.copbio.2014.01.001">https://doi.org/10.1016/j.copbio.2014.01.001</a>.</p>
</div>
<div id="ref-ko1991">
<p>Ko, Minoru S. H. 1991. “A Stochastic Model for Gene Induction.” <em>Journal of Theoretical Biology</em> 153 (2): 181–94. <a href="https://doi.org/10.1016/S0022-5193(05)80421-7">https://doi.org/10.1016/S0022-5193(05)80421-7</a>.</p>
</div>
<div id="ref-li2014a">
<p>Li, Gene-Wei, David Burkhardt, Carol Gross, and Jonathan S. Weissman. 2014. “Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources.” <em>Cell</em> 157 (3): 624–35. <a href="https://doi.org/10.1016/j.cell.2014.02.033">https://doi.org/10.1016/j.cell.2014.02.033</a>.</p>
</div>
<div id="ref-michel2010">
<p>Michel, Denis. 2010. “How Transcription Factors Can Adjust the Gene Expression Floodgates.” <em>Progress in Biophysics and Molecular Biology</em> 102 (1): 16–37. <a href="https://doi.org/10.1016/j.pbiomolbio.2009.12.007">https://doi.org/10.1016/j.pbiomolbio.2009.12.007</a>.</p>
</div>
<div id="ref-phillips2019">
<p>Phillips, Rob, Nathan M. Belliveau, Griffin Chure, Hernan G. Garcia, Manuel Razo-Mejia, and Clarissa Scholes. 2019. “Figure 1 Theory Meets Figure 2 Experiments in the Study of Gene Expression.” <em>Annual Review of Biophysics</em> 48 (1): 121–63. <a href="https://doi.org/10.1146/annurev-biophys-052118-115525">https://doi.org/10.1146/annurev-biophys-052118-115525</a>.</p>
</div>
<div id="ref-razo-mejia2018">
<p>Razo-Mejia, Manuel, Stephanie L. Barnes, Nathan M. Belliveau, Griffin Chure, Tal Einav, Mitchell Lewis, and Rob Phillips. 2018. “Tuning Transcriptional Regulation Through Signaling: A Predictive Theory of Allosteric Induction.” <em>Cell Systems</em> 6 (4): 456–469.e10. <a href="https://doi.org/10.1016/j.cels.2018.02.004">https://doi.org/10.1016/j.cels.2018.02.004</a>.</p>
</div>
<div id="ref-rosenfeld2002">
<p>Rosenfeld, N., M. B. Elowitz, and U. Alon. 2002. “Negative Autoregulation Speeds the Response Times of Transcription Networks.” <em>J Mol Biol</em> 323 (November): 785–93. <a href="https://doi.org/S0022283602009944\\\%0020%5Bpii%5D">https://doi.org/S0022283602009944\\\%0020[pii]</a>.</p>
</div>
<div id="ref-rosenfeld2005">
<p>Rosenfeld, Nitzan, Jonathan W. Young, Uri Alon, Peter S. Swain, and Michael B. Elowitz. 2005. “Gene Regulation at the Single-Cell Level.” <em>Science</em> 307 (5717): 1962–5. <a href="https://doi.org/10.1126/science.1106914">https://doi.org/10.1126/science.1106914</a>.</p>
</div>
<div id="ref-rydenfelt2014">
<p>Rydenfelt, Mattias, Hernan G. Garcia, Robert Sidney Cox, and Rob Phillips. 2014. “The Influence of Promoter Architectures and Regulatory Motifs on Gene Expression in Escherichia Coli.” Edited by Jordi Garcia-Ojalvo. <em>PLoS ONE</em> 9 (12): e114347. <a href="https://doi.org/10.1371/journal.pone.0114347">https://doi.org/10.1371/journal.pone.0114347</a>.</p>
</div>
<div id="ref-schaechter1958">
<p>Schaechter, M., O. Maaløe, and N. O. Kjeldgaard. 1958. “Dependency on Medium and Temperature of Cell Size and Chemical Composition During Balanced Growth of Salmonella Typhimurium.” <em>Microbiology</em> 19 (3): 592–606. <a href="https://doi.org/10.1099/00221287-19-3-592">https://doi.org/10.1099/00221287-19-3-592</a>.</p>
</div>
<div id="ref-schmidt2016">
<p>Schmidt, Alexander, Karl Kochanowski, Silke Vedelaar, Erik Ahrné, Benjamin Volkmer, Luciano Callipo, Kèvin Knoops, Manuel Bauer, Ruedi Aebersold, and Matthias Heinemann. 2016. “The Quantitative and Condition-Dependent Escherichia Coli Proteome.” <em>Nature Biotechnology</em> 34 (1): 104–10. <a href="https://doi.org/10.1038/nbt.3418">https://doi.org/10.1038/nbt.3418</a>.</p>
</div>
<div id="ref-scott2014">
<p>Scott, M., S. Klumpp, E. M. Mateescu, and T. Hwa. 2014. “Emergence of Robust Growth Laws from Optimal Regulation of Ribosome Synthesis.” <em>Molecular Systems Biology</em> 10 (8): 747–47. <a href="https://doi.org/10.15252/msb.20145379">https://doi.org/10.15252/msb.20145379</a>.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure_phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_03" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter III
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_05" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd/" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/scripts.min.js"></script>




		</div><!-- /.js-footer-area -->
	</body>
</html>
