<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter IX</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd/favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd/doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd/" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd/doks-theme/assets/images/waves.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter IX</h2>
								
								
									<h1 class="hero-subheader__title">Supplemental Information For Chapter V: How Bacteria Survive Osmotic Shocks
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/published_content" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_02" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/published_content" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Published Content
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_02" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<strong>Published as …</strong>
<hr/>
<p>A version of this chapter was published as Chure, G.* , Lee, H.J.* , Rasmussen, A., and Phillips, R. (2018). <em>Connecting the Dots between Mechanosensitive Channel Abundance, Osmotic Shock, and Survival at Single-Cell Resolution.</em> Journal of Bacteriology 200. (* contributed equally). G.C., H.J.L, and R.P. designed and planned experiments. G.C. and H.J.L performed experiments. H.J.L constructed bacterial strains. A.R. performed electrophysiology experiments. G.C. performed data analysis and figure generation. G.C. and R.P. wrote the manuscript.</p>
<h2 id="experimental-validation-of-mscl-sfgfp">Experimental validation of MscL-sfGFP</h2>
<p>Despite revolutionizing modern cell biology, tagging proteins with fluorophores can lead to myriad deleterious effects such as mislocalization, abrogation of functionality, or even cytotoxicity. In this section, we examine the stability and functionality of the MscL-sfGFP construct used in this work.</p>
<h3 id="comparing-functionality-of-wild-type-and-fluorescently-tagged-mscl">Comparing functionality of wild-type and fluorescently tagged MscL</h3>
<p>     To quantitatively compare the functionality between the wild-type MscL and MscL-sfGFP, patch-clamp electrophysiology experiments were conducted on each channel. Patch-clamp recordings were performed on membrane patches derived from giant protoplasts which were prepared as previously described <span class="citation" data-cites="blount1999">(Blount et al. <a href="#ref-blount1999" role="doc-biblioref">1999</a>)</span>. In brief, cells were grown in Luria-Bertani (LB) medium with 0.06 mg/ml cephalexin for 2.5 hours. The elongated cells were then collected by centrifugation and digested by 0.2 mg/ml lysozyme to form giant protoplasts.</p>
<p>     Excised, inside-out patches were analyzed at a membrane potential of -20 mV with pipette and bath solutions containing 200 mM KCl, 90 mM MgCl<span class="math inline"><em></em><sub>2</sub></span>, 10 mM CaCl<span class="math inline"><em></em><sub>2</sub></span>, and 5 mM HEPES buffer at pH 7. All data were acquired at a sampling rate of 50 kHz with 5 kHz filtration using an AxoPatch 200B amplifier and pClamp software (Molecular Devices). The pressure threshold for activation a single MscS channel (blue stripe in Fig. 1) was compared to that of single MscL channels (yellow strip in Fig. 1). The pressure threshold for activation of the MscL channels was referenced against the activation threshold of MscS to determine the pressure ratio (PL:PS) for gating as previously described <span class="citation" data-cites="blount1996">(Blount et al. <a href="#ref-blount1996" role="doc-biblioref">1996</a>)</span>. Recordings of the transmembrane current were made of three individual patches with an average PL:PS ratio of 1.56 for MscL-sfGFP. This ratio quantitatively agrees with the PL:PS ratio of 1.54 measured in a strain (MJF429 from the Booth laboratory) which expresses the wild-type MscL protein from the chromosome. The average transient current change from MscL openings (Fig. S1 shaded yellow region ) is 75 pA, corresponding to a single channel conductance of 3.7 nS, comparable to the reported values of wild-type MscL. The agreement between these two strains indicates that there is negligible difference in functionality between MscL and MscL-sfGFP, allowing us to make physiological conclusions of the wild-type channel from our experiments.</p>
<figure>
<img src="ch9_figS1.png" id="fig:ephys" data-short-caption="Characteristic MscL-sfGFP conductance obtained through patch-clamp electrophysiology" alt="" /><figcaption>Figure 1: <strong>Characteristic MscL-sfGFP conductance obtained through patch-clamp electrophysiology</strong>. Top panel presents a characteristic measurement of channel current obtained through a patch-clamp electrophysiology measurement of bacterial protoplasts. The bottom panel shows the applied pressure through the micropipette to facilitate opening of the mechanosensitive channels. The blue shaded region indicates opening of the mechanosensitive channel of small conductance (MscS). The shaded yellow region represents opening of single MscL channels. These regions were used to compute the PL:PS ratio.</figcaption>
</figure>
<h3 id="maturation-time-of-mscl-sfgfp">Maturation time of MscL-sfGFP</h3>
<p>     Reliable quantification of the channel copy number is paramount to this work. As such, it is important to verify that the detected fluorescence per cell accurately represents the total cellular MscL copy number. We have made the assumption that the total fluorescence per represents all MscL-sfGFP channels present. However, it is possible that there are more channels present per cell but are not detected as the fluorophores have not properly matured. This potential error becomes more significant with longer maturation times of the fluorophore as the mean expression level changes with the growth phase of the culture. With a maturation time much longer than the typical cell division time, it is possible that the measured channel copy number represents only a fraction of the total number inherited over generations.</p>
<p>     In our earlier work, we quantified the MscL-sfGFP channel copy number using fluorescence microscopy as well as with quantitative Western blotting. We found that these two methods agreed within 20% of the mean value, often with the counts resulting from microscopy being slightly larger than those measured through Western blotting <span class="citation" data-cites="bialecka-fornal2012">(Bialecka-Fornal et al. <a href="#ref-bialecka-fornal2012" role="doc-biblioref">2012</a>)</span>. This strongly suggests that a negligible amount of channels are not observed due to inactive fluorophores.</p>
<p>    Despite these suggestive data, we directly measured the maturation time of the superfolder GFP protein. We constructed a chromosomal integration of sfGFP expressed from a promoter under regulation from plasmid-borne TetR (<em>E. coli</em> MG1655 K12 <em><span class="math inline"><em>Δ</em></span>lacIZYA ybcN::sfGFP</em>) . These cells were allowed to grow in LB supplemented with 500 mM NaCl held at 37<span class="math inline"><em></em><sup>∘</sup></span>C to an OD<span class="math inline"><em></em><sub>600nm</sub></span> of approximately 0.3. At this time, transcription and translation of the sfGFP gene was induced by addition of 10 ng/mL of anhydrous tetracycline. This expression was allowed to occur for three minutes before the addition of 100 <span class="math inline"><em>μ</em></span>g/mL of kanamycin, ceasing proper protein synthesis. Three minutes of expression was chosen to provide enough time for transcription and translation. The sfGFP variant used in this work is 1155 base pairs. We can assume that the rate for transcription is 42 nucleotides per second (BNID 108488)<span class="citation" data-cites="milo2010">(Milo et al. <a href="#ref-milo2010" role="doc-biblioref">2010</a>)</span>, meaning approximately 28 seconds are needed to transcribe the gene. The translation rate is on the order of 10 amino acids per second, (12 - 42 amino acids / s, BNID 100059)<span class="citation" data-cites="milo2010">(Milo et al. <a href="#ref-milo2010" role="doc-biblioref">2010</a>)</span>. This means that 39 seconds are needed to complete translation. In total, approximately one minute is needed to complete expression of the genes. These numbers are not known for LB supplemented with 500 mM NaCl but may be reduced. For this reason, we extended the length of induction to three minutes before translation was ceased.</p>
<p>The excess anhydrous tetracycline was removed from the culture through centrifugation and washing with one volume of LB supplemented with 500 mM NaCl and 100 <span class="math inline"><em>μ</em></span>g/mL kanamycin at 37<span class="math inline"><em></em><sup>∘</sup></span>C. The maturation of sfGFP was then monitored through flow cytometry by measuring the mean expression of 100,000 cells every 60 to 90 seconds. The result of these measurements are shown in Fig. 2.</p>
<p>    We observe complete maturation of the protein within 20 minutes after translation of the sfGFP gene was ceased. While the growth rate in LB + 500mM NaCl varies depending on the expression of MscL-sfGFP, we typically observe doubling times between 30 and 40 minutes, as indicated by a yellow stripe in Fig. 2A. To examine the ``best case” scenario for cell growth in this medium, we measured the growth rate of the same <em>E. coli</em> strain used to measure the fluorophore maturation time (Fig. 2 B). We observed a doubling time of 35 <span class="math inline">±</span> 1 min, which falls in the middle of the yellow stripe shown in Fig. 2 A. These data, coupled with our previous quantification of MscL copy number using independent methods, suggests that the fluorescence measurements made in this work reflect the total amount of MscL protein expressed.</p>
<figure>
<img src="ch9_figS2.png" id="fig:maturation_time" data-short-caption="Measurement of sfGFP maturation as a function of time through flow cytometry." alt="" /><figcaption>Figure 2: <strong>Measurement of sfGFP maturation as a function of time through flow cytometry.</strong> (A) Measurement of sfGFP fluorescence intensity as a function of time after cessation of protein translation. Points and connected lines indicate means of gated flow cytometry intensity distributions. Yellow stripe indicates the range of doubling times observed for the various RBS mutant strains described in this work (B) Growth curve of <em>E. coli</em> MG1655 cells in LB + 500mM NaCl. Red points indicate individual absorbance measurements. Line of best fit is shown in black with the uncertainty shown in shaded gray. The measured doubling time was 35 <span class="math inline">±</span> 1 min.</figcaption>
</figure>
<h2 id="calibration-of-a-standard-candle">Calibration of a Standard Candle</h2>
<p>    To estimate the single-cell MscL abundance via microscopy, we needed to determine a calibration factor that could translate arbitrary fluorescence units to protein copy number. To compute this calibration factor, we relied on <em>a priori</em> knowledge of the mean copy number of MscL-sfGFP for a particular bacterial strain in specific growth conditions. In Bialecka-Fornal et al. 2012 <span class="citation" data-cites="bialecka-fornal2012">(Bialecka-Fornal et al. <a href="#ref-bialecka-fornal2012" role="doc-biblioref">2012</a>)</span>, the average MscL copy number for a population of cells expressing an MscL-sfGFP fusion (<em>E. coli</em> K-12 MG1655 <span class="math inline"><em>ϕ</em></span>(<em>mscL-sfGFP</em>)) cells was measured using quantitative Western blotting and single-molecule photobleaching assays. By growing this strain in identical growth and imaging conditions, we can make an approximate measure of this calibration factor. In this section, we derive a statistical model for estimating the most-likely value of this calibration factor and its associated error.</p>
<h3 id="definition-of-a-calibration-factor">Definition of a calibration factor</h3>
<p>    We assume that all detected fluorescence signal from a particular cell is derived from the MscL-sfGFP protein, after background subtraction and correction for autofluorescence. The arbitrary units of fluorescence can be directly related to the protein copy number via a calibration factor <span class="math inline"><em>α</em></span>, <span id="eq:mscl_ian"><br /><span class="math display"><em>I</em><sub>tot</sub> = <em>α</em><em>N</em><sub>tot</sub>,   (1)</span><br /></span> where <span class="math inline"><em>I</em><sub>tot</sub></span> is the total cell fluorescence and <span class="math inline"><em>N</em><sub>tot</sub></span> is the total number of MscL proteins per cell. <span class="citation" data-cites="bialecka-fornal2012">Bialecka-Fornal et al. (<a href="#ref-bialecka-fornal2012" role="doc-biblioref">2012</a>)</span> et al. report the average cell Mscl copy number for the population rather than the distribution. Knowing only the mean, we can rewrite Eq. 1 as <span id="eq:mscl_avg_ian"><br /><span class="math display">⟨<em>I</em><sub>tot</sub>⟩ = <em>α</em>⟨<em>N</em><sub>tot</sub>⟩,   (2)</span><br /></span> assuming that <span class="math inline"><em>α</em></span> is a constant value that does not change from cell to cell or fluorophore to fluorophore.</p>
<p>    The experiments presented in this work were performed using non-synchronously growing cultures. As there is a uniform distribution of growth phases in the culture, the cell size distribution is broad the the extremes being small, newborn cells and large cells in the process of division. As described in the main text, the cell size distribution of a population is broadened further by modulating the MscL copy number with low copy numbers resulting in aberrant cell morphology. To speak in the terms of an effective channel copy number, we relate the average areal intensity of the population to the average cell size, <span id="eq:area_conversion"><br /><span class="math display">⟨<em>I</em><sub>tot</sub>⟩ = ⟨<em>I</em><sub><em>A</em></sub>⟩⟨<em>A</em>⟩ = <em>α</em>⟨<em>N</em><sub>tot</sub>⟩,   (3)</span><br /></span> where <span class="math inline">⟨<em>I</em><sub><em>A</em></sub>⟩</span> is the average areal intensity in arbitrary units per pixel of the population and <span class="math inline">⟨<em>A</em>⟩</span> is the average area of a segmented cell. As only one focal plane was imaged in these experiments, we could not compute an appropriate volume for each cell given the highly aberrant morphology. We therefore opted to use the projected two-dimensional area of each cell as a proxy for cell size. Given this set of measurements, the calibration factor can be computed as <span id="eq:mscl_simple_cal_factor"><br /><span class="math display">$$
\alpha = {\langle I_A \rangle\langle A \rangle \over \langle N_\text{tot} \rangle}.
\qquad(4)$$</span><br /></span>     While it is tempting to use Eq. 4 directly, there are multiple sources of error that are important to propagate through the final calculation. The most obvious error to include is the measurement error reported in Bialecka-Fornal et al. 2012 for the average MscL channel count <span class="citation" data-cites="bialecka-fornal2012">(Bialecka-Fornal et al. <a href="#ref-bialecka-fornal2012" role="doc-biblioref">2012</a>)</span>. There are also slight variations in expression across biological replicates that arise from a myriad of day-to-day differences. Rather than abstracting all sources of error away into a systematic error budget, we used an inferential model derived from Bayes’ theorem that allows for the computation of the probability distribution of <span class="math inline"><em>α</em></span>.</p>
<h3 id="estimation-of-alpha-for-a-single-biological-replicate">Estimation of <span class="math inline"><em>α</em></span> for a single biological replicate</h3>
<p>    A single data set consists of several hundred single-cell measurements of intensity, area of the segmentation mask, and other morphological quantities. The areal density <span class="math inline"><em>I</em><sub><em>A</em></sub></span> is computed by dividing the total cell fluorescence by the cell area <span class="math inline"><em>A</em></span>. We are interested in computing the probability distributions for the calibration factor <span class="math inline"><em>α</em></span>, the average cell area <span class="math inline">⟨<em>A</em>⟩</span>, and the mean number of channels per cell <span class="math inline">⟨<em>N</em><sub>tot</sub>⟩</span> for the data set as a whole given only <span class="math inline"><em>I</em><sub><em>A</em></sub></span> and <span class="math inline"><em>A</em></span>. Using Bayes’ theorem, the probability distribution for these parameters given a single cell measurement, hereafter called the posterior distribution, can be written as <span id="eq:simple_bayes"><br /><span class="math display">$$
g(\alpha, \langle A \rangle, \langle N_\text{tot} \rangle\,\vert\, A, I_A) = {f(A, I_A\,\vert
\, \alpha,\langle A \rangle, \langle N_\text{tot} \rangle) g(\alpha,\langle A \rangle, \langle N_\text{tot} \rangle) \over f(\alpha, I_A)},
\qquad(5)$$</span><br /></span> where <span class="math inline"><em>g</em></span> and <span class="math inline"><em>f</em></span> represent probability density functions over parameters and data, respectively. The term <span class="math inline"><em>f</em>(<em>A</em>, <em>I</em><sub><em>A</em></sub> | <em>α</em>, ⟨<em>A</em>⟩, ⟨<em>N</em><sub>tot</sub>⟩)</span> in the numerator represents the likelihood of observing the areal intensity <span class="math inline"><em>I</em><sub><em>A</em></sub></span> and area <span class="math inline"><em>A</em></span> of a cell for a given values of <span class="math inline"><em>α</em></span>, <span class="math inline">⟨<em>A</em>⟩</span>, and <span class="math inline">⟨<em>N</em><sub>tot</sub>⟩</span>. The second term in the numerator <span class="math inline"><em>g</em>(<em>α</em>, ⟨<em>A</em>⟩, ⟨<em>N</em><sub>tot</sub>⟩)</span> captures all prior knowledge we have regarding the possible values of these parameters knowing nothing about the measured data. The denominator, <span class="math inline"><em>f</em>(<em>I</em><sub><em>A</em></sub>, <em>A</em>)</span> captures the probability of observing the data knowing nothing about the parameter values. This term, in our case, serves simply as a normalization constant and is neglected for the remainder of this section.</p>
<p>    To determine the appropriate functional form for the likelihood and prior, we must make some assumptions regarding the biological processes that generate them. As there are many independent processes that regulate the timing of cell division and cell growth, such as DNA replication and peptidoglycan synthesis, it is reasonable to assume that for a given culture the distribution of cell size would be normally distributed with a mean of <span class="math inline">⟨<em>A</em>⟩</span> and a variance <span class="math inline"><em>σ</em><sub>⟨<em>A</em>⟩</sub></span>. Mathematically, we can write this as <span id="eq:area_likelihood"><br /><span class="math display">$$
f(A\,\vert\,\langle A \rangle, \sigma_{\langle A \rangle}) \propto {1 \over \sigma_{\langle A \rangle} }\exp\left[-{(A - \langle A \rangle)^2 \over 2\sigma_{\langle A \rangle}^2}\right],
\qquad(6)$$</span><br /></span> where the proportionality results from dropping normalization constants for notational simplicity.</p>
<p>     While total cell intensity is intrinsically dependent on the cell area the areal intensity <span class="math inline"><em>I</em><sub><em>A</em></sub></span> is independent of cell size. The myriad processes leading to the detected fluorescence, such as translation and proper protein folding, are largely independent, allowing us to assume a normal distribution for <span class="math inline"><em>I</em><sub><em>A</em></sub></span> as well with a mean <span class="math inline">⟨<em>I</em><sub><em>A</em></sub>⟩</span> and a variance <span class="math inline"><em>σ</em><sub><em>I</em><sub><em>A</em></sub></sub><sup>2</sup></span>. However, we do not have knowledge of the average areal intensity for the standard candle strain <em>a priori</em>. This can be calculated knowing the calibration factor, total MscL channel copy number, and the average cell area as <span id="eq:avg_rho"><br /><span class="math display">$$
I_A =  {\alpha\langle N_\text{tot} \rangle \over \langle A \rangle}.
\qquad(7)$$</span><br /></span> Using Eq. 7 to calculate the expected areal intensity for the population, we can write the likelihood as a Gaussian distribution, <span id="eq:rho_likelihood"><br /><span class="math display">$$
f(I_A\,\vert\,\alpha,\langle A \rangle,\langle N_\text{tot} \rangle, \sigma_{I_A}) \propto {1 \over \sigma_{I_A} }\exp\left[-{\left(I_A - {\alpha \langle N_\text{tot} \rangle\over \langle A \rangle}\right)^2 \over 2 \sigma_{I_A}^2}\right].
\qquad(8)$$</span><br /></span></p>
<p>    With these two likelihoods in hand, we are tasked with determining the appropriate priors. As we have assumed normal distributions for the likelihoods of <span class="math inline">⟨<em>A</em>⟩</span> and <span class="math inline"><em>I</em><sub><em>A</em></sub></span>, we have included two additional parameters, <span class="math inline"><em>σ</em><sub>⟨<em>A</em>⟩</sub></span> and <span class="math inline"><em>σ</em><sub><em>I</em><sub><em>A</em></sub></sub></span>, each requiring their own prior probability distribution. It is common practice to assume maximum ignorance for these variances and use a Jeffreys prior <span class="citation" data-cites="sivia2006">(Sivia and Skilling <a href="#ref-sivia2006" role="doc-biblioref">2006</a>)</span>, <span id="eq:jeffreys"><br /><span class="math display">$$
g(\sigma_{\langle A \rangle}, \sigma_{I_A}) = {1 \over \sigma_{\langle A \rangle}\sigma_{I_A} }.
\qquad(9)$$</span><br /></span></p>
<p>    The next obvious prior to consider is for the average channel copy number <span class="math inline">⟨<em>N</em><sub>tot</sub>⟩</span>, which comes from Bialecka-Fornal et al. 2012. In this work, they report a mean <span class="math inline"><em>μ</em><sub><em>N</em></sub></span> and variance <span class="math inline"><em>σ</em><sub><em>N</em></sub><sup>2</sup></span>, allowing us to assume a normal distribution for the prior, <span id="eq:informative_prior"><br /><span class="math display">$$
g(\langle N_\text{tot}\rangle\,\vert\, \mu_N,\sigma_N) \propto {1 \over \sigma_N}\exp\left[-{(\langle N_\text{tot} \rangle - \mu_N)^2 \over 2 \sigma_N^2}\right].
\qquad(10)$$</span><br /></span> For <span class="math inline"><em>α</em></span> and <span class="math inline">⟨<em>A</em>⟩</span>, we have some knowledge of what these parameters can and cannot be. For example, we know that neither of these parameters can be negative. As we have been careful to not overexpose the microscopy images, we can say that the maximum value of <span class="math inline"><em>α</em></span> would be the bit-depth of our camera. Similarly, it is impossible to segment a single cell with an area larger than our camera’s field of view (although there are biological limitations to size below this extreme). To remain maximally uninformative, we can assume that the parameter values are uniformly distributed between these bounds, allowing us to state <span id="eq:alpha_uniform_prior"><br /><span class="math display">$$
g(\alpha) = \begin{cases} {1 \over \alpha_\text{max} - \alpha_\text{min} } &amp; \alpha_\text{min} \leq \alpha \leq \alpha_\text{max} \\
0 &amp; \text{otherwise}
\end{cases},
\qquad(11)$$</span><br /></span> for <span class="math inline"><em>α</em></span> and <span id="eq:area_uniform_prior"><br /><span class="math display">$$
g(\langle A \rangle) = \begin{cases} {1 \over \langle A \rangle_\text{max} - \langle A \rangle_\text{min} } &amp; \langle A \rangle_\text{min} \leq \langle A \rangle \leq \langle A \rangle_\text{max}\\
0 &amp; \text{otherwise}
\end{cases}
\qquad(12)$$</span><br /></span> for <span class="math inline">⟨<em>A</em>⟩</span>.</p>
<p>Piecing Eq. 6 through Eq. 12 together generates a complete posterior probability distribution for the parameters given a single cell measurement. This can be generalized to a set of <span class="math inline"><em>k</em></span> single cell measurements as <span id="eq:single_rep_post"><br /><span class="math display">$$
\begin{aligned}
g(\alpha,\langle A \rangle, \langle N_\text{tot} &amp; \rangle, \sigma_{I_A}, \sigma_{\langle A \rangle}\,\vert\, [I_A, A], \mu_N, \sigma_N) \propto {1 \over (\alpha_\text{max} - \alpha_\text{min})(\langle A \rangle_\text{max} - \langle A \rangle_\text{min})}{1 \over (\sigma_{I_A}\sigma_{\langle A \rangle})^{k+1} }\,\times\\\\
&amp;{1 \over \sigma_N}\exp\left[- {(\langle N_\text{tot}\rangle - \mu_N)^2 \over 2\sigma_N^2}\right]
\prod\limits_i^k\exp\left[-{(A^{(i)} - \langle A \rangle)^2 \over 2\sigma_{\langle A \rangle}^2} - {\left(I_A^{(i)} - {\alpha \langle N_\text{tot}\rangle \over \langle A \rangle}\right)^2 \over 2\sigma_{I_A}^2}\right] \end{aligned},
\qquad(13)$$</span><br /></span> where <span class="math inline">[<em>I</em><sub><em>A</em></sub>, <em>A</em>]</span> represents the set of <span class="math inline"><em>k</em></span> single-cell measurements.****</p>
<p>    As small variations in the day-to-day details of cell growth and sample preparation can alter the final channel count of the standard candle strain, it is imperative to perform more than a single biological replicate. However, properly propagating the error across replicates is non trivial. One option would be to pool together all measurements of <span class="math inline"><em>n</em></span> biological replicates and evaluate the posterior given in Eq. 13. However, this by definition assumes that there is no difference between replicates. Another option would be to perform this analysis on each biological replicate individually and then compute a mean and standard deviation of the resulting most-likely parameter estimates for <span class="math inline"><em>α</em></span> and <span class="math inline">⟨<em>A</em>⟩</span>. While this is a better approach than simply pooling all data together, it suffers a bias from giving each replicate equal weight, skewing the estimate of the most-likely parameter value if one replicate is markedly brighter or dimmer than the others. Given this type of data and a limited number of biological replicates (<span class="math inline"><em>n</em> = 6</span> in this work), we chose to extend the Bayesian analysis presented in this section to model the posterior probability distribution for <span class="math inline"><em>α</em></span> and <span class="math inline">⟨<em>A</em>⟩</span> as a hierarchical process in which <span class="math inline"><em>α</em></span> and <span class="math inline">⟨<em>A</em>⟩</span> for each replicate is drawn from the same distribution.</p>
<h3 id="a-hierarchical-model-for-estimating-alpha">A hierarchical model for estimating <span class="math inline"><em>α</em></span></h3>
<p>    In the previous section, we assumed maximally uninformative priors for the most-likely values of <span class="math inline"><em>α</em></span> and <span class="math inline">⟨<em>A</em>⟩</span>. While this is a fair approach to take, we are not completely ignorant with regard to how these values are distributed across biological replicates. A major assumption of our model is that the most-likely value of <span class="math inline"><em>α</em></span> and <span class="math inline">⟨<em>A</em>⟩</span> for each biological replicate are comparable, so long as the experimental error between them is minimized. In other words, we are assuming that the most-likely value for each parameter for each replicate is drawn from the same distribution. While each replicate may have a unique value, they are all related to one another. Unfortunately, proper sampling of this distribution requires an extensive amount of experimental work, making inferential approaches more attractive.</p>
<p>This approach, often called a multi-level or hierarchical model, is schematized in Fig. 3. Here, we use an informative prior for <span class="math inline"><em>α</em></span> and <span class="math inline">⟨<em>A</em>⟩</span> for each biological replicate. This informative prior probability distribution can be described by summary statistics, often called hyper-parameters, which are then treated as the “true” value and are used to calculate the channel copy number. As this approach allows us to get a picture of the probability distribution of the hyper-parameters, we are able to report a point estimate for the most-likely value along with an error estimate that captures all known sources of variation.</p>
<figure>
<img src="ch9_figS3.png" id="fig:hierarchical_model" data-short-caption="Schematic of hierarchical model structure for estimating MscL-sfGFP fluorescence calibration factor." alt="" /><figcaption>Figure 3: <strong>Schematic of hierarchical model structure.</strong> The hyper-parameter probability distributions (top panel) are used as an informative prior for the most-likely parameter values for each biological replicate (middle panel). The single-cell measurements of cell area and areal intensity (bottom panel) are used as data in the evaluation of the likelihood.</figcaption>
</figure>
<p>    The choice for the functional form for the informative prior is often not obvious and can require other experimental approaches or back-of-the-envelope estimates to approximate. Each experiment in this work was carefully constructed to minimize the day-to-day variation. This involved adhering to well-controlled growth temperatures and media composition, harvesting of cells at comparable optical densities, and ensuring identical imaging parameters. As the experimental variation is minimized, we can use our knowledge of the underlying biological processes to guess at the approximate functional form. For similar reasons presented in the previous section, cell size is controlled by a myriad of independent processes. As each replicate is independent of another, it is reasonable to assume a normal distribution for the average cell area for each replicate. This normal distribution is described by a mean <span class="math inline">$\tilde{\langle A \rangle}$</span> and variance <span class="math inline"><em>σ̃</em><sub>⟨<em>A</em>⟩</sub></span>. Therefore, the prior for <span class="math inline">⟨<em>A</em>⟩</span> for <span class="math inline"><em>n</em></span> biological replicates can be written as <span id="eq:hyperprior_area"><br /><span class="math display">$$
g(\langle A \rangle\, \vert\, \tilde{\langle A \rangle}, \tilde{\sigma}_{\langle A \rangle}) \propto {1 \over \tilde{\sigma}_{\langle A \rangle}^n}\prod\limits_{j=1}^{n}\exp\left[-{(\langle A \rangle_j - \tilde{\langle A \rangle})^2 \over 2 \tilde{\sigma}_{\langle A \rangle}^2}\right].
\qquad(14)$$</span><br /></span> In a similar manner, we can assume that the calibration factor for each replicate is normally distributed with a mean <span class="math inline"><em>α̃</em></span> and variance <span class="math inline"><em>σ̃</em><sub><em>α</em></sub></span>, <span id="eq:hyperprior_alpha"><br /><span class="math display">$$
g(\alpha\,\vert\,\tilde{\alpha}, \tilde{\sigma}_\alpha) \propto {1 \over \tilde{\sigma}_\alpha^n}\prod\limits_{j=1}^n \exp\left[-{(\alpha_j - \tilde{\alpha})^2 \over 2\tilde{\sigma}_\alpha^2}\right].
\qquad(15)$$</span><br /></span></p>
<p>    With the inclusion of two more normal distributions, we have introduced four new parameters, each of which needing their own prior. However, our knowledge of the reasonable values for the hyper-parameters has not changed from those described for a single replicate. We can therefore use the same maximally uninformative Jeffreys priors given in Eq. 9 for the variances and the uniform distributions given in Eq. 11 and Eq. 12 for the means. Stitching all of this work together generates the full posterior probability distribution for the best-estimate of <span class="math inline"><em>α̃</em></span> and <span class="math inline">$\tilde{\langle A \rangle}$</span> shown in Eq. 2 given <span class="math inline"><em>n</em></span> replicates of <span class="math inline"><em>k</em></span> single cell measurements, <span id="eq:cal_factor_posterior"><br /><span class="math display">$$
\begin{aligned}
g(\tilde{\alpha}, \tilde{\sigma}_\alpha, \tilde{\langle A \rangle}, \tilde{\sigma}_{\langle A \rangle}, &amp;\{\langle N_\text{tot} \rangle, \langle A \rangle, \alpha, \sigma_{I_A}\}\,\vert\, [I_A, A], \mu_N, \sigma_N) \propto\\
&amp;{1 \over (\tilde{\alpha}_\text{max} - \tilde{\alpha}_\text{min})(\tilde{\langle A \rangle}_\text{max} - \tilde{\langle A \rangle}_\text{min})\sigma_N^n(\tilde{\sigma}_\alpha\tilde{\sigma}_{\langle A \rangle})^{n + 1} }\,\times\\
&amp;\prod\limits_{j=1}^n\exp\left[-{(\langle N \rangle_j^{(i)} - \mu_N)^2 \over 2\sigma_N^2} - {(\alpha_j - \tilde{\alpha})^2 \over 2\tilde{\sigma}_\alpha^2} - {(\langle A \rangle_j - \tilde{\langle A \rangle})^2 \over 2\tilde{\sigma}_{\langle A \rangle}^2}\right]\,\times\,\\ 
&amp;{1 \over (\sigma_{ {I_A}_j}\sigma_{\langle A \rangle_j})^{k_j + 1} }\prod\limits_{i=1}^{k_j}\exp\left[-{(A_j^{(i)} - \langle A \rangle_j)^2 \over 2\sigma^{(i)2}_{\langle A \rangle_j} } - {\left({I_A}_{j}^{(i)} - {\alpha_j \langle N_\text{tot}\rangle_j \over \langle A \rangle_j}\right)\over 2\sigma_{ {I_A}_j}^{(i)2} }\right]
\end{aligned}
\qquad(16)$$</span><br /></span> where the braces <span class="math inline">{…}</span> represent the set of parameters for biological replicates and the brackets <span class="math inline">[…]</span> correspond to the set of single-cell measurements for a given replicate.</p>
<p>    While Eq. 16 is not analytically solvable, it can be easily sampled using Markov chain Monte Carlo (MCMC). The results of the MCMC sampling for <span class="math inline"><em>α̃</em></span> and <span class="math inline">$\tilde{\langle A \rangle}$</span> can be seen in Fig. 4. From this approach, we found the most-likely parameter values of <span class="math inline">3300<sub> − 700</sub><sup> + 700</sup></span> a.u. per MscL channel and <span class="math inline">5.4<sub> − 0.5</sub><sup> + 0.4</sup></span> <span class="math inline"><em>μ</em></span>m<span class="math inline"><em></em><sup>2</sup></span> for <span class="math inline"><em>α̃</em></span> and <span class="math inline">$\tilde{\langle A \rangle}$</span>, respectively. Here, we’ve reported the median value of the posterior distribution for each parameter with the upper and lower bound of the 95% credible region as superscript and subscript, respectively. These values and associated errors were used in the calculation of channel copy number.</p>
<figure>
<img src="ch9_figS4.png" id="fig:mscl_posterior_samples" data-short-caption="Posterior distributions for hyper-parameters and replicate parameters." alt="" /><figcaption>Figure 4: <strong>Posterior distributions for hyper-parameters and replicate parameters.</strong> (A) The posterior probability distribution for <span class="math inline"><em>α̃</em></span> and <span class="math inline">$\tilde{\langle A \rangle}$</span>. Probability increases from light to dark red. The replicate parameter (blue) and hyper-parameter (red) marginalized posterior probability distributions for <span class="math inline"><em>α</em></span> (B) and <span class="math inline">⟨<em>A</em>⟩</span> (C).</figcaption>
</figure>
<h3 id="effect-of-correction">Effect of correction</h3>
<p>     The posterior distributions for <span class="math inline"><em>α</em></span> and <span class="math inline">⟨<em>A</em>⟩</span> shown in Fig. <strong>¿fig:posterior_samples?</strong> were used directly to compute the most-likely channel copy number for each measurement of the Shine-Dalgarno mutant strains, as is described in the coming section. The importance of this correction can be seen in Fig. 5. Cells with low abundance of MscL channels exhibit notable morphological defects, as illustrated in Fig. 5 (A). While these would all be considered single cells, the two-dimensional area of each may be comparable to two or three wild-type cells. For all of the Shine-Dalgarno mutants, the distribution of projected cell area has a long tail, with the extremes reaching 35 <span class="math inline"><em>μ</em>m<sup>2</sup></span> per cell Fig. 5 (B). Calculating the total number of channels per cell does nothing to decouple this correlation between cell area and measured cell intensity. Fig. 5 (C) shows the correlation between cell area and the total number of channels without normalizing to an average cell size <span class="math inline">⟨<em>A</em>⟩</span> differentiated by their survival after an osmotic down-shock. This correlation is removed by calculating an effective channel copy number shown in Fig. 5 (D).</p>
<figure>
<img src="ch9_figS5.png" id="fig:area_correction" data-short-caption="Influence of area correction for Shine-Dalgarno mutants." alt="" /><figcaption>Figure 5: <strong>Influence of area correction for Shine-Dalgarno mutants.</strong> (A) Representative images of aberrant cell morphologies found in low-expressing Shine-Dalgarno mutants. (B) Empirical cumulative distribution of two-dimensional projected cell area for the standard candle strain MLG910 (gray line) and for all Shine-Dalgarno mutants (red line). (C) The correlation between channel copy number and cell area without the area correction. (D) The correlation between effective channel copy number and cell area with the area correction applied.</figcaption>
</figure>
<h2 id="classification-of-cell-fates">Classification of Cell Fates</h2>
<p>     We defined a survival event as a cell that went on to divide at least twice in the several hours following the applied osmotic shock. In nearly all of our experiments, cells which did not survive an osmotic shock exhibited necrosis with loss of phase contrast, extensive blebbing and bursting of the membrane, and the presence of dark aggregates at the cell poles. An example field across time is shown below in Fig. 6 where the cells are necrotic. On occasion, we observed cells which did not obviously display the aforementioned death criteria yet did not undergo one or two division events. These cells were not counted in our experiments and were not included in the final tally of survival versus death. Across our 2822 single cell measurements, such “no call” classifications were observed only 83 times, constituting only 3% of the total cell measurements. A breakdown of all classification types and their respective abundances can be seen in Table S1.</p>
<figure>
<img src="ch9_figS7.png" id="fig:phase_death" data-short-caption="Time lapse of a representative field after osmotic shock and the resulting classifications." alt="" /><figcaption>Figure 6: <strong>Time lapse of a representative field after osmotic shock and the resulting classifications.</strong> Each row shows an individual cell or pair of neighboring cells over time after the application of a fast osmotic shock. Cells classified as dead are denoted by red arrows. The lone surviving cell in this field (bottom row, top 1/4 of image) is marked in green.</figcaption>
</figure>
<table>
<caption>Cell fate classifications and their relative abundances in the complete data set.</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Classification</strong></th>
<th style="text-align: center;"><strong>Number of Observations</strong></th>
<th style="text-align: center;"><strong>Percentage of Measurements</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Dead-On-Arrival</td>
<td style="text-align: center;">11</td>
<td style="text-align: center;">0.4%</td>
</tr>
<tr class="even">
<td style="text-align: center;">No Call</td>
<td style="text-align: center;">83</td>
<td style="text-align: center;">3%</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Death</td>
<td style="text-align: center;">1246</td>
<td style="text-align: center;">44%</td>
</tr>
<tr class="even">
<td style="text-align: center;">Survival</td>
<td style="text-align: center;">1482</td>
<td style="text-align: center;">53%</td>
</tr>
</tbody>
</table>
<p>    To assess the validity of our morphology-based classification scheme, we performed a subset of the osmotic shock experiments described in the manuscript using propidium iodide staining to mark cells which had compromised membranes, identifying them as dead. Briefly, cells expressing on average <span class="math inline">≈</span> 80 MscL channels per cell were grown in LB + 500 mM NaCl to an OD<span class="math inline"><em></em><sub>600nm</sub></span> of approximately 0.25. The cells were then mounted in the flow cell as described in the Materials and Methods in the main text and subjected to a large osmotic shock. After the shock, the cells were monitored for two hours. The propidium iodide stain (LIVE/DEAD BacLight Bacterial Cell Viability Staining, Thermo Fisher) was then passed into the flow chamber and imaged. An example of image of the phase contrast and propidium iodide fluorescence images are shown in Fig. 7. We note that cells matching our death criteria, meaning loss of phase contrast and visible distortion of the cell membrane, were strongly marked with propidium iodide, confirming that these cells were dead. The few example of “no call” classification where survival or death could not be determined from morphology alone showed that these cells were in fact dead (see highlighted row in Fig. 7). Cells that went on to divide two or more times in this period were not significantly stained with propidium iodide, confirming their viability and effectiveness of the stain itself. Given this data set, we compared the classification breakdown using our morphology-based method with the conclusive results from the propidium iodide staining (Table 9.2). We found that the two approaches to defining death agreed within 1%. This agreement leads us to believe that our definition of cell survival as morphological regularity and sustained cell growth is sufficiently accurate to draw physiological conclusions from our experiments.</p>
<figure>
<img src="ch9_figS8.png" id="fig:pi_death" data-short-caption="Representative images of propidium iodide staining after a strong osmotic shock." alt="" /><figcaption>Figure 7: <strong>Representative images of propidium iodide staining after a strong osmotic shock.</strong> Phase contrast images of individual or pairs of cells as a function of time (columns). The final column corresponds to fluorescence from propidium iodide. Bright fluorescence indicates intercalation with DNA indicating cell death. Classification of survival based only from morphology is shown as text in the final column. Highlighted row indicates a “no call” event where morphology alone could not be used to determine survival or death.</figcaption>
</figure>
<table>
<caption>Comparison of morphology-based and dye-based survival classification.</caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Classification</strong></th>
<th style="text-align: center;"><strong>Observations via Morphology</strong></th>
<th style="text-align: center;"><strong>Observations via Propidium Iodide Staining</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Dead-On-Arrival</td>
<td style="text-align: center;">184</td>
<td style="text-align: center;">185</td>
</tr>
<tr class="even">
<td style="text-align: center;">No Call</td>
<td style="text-align: center;">2</td>
<td style="text-align: center;">1</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Survival</td>
<td style="text-align: center;">5</td>
<td style="text-align: center;">5</td>
</tr>
</tbody>
</table>
<h2 class="unnumbered" id="references" class="unnumbered">References</h2>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-bialecka-fornal2012">
<p>Bialecka-Fornal, Maja, Heun Jin Lee, Hannah A. DeBerg, Chris S. Gandhi, and Rob Phillips. 2012. “Single-Cell Census of Mechanosensitive Channels in Living Bacteria.” Edited by Arnold Driessen. <em>PLoS ONE</em> 7 (3): e33077. <a href="https://doi.org/10.1371/journal.pone.0033077">https://doi.org/10.1371/journal.pone.0033077</a>.</p>
</div>
<div id="ref-blount1999">
<p>Blount, P., S. I. Sukharev, P. C. Moe, B. Martinac, and C. Kung. 1999. “Mechanosensitive Channels of Bacteria.” <em>Methods in Enzymology</em> 294: 458–82.</p>
</div>
<div id="ref-blount1996">
<p>Blount, P., S. I. Sukharev, M. J. Schroeder, S. K. Nagle, and C. Kung. 1996. “Single Residue Substitutions That Change the Gating Properties of a Mechanosensitive Channel in <em>Escherichia Coli</em>.” <em>Proc Natl Acad Sci U S A</em> 93 (October): 11652–7.</p>
</div>
<div id="ref-milo2010">
<p>Milo, Ron, Paul Jorgensen, Uri Moran, Griffin Weber, and Michael Springer. 2010. “BioNumbersthe Database of Key Numbers in Molecular and Cell Biology.” <em>Nucleic Acids Research</em> 38 (suppl_1): D750–D753. <a href="https://doi.org/10.1093/nar/gkp889">https://doi.org/10.1093/nar/gkp889</a>.</p>
</div>
<div id="ref-sivia2006">
<p>Sivia, Devinderjit, and John Skilling. 2006. <em>Data Analysis: A Bayesian Tutorial</em>. OUP Oxford.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure_phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/published_content" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Published Content
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_02" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd/" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/scripts.min.js"></script>




		</div><!-- /.js-footer-area -->
	</body>
</html>
