<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter 3</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd/favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd/doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd/" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd/doks-theme/assets/images/switchboard.png); background-position: center;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter 3</h2>
								
								
									<h1 class="hero-subheader__title">Unknown Knowns, Known Unknowns, and Unforseen Consequences: Using Free Energy Shifts To Predict Mutant Phenotypes
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phdchapter_02" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phdchapter_04" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
						<div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
							<div class="align-container" data-mh>
								<div class="align-inner">
									<div class="hero-subheader__author">
										<p class="hero-subheader__author-title">
												Summary
												<i class="icon icon--chevron-down"></i>
										</p><!-- /.hero-subheader__author-title -->
										<p>
Here, I present a biophysical model of allosteric transcriptional regulation that directly links the location of a mutation within a repressor to the biophysical parameters that describe its behavior. Here, we explore the phenotypic space of a repressor with mutations in either the inducer binding or DNA binding domains. Using the LacI repressor in <i>Escherichia</i> <i>coli</i>, we make sharp, falsifiable predictions and use this framework to generate a null hypothesis for how double mutants behave, given knowledge of the single mutants. Linking mutations to the parameters which govern the system allows for quantitative predictions of how the free energy of the system changes as a result, permitting coarse graining of high-dimensional data into a single-parameter description of the mutational consequences.
</p>
									</div><!-- /.hero-subheader__author -->
								</div><!-- /.align-inner -->
							</div><!-- /.align-container -->
						</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phdchapter_02" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter II
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phdchapter_04" class="page-nav__item page-nav__item--next">
										Chapter IV
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<strong>Published as …</strong>
<hr/>
<p>A version of this chapter originally appeared as Chure, G; Razo-Mejia, M., Belliveau, N.M.; Kaczmarek, Zofii A.; Einav, T.; Barnes, Stephanie L.; Lewis, M., and Phillips, R. (2019). Predictive Shifts in Free Energy Couple Mutations to Their Phenotypic Consequences. PNAS 116(37), G.C., M.R.M, N.M.B., Z.A.K., and S.L.B designed the experiments and collected and analyzed data. G.C. developed theoretical treatment of free energy shifts. G.C., M.R.M, N.M.B., Z.A.K., T.E., S.L.B., and R.P. designed the research project. G.C. and R.P. wrote the paper. M.L. provided guidance and advice.</p>
<h2 id="introduction">Introduction</h2>
<p>Thermodynamic treatments of transcriptional regulation have been fruitful in their ability to generate quantitative predictions of gene expression as a function of a minimal set of physically meaningful parameters <span class="citation" data-cites="ackers1982 buchler2003 vilar2003 garcia2011 daber2009a brewster2014 weinert2014 rydenfelt2014b razo-mejia2014 razo-mejia2018 bintu2005 bintu2005a kuhlman2007">(Ackers and Johnson <a href="#ref-ackers1982" role="doc-biblioref">1982</a>; Buchler, Gerland, and Hwa <a href="#ref-buchler2003" role="doc-biblioref">2003</a>; Vilar and Leibler <a href="#ref-vilar2003" role="doc-biblioref">2003</a>; Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Daber, Sharp, and Lewis <a href="#ref-daber2009a" role="doc-biblioref">2009</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Weinert et al. <a href="#ref-weinert2014" role="doc-biblioref">2014</a>; Rydenfelt et al. <a href="#ref-rydenfelt2014b" role="doc-biblioref">2014</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2014" role="doc-biblioref">2014</a>, <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Bintu, Buchler, Garcia, Gerland, Hwa, Kondev, and Phillips <a href="#ref-bintu2005" role="doc-biblioref">2005</a>; Bintu, Buchler, Garcia, Gerland, Hwa, Kondev, Kuhlman, et al. <a href="#ref-bintu2005a" role="doc-biblioref">2005</a>; Kuhlman et al. <a href="#ref-kuhlman2007" role="doc-biblioref">2007</a>)</span>. These models quantitatively describe numerous properties of input-output functions, such as the leakiness, saturation, dynamic range, steepness of response, and the <span class="math inline"><em>E</em><em>C</em><sub>50</sub></span> – the concentration of inducer at which the response is half maximal. The mathematical forms of these phenotypic properties are couched in terms of a minimal set of experimentally accessible variables, such as the inducer concentration, transcription factor copy number, and the DNA sequence of the binding site <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>. While the amino acid sequence of the transcription factor is another controllable variable, it is seldom implemented in quantitative terms considering mutations with subtle changes in chemistry frequently yield unpredictable physiological consequences. In this work, we examine how a series of mutations in either the DNA binding or inducer binding domains of a transcriptional repressor influence the values of the biophysical parameters which govern its regulatory behavior.</p>
<p>We build upon the results presented in Chapter 1 of this thesis and present a theoretical framework for understanding how mutations in the amino acid sequenc of the repressor affect different parameters and alter the free energy of the system. We find that the parameters capturing the allosteric nature of the repressor, the repressor copy number, and the DNA binding specificity contribute independently to the free energy of the system with different degrees of sensitivity. Furthermore, changes restricted to one of these three groups of parameters result in characteristic changes in the free energy relative to the wild-type repressor, providing falsifiable predictions of how different classes of mutations should behave.</p>
<p>Next, we test these descriptions experimentally using the well-characterized transcriptional repressor of the <em>lac</em> operon LacI in <em>E. coli</em> regulating expression of a fluorescent reporter. We introduce a series of point mutations in either the inducer binding or DNA binding domain. We then measure the full induction profile of each mutant, determine the minimal set of parameters that are affected by the mutation, and predict how each mutation tunes the free energy at different inducer concentrations, repressor copy numbers, and DNA binding strengths. We find in general that mutations in the DNA binding domain only influence DNA binding strength, and that mutations within the inducer binding domain affect only the parameters which dictate the allosteric response. The degree to which these parameters are insulated is notable, as the very nature of allostery suggests that all parameters are intimately connected, thus enabling binding events at one domain to be “sensed” by another.</p>
<p>With knowledge of how a collection of DNA binding and inducer binding single mutants behave, we predict the induction profiles and the free energy changes of pairwise double mutants with quantitative accuracy. We find that the energetic effects of each individual mutation are additive, indicating that epistatic interactions are absent between the mutations examined here. Our model provides a means for identifying and quantifying the extent of epistatic interactions in a more complex set of mutations, and can shed light on how the protein sequence and general regulatory architecture coevolve.</p>
<h2 id="theoretical-model">Theoretical Model</h2>
<p>This work considers the inducible simple repression regulatory motif depicted in Fig. ?? (A) from a thermodynamic perspective which has been thoroughly dissected and tested experimentally <span class="citation" data-cites="garcia2011 brewster2014 razo-mejia2018">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> and is described in depth in Chapter 1. The result of this extensive theory-experiment dialogue is a succinct input-output function schematized in Fig. ?? (B) that computes the fold-change in gene expression relative to an unregulated promoter. This function is of the form</p>
<p><span id="eq:foldchange" class="eqnos"><br /><span class="math display">$$
\text{fold-change} = \left(1 + {R_A \over
N_{NS}}e^{-\beta\Delta\varepsilon_{RA}}\right)^{-1},
$$</span><br /><span class="eqnos-number">(1)</span></span></p>
<p>where <span class="math inline"><em>R</em><sub><em>A</em></sub></span> is the number of active repressors per cell, <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of non-specific binding sites for the repressor, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is the binding energy of the repressor to its specific binding site relative to the non-specific background, and <span class="math inline"><em>β</em></span> is defined as <span class="math inline">${1 \over k_B T}$</span> where <span class="math inline"><em>k</em><sub><em>B</em></sub></span> is the Boltzmann constant and <span class="math inline"><em>T</em></span> is the temperature. While this theory requires knowledge of the number of <em>active</em> repressors, we often only know the total number <span class="math inline"><em>R</em></span> which is the sum total of active and inactive repressors. We can define a prefactor <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> which captures the allosteric nature of the repressor and encodes the probability a repressor is in the active (repressive) state rather than the inactive state for a given inducer concentration <span class="math inline"><em>c</em></span>, namely, <span id="eq:pact" class="eqnos"><br /><span class="math display">$$
p_\text{act}(c) = {\left(1 + {c \over K_A}\right)^n \over \left(1 + {c \over
K_A}\right)^n + e^{-\beta\Delta\varepsilon_{AI}}\left(1 + {c \over
K_I}\right)^n}.
$$</span><br /><span class="eqnos-number">(2)</span></span></p>
<p>Here, <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are the dissociation constants of the inducer to the active and inactive repressor, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the energetic difference between the repressor active and inactive states, and <span class="math inline"><em>n</em></span> is the number of allosteric binding sites per repressor molecule (<span class="math inline"><em>n</em> = 2</span> for LacI). With this in hand, we can define <span class="math inline"><em>R</em><sub><em>A</em></sub></span> in Eq. <a href="#eq:foldchange">1</a> as <span class="math inline"><em>R</em><sub><em>A</em></sub> = <em>p</em><sub>act</sub>(<em>c</em>)<em>R</em></span>.</p>
<div id="fig:mut_induction_summary" class="fignos">
<figure>
<img src="ch3_fig1.png" data-short-caption="Summary of Chapter 2. A predictive framework for dissection of the simple repression motif" alt="" /><figcaption><span>Figure 1:</span> <strong>A predictive framework for phenotypic and energetic dissection of the simple repression motif.</strong> (A) The inducible simple repression architecture. When in the active state, the repressor (red) binds the cognate operator sequence of the DNA (orange box) with high specificity, preventing transcription by occluding binding of the RNA polymerase to the promoter (blue rectangle). Upon addition of an inducer molecule, the inactive state (purple) becomes energetically preferable, and the repressor no longer binds the operator sequence with appreciable specificity. Once unbound from the operator, binding of the RNA polymerase (blue) is no longer blocked, and transcription can occur. (B) The simple repression input–output function for an allosteric repressor with two inducer binding sites. The key parameters are identified in speech bubbles. (C) The fold change in gene expression collapses as a function of the free energy. Panel (C, left) shows measurements of the fold change in gene expression as a function of inducer concentration from <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>. Points and errors correspond to the mean and SEM of at least 10 biological replicates. The thin lines represent the line of best fit given the model shown in (B). This model can be rewritten as a Fermi function with an energetic parameter F, which is the energetic difference between the repressor bound and unbound states of the promoter, schematized in C, Middle. The points in (C), Bottom correspond to the data shown in (C, left) collapsed onto a master curve defined by their calculated free energy F. The solid black line is the master curve defined by the Fermi function shown in (C, Middle)</figcaption>
</figure>
</div>
<p>A key feature of Eq. <a href="#eq:foldchange">1</a> and Eq. <a href="#eq:pact">2</a> is that the diverse phenomenology of the gene expression induction profile can be collapsed onto a single master curve by rewriting the input-output function in terms of the free energy <span class="math inline"><em>F</em></span> also called the Bohr parameter <span class="citation" data-cites="phillips2015">(Phillips <a href="#ref-phillips2015" role="doc-biblioref">2015</a>)</span>, <span id="eq:colllapse" class="eqnos"><br /><span class="math display">$$
\text{fold-change} = \frac{1}{1 + e^{-\beta F}},
$$</span><br /><span class="eqnos-number">(3)</span></span></p>
<p>where</p>
<p><span id="eq:bohr" class="eqnos"><br /><span class="math display">$$
F = -k_BT \log p_\text{act}(c) - k_BT\log\left({R \over N_{NS}}\right) +
\Delta\varepsilon_{RA}.
$$</span><br /><span class="eqnos-number">(4)</span></span></p>
<p>Hence, if different combinations of parameters yield the same free energy, they will give rise to the same fold-change in gene expression, enabling us to collapse multiple regulatory scenarios onto a single curve. This can be seen in Fig. <a href="#fig:mut_induction_summary">1</a> (C) where eighteen unique inducer titration profiles of a LacI simple repression architecture collected and analyzed in <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> collapse onto a single master curve. The tight distribution about this curve reveals that the fold-change across a variety of genetically distinct individuals can be adequately described by a small number of parameters. Beyond predicting the induction profiles of different strains, the method of data collapse inspired by Eq. ?? and Eq. <a href="#eq:bohr">4</a> can be used as a tool to identify mechanistic changes in the regulatory architecture <span class="citation" data-cites="swem2008">(Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>)</span>. Similar data collapse approaches have been used previously in such a manner and have proved vital for distinguishing between changes in parameter values and changes in the fundamental behavior of the system<span class="citation" data-cites="swem2008 keymer2006">(Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>; <span class="citeproc-not-found" data-reference-id="keymer2006"><strong>???</strong></span>)</span>.</p>
<p>Assuming that a given mutation does not result in a non-functional protein, it is reasonable to say that any or all of the parameters in Eq. <a href="#eq:foldchange">1</a> can be affected by the mutation, changing the observed induction profile and therefore the free energy. To examine how the free energy of a mutant <span class="math inline"><em>F</em><sup>(mut)</sup></span> differs from that of the wild-type <span class="math inline"><em>F</em><sup>(wt)</sup></span>, we define <span class="math inline"><em>Δ</em><em>F</em> = <em>F</em><sup>(<em>m</em><em>u</em><em>t</em>)</sup> − <em>F</em><sup>(<em>w</em><em>t</em>)</sup></span>, which has the form</p>
<p><span id="eq:delF" class="eqnos"><br /><span class="math display">$$
\begin{aligned}
\Delta F = -k_BT\log\left({p_\text{act}^\mathrm{(mut)}(c) \over  p_\text{act}^\mathrm{(wt)}(c)}\right) &amp;- k_BT \log\left({R^\mathrm{(mut)}\over R^\mathrm{(wt)}}\right)\\
 &amp;+ (\Delta\varepsilon_{RA}^\mathrm{(mut)}- \Delta\varepsilon_{RA}^\mathrm{(wt)}).
    \end{aligned}
$$</span><br /><span class="eqnos-number">(5)</span></span></p>
<p><span class="math inline"><em>Δ</em><em>F</em></span> describes how a mutation translates a point across the master curve shown in Fig. ?? (C). As we will show in the coming paragraphs (illustrated in Fig. ??), this formulation coarse grains the myriad parameters shown in Eq. <a href="#eq:foldchange">1</a> and Eq. <a href="#eq:pact">2</a> into three distinct quantities, each with different sensitivities to parametric changes. By examining how a mutation changes the <span class="math inline"><em>Δ</em><em>F</em></span> as a function of the inducer concentration, one can draw conclusions as to which parameters have been modified based solely on the shape of the curve. To help the reader understand how various perturbations to the parameters tune the free energy, we have hosted an interactive figure on the dedicated <a href="http://www.rpgroup.caltech.edu/mwc_mutants">website for the publication</a> which makes exploration of parameter space a simpler task.</p>
<p>The first term in Eq. <a href="#eq:delF">5</a> is the log ratio of the probability of a mutant repressor being active relative to the wild type at a given inducer concentration <span class="math inline"><em>c</em></span>. This quantity defines how changes to any of the allosteric parameters – such as inducer binding constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> or active/inactive state energetic difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> – alter the free energy <span class="math inline"><em>F</em></span>, which can be interpreted as the free energy difference between the repressor bound and unbound states of the promoter. Fig. ?? (A) illustrates how changes to the inducer binding constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> alone alter the induction profiles and resulting free energy as a function of the inducer concentration. In the limit where <span class="math inline"><em>c</em> = 0</span>, the values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> do not factor into the calculation of <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> given by Eq. <a href="#eq:pact">2</a> meaning that <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the lone parameter setting the residual activity of the repressor. Thus, if only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are altered by a mutation, then <span class="math inline"><em>Δ</em><em>F</em></span> should be <span class="math inline">0 <em>k</em><sub><em>B</em></sub><em>T</em></span> when <span class="math inline"><em>c</em> = 0</span>, illustrated by the overlapping red, purple, and grey curves in the right-hand plot of Fig. ?? (A). However, if <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is influenced by the mutation (either alone or in conjunction with <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>), the leakiness will change, resulting in a non-zero <span class="math inline"><em>Δ</em><em>F</em></span> when <span class="math inline"><em>c</em> = 0</span>. This is illustrated in Fig. ?? (B) where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the only parameter affected by the mutation.</p>
<p>It is important to note that for a mutation which perturbs only the inducer binding constants, the dependence of <span class="math inline"><em>Δ</em><em>F</em></span> on the inducer concentration can be non-monotonic. While the precise values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> control the sensitivity of the repressor to inducer concentration, it is the ratio <span class="math inline"><em>K</em><sub><em>A</em></sub>/<em>K</em><sub><em>I</em></sub></span> that defines whether this non-monotonic behavior is observed. This can be seen more clearly when we consider the limit of saturating inducer concentration, <span id="eq:kaki_sat_c" class="eqnos"><br /><span class="math display">$$
\lim\limits_{c \rightarrow \infty} \log\left({p_\text{act}^\mathrm{(mut)}\over
p_\text{act}^\mathrm{(wt)}}\right) \approx \log\left[{1 +
e^{-\beta\Delta\varepsilon_{AI}^\mathrm{(wt)}} \left({K_A^\mathrm{(wt)}\over
K_I^\mathrm{(wt)}}\right)^n \over 1 +
e^{-\beta\Delta\varepsilon_{AI}^\mathrm{(wt)}} \left({K_A^\mathrm{(mut)}\over
K_I^\mathrm{(mut)}}\right)^n}\right],
$$</span><br /><span class="eqnos-number">(6)</span></span></p>
<p>which illustrates that <span class="math inline"><em>Δ</em><em>F</em></span> returns to zero at saturating inducer concentration when the ratio <span class="math inline"><em>K</em><sub><em>A</em></sub>/<em>K</em><sub><em>I</em></sub></span> is the same for both the mutant and wild-type repressors, so long as <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is unperturbed. Non-monotonicity can <em>only</em> be achieved by changing <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> and therefore serves as a diagnostic for classifying mutational effects reliant solely on measuring the change in free energy. A rigorous proof of this non-monotonic behavior given changing <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> can be found in supplemental Chapter 7.</p>
<p>The second term in Eq. <a href="#eq:delF">5</a> captures how changes in the repressor copy number contributes to changes in free energy. It is important to note that this contribution to the free energy change depends on the total number of repressors in the cell, not just those in the active state. This emphasizes that changes in the expression of the repressor are energetically divorced from changes to the allosteric nature of the repressor. As a consequence, the change in free energy is constant for all inducer concentrations, as is schematized in Fig. Fig. ?? (C). Because the magnitude of the change in free energy scales logarithmically with changing repressor copy number, a mutation which increases expression from 1 to 10 repressors per cell is more impactful from an energetic standpoint (<span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em>log (10) ≈ 2.3 <em>k</em><sub><em>B</em></sub><em>T</em></span>) than an increase from 90 to 100 (<span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em>log (100/90) ≈ 0.1 <em>k</em><sub><em>B</em></sub><em>T</em></span>). Appreciable changes in the free energy only arise when variations in the repressor copy number are larger than or comparable to an order of magnitude. Changes of this magnitude are certainly possible from a single point mutation, as it has been shown that even synonymous substitutions can drastically change translation efficiency <span class="citation" data-cites="frumkin2018">(Frumkin et al. <a href="#ref-frumkin2018" role="doc-biblioref">2018</a>)</span>.</p>
<p>The third and final term in Eq. <a href="#eq:delF">5</a> is the difference in the DNA binding energy between the mutant and wild-type repressors. All else being equal, if the mutated state binds more tightly to the DNA than the wild type (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>w</em><em>t</em>)</sup> &gt; <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>m</em><em>u</em><em>t</em>)</sup></span>), the net change in the free energy is negative, indicating that the repressor bound states become more energetically favorable due to the mutation. Much like in the case of changing repressor copy number, this quantity is independent of inducer concentration and is therefore also constant Fig. ?? (D). However, the magnitude of the change in free energy is linear with DNA binding affinity while it is logarithmic with respect to changes in the repressor copy number. Thus, to change the free energy by <span class="math inline">1 <em>k</em><sub><em>B</em></sub><em>T</em></span>, the repressor copy number must change by a factor of <span class="math inline"> ≈ 2.3</span> whereas the DNA binding energy must change by <span class="math inline">1 <em>k</em><sub><em>B</em></sub><em>T</em></span>.</p>
<p>The unique behavior of each quantity in Eq. <a href="#eq:delF">5</a> and its sensitivity with respect to the parameters makes <span class="math inline"><em>Δ</em><em>F</em></span> useful as a diagnostic tool to classify mutations. Given a set of fold-change measurements, a simple rearrangement of Eq. ?? permits the direct calculation of the free energy, assuming that the underlying physics of the regulatory architecture has not changed. Thus, it becomes possible to experimentally test the general assertions made in Fig. ??.</p>
<figure>
<img src="ch3_fig2.png" id="eq:deltaF_theory" data-short-caption="Parametric changes due to mutations and their corresponding free-energy shifts." alt="" /><figcaption><strong>Parametric changes due to mutations and the corresponding free-energy changes for (A) perturbations to <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, (B) changes to the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, (C) changes to repressor copy number, and (D) changes in DNA binding affinity</strong>. The first column schematizes the changed parameters and the second column reflects which quantity in Eq. <a href="#eq:delF">5</a> is affected. The third column shows representative induction profiles from mutants which have smaller (purple) and larger (orange) values for the parameters than the wild type (gray). The fourth and fifth columns illustrate how the free energy is changed as a result. Purple and red arrows indicate the direction in which the points are translated about the master curve. Three concentrations (points labeled 1, 2, and 3) are shown to illustrate how each point is moved in free-energy space. An interactive version of this figure can be found on the paper website (https://www.rpgroup.caltech.edu/mwc_mutants).</figcaption>
</figure>
<h2 id="dna-binding-domain-mutants">DNA Binding Domain Mutants</h2>
<p>With this arsenal of analytic diagnostics, we can begin to explore the mutational space of the repressor and map these mutations to the biophysical parameters they control. As one of the most thoroughly studied transcription factors, LacI has been subjected to numerous crystallographic and mutational studies <span class="citation" data-cites="lewis1996 daber2009a daber2011a swerdlow2014">(Lewis et al. <a href="#ref-lewis1996" role="doc-biblioref">1996</a>; Daber, Sharp, and Lewis <a href="#ref-daber2009a" role="doc-biblioref">2009</a>; Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>; <span class="citeproc-not-found" data-reference-id="swerdlow2014"><strong>???</strong></span>)</span>. One such work generated a set of point mutations in the LacI repressor and examined the diversity of the phenotypic response to different allosteric effectors <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span>. However, several experimental variables were unknown, precluding precise calculation of <span class="math inline"><em>Δ</em><em>F</em></span> as presented in the previous section. In Ref. <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span>, the repressor variants and the fluorescence reporter were expressed from separate plasmids. As the copy numbers of these plasmids fluctuate in the population, both the population average repressor copy number and the number of regulated promoters were unknown. Both of these quantities have been shown previously to significantly alter the measured gene expression and calculation of <span class="math inline"><em>Δ</em><em>F</em></span> is dependent on knowledge of their values. While the approach presented in Ref. <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> considers the Lac repressor as an MWC molecule, the copy numbers of the repressor and the reporter gene were swept into an effective parameter <span class="math inline">${R \over K_{DNA}}$</span>, hindering our ability to distinguish between changes in repressor copy number or in DNA binding energy. To test our hypothesis of free energy differences resulting from various parameter perturbations, we used the dataset in Ref. <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> as a guide and chose a subset of the mutations to quantitatively dissect. To control copy number variation, the mutant repressors and the reporter gene were integrated into the <em>E. coli</em> chromosome where the copy numbers are known and tightly controlled <span class="citation" data-cites="razo-mejia2018 garcia2011">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>. Furthermore, the mutations were paired with ribosomal binding sties where the level of translation of the wild-type repressor had been directly measured previously <span class="citation" data-cites="garcia2011">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>.</p>
<p>We made three amino acid substitutions (Y17I, Q18A, and Q18M) that are critical for the DNA-repressor interaction. These mutations were introduced into the <em>lacI</em> sequence used in <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span> with four different ribosomal binding site sequences that were shown (via quantitative Western blotting) to tune the wild-type repressor copy number across three orders of magnitude. These mutant constructs were integrated into the <em>E. coli</em> chromosome harboring a Yellow Fluorescent Protein (YFP) reporter. The YFP promoter included the native O2 LacI operator sequence which the wild-type LacI repressor binds with high specificity (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>). The fold-change in gene expression for each mutant across twelve concentrations of IPTG was measured via flow cytometry. As we mutated only a single amino acid with the minimum number of base pair changes to the codons from the wild-type sequence, we find it unlikely that the repressor copy number was drastically altered from those reported in Ref. <span class="citation" data-cites="garcia2011">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span> for the wild-type sequence paired with the same ribosomal binding site sequence. In characterizing the effects of these DNA binding mutations, we take the repressor copy number to be unchanged. Any error introduced by this assumption should be manifest as a larger than predicted systematic shift in the free energy change when the repressor copy number is varied.</p>
<div id="fig:DNA_muts" class="fignos">
<figure>
<img src="ch3_fig3.png" data-short-caption="Induction profiles and free-energy differences of DNA binding domain mutations in the *lac* repressor." alt="" /><figcaption><span>Figure 2:</span> <strong>Induction profiles and free-energy differences of DNA binding domain mutations.</strong> Each column corresponds to the highlighted mutant at the top of the figure. Each strain was paired with the native O2 operator sequence. Open points correspond to the strain for each mutant from which the DNA binding energy was estimated. (A) Induction profiles of each mutant at four different repressor copy numbers as a function of the inducer concentration. Points correspond to the mean fold change in gene expression of 6–10 biological replicates. Error bars are the SEM. Shaded regions demarcate the 95% credible region of the induction profile generated by the estimated DNA binding energy. (B) Data collapse of all points for each mutant shown in A using only the DNA binding energy estimated from a single repressor copy number. Points correspond to the average fold change in gene expression of 6–10 biological replicates. Error bars are SEM. Where error bars are not visible, the relative error in measurement is smaller than the size of the marker. (C) The change in the free energy resulting from each mutation as a function of the inducer concentration. Points correspond to the median of the marginal posterior distribution for the free energy. Error bars represent the upper and lower bounds of the 95% credible region. Points in A at the detection limits of the flow cytometer (near fold-change values of 0 and 1) were neglected for calculation of the <span class="math inline"><em>Δ</em><em>F</em></span>. The IPTG concentration is shown on a symmetric log scale with linear scaling ranging from 0 to <span class="math inline">10<sup> − 2</sup></span> <span class="math inline"><em>μ</em></span>M and log scaling elsewhere. The shaded red lines in C correspond to the 95% credible region of our predictions for <span class="math inline"><em>Δ</em><em>F</em></span> based solely on estimation of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> from the strain with <span class="math inline"><em>R</em> = 260</span> repressors per cell.</figcaption>
</figure>
</div>
<p><span id="fig:DNA_muts" label="fig:DNA_muts">[fig:DNA_muts]</span></p>
<p>A naïve hypothesis for the effect of a mutation in the DNA binding domain is that <em>only</em> the DNA binding energy is affected. This hypothesis appears to contradict the core principle of allostery in that ligand binding in one domain influences binding in another, suggesting that changing parameter modifies them all. The characteristic curves summarized in Fig. <a href="#fig:deltaF_theory">[fig:deltaF_theory]</a> give a means to discriminate between these two hypotheses by examining the change in the free energy. Using a single induction profile (white-faced points in Fig. <a href="#fig:DNA_muts">[fig:DNA_muts]</a>), we estimated the DNA binding energy using Bayesian inferential methods, the details of which are thoroughly discussed in the Materials and Methods as well as the SI text. The shaded red region for each mutant in Fig. <a href="#fig:DNA_muts">[fig:DNA_muts]</a> represents the 95% credible region of this fit whereas all other shaded regions are 95% credible regions of the predictions for other repressor copy numbers. We find that redetermining only the DNA binding energy accurately captures the majority of the induction profiles, indicating that other parameters are unaffected. One exception is for the lowest repressor copy numbers (<span class="math inline"><em>R</em> = 60</span> and <span class="math inline"><em>R</em> = 124</span> per cell) of mutant Q18A at low concentrations of IPTG. However, we note that this disagreement is comparable to that observed for the wild-type repressor binding to the weakest operator in Razo-Mejia <em>et al.</em> 2018 , illustrating that our model is imperfect in characterizing weakly repressing architectures. Including other parameters in the fit (such as <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>) does not significantly improve the accuracy of the predictions. Furthermore, the magnitude of this disagreement also depends on the choice of the fitting strain (see SI text).</p>
<p>Mutations Y17I and Q18A both weaken the affinity of the repressor to the DNA relative to the wild type strain with binding energies of <span class="math inline"> − 9.9<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span> and <span class="math inline"> − 11.0<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span>, respectively. Here we report the median of the inferred posterior probability distribution with the superscripts and subscripts corresponding to the upper and lower bounds of the 95% credible region. These binding energies are comparable to that of the wild-type repressor affinity to the native LacI operator sequence O3, with a DNA binding energy of <span class="math inline"> − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The mutation Q18M increases the strength of the DNA-repressor interaction relative to the wild-type repressor with a binding energy of <span class="math inline"> − 15.43<sub> − 0.06</sub><sup> + 0.07</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span>, comparable to the affinity of the wild-type repressor to the native O1 operator sequence (<span class="math inline"> − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span>). It is notable that a single amino acid substitution of the repressor is capable of changing the strength of the DNA binding interaction well beyond that of many single base-pair mutations in the operator sequence .</p>
<p>Using the new DNA binding energies, we can collapse all measurements of fold-change as a function of the free energy as shown in Fig. <a href="#fig:DNA_muts">[fig:DNA_muts]</a>(B). This allows us to test the diagnostic power of the decomposition of the free energy described in Fig. <a href="#fig:deltaF_theory">[fig:deltaF_theory]</a>. To compute the <span class="math inline"><em>Δ</em><em>F</em></span> for each mutation, we inferred the observed mean free energy of the mutant strain for each inducer concentration and repressor copy number (see Materials and Methods as well as the SI text for a detailed explanation of the inference). We note that in the limit of extremely low or high fold-change, the inference of the free energy is either over- or under-estimated, respectively, introducing a systematic error. Thus, points which are close to these limits are omitted in the calculation of <span class="math inline"><em>Δ</em><em>F</em></span>. We direct the reader to the SI text for a detailed discussion of this systematic error. With a measure of <span class="math inline"><em>F</em><sup>(<em>m</em><em>u</em><em>t</em>)</sup></span> for each mutant at each repressor copy number, we compute the difference in free energy relative to the wild-type strain with the same repressor copy number and operator sequence, restricting all variability in <span class="math inline"><em>Δ</em><em>F</em></span> solely to changes in <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>.</p>
<p>The change in free energy for each mutant is shown in Fig. <a href="#fig:DNA_muts">[fig:DNA_muts]</a>(C). It can be seen that the <span class="math inline"><em>Δ</em><em>F</em></span> for each mutant is constant as a function of the inducer concentration and is concordant with the prediction generated from fitting <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> to a single repressor copy number [red lines Fig. <a href="#fig:DNA_muts">[fig:DNA_muts]</a>(C)]. This is in line with the predictions outlined in Fig. <a href="#fig:deltaF_theory">[fig:deltaF_theory]</a>(C) and (D), indicating that the allosteric parameters are “insulated”, meaning they are not affected by the DNA binding domain mutations. As the <span class="math inline"><em>Δ</em><em>F</em></span> for all repressor copy numbers collapses onto the prediction, we can say that the expression of the repressor itself is the same or comparable with that of the wild type. If the repressor copy number were perturbed in addition to <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, one would expect a shift away from the prediction that scales logarithmically with the change in repressor copy number. However, as the <span class="math inline"><em>Δ</em><em>F</em></span> is approximately the same for each repressor copy number, it can be surmised that the mutation does not significantly change the expression or folding efficiency of the repressor itself. These results allow us to state that the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is the only parameter modified by the DNA mutants examined.</p>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-ackers1982">
<p>Ackers, Gary K, and Alexander D Johnson. 1982. “Quantitative Model for Gene Regulation by A Phage Repressor.” <em>Proc. Natl. Acad. Sci. USA</em>, 5.</p>
</div>
<div id="ref-bintu2005a">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, Thomas Kuhlman, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Applications.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 125–35. <a href="https://doi.org/10.1016/j.gde.2005.02.006">https://doi.org/10.1016/j.gde.2005.02.006</a>.</p>
</div>
<div id="ref-bintu2005">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Models.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 116–24. <a href="https://doi.org/10.1016/j.gde.2005.02.007">https://doi.org/10.1016/j.gde.2005.02.007</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-buchler2003">
<p>Buchler, N. E., U. Gerland, and T. Hwa. 2003. “On Schemes of Combinatorial Transcription Logic.” <em>Proceedings of the National Academy of Sciences</em> 100 (9): 5136–41. <a href="https://doi.org/10.1073/pnas.0930314100">https://doi.org/10.1073/pnas.0930314100</a>.</p>
</div>
<div id="ref-daber2009a">
<p>Daber, Robert, Kim Sharp, and Mitchell Lewis. 2009. “One Is Not Enough.” <em>Journal of Molecular Biology</em> 392 (5): 1133–44. <a href="https://doi.org/10.1016/j.jmb.2009.07.050">https://doi.org/10.1016/j.jmb.2009.07.050</a>.</p>
</div>
<div id="ref-daber2011a">
<p>Daber, Robert, Matthew A. Sochor, and Mitchell Lewis. 2011. “Thermodynamic Analysis of Mutant Lac Repressors.” <em>Journal of Molecular Biology</em>, The Operon Model and its Impact on Modern Molecular Biology, 409 (1): 76–87. <a href="https://doi.org/10.1016/j.jmb.2011.03.057">https://doi.org/10.1016/j.jmb.2011.03.057</a>.</p>
</div>
<div id="ref-frumkin2018">
<p>Frumkin, Idan, Marc J. Lajoie, Christopher J. Gregg, Gil Hornung, George M. Church, and Yitzhak Pilpel. 2018. “Codon Usage of Highly Expressed Genes Affects Proteome-Wide Translation Efficiency.” <em>Proceedings of the National Academy of Sciences</em> 115 (21): E4940–E4949. <a href="https://doi.org/10.1073/pnas.1719375115">https://doi.org/10.1073/pnas.1719375115</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-kuhlman2007">
<p>Kuhlman, T., Z. Zhang, M. H. Saier, and T. Hwa. 2007. “Combinatorial Transcriptional Control of the Lactose Operon of Escherichia Coli.” <em>Proceedings of the National Academy of Sciences</em> 104 (14): 6043–8. <a href="https://doi.org/10.1073/pnas.0606717104">https://doi.org/10.1073/pnas.0606717104</a>.</p>
</div>
<div id="ref-lewis1996">
<p>Lewis, Mitchell, Geoffrey Chang, Nancy C. Horton, Michele A. Kercher, Helen C. Pace, Maria A. Schumacher, Richard G. Brennan, and Ponzy Lu. 1996. “Crystal Structure of the Lactose Operon Repressor and Its Complexes with DNA and Inducer.” <em>Science</em> 271 (5253): 1247–54.</p>
</div>
<div id="ref-phillips2015">
<p>Phillips, Rob. 2015. “Napoleon Is in Equilibrium.” <em>Annual Review of Condensed Matter Physics</em> 6 (1): 85–111. <a href="https://doi.org/10.1146/annurev-conmatphys-031214-014558">https://doi.org/10.1146/annurev-conmatphys-031214-014558</a>.</p>
</div>
<div id="ref-razo-mejia2018">
<p>Razo-Mejia, Manuel, Stephanie L. Barnes, Nathan M. Belliveau, Griffin Chure, Tal Einav, Mitchell Lewis, and Rob Phillips. 2018. “Tuning Transcriptional Regulation Through Signaling: A Predictive Theory of Allosteric Induction.” <em>Cell Systems</em> 6 (4): 456–469.e10. <a href="https://doi.org/10.1016/j.cels.2018.02.004">https://doi.org/10.1016/j.cels.2018.02.004</a>.</p>
</div>
<div id="ref-razo-mejia2014">
<p>Razo-Mejia, M., J. Q. Boedicker, D. Jones, A. DeLuna, J. B. Kinney, and R. Phillips. 2014. “Comparison of the Theoretical and Real-World Evolutionary Potential of a Genetic Circuit.” <em>Physical Biology</em> 11 (2): 026005. <a href="https://doi.org/10.1088/1478-3975/11/2/026005">https://doi.org/10.1088/1478-3975/11/2/026005</a>.</p>
</div>
<div id="ref-rydenfelt2014b">
<p>Rydenfelt, Mattias, Robert Sidney Cox, Hernan Garcia, and Rob Phillips. 2014. “Statistical Mechanical Model of Coupled Transcription from Multiple Promoters Due to Transcription Factor Titration.” <em>Physical Review E</em> 89 (1): 012702. <a href="https://doi.org/10.1103/PhysRevE.89.012702">https://doi.org/10.1103/PhysRevE.89.012702</a>.</p>
</div>
<div id="ref-swem2008">
<p>Swem, Lee R., Danielle L. Swem, Ned S. Wingreen, and Bonnie L. Bassler. 2008. “Deducing Receptor Signaling Parameters from in Vivo Analysis: LuxN/AI-1 Quorum Sensing in Vibrio Harveyi.” <em>Cell</em> 134 (3): 461–73. <a href="https://doi.org/10.1016/j.cell.2008.06.023">https://doi.org/10.1016/j.cell.2008.06.023</a>.</p>
</div>
<div id="ref-vilar2003">
<p>Vilar, José M. G., and Stanislas Leibler. 2003. “DNA Looping and Physical Constraints on Transcription Regulation.” <em>Journal of Molecular Biology</em> 331 (5): 981–89. <a href="https://doi.org/10.1016/S0022-2836(03)00764-2">https://doi.org/10.1016/S0022-2836(03)00764-2</a>.</p>
</div>
<div id="ref-weinert2014">
<p>Weinert, Franz M., Robert C. Brewster, Mattias Rydenfelt, Rob Phillips, and Willem K. Kegel. 2014. “Scaling of Gene Expression with Transcription-Factor Fugacity.” <em>Physical Review Letters</em> 113 (25). <a href="https://doi.org/10.1103/PhysRevLett.113.258101">https://doi.org/10.1103/PhysRevLett.113.258101</a>.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure_phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phdchapter_02" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter II
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phdchapter_04" class="page-nav__item page-nav__item--next">
										Chapter IV
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd/" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/scripts.min.js"></script>




		</div><!-- /.js-footer-area -->
	</body>
</html>
