<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter 3</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd//favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd//doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd//doks-theme/assets/images/islands.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter 3</h2>
								
								
									<h1 class="hero-subheader__title">Unknown Knowns, Known Unknowns, and Unforeseen Consequences: Using Free Energy Shifts To Predict Mutant Phenotypes
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phd/chapter_02" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phd/chapter_04" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
						<div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
							<div class="align-container" data-mh>
								<div class="align-inner">
									<div class="hero-subheader__author">
										<p class="hero-subheader__author-title">
												Summary
												<i class="icon icon--chevron-down"></i>
										</p><!-- /.hero-subheader__author-title -->
										<p>
Here, I present a biophysical model of allosteric transcriptional regulation that directly links the location of a mutation within a repressor to the biophysical parameters that describe its behavior. Here, we explore the phenotypic space of a repressor with mutations in either the inducer binding or DNA binding domains. Using the LacI repressor in <i>Escherichia</i> <i>coli</i>, we make sharp, falsifiable predictions and use this framework to generate a null hypothesis for how double mutants behave, given knowledge of the single mutants. Linking mutations to the parameters which govern the system allows for quantitative predictions of how the free energy of the system changes as a result, permitting coarse graining of high-dimensional data into a single-parameter description of the mutational consequences.
</p>
									</div><!-- /.hero-subheader__author -->
								</div><!-- /.align-inner -->
							</div><!-- /.align-container -->
						</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/chapter_02" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter II
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_04" class="page-nav__item page-nav__item--next">
										Chapter IV
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<strong>Published as …</strong>
<hr/>
<p>A version of this chapter originally appeared as Chure, G, Razo-Mejia, M., Belliveau, N.M., Kaczmarek, Zofii A., Einav, T., Barnes, Stephanie L., Lewis, M., and Phillips, R. (2019). Predictive shifts in free energy couple mutations to their phenotypic consequences. Proceedings of the National Academies of Sciences 116(37) DOI: https://doi.org/10.1073/pnas.1907869116. G.C., M.R.M, N.M.B., Z.A.K., and S.L.B designed the experiments and collected and analyzed data. G.C. developed the theoretical treatment of free energy shifts. G.C., M.R.M, N.M.B., Z.A.K., T.E., S.L.B., and R.P. designed the research project. G.C. and R.P. wrote the paper. M.L. provided guidance and advice.</p>
<h2 id="abstract">Abstract</h2>
<p>Mutation is a critical mechanism by which evolution explores the functional landscape of proteins. Despite our ability to experimentally inflict mutations at will, it remains difficult to link sequence-level perturbations to systems-level responses. Here, we present a framework centered on measuring changes in the free energy of the system to link individual mutations in an allosteric transcriptional repressor to the parameters which govern its response. We find that the energetic effects of the mutations can be categorized into several classes which have characteristic curves as a function of the inducer concentration. We experimentally test these diagnostic predictions using the well-characterized LacI repressor of <em>Escherichia coli</em>, probing several mutations in the DNA binding and inducer binding domains. We find that the change in gene expression due to a point mutation can be captured by modifying only the model parameters that describe the respective domain of the wild-type protein. These parameters appear to be insulated, with mutations in the DNA binding domain altering only the DNA affinity and those in the inducer binding domain altering only the allosteric parameters. Changing these subsets of parameters tunes the free energy of the system in a way that is concordant with theoretical expectations. Finally, we show that the induction profiles and resulting free energies associated with pairwise double mutants can be predicted with quantitative accuracy given knowledge of the single mutants, providing an avenue for identifying and quantifying epistatic interactions.</p>
<h2 id="introduction">Introduction</h2>
<p>Thermodynamic treatments of transcriptional regulation have been fruitful in their ability to generate quantitative predictions of gene expression as a function of a minimal set of physically meaningful parameters <span class="citation" data-cites="ackers1982 buchler2003 vilar2003 garcia2011 daber2009a brewster2014 weinert2014 rydenfelt2014b razo-mejia2014 razo-mejia2018 bintu2005 bintu2005a kuhlman2007">(Ackers and Johnson <a href="#ref-ackers1982" role="doc-biblioref">1982</a>; Buchler, Gerland, and Hwa <a href="#ref-buchler2003" role="doc-biblioref">2003</a>; Vilar and Leibler <a href="#ref-vilar2003" role="doc-biblioref">2003</a>; Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Daber, Sharp, and Lewis <a href="#ref-daber2009a" role="doc-biblioref">2009</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Weinert et al. <a href="#ref-weinert2014" role="doc-biblioref">2014</a>; Rydenfelt et al. <a href="#ref-rydenfelt2014b" role="doc-biblioref">2014</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2014" role="doc-biblioref">2014</a>, <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Bintu, Buchler, Garcia, Gerland, Hwa, Kondev, and Phillips <a href="#ref-bintu2005" role="doc-biblioref">2005</a>; Bintu, Buchler, Garcia, Gerland, Hwa, Kondev, Kuhlman, et al. <a href="#ref-bintu2005a" role="doc-biblioref">2005</a>; Kuhlman et al. <a href="#ref-kuhlman2007" role="doc-biblioref">2007</a>)</span>. These models quantitatively describe numerous properties of input-output functions, such as the leakiness, saturation, dynamic range, steepness of response, and the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> – the concentration of inducer at which the response is half maximal. The mathematical forms of these phenotypic properties are couched in terms of a minimal set of experimentally accessible variables, such as the inducer concentration, transcription factor copy number, and the DNA sequence of the binding site (see Chapter 2 and <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>). While the amino acid sequence of the transcription factor is another controllable variable, it is seldom implemented in quantitative terms considering mutations with subtle changes in chemistry frequently yield unpredictable physiological consequences. In this work, we examine how a series of mutations in either the DNA binding or inducer binding domains of a transcriptional repressor influence the values of the biophysical parameters which govern its regulatory behavior.</p>
<p>     We build upon the results presented in Chapter 2 of this thesis and present a theoretical framework for understanding how mutations in the amino acid sequence of the repressor affect different parameters and alter the free energy of the system. We find that the parameters capturing the allosteric nature of the repressor, the repressor copy number, and the DNA binding specificity contribute independently to the free energy of the system with different degrees of sensitivity. Furthermore, changes restricted to one of these three groups of parameters result in characteristic changes in the free energy relative to the wild-type repressor, providing falsifiable predictions of how different classes of mutations should behave.</p>
<p>     Next, we test these descriptions experimentally using the well-characterized transcriptional repressor of the <em>lac</em> operon LacI in <em>E. coli</em> regulating expression of a fluorescent reporter. We introduce a series of point mutations in either the inducer binding or DNA binding domain. We then measure the full induction profile of each mutant, determine the minimal set of parameters that are affected by the mutation, and predict how each mutation tunes the free energy at different inducer concentrations, repressor copy numbers, and DNA binding strengths. We find in general that mutations in the DNA binding domain only influence DNA binding strength, and that mutations within the inducer binding domain affect only the parameters which dictate the allosteric response. The degree to which these parameters are insulated is notable, as the very nature of allostery suggests that all parameters are intimately connected, thus enabling binding events at one domain to be “sensed” by another.</p>
<p>     With knowledge of how a collection of DNA binding and inducer binding single mutants behave, we predict the induction profiles and the free energy changes of pairwise double mutants with quantitative accuracy. We find that the energetic effects of each individual mutation are additive, indicating that epistatic interactions are absent between the mutations examined here. Our model provides a means for identifying and quantifying the extent of epistatic interactions in a more complex set of mutations, and can shed light on how the protein sequence and general regulatory architecture coevolve.</p>
<h2 id="theoretical-model">Theoretical Model</h2>
<p>This work considers the inducible simple repression regulatory motif depicted in Fig. 1 (A) from a thermodynamic perspective which has been thoroughly dissected and tested experimentally <span class="citation" data-cites="garcia2011 brewster2014 razo-mejia2018">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> and is described in depth in Chapter 2. The result of this extensive theory-experiment dialogue is a succinct input-output function schematized in Fig. 1 (B) that computes the fold-change in gene expression relative to an unregulated promoter. This function is of the form <span id="eq:mut_foldchange"><br /><span class="math display">$$
\text{fold-change} = \left(1 + {R_A \over
N_{NS}}e^{-\beta\Delta\varepsilon_{RA}}\right)^{-1},
\qquad(1)$$</span><br /></span> where <span class="math inline"><em>R</em><sub><em>A</em></sub></span> is the number of active repressors per cell, <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of non-specific binding sites for the repressor, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is the binding energy of the repressor to its specific binding site relative to the non-specific background, and <span class="math inline"><em>β</em></span> is defined as <span class="math inline">${1 \over k_B T}$</span> where <span class="math inline"><em>k</em><sub><em>B</em></sub></span> is the Boltzmann constant and <span class="math inline"><em>T</em></span> is the temperature. While this theory requires knowledge of the number of <em>active</em> repressors, we often only know the total number <span class="math inline"><em>R</em></span> which is the sum total of active and inactive repressors. We can define a prefactor <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> which captures the allosteric nature of the repressor and encodes the probability that a repressor is in the active (repressive) state rather than the inactive state for a given inducer concentration <span class="math inline"><em>c</em></span>, namely, <span id="eq:mut_pact"><br /><span class="math display">$$
p_\text{act}(c) = {\left(1 + {c \over K_A}\right)^n \over \left(1 + {c \over
K_A}\right)^n + e^{-\beta\Delta\varepsilon_{AI}}\left(1 + {c \over
K_I}\right)^n}.
\qquad(2)$$</span><br /></span></p>
<p>Here, <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are the dissociation constants of the inducer to the active and inactive repressor, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the energetic difference between the repressor active and inactive states, and <span class="math inline"><em>n</em></span> is the number of allosteric binding sites per repressor molecule (<span class="math inline"><em>n</em> = 2</span> for LacI). With this in hand, we can define <span class="math inline"><em>R</em><sub><em>A</em></sub></span> in Eq. 1 as <span class="math inline"><em>R</em><sub><em>A</em></sub> = <em>p</em><sub>act</sub>(<em>c</em>)<em>R</em></span>.</p>
<figure>
<img src="ch3_fig1.png" id="fig:mut_induction_summary" data-short-caption="Summary of Chapter 2. A predictive framework for dissection of the simple repression motif" alt="" /><figcaption>Figure 1: <strong>A predictive framework for phenotypic and energetic dissection of the simple repression motif.</strong> (A) The inducible simple repression architecture. When in the active state, the repressor (red) binds the cognate operator sequence of the DNA (orange box) with high specificity, preventing transcription by occluding binding of the RNA polymerase to the promoter (blue rectangle). Upon addition of an inducer molecule, the inactive state (purple) becomes energetically preferable, and the repressor no longer binds the operator sequence with appreciable specificity. Once unbound from the operator, binding of the RNA polymerase (blue) is no longer blocked, and transcription can occur. (B) The simple repression input–output function for an allosteric repressor with two inducer binding sites. The key parameters are identified in speech bubbles. (C) The fold change in gene expression collapses as a function of the free energy. Panel (C, left) shows measurements of the fold change in gene expression as a function of inducer concentration from <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>. Points and errors correspond to the mean and SEM of at least 10 biological replicates. The thin lines represent the line of best fit given the model shown in (B). This model can be rewritten as a Fermi function with an energetic parameter <span class="math inline"><em>F</em></span>, which is the energetic difference between the repressor bound and unbound states of the promoter, schematized in C, Middle. The points in (C), Bottom correspond to the data shown in (C, left) collapsed onto a master curve defined by their calculated free energy <span class="math inline"><em>F</em></span>. The solid black line is the master curve defined by the Fermi function shown in (C, Middle). The <a href="https://github.com/gchure/phd/blob/master/src/chapter_03/code/ch3_fig1.py">Python code (<code>ch3_fig1.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     A key feature of Eq. 1 and Eq. 2 is that the diverse phenomenology of the gene expression induction profile can be collapsed onto a single master curve by rewriting the input-output function in terms of the free energy <span class="math inline"><em>F</em></span> also called the Bohr parameter <span class="citation" data-cites="phillips2015">(Phillips <a href="#ref-phillips2015" role="doc-biblioref">2015</a>)</span>, <span id="eq:mut_collapse"><br /><span class="math display">$$
\text{fold-change} = \frac{1}{1 + e^{-\beta F}},
\qquad(3)$$</span><br /></span> where <span id="eq:mut_bohr"><br /><span class="math display">$$
F = -k_BT \log p_\text{act}(c) - k_BT\log\left({R \over N_{NS}}\right) +
\Delta\varepsilon_{RA}.
\qquad(4)$$</span><br /></span> Hence, if different combinations of parameters yield the same free energy, they will give rise to the same fold-change in gene expression, enabling us to collapse multiple regulatory scenarios onto a single curve. This can be seen in Fig. 1 (C) where eighteen unique inducer titration profiles of a LacI simple repression architecture collected and analyzed in <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> collapse onto a single master curve. The tight distribution about this curve reveals that the fold-change across a variety of genetically distinct individuals can be adequately described by a small number of parameters. Beyond predicting the induction profiles of different strains, the method of data collapse inspired by Eq. 3 and Eq. 4 can be used as a tool to identify mechanistic changes in the regulatory architecture <span class="citation" data-cites="swem2008">(Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>)</span>. Similar data collapse approaches have been used previously in such a manner and have proved vital for distinguishing between changes in parameter values and changes in the fundamental behavior of the system <span class="citation" data-cites="swem2008 keymer2006">(Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>; Keymer et al. <a href="#ref-keymer2006" role="doc-biblioref">2006</a>)</span>.</p>
<p>     Assuming that a given mutation does not result in a non-functional protein, it is reasonable to say that any or all of the parameters in Eq. 1 can be affected by the mutation, changing the observed induction profile and therefore the free energy. To examine how the free energy of a mutant <span class="math inline"><em>F</em><sup>(mut)</sup></span> differs from that of the wild-type <span class="math inline"><em>F</em><sup>(wt)</sup></span>, we define <span class="math inline"><em>Δ</em><em>F</em> = <em>F</em><sup>(<em>m</em><em>u</em><em>t</em>)</sup> − <em>F</em><sup>(<em>w</em><em>t</em>)</sup></span>, which has the form <span id="eq:delF"><br /><span class="math display">$$
\begin{aligned}
\Delta F = -k_BT\log\left({p_\text{act}^\mathrm{(mut)}(c) \over  p_\text{act}^\mathrm{(wt)}(c)}\right) &amp;- k_BT \log\left({R^\mathrm{(mut)}\over R^\mathrm{(wt)}}\right)\\
 &amp;+ (\Delta\varepsilon_{RA}^\mathrm{(mut)}- \Delta\varepsilon_{RA}^\mathrm{(wt)}).
    \end{aligned}
\qquad(5)$$</span><br /></span> <span class="math inline"><em>Δ</em><em>F</em></span> describes how a mutation translates a point across the master curve shown in Fig. 1 (C). As we will show in the coming paragraphs (illustrated in Fig. 2), this formulation coarse grains the myriad parameters shown in Eq. 1 and Eq. 2 into three distinct quantities, each with different sensitivities to parametric changes. By examining how a mutation changes the <span class="math inline"><em>Δ</em><em>F</em></span> as a function of the inducer concentration, one can draw conclusions as to which parameters have been modified based solely on the shape of the curve. To help the reader understand how various perturbations to the parameters tune the free energy, we have hosted an interactive figure on the dedicated <a href="http://www.rpgroup.caltech.edu/mwc_mutants">website for the publication</a> which makes exploration of parameter space a simpler task.</p>
<p>     The first term in Eq. 5 is the log ratio of the probability of a mutant repressor being active relative to the wild type at a given inducer concentration <span class="math inline"><em>c</em></span>. This quantity defines how changes to any of the allosteric parameters – such as inducer binding constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> or active/inactive state energetic difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> – alter the free energy <span class="math inline"><em>F</em></span>, which can be interpreted as the free energy difference between the repressor bound and unbound states of the promoter. Fig. 2 (A) illustrates how perturbations to the inducer binding constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> alone alter the induction profiles and free energy as a function of the inducer concentration. In the limit where <span class="math inline"><em>c</em> = 0</span>, the values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> do not factor into the calculation of <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> given by Eq. 2 meaning that <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the lone parameter setting the residual activity of the repressor. Thus, if only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are altered by a mutation, then <span class="math inline"><em>Δ</em><em>F</em></span> should be <span class="math inline">0 <em>k</em><sub><em>B</em></sub><em>T</em></span> when <span class="math inline"><em>c</em> = 0</span>, illustrated by the overlapping red, purple, and grey curves in the right-hand plot of Fig. 2 (A). However, if <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is influenced by the mutation (either alone or in conjunction with <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>), the leakiness will change, resulting in a non-zero <span class="math inline"><em>Δ</em><em>F</em></span> when <span class="math inline"><em>c</em> = 0</span>. This is illustrated in Fig. 2 (B) where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the only parameter affected by the mutation.</p>
<p>     It is important to note that for a mutation which perturbs only the inducer binding constants, the dependence of <span class="math inline"><em>Δ</em><em>F</em></span> on the inducer concentration can be non-monotonic. While the precise values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> control the sensitivity of the repressor to inducer concentration, it is the ratio <span class="math inline"><em>K</em><sub><em>A</em></sub>/<em>K</em><sub><em>I</em></sub></span> that defines whether this non-monotonic behavior is observed. This can be seen more clearly when we consider the limit of saturating inducer concentration, <span id="eq:kaki_sat_c"><br /><span class="math display">$$
\lim\limits_{c \rightarrow \infty} \log\left({p_\text{act}^\mathrm{(mut)}\over
p_\text{act}^\mathrm{(wt)}}\right) \approx \log\left[{1 +
e^{-\beta\Delta\varepsilon_{AI}^\mathrm{(wt)}} \left({K_A^\mathrm{(wt)}\over
K_I^\mathrm{(wt)}}\right)^n \over 1 +
e^{-\beta\Delta\varepsilon_{AI}^\mathrm{(wt)}} \left({K_A^\mathrm{(mut)}\over
K_I^\mathrm{(mut)}}\right)^n}\right],
\qquad(6)$$</span><br /></span> which illustrates that <span class="math inline"><em>Δ</em><em>F</em></span> returns to zero at saturating inducer concentration when the ratio <span class="math inline"><em>K</em><sub><em>A</em></sub>/<em>K</em><sub><em>I</em></sub></span> is the same for both the mutant and wild-type repressors, so long as <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is unperturbed. Non-monotonicity can <em>only</em> be achieved by changing <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> and therefore serves as a diagnostic for classifying mutational effects reliant solely on measuring the change in free energy. A rigorous proof of this non-monotonic behavior given changing <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> can be found in supplemental Chapter 7.</p>
<p>     The second term in Eq. 5 captures how changes in the repressor copy number contributes to changes in free energy. It is important to note that this contribution to the free energy change depends on the total number of repressors in the cell, not just those in the active state. This emphasizes that changes in the expression of the repressor are energetically divorced from changes to the allosteric nature of the repressor. As a consequence, the change in free energy is constant for all inducer concentrations, as is schematized in Fig. 2 (C). Because the magnitude of the free energy shift logarithmically proportional to the relative change in repressor copy number, a mutation which increases expression from 1 to 10 repressors per cell is more impactful from an energetic standpoint (<span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em>log (10) ≈ 2.3 <em>k</em><sub><em>B</em></sub><em>T</em></span>) than an increase from 90 to 100 (<span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em>log (100/90) ≈ 0.1 <em>k</em><sub><em>B</em></sub><em>T</em></span>). Appreciable changes in the free energy only arise when variations in the repressor copy number are larger than or comparable to an order of magnitude. Changes of this magnitude are certainly possible from a single point mutation, as it has been shown that even synonymous substitutions can drastically change translation efficiency <span class="citation" data-cites="frumkin2018">(Frumkin et al. <a href="#ref-frumkin2018" role="doc-biblioref">2018</a>)</span>.</p>
<p>     The third and final term in Eq. 5 is the difference in the DNA binding energy between the mutant and wild-type repressors. All else being equal, if the mutated state binds more tightly to the DNA than the wild type (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>w</em><em>t</em>)</sup> &gt; <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>m</em><em>u</em><em>t</em>)</sup></span>), the net change in the free energy is negative, indicating that the repressor bound states become more energetically favorable due to the mutation. Much like in the case of changing repressor copy number, this quantity is independent of inducer concentration and is therefore also constant (Fig. 2 (D)). However, the magnitude of the change in free energy is linear with DNA binding affinity while it is logarithmic with respect to changes in the repressor copy number. Thus, to change the free energy by <span class="math inline">1 <em>k</em><sub><em>B</em></sub><em>T</em></span>, the repressor copy number must change by a factor of <span class="math inline"> ≈ 2.3</span> whereas the DNA binding energy must change by <span class="math inline">1 <em>k</em><sub><em>B</em></sub><em>T</em></span>.</p>
<p>     The unique behavior of each quantity in Eq. 5 and its sensitivity with respect to the parameters makes <span class="math inline"><em>Δ</em><em>F</em></span> useful as a diagnostic tool to classify mutations. Given a set of fold-change measurements, a simple rearrangement of Eq. 3 permits the direct calculation of the free energy, assuming that the underlying physics of the regulatory architecture has not changed. Thus, it becomes possible to experimentally test the general assertions made in Fig. 2.</p>
<figure>
<img src="ch3_fig2.png" id="fig:deltaF_theory" data-short-caption="Parametric changes due to mutations and their corresponding free-energy shifts." alt="" /><figcaption>Figure 2: <strong>Parametric changes due to mutations and the corresponding free-energy changes for (A) perturbations to <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, (B) changes to the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, (C) changes to repressor copy number, and (D) changes in DNA binding affinity</strong>. The first column schematizes the changed parameters and the second column reflects which quantity in Eq. 5 is affected. The third column shows representative induction profiles from mutants which have smaller (purple) and larger (orange) values for the parameters than the wild type (gray). The fourth and fifth columns illustrate how the free energy is changed as a result. Purple and red arrows indicate the direction in which the points are translated about the master curve. Three concentrations (points labeled 1, 2, and 3) are shown to illustrate how each point is moved in free-energy space. An interactive version of this figure can be found on the paper website (https://www.rpgroup.caltech.edu/mwc_mutants).</figcaption>
</figure>
<h2 id="results">Results</h2>
<h3 id="dna-binding-domain-mutants">DNA Binding Domain Mutants</h3>
<p>With this arsenal of analytic diagnostics, we can begin to explore the mutational space of the repressor and map these mutations to the biophysical parameters they control. As one of the most thoroughly studied transcription factors, LacI has been subjected to numerous crystallographic and mutational studies <span class="citation" data-cites="lewis1996 daber2009a daber2011">(Lewis et al. <a href="#ref-lewis1996" role="doc-biblioref">1996</a>; Daber, Sharp, and Lewis <a href="#ref-daber2009a" role="doc-biblioref">2009</a>; Daber, Sochor, and Lewis <a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span>. One such work generated a set of point mutations in the LacI repressor and examined the diversity of the phenotypic response to different allosteric effectors <span class="citation" data-cites="daber2011">(Daber, Sochor, and Lewis <a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span>. However, several experimental variables were unknown, precluding precise calculation of <span class="math inline"><em>Δ</em><em>F</em></span> as presented in the previous section. In <span class="citation" data-cites="daber2011">Daber, Sochor, and Lewis (<a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span>, the repressor variants and the fluorescence reporter were expressed from separate plasmids. As the copy numbers of these plasmids fluctuate in the population, both the population average repressor copy number and the number of regulated promoters were unknown. Both of these quantities have been shown previously to significantly alter the measured gene expression, and calculation of <span class="math inline"><em>Δ</em><em>F</em></span> is dependent on knowledge of their values. While the approach presented in <span class="citation" data-cites="daber2011">Daber, Sochor, and Lewis (<a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span> considers the Lac repressor as an MWC molecule, the copy numbers of the repressor and the reporter gene were swept into an effective parameter <span class="math inline"><em>R</em>/<em>K</em><sub><em>D</em><em>N</em><em>A</em></sub></span>, hindering our ability to distinguish between changes in repressor copy number or in DNA binding energy. To test our hypothesis of free energy differences resulting from various parameter perturbations, we used the data set in <span class="citation" data-cites="daber2011">Daber, Sochor, and Lewis (<a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span> as a guide and chose a subset of the mutations to quantitatively dissect. To control copy number variation, the mutant repressors and the reporter gene were integrated into the <em>E. coli</em> chromosome where the copy numbers are known and tightly controlled <span class="citation" data-cites="razo-mejia2018 garcia2011">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>. Furthermore, the mutations were paired with ribosomal binding sites where the level of translation of the wild-type repressor had been directly measured previously <span class="citation" data-cites="garcia2011">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>.</p>
<p>     We made three amino acid substitutions (Y17I, Q18A, and Q18M) that are critical for the DNA-repressor interaction. These mutations were introduced into the <em>lacI</em> sequence used in <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span> with four different ribosomal binding site sequences that were shown (via quantitative Western blotting) to tune the wild-type repressor copy number across three orders of magnitude. These mutant constructs were integrated into the <em>E. coli</em> chromosome harboring a Yellow Fluorescent Protein (YFP) reporter. The YFP promoter included the native O2 LacI operator sequence which the wild-type LacI repressor binds with high specificity (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>). The fold-change in gene expression for each mutant across twelve concentrations of IPTG was measured via flow cytometry. As we mutated only a single amino acid with the minimum number of base pair changes to the codons from the wild-type sequence, we find it unlikely that the repressor copy number was drastically altered from those reported in <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span> for the wild-type sequence paired with the same ribosomal binding site sequence. In characterizing the effects of these DNA binding mutations, we take the repressor copy number to be unchanged. Any error introduced by this assumption should be manifest as a larger than predicted systematic shift in the free energy change when the repressor copy number is varied.</p>
<figure>
<img src="ch3_fig3.png" id="fig:DNA_muts" data-short-caption="Induction profiles and free-energy differences of DNA binding domain mutations in the *lac* repressor." alt="" /><figcaption>Figure 3: <strong>Induction profiles and free-energy differences of DNA binding domain mutations.</strong> Each column corresponds to the highlighted mutant at the top of the figure. Each strain was paired with the native O2 operator sequence. Open points correspond to the strain for each mutant from which the DNA binding energy was estimated. (A) Induction profiles of each mutant at four different repressor copy numbers as a function of the inducer concentration. Shaded regions demarcate the 95% credible region of the induction profile generated by the estimated DNA binding energy. (B) Data collapse of all points for each mutant shown in A using only the DNA binding energy estimated from a single repressor copy number. Points correspond to the average fold change in gene expression of 6–10 biological replicates. Error bars are SEM. Where error bars are not visible, the relative error in measurement is smaller than the size of the marker. (C) The change in the free energy resulting from each mutation as a function of the inducer concentration. Points correspond to the median of the marginal posterior distribution for the free energy. Error bars represent the upper and lower bounds of the 95% credible region. Points in A at the detection limits of the flow cytometer (near fold-change values of 0 and 1) were neglected for calculation of the <span class="math inline"><em>Δ</em><em>F</em></span>. The IPTG concentration is shown on a symmetric log scale with linear scaling ranging from 0 to <span class="math inline">10<sup> − 2</sup></span> <span class="math inline"><em>μ</em></span>M and log scaling elsewhere. The shaded red lines in C correspond to the 95% credible region of our predictions for <span class="math inline"><em>Δ</em><em>F</em></span> based solely on estimation of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> from the strain with <span class="math inline"><em>R</em> = 260</span> repressors per cell. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_03/code/ch3_fig3.py">Python code (<code>ch3_fig3.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     A naïve hypothesis for the effect of a mutation in the DNA binding domain is that <em>only</em> the DNA binding energy is affected. This hypothesis appears to contradict the core principle of allostery in that ligand binding in one domain influences binding in another, suggesting that changing parameter modifies them all. The characteristic curves summarized in Fig. 2 give a means to discriminate between these two hypotheses by examining the change in the free energy. Using a single induction profile (white-faced points in Fig. 3), we estimated the DNA binding energy using Bayesian inferential methods, the details of which are thoroughly discussed in the Materials &amp; Methods as well as in the supplemental Chapter 7. The shaded red region for each mutant in Fig. 3 represents the 95% credible region of this fit whereas all other shaded regions are 95% credible regions of the predictions for other repressor copy numbers. We find that redetermining only the DNA binding energy accurately captures the majority of the induction profiles, indicating that other parameters are unaffected. One exception is for the lowest repressor copy numbers (<span class="math inline"><em>R</em> = 60</span> and <span class="math inline"><em>R</em> = 124</span> per cell) of mutant Q18A at low concentrations of IPTG. However, we note that this disagreement is comparable to that observed for the wild-type repressor binding to the weakest operator in <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>, illustrating that our model is imperfect in characterizing weakly repressing architectures. Including other parameters in the fit (such as <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>) does not significantly improve the accuracy of the predictions. Furthermore, the magnitude of this disagreement also depends on the choice of the fitting strain (see supplemental Chapter 7).</p>
<p>     Mutations Y17I and Q18A both weaken the affinity of the repressor to the DNA relative to the wild type strain with binding energies of <span class="math inline"> − 9.9<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span> and <span class="math inline"> − 11.0<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span>, respectively. Here we report the median of the inferred posterior probability distribution with the superscripts and subscripts corresponding to the upper and lower bounds of the 95% credible region. These binding energies are comparable to that of the wild-type repressor affinity to the native LacI operator sequence O3, with a DNA binding energy of <span class="math inline"> − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The mutation Q18M increases the strength of the DNA-repressor interaction relative to the wild-type repressor with a binding energy of <span class="math inline"> − 15.43<sub> − 0.06</sub><sup> + 0.07</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span>, comparable to the affinity of the wild-type repressor to the native O1 operator sequence (<span class="math inline"> − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span>). It is notable that a single amino acid substitution of the repressor is capable of changing the strength of the DNA binding interaction well beyond that of many single base-pair mutations in the operator sequence <span class="citation" data-cites="barnes2019">(Barnes et al. <a href="#ref-barnes2019" role="doc-biblioref">2019</a>)</span>.</p>
<p>     Using the new DNA binding energies, we can collapse all measurements of fold-change as a function of the free energy as shown in Fig. 3 (B). This allows us to test the diagnostic power of the decomposition of the free energy described in Fig. 2. To compute the <span class="math inline"><em>Δ</em><em>F</em></span> for each mutation, we inferred the observed mean free energy of the mutant strain for each inducer concentration and repressor copy number (see Materials &amp; Methods as well as the supplemental Chapter 7 for a detailed explanation of the inference). We note that in the limit of extremely low or high fold-change, the inference of the free energy is either over- or under-estimated, respectively, introducing a systematic error. Thus, points which are close to these limits are omitted in the calculation of <span class="math inline"><em>Δ</em><em>F</em></span>. We direct the reader to the supplemental Chapter 7 for a detailed discussion of this systematic error. With a measure of <span class="math inline"><em>F</em><sup>(<em>m</em><em>u</em><em>t</em>)</sup></span> for each mutant at each repressor copy number, we compute the difference in free energy relative to the wild-type strain with the same repressor copy number and operator sequence, restricting all variability in <span class="math inline"><em>Δ</em><em>F</em></span> solely to changes in <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>.</p>
<p>     The change in free energy for each mutant is shown in Fig. 3 (C). It can be seen that the <span class="math inline"><em>Δ</em><em>F</em></span> for each mutant is constant as a function of the inducer concentration and is concordant with the prediction generated from fitting <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> to a single repressor copy number (orange lines Fig. 3 (C)]) This is in line with the predictions outlined in Fig. 2 (C) and (D), indicating that the allosteric parameters are “insulated,” meaning they are not affected by the DNA binding domain mutations. As the <span class="math inline"><em>Δ</em><em>F</em></span> for all repressor copy numbers collapses onto the prediction, we can say that the expression of the repressor itself is the same or comparable with that of the wild type. If the repressor copy number were perturbed in addition to <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, one would expect a shift away from the prediction that scales logarithmically with the change in repressor copy number. However, as the <span class="math inline"><em>Δ</em><em>F</em></span> is approximately the same for each repressor copy number, it can be surmised that the mutation does not significantly change the expression or folding efficiency of the repressor itself. These results allow us to state that the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is the only parameter modified by the DNA mutants examined.</p>
<h3 id="inducer-binding-domain-mutants">Inducer Binding Domain Mutants</h3>
<p>Much as in the case of the DNA binding mutants, we cannot safely assume <em>a priori</em> that a given mutation in the inducer binding domain affects only the inducer binding constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. While it is easy to associate the inducer binding constants with the inducer binding domain, the critical parameter in our allosteric model <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is harder to restrict to a single spatial region of the protein. As <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> are all parameters dictating the allosteric response, we consider two hypotheses in which inducer binding mutations alter either all three parameters or only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>.</p>
<p>     We made four point mutations within the inducer binding domain of LacI (F161T, Q291V, Q291R, and Q291K) that have been shown previously to alter binding to multiple allosteric effectors <span class="citation" data-cites="daber2009a">(Daber, Sharp, and Lewis <a href="#ref-daber2009a" role="doc-biblioref">2009</a>)</span>. In contrast to the DNA binding domain mutants, we paired the inducer binding domain mutations with the three native LacI operator sequences (which have various affinities for the repressor) and a single ribosomal binding site sequence. This ribosomal binding site sequence, as reported in <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>, expresses the wild-type LacI repressor to an average copy number of approximately <span class="math inline">260</span> per cell. As the free energy differences resulting from point mutations in the DNA binding domain can be described solely by changes to <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, we continue under the assumption that the inducer binding domain mutations do not significantly alter the repressor copy number.</p>
<figure>
<img src="ch3_fig4.png" id="fig:IND_muts" data-short-caption="Induction profiles and free-energy differences of inducer binding domain mutants." alt="" /><figcaption>Figure 4: <strong>Induction profiles and free-energy differences of inducer binding domain mutants.</strong> Open points represent the strain to which the parameters were fit — namely, the O2 operator sequence. Each column corresponds to the mutant highlighted at the top of the figure. All strains have <span class="math inline"><em>R</em> = 260</span> per cell. (A) The fold change in gene expression as a function of the inducer concentration for three operator sequences of varying strength. Dashed lines correspond to the curve of best fit resulting from fitting <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> alone. Shaded curves correspond to the 95% credible region of the induction profile determined from fitting <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. Points correspond to the mean measurement of 6–12 biological replicates. Error bars are the SEM. (B) Points in A collapsed as a function of the free energy calculated from redetermining <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. (C) Change in free energy resulting from each mutation as a function of the inducer concentration. Points correspond to the median of the posterior distribution for the free energy. Error bars represent the upper and lower bounds of the 95% credible region. Shaded curves are the predictions. IPTG concentration is shown on a symmetric log scaling axis with the linear region spanning from 0 to <span class="math inline">10<sup> − 2</sup></span> <span class="math inline"><em>μ</em></span>M and log scaling elsewhere. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_03/code/ch3_fig4.py">Python code (<code>ch3_fig4.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     The induction profiles for these four mutants are shown in Fig. 4 (A). Of the mutations chosen, Q291R and Q291K appear to have the most significant impact, with Q291R abolishing the characteristic sigmoidal titration curve entirely. It is notable that both Q291R and Q291K have elevated expression in the absence of inducer compared to the other two mutants paired with the same operator sequence. Panel (A) in Fig. 2 illustrates that if only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> were being affected by the mutations, the fold-change should be identical for all mutants in the absence of inducer. This discrepancy in the observed leakiness immediately suggests that more than <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are affected for Q291K and Q291R.</p>
<p>     Using a single induction profile for each mutant (shown in Fig. 4 as white-faced circles), we inferred the parameter combinations for both hypotheses and drew predictions for the induction profiles with other operator sequences. We find that the simplest hypothesis (in which only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are altered) does not permit accurate prediction of most induction profiles. These curves, shown as dotted lines in Fig. 4 (A), fail spectacularly in the case of Q291R and Q291K, and undershoot the observed profiles for F161T and Q291V, especially when paired with the weak operator sequence O3. The change in the leakiness for Q291R and Q291K is particularly evident as the expression at <span class="math inline"><em>c</em> = 0</span> should be identical to the wild-type repressor under this hypothesis. Altering only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> is not sufficient to accurately predict the induction profiles for F161T and Q291V, but not to the same degree as Q291K and Q291R. The disagreement is most evident for the weakest operator O3 green lines in 4 (A), though we have discussed previously that the induction profiles for weak operators are difficult to accurately describe and can result in comparable disagreement for the wild-type repressor <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>.</p>
<p>     Including <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> as a perturbed parameter in addition to <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> improves the predicted profiles for all four mutants. By fitting these three parameters to a single strain, we are able to accurately predict the induction profiles of other operators as seen by the shaded lines in Fig. 4 (A). With these modified parameters, all experimental measurements collapse as a function of their free energy as prescribed by Eq. 3 (Fig. 4 (B)). All four mutations significantly diminish the binding affinity of both states of the repressor to the inducer, as seen by the estimated parameter values reported in Table 3.1. As evident in the data alone, Q291R abrogates inducibility outright (<span class="math inline"><em>K</em><sub><em>A</em></sub> ≈ <em>K</em><sub><em>I</em></sub></span>). For Q291K, the active state of the repressor can no longer bind inducer whereas the inactive state binds with weak affinity. The remaining two mutants, Q291V and F161T, both show diminished binding affinity of the inducer to both the active and inactive states of the repressor relative to the wild-type.</p>
<table>
<caption>Inferred values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> for inducer binding mutants</caption>
<thead>
<tr class="header">
<th style="text-align: left;">Mutant</th>
<th style="text-align: center;"><span class="math inline"><em>K</em><sub><em>A</em></sub></span></th>
<th style="text-align: center;"><span class="math inline"><em>K</em><sub><em>I</em></sub></span></th>
<th style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> [<span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span>]</th>
<th style="text-align: right;">Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">WT</td>
<td style="text-align: center;"><span class="math inline">139<sub> − 22</sub><sup> + 29</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">0.53<sub> − 0.04</sub><sup> + 0.04</sup> <em>μ</em></span>M</td>
<td style="text-align: center;">4.5</td>
<td style="text-align: right;"><span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: left;">F161T</td>
<td style="text-align: center;"><span class="math inline">165<sub> − 65</sub><sup> + 90</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">3<sub> − 3</sub><sup> + 6</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">1<sub> − 2</sub><sup> + 5</sup></span></td>
<td style="text-align: right;">This study</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Q291V</td>
<td style="text-align: center;"><span class="math inline">650<sub> − 250</sub><sup> + 450</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">8<sub> − 8</sub><sup> + 8</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">3<sub> − 3</sub><sup> + 6</sup></span></td>
<td style="text-align: right;">This study</td>
</tr>
<tr class="even">
<td style="text-align: left;">Q291K</td>
<td style="text-align: center;"><span class="math inline"> &gt; 1</span> mM</td>
<td style="text-align: center;"><span class="math inline">310<sub> − 60</sub><sup> + 70</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline"> − 3.11<sub> − 0.07</sub><sup> + 0.07</sup></span></td>
<td style="text-align: right;">This study</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Q291R</td>
<td style="text-align: center;"><span class="math inline">9<sub> − 9</sub><sup> + 20</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">8<sub> − 8</sub><sup> + 20</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline"> − 2.35<sub> − 0.09</sub><sup> + 0.01</sup></span></td>
<td style="text-align: right;">This study</td>
</tr>
</tbody>
</table>
<p>     Given the collection of fold-change measurements, we computed the <span class="math inline"><em>Δ</em><em>F</em></span> relative to the wild-type strain with the same operator and repressor copy number. This leaves differences in <span class="math inline"><em>p</em><sub><em>a</em><em>c</em><em>t</em></sub>(<em>c</em>)</span> as the sole contributor to the free energy difference, assuming our hypothesis that <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> are the only perturbed parameters is correct. The change in free energy can be seen in Fig. 4 (C). For all mutants, the free energy difference inferred from the observed fold-change measurements falls within error of the predictions generated under the hypothesis that <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> are all affected by the mutation (shaded curves in Fig. 4 (C)). The profile of the free energy change exhibits some of the rich phenomenology illustrated in Fig. 2 (A) and (B). Q291K, F161T, and Q291V exhibit a non-monotonic dependence on the inducer concentration, a feature that can only appear when <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are altered. The non-zero <span class="math inline"><em>Δ</em><em>F</em></span> at <span class="math inline"><em>c</em> = 0</span> for Q291R and Q291K coupled with an inducer concentration dependence is a telling sign that <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> must be significantly modified. This shift in <span class="math inline"><em>Δ</em><em>F</em></span> is positive in all cases, indicating that <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> must have decreased, and that the inactive state has become more energetically favorable for these mutants than for the wild-type protein. Indeed the estimates for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> (Table 3.1) reveal both mutations Q291R and Q291K make the inactive state more favorable than the active state. Thus, for these two mutations, only <span class="math inline"> ≈ 10%</span> of the repressors are active in the absence of inducer, whereas the basal active fraction is <span class="math inline"> ≈ 99%</span> for the wild-type repressor <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>.</p>
<p>     We note that the parameter values reported here disagree with those reported in <span class="citation" data-cites="daber2011">Daber, Sochor, and Lewis (<a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span>. This disagreement stems from different assumptions regarding the residual activity of the repressor in the absence of inducer and the parametric degeneracy of the MWC model without a concrete independent measure of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. A detailed discussion of the difference in parameter values between our previous work <span class="citation" data-cites="garcia2011 razo-mejia2018">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>, that of <span class="citation" data-cites="daber2011">Daber, Sochor, and Lewis (<a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span>, and those of other seminal works can be found in the supplemental Chapter 7.</p>
<p>     Taken together, these parametric changes diminish the response of the regulatory architecture as a whole to changing inducer concentrations. They furthermore reveal that the parameters which govern the allosteric response are interdependent and no single parameter is insulated from the others. However, as <em>only</em> the allosteric parameters are changed, one can say that the allosteric parameters as a whole are insulated from the other components which define the regulatory response, such as repressor copy number and DNA binding affinity.</p>
<h3 id="predicting-effects-of-pairwise-double-mutations">Predicting Effects of Pairwise Double Mutations</h3>
<p>Given full knowledge of each mutation individually, we can draw predictions of the behavior of the pairwise double mutants with no free parameters based on the simplest null hypothesis of no epistasis. The formalism of <span class="math inline"><em>Δ</em><em>F</em></span> defined by Eq. 5 explicitly states that the contribution to the free energy of the system from the difference in DNA binding energy and the allosteric parameters are strictly additive. Thus, deviations from the predicted change in free energy would suggest epistatic interactions between the two mutations.</p>
<p>      To test this additive model, we constructed nine double mutant strains, each having a unique inducer binding (F161T, Q291V, Q291K) and DNA binding mutation (Y17I, Q18A, Q18M). To make predictions with an appropriate representation of the uncertainty, we computed a large array of induction profiles given random draws from the posterior distribution for the DNA binding energy (determined from the single DNA binding mutants) as well as from the joint posterior for the allosteric parameters (determined from the single inducer binding mutants). These predictions, shown in Fig. 5 (A) and (B) as shaded blue curves, capture all experimental measurements of the fold-change (Fig. 5 (A)) and the inferred difference in free energy (Fig. 5 (B)). The latter indicates that there are no epistatic interactions between the mutations queried in this work, though if there were, systematic deviations from these predictions would shed light on how the epistasis is manifested.</p>
<p>     The precise agreement between the predictions and measurements for Q291K paired with either Q18A or Q18M is striking as Q291K drastically changed <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> in addition to <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. Our ability to predict the induction profile and free energy change underscores the extent to which the DNA binding energy and the allosteric parameters are insulated from one another. Despite this insulation, the repressor still functions as an allosteric molecule, emphasizing that the mutations we have inserted do not alter the pathway of communication between the two domains of the protein. As the double mutant Y17I-Q291K exhibits fold-change of approximately <span class="math inline">1</span> across all IPTG concentrations (Fig. 5 (A)), these mutations in tandem make repression so weak that it is beyond the limits which are detectable by our experiments. As a consequence, we are unable to estimate <span class="math inline"><em>Δ</em><em>F</em></span> nor experimentally verify the corresponding prediction (grey box in Fig. 5 (B)). However, as the predicted fold-change in gene expression is also approximately <span class="math inline">1</span> for all <span class="math inline"><em>c</em></span>, we believe that the prediction shown for <span class="math inline"><em>Δ</em><em>F</em></span> is likely accurate. One would be able to infer the <span class="math inline"><em>Δ</em><em>F</em></span> to confirm these predictions using a more sensitive method for measuring the fold-change, such as single-cell microscopy or colorimetric assays.</p>
<figure>
<img src="ch3_fig5.png" id="fig:dbl_muts" data-short-caption="Induction and free-energy profiles of DNA binding and inducer binding double mutants." alt="" /><figcaption>Figure 5: <strong>Induction and free-energy profiles of DNA binding and inducer binding double mutants.</strong> (A) Fold change in gene expression for each double mutant as a function of IPTG. Points and errors correspond to the mean and standard error of 6–10 biological replicates. Where not visible, error bars are smaller than the corresponding marker. Shaded regions correspond to the 95% credible region of the prediction given knowledge of the single mutants. These were generated by drawing <span class="math inline">10<sup>4</sup></span> samples from the <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> posterior distribution of the single DNA binding domain mutants and the joint probability distribution of <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> from the single inducer binding domain mutants. (B) The difference in free energy of each double mutant as a function of the reference free energy. Points and errors correspond to the median and bounds of the 95% credible region of the posterior distribution for the inferred <span class="math inline"><em>Δ</em><em>F</em></span>. Shaded regions are the predicted change in free energy, generated in the same manner as the shaded lines in (A). All measurements were taken from a strain with 260 repressors per cell paired with a reporter with the native O2 LacI operator sequence. In all plots, the IPTG concentration is shown on a symmetric log axis with linear scaling between 0 and <span class="math inline">10<sup> − 2</sup> <em>μ</em></span>M and log scaling elsewhere. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_03/code/ch3_fig5.py">Python code (<code>ch3_fig5.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<p>Allosteric regulation is often couched as “biological action at a distance.” Despite extensive knowledge of protein structure and function, it remains difficult to translate the coordinates of the atomic constituents of a protein to the precise parameter values which define the functional response, making each mutant its own intellectual adventure. Bioinformatic approaches to understanding the sequence-structure relationship have permitted us to examine how the residues of allosteric proteins evolve, revealing conserved regions which hint to their function. Co-evolving residues reveal sectors of conserved interactions which traverse the protein that act as the allosteric communication channel between domains <span class="citation" data-cites="suel2003 mclaughlinjr2012 reynolds2011">(Süel et al. <a href="#ref-suel2003" role="doc-biblioref">2003</a>; McLaughlin Jr et al. <a href="#ref-mclaughlinjr2012" role="doc-biblioref">2012</a>; Reynolds, McLaughlin, and Ranganathan <a href="#ref-reynolds2011" role="doc-biblioref">2011</a>)</span>. Elucidating these sectors has advanced our understanding of how distinct domains “talk” to one another and has permitted direct engineering of allosteric responses into non-allosteric enzymes <span class="citation" data-cites="poelwijk2011a raman2016">(Poelwijk et al. <a href="#ref-poelwijk2011a" role="doc-biblioref">2011</a>; Raman, White, and Ranganathan <a href="#ref-raman2016" role="doc-biblioref">2016</a>)</span>. Even so, we are left without a quantitative understanding of how these admittedly complex networks set the energetic difference between active and inactive states or how a given mutation influences binding affinity. In this context, a biophysical model in which the various parameters are intimately connected to the molecular details can be of use and can lead to quantitative predictions of the interplay between amino-acid identity and system-level response.</p>
<p>     By considering how each parameter contributes to the observed change in free energy, we are able to tease out different classes of parameter perturbations which result in stereotyped responses to changing inducer concentration. These characteristic changes to the free energy can be used as a diagnostic tool to classify mutational effects. For example, we show in Fig. 2 that modulating the inducer binding constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> results in non-monotonic free energy changes that are dependent on the inducer concentration, a feature observed in the inducer binding mutants examined in this work. Simply looking at the inferred <span class="math inline"><em>Δ</em><em>F</em></span> as a function of inducer concentration, which requires no fitting of the biophysical parameters, indicates that <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> must be modified considering those are the only parameters which can generate such a response.</p>
<p>     Another key observation is that a perturbation to only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> requires that the <span class="math inline"><em>Δ</em><em>F</em> = 0 <em>k</em><sub><em>B</em></sub><em>T</em></span> at <span class="math inline"><em>c</em> = 0</span>. Deviations from this condition imply that more than the inducer binding constants must have changed. If this shift in <span class="math inline"><em>Δ</em><em>F</em></span> off of <span class="math inline">0 <em>k</em><sub><em>B</em></sub><em>T</em></span> at <span class="math inline"><em>c</em> = 0</span> is not constant across all inducer concentrations, we can surmise that the energy difference between the allosteric states <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> must also be modified. We again see this effect for all of our inducer mutants. By examining the inferred <span class="math inline"><em>Δ</em><em>F</em></span>, we can immediately say that in addition to <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> must decrease relative to the wild-type value as <span class="math inline"><em>Δ</em><em>F</em> &gt; 0</span> at <span class="math inline"><em>c</em> = 0</span>. When the allosteric parameters are fit to the induction profiles, we indeed see that this is the case, with all four mutations decreasing the energy gap between the active and inactive states. Two of these mutations, Q291R and Q291K, make the inactive state of the repressor <em>more</em> stable than the active state, which is not the case for the wild-type repressor <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>.</p>
<p>      Our formulation of <span class="math inline"><em>Δ</em><em>F</em></span> indicates that shifts away from <span class="math inline">0 <em>k</em><sub><em>B</em></sub><em>T</em></span> that are independent of the inducer concentration can only arise from changes to the repressor copy number and/or DNA binding specificity, indicating that the allosteric parameters are untouched. We see that for three mutations in the DNA binding domain, <span class="math inline"><em>Δ</em><em>F</em></span> is the same irrespective of the inducer concentration. Measurements of <span class="math inline"><em>Δ</em><em>F</em></span> for these mutants with repressor copy numbers across three orders of magnitude yield approximately the same value, revealing that <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is the sole parameter altered via the mutations.</p>
<p>     We note that the conclusions stated above can be qualitatively drawn without resorting to fitting various parameters and measuring the goodness-of-fit. Rather, the distinct behavior of <span class="math inline"><em>Δ</em><em>F</em></span> is sufficient to determine which parameters are changing. Here, these conclusions are quantitatively confirmed by fitting these parameters to the induction profile, which results in accurate predictions of the fold-change and <span class="math inline"><em>Δ</em><em>F</em></span> for nearly every strain across different mutations, repressor copy numbers, and operator sequence, all at different inducer concentrations. With a collection of evidence as to what parameters are changing for single mutations, we put our model to the test and drew predictions of how double mutants would behave both in terms of the titration curve and free energy profile.</p>
<p>     A hypothesis that arises from our formulation of <span class="math inline"><em>Δ</em><em>F</em></span> is that a simple summation of the energetic contribution of each mutation should be sufficient to predict the double mutants (as long as they are in separate domains). We find that such a calculation permits precise and accurate predictions of the double mutant phenotypes, indicating that there are no epistatic interactions between the mutations examined in this work. With an expectation of what the free energy differences should be, epistatic interactions could be understood by looking at how the measurements deviate from the prediction. For example, if epistatic interactions exist which appear as a systematic shift from the predicted <span class="math inline"><em>Δ</em><em>F</em></span> independent of inducer concentration, one could conclude that DNA binding energy is not equal to that of the single mutation in the DNA binding domain alone. Similarly, systematic shifts that are dependent on the inducer concentration (i.e. not constant) indicate that the allosteric parameters must be influenced. If the expected difference in free energy is equal to <span class="math inline">0 <em>k</em><sub><em>B</em></sub><em>T</em></span> when <span class="math inline"><em>c</em> = 0</span>, one could surmise that the modified parameter must not be <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> nor <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> as these would both result in a shift in leakiness, indicating that <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are further modified.</p>
<p>     Ultimately, we present this work as a proof-of-principle for using biophysical models to investigate how mutations influence the response of allosteric systems. We emphasize that such a treatment allows one to boil down the complex phenotypic responses of these systems to a single-parameter description which is easily interpretable as a free energy. The general utility of this approach is illustrated in Fig. 6 where gene expression data from previous work along with all of the measurements presented in this work collapse onto the master curve defined by Eq. 3. While our model coarse grains many of the intricate details of transcriptional regulation into two states (one in which the repressor is bound to the promoter and one where it is not), it is sufficient to describe a swath of regulatory scenarios. As discussed in the supplemental Chapter 7, any architecture in which the transcription-factor bound and transcriptionally active states of the promoter can be separated into two distinct coarse-grained states can be subjected to such an analysis.</p>
<p>     Given enough parametric knowledge of the system, it becomes possible to examine how modifications to the parameters move the physiological response along this reduced one-dimensional parameter space. This approach offers a glimpse at how mutational effects can be described in terms of energy rather than Hill coefficients and arbitrary prefactors. While we have explored a very small region of sequence space in this work, coupling of this approach with high-throughput sequencing-based methods to query a library of mutations within the protein will shed light on the phenotypic landscape centered at the wild-type sequence. Furthermore, pairing libraries of protein and operator sequence mutants will provide insight as to how the protein and regulatory sequence coevolve, a topic rich with opportunity for a dialogue between theory and experiment.</p>
<figure>
<img src="ch3_fig6.png" id="fig:all_data_collapse" data-short-caption="Data collapse of the simple repression regulatory architecture. All data are means of biological replicates." alt="" /><figcaption>Figure 6: <strong>Data collapse of the simple repression regulatory architecture.</strong> All data are means of biological replicates. Where present, error bars correspond to the standard error of the mean of five to fifteen biological replicates. Red triangles indicate data from <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span> obtained by colorimetric assays. Blue squares are data from <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> acquired from video microscopy. Green circles are data from <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>. obtained via flow cytometry. All other symbols correspond to the work presented here. An interactive version of this figure can be found on the <a href="https://www.rpgroup.caltech.edu/mwc_mutants">website</a> associated with the publication of this chapter where the different data sets can be viewed in more detail. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_03/code/ch3_fig6.py">Python code (<code>ch3_fig6.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="materials-methods">Materials &amp; Methods</h2>
<h3 id="bacterial-strains-and-dna-constructs">Bacterial Strains and DNA Constructs</h3>
<p>All wild-type strains from which the mutants were derived were generated in previous work from the Phillips group <span class="citation" data-cites="garcia2011 razo-mejia2018">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>. Briefly, mutations were first introduced into the <em>lacI</em> gene of our pZS3*1-lacI plasmid <span class="citation" data-cites="garcia2011">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span> using a combination of overhang PCR Gibson assembly as well as QuickChange mutagenesis (Agligent Technologies). The oligonucleotide sequences used to generate each mutant as well as the method are provided in the supplemental Chapter 7.</p>
<p>     For mutants generated through overhang PCR and Gibson assembly, oligonucleotide primers were purchased containing an overhang with the desired mutation and used to amplify the entire plasmid. Using the homology of the primer overhang, Gibson assembly was performed to circularize the DNA prior to electroporation into MG1655 <em>E. coli</em> cells. Integration of LacI mutants was performed with <span class="math inline"><em>λ</em></span> Red recombineering as described in <span class="citation" data-cites="sharan2009">Sharan et al. (<a href="#ref-sharan2009" role="doc-biblioref">2009</a>)</span> and <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>.</p>
<p>      The mutants studied in this work were chosen from data reported in <span class="citation" data-cites="daber2011">Daber, Sochor, and Lewis (<a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span>. In selecting mutations, we looked for mutants which suggested moderate to strong deviations from the behavior of the wild-type repressor. We note that the variant of LacI used in this work has an additional three amino acids (Met-Val-Asn) added to the N-terminus than the canonical LacI sequence reported in <span class="citation" data-cites="farabaugh1978">Farabaugh (<a href="#ref-farabaugh1978" role="doc-biblioref">1978</a>)</span>. To remain consistent with the field, we have identified the mutations with respect to their positions in the canonical sequence and those in <span class="citation" data-cites="daber2011">Daber, Sochor, and Lewis (<a href="#ref-daber2011" role="doc-biblioref">2011</a>)</span>. However, their positions in the raw data files correspond to that of our LacI variant and is noted in the <code>README</code> files associated with the data.</p>
<h3 id="flow-cytometry">Flow Cytometry</h3>
<p>All fold-change measurements were performed on a MACSQuant flow cytometer as described in <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>. Briefly, saturated overnight cultures 500 <span class="math inline"><em>μ</em></span>L in volume were grown in deep-well 96-well plates covered with a breathable nylon cover (Lab Pak - Nitex Nylon, Sefar America, Cat. No. 241205). After approximately 12 to 15 hr, the cultures reached saturation and were diluted 1000-fold into a second 2 mL 96-deep-well plate where each well contained 500 <span class="math inline"><em>μ</em></span>L of M9 minimal media supplemented with 0.5% w/v glucose (anhydrous D-Glucose, Macron Chemicals) and the appropriate concentration of IPTG (Isopropyl <span class="math inline"><em>β</em></span>-D-1-thiogalactopyranoside, Dioxane Free, Research Products International). These were sealed with a breathable cover and were allowed to grow for approximately 8 hours until the OD<span class="math inline"><em></em><sub>600nm</sub> ≈ 0.3</span>. Cells were then diluted ten-fold into a round-bottom 96-well plate (Corning Cat. No. 3365) containing 90 <span class="math inline"><em>μ</em></span>L of M9 minimal media supplemented with 0.5% w/v glucose along with the corresponding IPTG concentrations.</p>
<p>     The flow cytometer was calibrated prior to use with MACSQuant Calibration Beads (Cat. No. 130-093-607). During measurement, the cultures were held at approximately 4<span class="math inline"><em></em><sup>∘</sup></span> C by placing the 96-well plate on a MACSQuant ice block. All fluorescence measurements were made using a 488 nm excitation wavelength with a 525/50 nm emission filter. The photomultiplier tube voltage settings for the instrument are the same as those used in <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>, and are listed in supplemental Chapter 6.</p>
<p>     The data were processed using an automatic unsupervised gating procedure based on the front and side-scattering values, where we fit a two-dimensional Gaussian function to the <span class="math inline">log<sub>10</sub></span> forward-scattering (FSC) and the <span class="math inline">log<sub>10</sub></span> side-scattering (SSC) data. Here we assume that the region with highest density of points in these two channels corresponds to single-cell measurements and consider data points that fall within 40% of the highest density region of the two-dimensional Gaussian function. We direct the reader to <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> and supplemental Chapter 6 for further detail and comparison of flow cytometry with single-cell microscopy.</p>
<h3 id="bayesian-parameter-estimation">Bayesian Parameter Estimation</h3>
<p>We used a Bayesian definition of probability in the statistical analysis of all mutants in this work. In supplemental Chapter 7, we derive in detail the statistical models used for the various parameters as well as multiple diagnostic tests. Here, we give a generic description of our approach. To be succinct in notation, we consider a generic parameter <span class="math inline"><em>θ</em></span> which represents <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and/or <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> depending on the specific LacI mutant.</p>
<p>     As prescribed by Bayes’ theorem, we are interested in the posterior probability distribution <span id="eq:bayes_generic"><br /><span class="math display"><em>g</em>(<em>θ</em> | <em>y</em>) ∝ <em>f</em>(<em>y</em> | <em>θ</em>)<em>g</em>(<em>θ</em>),   (7)</span><br /></span> where we use <span class="math inline"><em>g</em></span> and <span class="math inline"><em>f</em></span> to represent probability densities over parameters and data, respectively, and <span class="math inline"><em>y</em></span> to represent a set of fold-change measurements. The likelihood of observing our dataset <span class="math inline"><em>y</em></span> given a value of <span class="math inline"><em>θ</em></span> is captured by <span class="math inline"><em>f</em>(<em>y</em> | <em>θ</em>)</span>. All prior information we have about the possible values of <span class="math inline"><em>θ</em></span> are described by <span class="math inline"><em>g</em>(<em>θ</em>)</span>.</p>
<p>     In all inferential models used in this work, we assumed that all experimental measurements at a given inducer concentration were normally distributed about a mean value <span class="math inline"><em>μ</em></span> dictated by Eq. 1 with a variance <span class="math inline"><em>σ</em><sup>2</sup></span>, <span id="eq:generic_likelihood"><br /><span class="math display">$$
f(y\,\vert\, \theta) = {1 \over (2\pi\sigma^2)^{N/2}}\prod\limits_i^N \exp\left[-{(y_i - \mu(\theta))^2 \over 2\sigma^2}\right], 
\qquad(8)$$</span><br /></span> where <span class="math inline"><em>N</em></span> is the number of measurements in the data set <span class="math inline"><em>y</em></span>.</p>
<p>     This choice of likelihood is justified as each individual measurement at a given inducer concentration is a biological replicate and independent of all other experiments. By using a Gaussian likelihood, we introduce another parameter <span class="math inline"><em>σ</em></span>. As <span class="math inline"><em>σ</em></span> must be positive and greater than zero, we define as a prior distribution a half-normal distribution with a standard deviation <span class="math inline"><em>ϕ</em></span>, <span id="eq:sigma_prior"><br /><span class="math display">$$
g(\sigma) = { {1 \over \phi}\sqrt{2 \over \pi} }\exp\left[-{x \over 2\phi^2}\right]\,;\, x \geq 0,
\qquad(9)$$</span><br /></span> where <span class="math inline"><em>x</em></span> is a given range of values for <span class="math inline"><em>σ</em></span>. A standard deviation of <span class="math inline"><em>ϕ</em> = 0.1</span> was chosen given our knowledge of the scale of our measurement error from other experiments. As the absolute measurement of fold-change is restricted between <span class="math inline">0</span> and <span class="math inline">1</span>, and given our knowledge of the sensitivity of the experiment, it is reasonable to assume that the error will be closer to <span class="math inline">0</span> than to <span class="math inline">1</span>. Further justification of this choice of prior through simulation based methods are given in the supplemental Chapter 7. The prior distribution for <span class="math inline"><em>θ</em></span> is dependent on the parameter and its associated physical and physiological restrictions. Detailed discussion of our chosen prior distributions for each model can also be found in the supplemental Chapter 7.</p>
<p>     All statistical modeling and parameter inference was performed using Markov chain Monte Carlo (MCMC). Specifically, Hamiltonian Monte Carlo sampling was used as is implemented in the Stan probabilistic programming language <span class="citation" data-cites="carpenter2017">(Carpenter et al. <a href="#ref-carpenter2017" role="doc-biblioref">2017</a>)</span>. All statistical models saved as <code>.stan</code> models and can be accessed at the <a href="https://www.github.com/rpgroup-pboc/mwc_mutants">GitHub repository</a> associated with this work (DOI: 10.5281/zenodo.2721798) or can be downloaded directly from the <a href="https://www.rpgroup.caltech.edu/mwc_mutants">website</a> associated with the publication this chapter is based on.</p>
<h3 id="inference-of-free-energy-from-fold-change-data">Inference of Free Energy From Fold-Change Data</h3>
<p>While the fold-change in gene expression is restricted to be between 0 and 1, experimental noise can generate fold-change measurements beyond these bounds. To determine the free energy for a given set of fold-change measurements (for one unique strain at a single inducer concentration), we modeled the observed fold-change measurements as being drawn from a normal distribution with a mean <span class="math inline"><em>μ</em></span> and standard deviation <span class="math inline"><em>σ</em></span>. Using Bayes’ theorem, we can write the posterior distribution as <span id="eq:post_fc_mu"><br /><span class="math display">$$
g(\mu, \sigma\,\vert y) \propto
g(\mu)g(\sigma){1\over(2\pi\sigma^2)^{N/2}}\prod\limits_i^N
\exp\left[{-(y_i - \mu)^2 \over 2\sigma^2}\right]
\qquad(10)$$</span><br /></span> where <span class="math inline"><em>y</em></span> is a collection of fold-change measurements. The prior distribution for <span class="math inline"><em>μ</em></span> was chosen to be uniform between 0 and 1 while the prior on <span class="math inline"><em>σ</em></span> was chosen to be half normal, as written in Eq. 9. The posterior distribution was sampled independently for each set of fold-change measurements using MCMC. The <code>.stan</code> model for this inference is available on the <a href="http://www.rpgroup.caltech.edu/mwc_mutants">paper website</a>.</p>
<p>     For each MCMC sample of <span class="math inline"><em>μ</em></span>, the free energy was calculated as <span id="eq:empirical_F"><br /><span class="math display"><em>F</em> =  − log (<em>μ</em><sup> − 1</sup>−1)   (11)</span><br /></span> which is simply the rearrangement of Eq. 3. Using simulated data, we determined that when <span class="math inline"><em>μ</em> &lt; <em>σ</em></span> or <span class="math inline">(1 − <em>μ</em>) &lt; <em>σ</em></span>, the mean fold-change in gene expression was over or underestimated for the lower and upper limit, respectively. This means that there are maximum and minimum levels of fold-changes that can be detected using flow cytometry which are set by the distribution of fold-change measurements resulting from various sources of day-to-day variation. This results in a systematic error in the calculation of the free energy, making proper inference beyond these limits difficult. This bounds the range in which we can confidently infer this quantity with flow cytometry. We hypothesize that more sensitive methods, such as single cell microscopy, colorimetric assays, or direct counting of mRNA transcripts via Fluorescence <em>In Situ</em> Hybridization (FISH) would improve the measurement of <span class="math inline"><em>Δ</em><em>F</em></span>. We further discuss details of this limitation in the supplemental Chapter 7.</p>
<h3 id="data-and-code-availability">Data and Code Availability</h3>
<p>All data was collected, stored, and preserved using the Git version control software. Code for data processing, analysis, and figure generation is available on the [GitHub repository] (https://www.github.com/rpgroup-pboc/mwc_mutants) or can be accessed via the <a href="http://www.rpgroup.caltech.edu/mwc_mutants">paper website</a>. Raw flow cytometry data is stored on the CaltechDATA data repository and can be accessed via DOI 10.22002/D1.1241.</p>
<h2 class="unnumbered" id="references" class="unnumbered">References</h2>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-ackers1982">
<p>Ackers, Gary K, and Alexander D Johnson. 1982. “Quantitative Model for Gene Regulation by A Phage Repressor.” <em>Proceedings of the National Academy of Sciences</em> 79: 1129–33.</p>
</div>
<div id="ref-barnes2019">
<p>Barnes, Stephanie L., Nathan M. Belliveau, William T. Ireland, Justin B. Kinney, and Rob Phillips. 2019. “Mapping DNA Sequence to Transcription Factor Binding Energy <em>in Vivo</em>.” Edited by Gary D. Stormo. <em>PLOS Computational Biology</em> 15 (2). <a href="https://doi.org/10.1371/journal.pcbi.1006226">https://doi.org/10.1371/journal.pcbi.1006226</a>.</p>
</div>
<div id="ref-bintu2005a">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, Thomas Kuhlman, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Applications.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 125–35. <a href="https://doi.org/10.1016/j.gde.2005.02.006">https://doi.org/10.1016/j.gde.2005.02.006</a>.</p>
</div>
<div id="ref-bintu2005">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Models.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 116–24. <a href="https://doi.org/10.1016/j.gde.2005.02.007">https://doi.org/10.1016/j.gde.2005.02.007</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates the Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-buchler2003">
<p>Buchler, N. E., U. Gerland, and T. Hwa. 2003. “On Schemes of Combinatorial Transcription Logic.” <em>Proceedings of the National Academy of Sciences</em> 100 (9): 5136–41. <a href="https://doi.org/10.1073/pnas.0930314100">https://doi.org/10.1073/pnas.0930314100</a>.</p>
</div>
<div id="ref-carpenter2017">
<p>Carpenter, Bob, Andrew Gelman, Matthew D. Hoffman, Daniel Lee, Ben Goodrich, Michael Betancourt, Marcus Brubaker, Jiqiang Guo, Peter Li, and Allen Riddell. 2017. “Stan: A Probabilistic Programming Language.” <em>Journal of Statistical Software</em> 76 (1): 1–32. <a href="https://doi.org/10.18637/jss.v076.i01">https://doi.org/10.18637/jss.v076.i01</a>.</p>
</div>
<div id="ref-daber2009a">
<p>Daber, Robert, Kim Sharp, and Mitchell Lewis. 2009. “One Is Not Enough.” <em>Journal of Molecular Biology</em> 392 (5): 1133–44. <a href="https://doi.org/10.1016/j.jmb.2009.07.050">https://doi.org/10.1016/j.jmb.2009.07.050</a>.</p>
</div>
<div id="ref-daber2011">
<p>Daber, Robert, Matthew A. Sochor, and Mitchell Lewis. 2011. “Thermodynamic Analysis of Mutant Lac Repressors.” <em>Journal of Molecular Biology</em>, The Operon Model and its Impact on Modern Molecular Biology, 409 (1): 76–87. <a href="https://doi.org/10.1016/j.jmb.2011.03.057">https://doi.org/10.1016/j.jmb.2011.03.057</a>.</p>
</div>
<div id="ref-farabaugh1978">
<p>Farabaugh, Phillip J. 1978. “Sequence of the <em>lacI</em> Gene.” <em>Nature</em> 274 (24): 5.</p>
</div>
<div id="ref-frumkin2018">
<p>Frumkin, Idan, Marc J. Lajoie, Christopher J. Gregg, Gil Hornung, George M. Church, and Yitzhak Pilpel. 2018. “Codon Usage of Highly Expressed Genes Affects Proteome-Wide Translation Efficiency.” <em>Proceedings of the National Academy of Sciences</em> 115 (21): E4940–E4949. <a href="https://doi.org/10.1073/pnas.1719375115">https://doi.org/10.1073/pnas.1719375115</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-keymer2006">
<p>Keymer, Juan E., Robert G. Endres, Monica Skoge, Yigal Meir, and Ned S. Wingreen. 2006. “Chemosensing in <em>Escherichia Coli</em>: Two Regimes of Two-State Receptors.” <em>Proceedings of the National Academy of Sciences</em> 103 (6): 1786–91. <a href="https://doi.org/10.1073/pnas.0507438103">https://doi.org/10.1073/pnas.0507438103</a>.</p>
</div>
<div id="ref-kuhlman2007">
<p>Kuhlman, T., Z. Zhang, M. H. Saier, and T. Hwa. 2007. “Combinatorial Transcriptional Control of the Lactose Operon of <em>Escherichia Coli</em>.” <em>Proceedings of the National Academy of Sciences</em> 104 (14): 6043–8. <a href="https://doi.org/10.1073/pnas.0606717104">https://doi.org/10.1073/pnas.0606717104</a>.</p>
</div>
<div id="ref-lewis1996">
<p>Lewis, Mitchell, Geoffrey Chang, Nancy C. Horton, Michele A. Kercher, Helen C. Pace, Maria A. Schumacher, Richard G. Brennan, and Ponzy Lu. 1996. “Crystal Structure of the Lactose Operon Repressor and Its Complexes with DNA and Inducer.” <em>Science</em> 271 (5253): 1247–54.</p>
</div>
<div id="ref-mclaughlinjr2012">
<p>McLaughlin Jr, Richard N., Frank J. Poelwijk, Arjun Raman, Walraj S. Gosal, and Rama Ranganathan. 2012. “The Spatial Architecture of Protein Function and Adaptation.” <em>Nature</em> 491 (7422): 138–42. <a href="https://doi.org/10.1038/nature11500">https://doi.org/10.1038/nature11500</a>.</p>
</div>
<div id="ref-phillips2015">
<p>Phillips, Rob. 2015. “Napoleon Is in Equilibrium.” <em>Annual Review of Condensed Matter Physics</em> 6 (1): 85–111. <a href="https://doi.org/10.1146/annurev-conmatphys-031214-014558">https://doi.org/10.1146/annurev-conmatphys-031214-014558</a>.</p>
</div>
<div id="ref-poelwijk2011a">
<p>Poelwijk, Frank J., Philip D. Heyning, Marjon GJ de Vos, Daniel J. Kiviet, and Sander J. Tans. 2011. “Optimality and Evolution of Transcriptionally Regulated Gene Expression.” <em>BMC Systems Biology</em> 5 (August): 128. <a href="https://doi.org/10.1186/1752-0509-5-128">https://doi.org/10.1186/1752-0509-5-128</a>.</p>
</div>
<div id="ref-raman2016">
<p>Raman, Arjun S., K. Ian White, and Rama Ranganathan. 2016. “Origins of Allostery and Evolvability in Proteins: A Case Study.” <em>Cell</em> 166 (2): 468–80. <a href="https://doi.org/10.1016/j.cell.2016.05.047">https://doi.org/10.1016/j.cell.2016.05.047</a>.</p>
</div>
<div id="ref-razo-mejia2018">
<p>Razo-Mejia, Manuel, Stephanie L. Barnes, Nathan M. Belliveau, Griffin Chure, Tal Einav, Mitchell Lewis, and Rob Phillips. 2018. “Tuning Transcriptional Regulation Through Signaling: A Predictive Theory of Allosteric Induction.” <em>Cell Systems</em> 6 (4): 456–469.e10. <a href="https://doi.org/10.1016/j.cels.2018.02.004">https://doi.org/10.1016/j.cels.2018.02.004</a>.</p>
</div>
<div id="ref-razo-mejia2014">
<p>Razo-Mejia, M., J. Q. Boedicker, D. Jones, A. DeLuna, J. B. Kinney, and R. Phillips. 2014. “Comparison of the Theoretical and Real-World Evolutionary Potential of a Genetic Circuit.” <em>Physical Biology</em> 11 (2): 026005. <a href="https://doi.org/10.1088/1478-3975/11/2/026005">https://doi.org/10.1088/1478-3975/11/2/026005</a>.</p>
</div>
<div id="ref-reynolds2011">
<p>Reynolds, Kimberly A., Richard N. McLaughlin, and Rama Ranganathan. 2011. “Hot Spots for Allosteric Regulation on Protein Surfaces.” <em>Cell</em> 147 (7): 1564–75. <a href="https://doi.org/10.1016/j.cell.2011.10.049">https://doi.org/10.1016/j.cell.2011.10.049</a>.</p>
</div>
<div id="ref-rydenfelt2014b">
<p>Rydenfelt, Mattias, Robert Sidney Cox, Hernan Garcia, and Rob Phillips. 2014. “Statistical Mechanical Model of Coupled Transcription from Multiple Promoters Due to Transcription Factor Titration.” <em>Physical Review E</em> 89 (1): 012702. <a href="https://doi.org/10.1103/PhysRevE.89.012702">https://doi.org/10.1103/PhysRevE.89.012702</a>.</p>
</div>
<div id="ref-sharan2009">
<p>Sharan, S. K., L. C. Thomason, S. G. Kuznetsov, and D. L. Court. 2009. “Recombineering: A Homologous Recombination-Based Method of Genetic Engineering.” <em>Nature Protocols</em> 4: 206–23. <a href="https://doi.org/nprot.2008.227\\\%0020%5Bpii%5D\\\%002010.1038/nprot.2008.227">https://doi.org/nprot.2008.227\\\%0020[pii]\\\%002010.1038/nprot.2008.227</a>.</p>
</div>
<div id="ref-suel2003">
<p>Süel, Gürol M., Steve W. Lockless, Mark A. Wall, and Rama Ranganathan. 2003. “Evolutionarily Conserved Networks of Residues Mediate Allosteric Communication in Proteins.” <em>Nature Structural Biology</em> 10 (1): 59–69. <a href="https://doi.org/10.1038/nsb881">https://doi.org/10.1038/nsb881</a>.</p>
</div>
<div id="ref-swem2008">
<p>Swem, Lee R., Danielle L. Swem, Ned S. Wingreen, and Bonnie L. Bassler. 2008. “Deducing Receptor Signaling Parameters from <em>in Vivo</em> Analysis: LuxN/AI-1 Quorum Sensing in <em>Vibrio Harveyi</em>.” <em>Cell</em> 134 (3): 461–73. <a href="https://doi.org/10.1016/j.cell.2008.06.023">https://doi.org/10.1016/j.cell.2008.06.023</a>.</p>
</div>
<div id="ref-vilar2003">
<p>Vilar, José M. G., and Stanislas Leibler. 2003. “DNA Looping and Physical Constraints on Transcription Regulation.” <em>Journal of Molecular Biology</em> 331 (5): 981–89. <a href="https://doi.org/10.1016/S0022-2836(03)00764-2">https://doi.org/10.1016/S0022-2836(03)00764-2</a>.</p>
</div>
<div id="ref-weinert2014">
<p>Weinert, Franz M., Robert C. Brewster, Mattias Rydenfelt, Rob Phillips, and Willem K. Kegel. 2014. “Scaling of Gene Expression with Transcription-Factor Fugacity.” <em>Physical Review Letters</em> 113 (25). <a href="https://doi.org/10.1103/PhysRevLett.113.258101">https://doi.org/10.1103/PhysRevLett.113.258101</a>.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/chapter_02" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter II
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_04" class="page-nav__item page-nav__item--next">
										Chapter IV
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd//" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>


<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-109526846-4', 'auto');
	ga('send', 'pageview');
</script>



		</div><!-- /.js-footer-area -->
	</body>
</html>
