<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter I</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd//favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd//doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd//doks-theme/assets/images/migration.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter I</h2>
								
								
									<h1 class="hero-subheader__title">The Phenomenon of Adaptation Across The Biological Scales 
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phd/abstract" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phd/chapter_02" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
						<div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
							<div class="align-container" data-mh>
								<div class="align-inner">
									<div class="hero-subheader__author">
										<p class="hero-subheader__author-title">
												Summary
												<i class="icon icon--chevron-down"></i>
										</p><!-- /.hero-subheader__author-title -->
										<p>All forms of life are united in their obedience to the forces of evolution. The ability to adapt -- either at the level of molecules or at the level of populations -- is central to evolution. This chapter outlines the focus of my dissertation and places the results in a historical context.
</p>
									</div><!-- /.hero-subheader__author -->
								</div><!-- /.align-inner -->
							</div><!-- /.align-container -->
						</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/abstract" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Abstract
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_02" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<h2 id="introduction">Introduction</h2>
<p>From archaea thriving in hydrothermal vents on the ocean floor to aspen trees dominating a Coloradan mountainside, all forms of life are unified in their obedience to (and influence on) their environment. Whether it be the stochastic events that can trigger mass extinctions <span class="citation" data-cites="jablonski2001">(Jablonski <a href="#ref-jablonski2001" role="doc-biblioref">2001</a>)</span> or the rapid “terraforming” of environments by their inhabitants (such as the rusting of the oceans <span class="citation" data-cites="luo2018">(Luo et al. <a href="#ref-luo2018" role="doc-biblioref">2018</a>)</span>), organisms and their environments are continually engaged in a dialogue that demands the ability to adapt. Over the past 3.5 billion years of evolution, life has evolved myriad clever ways to combat (and exploit) environmental fluctuations to amplify reproductive success. The mechanisms behind this adaptation are diverse and traverse the biological scales, ranging from nanosecond-scale conformational switching of proteins (Fig. 1 (A)), to reconfiguration of metabolic networks to consume different sugars (Fig. 1 (B)), to evolutionary trajectories that only become visible over many generations (Fig. 1 (C)). While “adaptation” is typically only associated with organisms (at least colloquially), one can use the same language to describe the microscopic operations of molecules.</p>
<figure>
<img src="ch1_fig1.png" id="fig:adaptation_levels" data-short-caption="The spatial, temporal, and mechanistic scales of adaptation." alt="" /><figcaption>Figure 1: <strong>The spatial, temporal, and mechanistic scales of adaptation.</strong> (A) Molecular adaptation in this work is defined through the lens of allostery where the activity of a protein complex is modulated the reversible binding of a small molecule. These binding and unbinding events lead to rapid changes in protein conformation whose behavior (both energetic and temporal) is comparable to that of thermal motion. (B) Physiological adaptation here is defined as the rewiring of biochemical reaction networks that lead to changes in cellular behavior (such as chemotaxis) or metabolic capacity (such as aerobic to fermentative metabolism). (C) Evolutionary adaptation is recorded in the variation in the genetic sequence of regulatory molecules. Variations in sequence influence the function of the proteins and RNAs they encode which ultimately define cellular fitness.</figcaption>
</figure>
<p>     The idea of molecular adaption is not novel and demands a brief foray into the history of bacterial growth and the dawn of regulatory biology. In the late 1890s, Emilé Duclaux and his graduate student Frédéric Diénert performed a series of experiments illustrating that the common yeast could only consume galactose after an incubation period with the sugar. This led to a general conclusion that “the production of diastases [enzymes] depends on the manner of nutrition” in which the cultures were grown <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>, a phenomenon later coined <em>enzymatic adaptation</em>. This is one of the first observations of the fact that, while an organism may be able to digest a certain sugar, it may not <em>always</em> be able to do so. Rather, there seemed to be certain conditions in which the production or formation of these enzymes could occur. In his doctoral thesis in 1900, Diénert proposed two mechanisms for the origin of enzymatic adaptation observed for galactozymase in <em>S. cerevisiae</em> <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. Either (a) the presence of galactose <em>directly</em> transformed enzymes already present in the cell into galactozymase or (b) that the galactose activated the production of galactozymase <em>de novo</em> <span class="citation" data-cites="dienert1900 loison2013">(Diénert <a href="#ref-dienert1900" role="doc-biblioref">1900</a>; Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>.</p>
<p>     Nearly half a century later, Jacques Monod would rediscover the phenomenon of enzymatic adaptation, this time in the context of bacterial growth. In his 1941 work, Monod for the first time reported on the phenomenon of diauxic or biphasic growth, shown in Fig. 2 (A). He noted that for some mixtures of carbon sources, the culture grew “kinetically normal” meaning they grew exponentially to saturation (blue points, Fig. 2 (A)). However, some mixtures (such as sucrose and arabinose) led to biphasic growth where the culture would grow exponentially, undergo a period where growth had ceased (lasting typically 20 - 60 minutes), followed again by another round of exponential growth (green points, Fig. 2 (A)). Additionally, Monod showed that the onset of this diauxic shift could be tuned by varying the relative concentrations of the carbon sources, revealing a controllable chemical basis for the adaptation (Fig. 2 (B)).</p>
<p>     Monod immediately made the connection between diauxic growth and enzymatic adaptation <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. Despite his work appearing 40 years after the pioneering work of Ducleaux and Diénért, there had been little progress towards a mechanistic, needless to say quantitative, explanation for the phenomenon. In fact, Monod was particularly disappointed by the teleological explanations where the cells simply changed their behavior to perform only the chemical reactions that were “needed” <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. The teleological approach to much of biology during this time period, especially in the French scientific community, severely bothered Monod. To him, this kind of approach belonged to a pre-scientific era and lacked the “postulate of objectivity” that other fields of science (particularly physics) had adopted <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. Near the middle of the 20th century, Monod published a 60-page treatise on the phenomenon of enzymatic adaptation with the level of quantitative rigor he thought it deserved <span class="citation" data-cites="monod1947">(Monod <a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. In this work, he set out to progressively deconstruct and invalidate a series of hypotheses for the phenomenon of enzymatic adaptation. In doing so, he laid the groundwork for what would become (in his opinion) his greatest contribution to science, the nature of allosteric transitions <span class="citation" data-cites="loison2013 monod1963 monod1965">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>; Monod, Changeux, and Jacob <a href="#ref-monod1963" role="doc-biblioref">1963</a>; Monod, Wyman, and Changeux <a href="#ref-monod1965" role="doc-biblioref">1965</a>)</span>, a topic that will feature prominently in the remainder of this thesis.</p>
<figure>
<img src="ch1_fig2.png" id="fig:diauxie_fig" data-short-caption="The phenomenon of enzymatic adaptation revealed in bacterial growth curves." alt="" /><figcaption>Figure 2: <strong>The phenomenon of enzymatic adaptation revealed in bacterial growth curves.</strong> (A) Optical density measurements of <em>Bacillus subtilis</em> cultures grown in a mixture of sucrose and either glucose (blue points) or arabinose (green points). Biphasic growth can be observed in the sucrose/arabinose mixture where the pause in growth (white shaded region) corresponds to enzymatic adaptation. Data digitized from <span class="citation" data-cites="monod1941">Monod (<a href="#ref-monod1941" role="doc-biblioref">1941</a>)</span>. (B) Diauxic growth curves of <em>Escherichia coli</em> cells grown on a mixture of glucose and sorbitol in different proportions. Data digitized from <span class="citation" data-cites="monod1947">Monod (<a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. Periods of enzymatic adaptation are highlighted by white vertical lines. In both plots, a univariate spline interpolation was used to draw lines to reflect data presentation of original literature. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig2.py">Python code (<code>ch1_fig2.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     The diauxic growth transitions shown in Fig. 2 illustrate adaptive processes across the biological scales, as were schematized in Fig. 1. While it was not known to Monod at the time, we now know that many cases of enzymatic adaptation are driven by the regulation of gene expression. As the bacterial culture approaches the auxic shift, the presence or absence of the substrate is sensed by regulatory molecules that control whether the genes encoding the enzymes for metabolism of the substrate are expressed. This represents the level of <em>molecular adaptation</em> where, given binding or unbinding of the substrate molecule, the activity of the regulatory protein is modulated. The amino acid sequence of these proteinaceous regulators are the product of billions of years of evolution with regions of the protein (such as the DNA-binding and inducer-binding domains) undergoing <em>evolutionary adaptation</em> that define how the regulatory molecule senses and responds to these signals. Finally, the precision with which these genes are regulated is determined by their sensitivity to physiological states, capturing the level of <em>physiological adaptation</em>.</p>
<p>     The central aim of this dissertation is to explore the biophysical mechanisms by which these levels of adaptation – molecular, physiological, and evolutionary – are interconnected. Furthermore, in the spirit of Monod, we seek to make our exploration quantitative and leverage the tools of statistical physics to provide precise predictions from pen-and-paper theory that can be rigorously tested through experiment. The remaining sections of this chapter will outline the major topics of this thesis and place them in a historical context alongside the work of Monod. This thesis is structured such that <strong>Chapters 1 – 5</strong> present a self-contained summary of how quantitative methods can be used to interrogate and understand the molecular biophysics of adaptation. The remainder, <strong>Chapters 6 – 9</strong>, are detailed supplements to Chapters 1 – 5,s and are targeted to readers who want to dig into the weeds of statistical inference, error estimation, and analytic properties of the theoretical models.</p>
<h2 id="on-the-janus-face-of-molecules">On The Janus Face of Molecules</h2>
<p>Monod is perhaps most famous for his discovery of allostery which he famously referred to as “the second secret of life” <span class="citation" data-cites="ullmann2011 monod1965">(Ullmann <a href="#ref-ullmann2011" role="doc-biblioref">2011</a>; Monod, Wyman, and Changeux <a href="#ref-monod1965" role="doc-biblioref">1965</a>)</span>. It is fair to say that this “secret” has now been declassified. Allosteric regulation is present in all domains of life across various types of biological processes. Allostery can be found governing the behavior of ion channels <span class="citation" data-cites="einav2017 auerbach2012">(Einav and Phillips <a href="#ref-einav2017" role="doc-biblioref">2017</a>; Auerbach <a href="#ref-auerbach2012" role="doc-biblioref">2012</a>)</span>, enzymatic reactions <span class="citation" data-cites="einav2016">(Einav, Mazutis, and Phillips <a href="#ref-einav2016" role="doc-biblioref">2016</a>)</span>, chemotaxis <span class="citation" data-cites="keymer2006">(Keymer et al. <a href="#ref-keymer2006" role="doc-biblioref">2006</a>)</span>, G-protein coupled receptors <span class="citation" data-cites="canals2012">(Canals et al. <a href="#ref-canals2012" role="doc-biblioref">2012</a>)</span>, quorum sensing <span class="citation" data-cites="swem2008">(Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>)</span>, and transcriptional regulation <span class="citation" data-cites="huang2018 lindsley2006a">(Huang et al. <a href="#ref-huang2018" role="doc-biblioref">2018</a>; Lindsley and Rutter <a href="#ref-lindsley2006a" role="doc-biblioref">2006</a>)</span>, to name a few of many examples. Despite the objective complexity in the molecular structures of all of these allosteric molecules, they can frequently be reduced to simple cartoons where the details of conformational changes, substrate-binding affinities, and more can be massaged into a small set of key details. Fig. 3 (A) shows the molecular structures of a variety of allosteric transcriptional repressors (top). While each has its own fascinating structure and continuum of conformational states, all can be coarse grained into a simple cartoon representation (bottom) with an active (red) and inactive (purple) state, both of which possess binding pockets (semicircular notches) for an inducer molecule (orange).</p>
<p>     Much as we can reduce the complexity of allosteric molecules schematically, we can enumerate simple mathematical models that describe their behavior. Thermodynamic models built on an assumption of quasi-equilibrium are routinely used to describe complex biological phenomena despite the reality that being in thermodynamic equilibrium is synonymous with being dead. Even with this glaring assumption, such models have been shown to be exceptionally predictive for a variety of complex systems, especially in modeling molecular binding reactions <span class="citation" data-cites="dill2010">(Dill and Bromberg <a href="#ref-dill2010" role="doc-biblioref">2010</a>)</span> and allostery in general <span class="citation" data-cites="swem2008 keymer2006 einav2017 einav2016 phillips2015">(Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>; Keymer et al. <a href="#ref-keymer2006" role="doc-biblioref">2006</a>; Einav and Phillips <a href="#ref-einav2017" role="doc-biblioref">2017</a>; Einav, Mazutis, and Phillips <a href="#ref-einav2016" role="doc-biblioref">2016</a>; Phillips <a href="#ref-phillips2015" role="doc-biblioref">2015</a>)</span>. As the timescales of binding and unbinding reactions are orders of magnitude smaller than that of many other processes in the cell, it is fair to make the approximation that molecular binding is in equilibrium. Under this assumption, we are granted the powerful mathematical privilege to say that the probability of a given state of the system <span class="math inline"><em>P</em><sub>state</sub></span> follows a Boltzmann distribution, <span id="eq:boltzmann"><br /><span class="math display">$$
P_\text{state} = \frac{e^{-\frac{\epsilon_\text{state}}{k_BT}}}{\mathcal{Z}},
\qquad(1)$$</span><br /></span> where <span class="math inline"><em>ϵ</em><sub>state</sub></span> is the energy of that state, <span class="math inline"><em>k</em><sub><em>B</em></sub></span> is the Boltzmann constant, and <span class="math inline"><em>T</em></span> is the system temperature. The denominator <span class="math inline">𝒵</span> is the partition function of the system and is the sum <span id="eq:partition_function_def"><br /><span class="math display">$$
\mathcal{Z} = \sum\limits_{i \in \text{states}} e^{-\frac{\epsilon_i}{k_BT}},
\qquad(2)$$</span><br /></span> ensuring that the distribution is normalized. Therefore, if we are interested in computing the probability of a given allosteric protein being in the active state, we merely have to enumerate all of the Boltzmann weights (given by the numerator in Eq. 1) and compute <span id="eq:verbal_pact"><br /><span class="math display">$$
P_\text{active}  = \frac{\text{sum over all possible active states}}{\text{sum
over all possible states}}.
\qquad(3)$$</span><br /></span> This probability, defined as a function of the inducer concentration, is shown schematically in Fig. 3 (B). While we have passed over some of the more subtle details of this calculation, the plot in Fig. 3 (B) presents a <em>quantitative</em> prediction of how the activity of an allosteric molecule should scale as a function of the inducer, in this case becoming less active as more inducer molecules are present.</p>
<figure>
<img src="ch1_fig3.png" id="fig:allostery" data-short-caption="A coarse grained representation of an allosteric molecule." alt="" /><figcaption>Figure 3: <strong>A coarse grained representation of an allosteric molecule.</strong> (A) Crystal structures of a variety of allosteric transcription factors are shown at the top. In this thesis, we coarse grain away many of the atomistic details to construct a minimal representation (bottom) where the protein can be represented as being either active (red) or inactive (purple), both of which can bind an inducer molecule (orange). (B) By making an assumption of quasi-equilibrium, we can compute a mathematical description of the active probability of an allosteric protein as a function of the inducer concentration (top). In this particular case, the inactive state becomes more probable relative to the active state at higher concentrations of inducer molecules.</figcaption>
</figure>
<p>     In <strong>Chapter 2</strong> and the associated supplementary <strong>Chapter 6</strong> of this dissertation, we use the Monod-Wyman-Changeux model of allostery <span class="citation" data-cites="monod1965">(Monod, Wyman, and Changeux <a href="#ref-monod1965" role="doc-biblioref">1965</a>)</span> to build a predictive model of transcriptional regulation where the level of gene expression changes in response to changing activity of an allosteric transcriptional repressor. Using the same tricks given by Eq. 1 and Eq. 3, we expand upon a previously characterized thermodynamic model of the simple repression motif. This motif, schematized in Fig. 4 (A), is not just a convenient abstraction of a regulatory architecture. Rather, this motif is the most ubiquitous regulatory scheme in <em>E. coli</em> <span class="citation" data-cites="gama-castro2016 ireland2020">(Gama-Castro et al. <a href="#ref-gama-castro2016" role="doc-biblioref">2016</a>; Ireland et al. <a href="#ref-ireland2020" role="doc-biblioref">2020</a>)</span> and has been the target of much theoretical and experimental dissection <span class="citation" data-cites="vilar2003 buchler2003 bintu2005 garcia2011 brewster2014 phillips2019">(Vilar and Leibler <a href="#ref-vilar2003" role="doc-biblioref">2003</a>; Buchler, Gerland, and Hwa <a href="#ref-buchler2003" role="doc-biblioref">2003</a>; Bintu et al. <a href="#ref-bintu2005" role="doc-biblioref">2005</a>; Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span>. However, inclusion of allostery in a mathematical sense had yet to be explored experimentally.</p>
<p>     At the beginning of 2016, Manuel Razo-Mejia, Stephanie Barnes, Nathan Belliveau, Tal Einav, and I joined forces and set out to build a complete theoretical model for allosteric transcriptional regulation coupled with a thorough experimental dissection. This was no small task and would have likely taken a full Ph.D.’s worth of effort for a single person to do. Yet, within a year of project inception we had submitted a manuscript to preprint servers where all of us were annotated as equal contributors. This experience defined how I view collaboration in scientific research and serves as a shining example of scientific socialism.</p>
<p>     Together, we enumerated a complete thermodynamic model for the inducible simple repression motif and defined a succinct input-output function for the fold-change in gene expression (schematized in Fig. 4 (B)). Here, we define the fold-change in gene expression as the level of gene expression in the presence of a transcriptional repressor relative to the level of expression when the repressor is absent from the system. Therefore, the value of the fold-change is restricted from 0 to 1, corresponding to high and low levels of regulation respectively.</p>
<p>      This model, which is explored in depth in Chapter 2, is defined by a minimal set of biophysical parameters, many of which can be directly measured using standard tricks of molecular biology and biochemistry. With a model in hand, we turned to a collection of 17 unique <em>E. coli</em> strains, each with different copy numbers of the repressor protein and different regulatory DNA sequences. Using our theoretical model, we inferred the lone two biophysical parameters, which we did not know <em>a priori</em>, from a single experimental strain (white points in middle panel of Fig. 4 (C)). We then applied these inferred parameters to our model and drew predictions for all other experimental strains. We found the model to be remarkably predictive, suggesting that our “toy” model of an allosteric repressor captured the underlying physics of the system.</p>
<figure>
<img src="ch1_fig4.png" id="fig:induction_intro" data-short-caption="Experimental dissection of the inducible simple repression input-output function." alt="" /><figcaption>Figure 4: <strong>Experimental dissection of the inducible simple repression input-output function.</strong> (A) Schematic diagram of the inducible simple repression motif. (B) Schematic diagram of the input-output function as derived in Chapter 2. (C) Experimental measurements of the fold-change in gene expression using the <em>lac</em> repressor from <em>E coli</em>. Different rows correspond to different operator sequences and therefore different values for the DNA affinity parameter, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. Different colors correspond to different values for the average repressor copy number <span class="math inline"><em>R</em></span>. Open-faced points in the middle panel represent the experimental strain used to infer the values of the inducer dissociation constants. Points and errors correspond to the mean and standard error of ten to fifteen biological replicates. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig4.py">Python code (<code>ch1_fig4.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     A key feature of this work is the derivation of thermodynamic state variable termed the <em>free energy</em> <span class="citation" data-cites="keymer2006 swem2008">(Keymer et al. <a href="#ref-keymer2006" role="doc-biblioref">2006</a>; Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>)</span>. This parameter provides an intuition for the effective free energy difference between states of the promoter in which the repressor is bound and those states in which the repressor is not bound. This parameter accounts for all of the ways in which one can tune the variables and still achieve the same fold-change in gene expression, as is diagrammed in Fig. 5 (A). While we leave the details of this derivation to Chapter 2, we emphasize that this formalism provides a means by which all of the experimental measurements plotted in Fig. 4 (C) can be collapsed onto a master curve defined solely by the free energy, which is illustrated in Fig. 5 (B). This scaling, often referred to as “data collapse” in physics, concretely shows that one has identified the <em>natural variable</em> of the system. With this scaling function in hand, we are able to make a measurement of the biophysical parameters, compute the free energy, and make a concrete prediction of what the fold-change in gene expression will be. Or, as we will see in the following section, details of the biophysical parameters can be determined directly from an empirical measurement of the free energy.</p>
<figure>
<img src="ch1_fig5.png" id="fig:collapse_intro" data-short-caption="Collapse of individual induction profiles onto a simple scaling function." alt="" /><figcaption>Figure 5: <strong>Collapse of individual induction profiles onto a simple scaling function.</strong> (A) Many different combinations of parameter values yield the same value of the fold-change in gene expression, shown as red and blue horizontal planes. Any point on those planes corresponds to a single value of the free energy (middle) and will appear on the master curve. (B) Data presented in Fig. 4 (C) collapsed onto the master curve defined by the predicted value of the free energy. Points and errors correspond to the mean and standard error of ten to fifteen biological replicates. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_02/code/ch2_fig8.py">Python code (<code>ch2_fig8.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="on-using-free-energy-to-examine-evolutionary-adaptation">On Using Free Energy to Examine Evolutionary Adaptation</h2>
<p>     Allow us to briefly return to Monod and his biphasic growth curves in the mid 1940s. At this point in scientific history, the French vision of biology had taken a strongly finalistic and vitalistic turn <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. In particular, a neo-Lamarckian view had been employed to explain the phenomenon of enzymatic adaptation where the enzymes appropriate for digesting the substrate could be spontaneously formed out of the bacterial cytoplasm and inherited by the cell’s descendants, completely independent of genes. In general, this approach to biology deeply frustrated Monod and strongly influenced his desire to “physicalize” the science <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. One tool he knew was critical to this mission was the burgeoning field of genetics. In the mid 1930s Monod undertook a short retreat to Thomas Hunt Morgan’s lab at Caltech where he was introduced to genetics which he later remarked as “biology’s first discipline” <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. This visit had a profound impact on Monod, who reflected upon it some three decades later:</p>
<blockquote>
<p><em>“Upon my return to France, I had again taken up the study of bacterial growth. But my mind remained full of the concepts of genetics and I was confident of its ability to analyze and convinced that one day these ideas would be applied to bacteria.”</em> <span class="citation" data-cites="monod1966">(Monod <a href="#ref-monod1966" role="doc-biblioref">1966</a>)</span></p>
</blockquote>
<p>     Once he returned to his study of bacterial growth and enzymatic adaptation, he was confronted with incorporating the role of genetic inheritance into his mechanistic explanations. In the mid 1940s, Monod and his coworkers had begun experimenting with a strain of <em>E. coli</em> which was unable to digest lactose, termed <span class="math inline"><em>L</em>−</span>. When grown on a mixture of glucose and lactose, this strain would not display a diauxic shift and would only be able to consume the glucose in the medium (Fig. 6, black). However, Monod and his coworker Alice Audureau discovered a mutation in this strain which <em>enabled</em> the digestion of lactose, termed <span class="math inline"><em>L</em>+</span> <span class="citation" data-cites="monod1947">(Monod <a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. The growth curve of this strain had the striking feature of diauxic growth. Rather than this mutation merely enabling the digestion of lactose, it did so in a non-constitutive manner and preserved the phenomenon of adaptation. This was an important step forward in Monod’s understanding of enzymatic adaptation <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>, revealing that it was a “truly genetic property” <span class="citation" data-cites="monod1966">(Monod <a href="#ref-monod1966" role="doc-biblioref">1966</a>)</span>.</p>
<figure>
<img src="ch1_fig6.png" id="fig:lacneg" data-short-caption="Growth curves of lactose-positive and lactose-negative *E. coli* strains." alt="" /><figcaption>Figure 6: <strong>Growth curves of lactose-positive and lactose-negative <em>E. coli</em> strains on a glucose/lactose mixture.</strong> Black curve shows the growth curve of an <em>E. coli</em> strain unable to digest lactose grown on a glucose/lactose mixed medium. Red curve shows a mutant of the same <em>E. coli</em> strain which is able to consume lactose. The latter displays a diauxic growth cycle with an adaptive period, illustrating that enzymatic adaptation is a truly genetic property. Figure adapted from <span class="citation" data-cites="monod1947">Monod (<a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. Lines correspond to univariate splines fit to the data to retain the data presentation from the literature. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig6.py">Python code (<code>ch1_fig6.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     This finding illustrates the level of evolutionary adaptation operating at the level of molecules. While it is difficult to find any literature dissecting this particular <span class="math inline"><em>L</em>+</span> mutation, it is not difficult to imagine several different mechanisms by which it could be manifest. One such explanation is that this <span class="math inline"><em>L</em>+</span> mutation is within a transcriptional regulator itself where a deficiency in the ability to respond to the presence of lactose (and decreasing glucose concentration) had been restored. Such mutations are the crux of <strong>Chapter 3</strong> and the corresponding supplemental <strong>Chapter 7</strong> of this dissertation.</p>
<p>     As summarized in the previous section and discussed in depth in Chapter 2, Chapter 3 and the associated supplemental Chapter 7 of this dissertation focus on the influence of mutation within the allosteric transcription factor. Furthermore, Chapter 3 presents a generic mechanism by which shifts in the free energy can be mapped directly to changes in values of the biophysical parameters. This chapter, much like Chapter 2, was borne out of a wonderful collaboration with Manuel Razo-Mejia, Stephanie Barnes, Nathan Belliveau, Tal Einav, and Zofii Kaczmarek. Being able to launch another collaborative effort afforded us the opportunity to both develop a new theoretical interpretation for how mutations influence the free energy and acquire enough experimental data to thoroughly test it.</p>
<figure>
<img src="ch1_fig7.png" id="fig:pedagogical_delF_intro" data-short-caption="Mutations lead to predictive shifts in free energy." alt="" /><figcaption>Figure 7: <strong>Mutations lead to shifts in free energy, permitting prediction of double mutant phenotypes.</strong> Consider a wild-type bacterium which, on average, exhibits a fold-change of <span class="math inline">≈</span> 0.3 and a free energy of <span class="math inline"> − 1 <em>k</em><sub><em>B</em></sub><em>T</em></span> (grey point in (B)). We can consider that a single mutation (either orange or purple) changes the mean fold-change and therefore the free energy, translating the measurement elsewhere along the master curve (black line in (B)). Assuming there are no interactions between the two single mutations, a null hypothesis predicts that for the double mutant (blue bacterium in (A) and point in (B)), the net free energy is simply the sum of the individual free energy shifts.The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig7.py">Python code (<code>ch1_fig7.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     The primary conceptual development of Chapter 3 is illustrated in Fig. 7. Theoretically, we consider a bacterial strain with an allosteric repressor (which we term the “wild-type” repressor) that has been characterized well enough to have sufficient parametric knowledge of the system. We can use these known parameters of the mutant to easily compute both its predicted average fold-change in gene expression along with the corresponding free energy. However, once a mutation has been introduced <em>into the repressor protein</em> (resulting in a non-synonymous amino acid change), we are once again ignorant <em>a priori</em> of what changes, if any, that mutation may have imparted on the system. In Fig. 7, we examine two separate hypothetical mutations, shown in purple and orange, which significantly change the character of the system by either increasing or decreasing the fold-change in gene expression, respectively. If we assume that these mutations do not change the underlying physics of the system, we are permitted to use the theoretical framework outlined in Fig. 4 and elaborated on in Chapter 2 to characterize each mutation and determine what biophysical parameters have been changed. This framework allows us to calculate the new free energy of the system (<span class="math inline"><em>F</em><sub>mutation 1</sub></span>) as well as the shift in free energy from the wild-type value, <span id="eq:intro_delF_definition"><br /><span class="math display"><em>Δ</em><em>F</em><sub>mutation 1</sub> = <em>F</em><sub>mutation 1</sub> − <em>F</em><sub>wt</sub>.   (4)</span><br /></span> As will be described in detail in Chapter 3 and the supplemental Chapter 7, the precise value of this free energy shift <span class="math inline"><em>Δ</em><em>F</em></span> can be directly computed given sufficient parametric knowledge.</p>
<p>     This formalism provides a mathematical hypothesis for how double mutants may behave. Given known values for <span class="math inline"><em>Δ</em><em>F</em></span> of each mutation in isolation, can we compute the shift in free energy of the pairwise double mutant <span class="math inline"><em>Δ</em><em>F</em><sub>mutations 1 &amp;2</sub></span>? Eq. 4 allows us to make a mathematical null hypothesis that, assuming there are no interactions between the mutants, the net shift in the free energy is simply the sum of the individual shifts in free energy, <span id="eq:delF_null_intro"><br /><span class="math display"><em>Δ</em><em>F</em><sub>mutations 1 &amp;2</sub> = <em>Δ</em><em>F</em><sub>mutation 1</sub> + <em>Δ</em><em>F</em><sub>mutation 2</sub>.   (5)</span><br /></span></p>
<p>Given the fact that we can compute the fold-change in gene expression with knowledge of the free energy, we can therefore predict the double mutant phenotype <em>a priori</em>, a prediction not possible prior to this work.</p>
<p>     Over the course of two years (while this theory was in the works), the experimental cast of characters (Stephanie Barnes, Nathan Belliveau, Manuel Razo-Mejia, and Zofii Kaczmarek) and I made a series of mutations in the LacI repressor that we had characterized in the work presented in Chapter 2. These mutations included three point mutations in the DNA binding domain of the repressor, four mutations in the inducer-binding domain, nine double mutants (one inducer binding and one DNA binding each), across four repressor copy numbers and three operator sequences. While this process of strain generation and data collection is not the primary focus of the work, it took <span class="math inline"> ≈ 80%</span> of the effort. Without them, this work would have remained an untested theoretical novelty. While we leave many of the rich details of this prediction to the reader in Chapter 3, we showcase our experimental success in Fig. 8 (B) where the predicted induction profiles of nine double mutants (light blue shaded regions) are overlaid with their experimental measurements (points). The intimate agreement between theory and experiment illustrates the utility of using free energy shifts as a means to predict new phenotypes.</p>
<figure>
<img src="ch1_fig8.png" id="fig:double_muts_intro" data-short-caption="Theoretical prediction and experimental validation of double mutant phenotypes." alt="" /><figcaption>Figure 8: <strong>Theoretical prediction and experimental validation of double mutant phenotypes.</strong> (A) Cartoon representation of the LacI repressor with mutations in the inducer-binding domain and DNA binding domain represented by hats and socks, respectively. While the mutations have known chemical features, we characterize each mutation as potentially modifying four biophysical parameters, the dissociation constants (<span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>), the relative energy difference between active and inactive states of the repressor (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>) for inducer binding mutants, or the DNA affinity (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>) for DNA binding mutants. (B) Predicted induction profiles for pairwise double mutants are shown as blue shaded regions representing the uncertainty in our predictions. Experimental measurements are shown as blue points (means of at least 10 biological replicates). Each row corresponds to a single DNA binding domain mutation and each column to a single inducer binding domain mutation. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_03/code/ch3_fig5.py">Python code (<code>ch3_fig5.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="on-the-physiological-adaptability-of-transient-molecular-interactions">On The Physiological Adaptability of Transient Molecular Interactions</h2>
<p>In <strong>Chapers 4 and 5</strong> (and the associated supplementary <strong>Chapters 8 and 9</strong>), we explore the final level of adaptation in Fig. 1 – physiological adaptation. We do so in two distinctly different systems. The first (discussed in Chapter 4) builds upon our discussion of transcriptional regulation, but now examines how robust the biophysical parameters of the thermodynamic model are to changes in physiology, either by changing the available carbon source or by changing the temperature. Secondly, in Chapter 5, we examine physiological adaptation in the context of osmoregulation – a true matter of life and death in the single-celled world.</p>
<p>     Up to this point in our travels through scientific history, we have examined Monod’s growth curves in various pairwise combinations of sugars. A feature of note is that the presence of diauxic shifts can be seen in various organisms and for many different types of sugars such as sucrose/arabinose, glucose/sorbitol, and glucose/lactose pairings <span class="citation" data-cites="monod1947">(Monod <a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. These combinations reveal that cells are able to juggle dual-input logic systems where the “decision” to digest one carbon source or another relies on monitoring changes in concentrations of either sugar. In his 1947 treatise, Monod showed that this phenomenon was not limited to dual-carbon mixtures and presented a “triauxic” growth curve of <em>E. coli</em> grown on a glucose/sorbitol/glycerol mixture, shown in Fig. 9. This result illustrated to Monod that the mechanisms underlying enzymatic adaptation “have the character of <em>competitive interactions</em> between different specific enzyme forming systems” <span class="citation" data-cites="monod1947">(Monod <a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>.</p>
<figure>
<img src="ch1_fig9.png" id="fig:triauxic_growth" data-short-caption="A metabolic hierarchy in a growth medium containing glucose, sorbitol, and glycerol." alt="" /><figcaption>Figure 9: <strong>A metabolic hierarchy in a three-component growth mixture of glucose, sorbitol, and glycerol.</strong> A “triauxic” growth curve illustrating a hierarchy of carbon source metabolism. An <em>E. coli</em> culture was grown in a medium with equal parts glucose, sorbitol, and glycerol with utilization in that order. Auxic transitions are shown as black points and white shaded regions. Regions of the growth curve where glucose, sorbitol, and glycerol are primarily consumed are colored in blue, orange, and green, respectively. Data digitized from <span class="citation" data-cites="monod1947">Monod (<a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. Smooth line corresponds to a univariate spline applied to the data to retain the presentation in the literature. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig9.py">Python code (<code>ch1_fig9.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     These competing interactions must be resilient to a variety of physiological states. Despite the fact that the carbon atoms in glucose, sorbitol, and glycerol are all ultimately incorporated into the same biomolecules, their pathways to utilization are all distinct and include a variety of different metabolic intermediates. Furthermore, the exponential growth phases in Fig. 9 for each carbon source have different growth rates which itself results in large changes in cell volume <span class="citation" data-cites="taheri-araghi2015 jun2018">(Taheri-Araghi et al. <a href="#ref-taheri-araghi2015" role="doc-biblioref">2015</a>; Jun et al. <a href="#ref-jun2018" role="doc-biblioref">2018</a>)</span>, genome copy number <span class="citation" data-cites="nordstrom2006">(Nordström and Dasgupta <a href="#ref-nordstrom2006" role="doc-biblioref">2006</a>)</span>, and global gene expression patterns <span class="citation" data-cites="li2014 schmidt2016 hui2015">(Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Hui et al. <a href="#ref-hui2015" role="doc-biblioref">2015</a>)</span>. Despite these changes in cellular physiology, the regulatory systems underlying enzymatic adaptation still function with binding of transcription factors being ignorant of the majority of possible metabolic states of the cell.</p>
<p>     Despite this empirical observation, it has been commonly assumed that the utility of thermodynamic models of gene expression are limited and that the precise values of the biophysical parameters are directly tied to the physiological state in which they were determined. It has even been said that thermodynamic models of gene expression have been a “tactical success, yet strategic failure” in building an understanding of how genomes operate <span class="citation" data-cites="phillips2019">(Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span>.</p>
<p>     In <strong>Chapter 4</strong> of this dissertation, we quantitatively assess these assumptions in the context of gene expression by considering the theoretical models built in Chapters 2 and 3 and directly measuring the adaptability of the inducible simple repression regulatory architecture across different physiological states. Namely, we explore how predictive our thermodynamic model can be when modulating either the quality of the carbon source (glucose, glycerol, or acetate, Fig. 10 (A)) or by changing the temperature of the growth medium (32<span class="math inline"><em></em><sup>∘</sup></span> C, 37<span class="math inline"><em></em><sup>∘</sup></span> C, or 42<span class="math inline"><em></em><sup>∘</sup></span> C, Fig. 10 (B)). The culture doubling time varies by nearly a factor of four across the different conditions, illustrating the diversity in physiological states.</p>
<figure>
<img src="ch1_fig10.png" id="fig:growth_intro" data-short-caption="Methods of physiological control used in Chapter 4." alt="" /><figcaption>Figure 10: <strong>Control of cellular physiology via carbon source and temperature variation.</strong> (A) Carbon sources used in work presented in Chapter 4 (left) with “star rating” indicating quality of the carbon source. Growth curves for the three carbon sources, all at 37<span class="math inline"><em></em><sup>∘</sup></span> C are shown on right-hand panel. (B) Growth temperatures explored in Chapter 4 (left) with “star rating” indicating fastest growth rate. Growth curves (right) are shown for the three temperatures, all of which use glucose as the sole carbon source. For right-hand panels in (A) and (B), optical density is computed relative to the initial optical density of the culture. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig10.py">Python code (<code>ch1_fig10.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>      How could this variation in cellular physiology be incorporated into our thermodynamic model? Up to this point in this thesis, all experiments have been conducted in a growth medium supplemented with glucose held at a balmy 37<span class="math inline"><em></em><sup>∘</sup></span> C. However, nowhere in the thermodynamic model schematized in Fig. 4 (B) is it specified <em>which</em> carbon source must be present whereas the temperature of the system is explicitly included as a multiplicative factor <span class="math inline"><em>β</em> = (<em>k</em><sub><em>B</em></sub><em>T</em>)<sup> − 1</sup></span> in front of the exponentiated terms. These features of the model allow us to make explicit predictions of how these perturbations should influence the observed fold-change in gene expression, if at all.</p>
<p>     The parameter that we can say <em>a priori</em> is very likely to change is the repressor copy number <span class="math inline"><em>R</em></span>. In Chapters 2 and 3, we had <em>a priori</em> knowledge of the total repressor copy number as previous work directly measured their values in a particular physiological state via quantitative Western blotting <span class="citation" data-cites="garcia2011">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>. However, it has been known for nearly three-quarters of a century that the total protein content of the cell scales linearly with the growth rate <span class="citation" data-cites="schaechter1958 jun2018">(Schaechter, Maaløe, and Kjeldgaard <a href="#ref-schaechter1958" role="doc-biblioref">1958</a>; Jun et al. <a href="#ref-jun2018" role="doc-biblioref">2018</a>)</span>, a phenomenon that has recently been queried at the single-protein level through proteomic methods <span class="citation" data-cites="schmidt2016 li2014 valgepea2013a peebo2015">(Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>; Valgepea et al. <a href="#ref-valgepea2013a" role="doc-biblioref">2013</a><a href="#ref-valgepea2013a" role="doc-biblioref">a</a>; Peebo et al. <a href="#ref-peebo2015" role="doc-biblioref">2015</a>)</span>. Thus, we cannot assume that the protein copy number of the strains used in Chapters 2 and 3 can be left unperturbed. To account for this fact, we used a fluorescence-based method to directly count the number of LacI repressors per cell in each growth condition, a method which is discussed in extensive detail in Chapter 9. This experimental approach, while necessary, is extremely laborious. I am indebted to the work of Zofii A. Kaczmarek for her heroic efforts in conducting a large number of the experiments presented in Chapter 4.</p>
<p>     Our work revealed two key features of this thermodynamic model of gene expression. First, we found that the values of the biophysical parameters inferred from a single physiological state were remarkably predictive when the quality of the carbon source was decreased (Fig. 11 (A, left)). This indicates that this genetic circuit is largely insulated from the metabolic state of the cell. This is exemplified in our ability to yet again collapse the measurements, this time across different carbon sources as a function of the free energy, shown in Fig. 11 (A, right). For the carbon sources studied in this chapter, we conclude that this simple thermodynamic model can be considered both tactically and strategically successful.</p>
<p>     Yet when it comes to temperature, we find that a simple rescaling of the thermal energy of the system is <em>not</em> sufficient to predict the output of this genetic circuit when the temperature is varied (dashed lines in left-hand side of Fig. 11, (B)). This is not necessarily a surprising result as binding of transcription factors is not strictly an enthalpic process. Temperature is known to have a strong influence on many material properties of DNA, such as persistence length and salt release <span class="citation" data-cites="goethe2015">(Goethe, Fita, and Rubi <a href="#ref-goethe2015" role="doc-biblioref">2015</a>)</span>, excluded volume effects <span class="citation" data-cites="driessen2014">(Driessen et al. <a href="#ref-driessen2014" role="doc-biblioref">2014</a>)</span>, and repressor-DNA solubility <span class="citation" data-cites="elf2007">(Elf, Li, and Xie <a href="#ref-elf2007" role="doc-biblioref">2007</a>)</span>, to name a few of many effects. To phenomenologically characterize the influence of temperature on the fold-change in gene expression, we considered that there was a constant entropic penalty (though inclusion of a temperature-dependent entropic cost is discussed in Chapter 9). We found that inclusion of this parameter markedly improved the description of the data (solid lines in left-hand side of Fig. 11 (B)) and permitted data collapse within experimental noise of data collected at 37<span class="math inline"><em></em><sup>∘</sup></span> C (right-hand side of Fig. 11 (B)).</p>
<figure>
<img src="ch1_fig11.png" id="fig:growth_results_intro" data-short-caption="Performance of a simple thermodynamic model of simple repression in diverse physiological states." alt="" /><figcaption>Figure 11: <strong>Performance of a thermodynamic model of simple repression in diverse physiological states.</strong> (A) Fold-change in gene expression measurements in different carbon sources plotted against the average repressor copy number (left) and free energy (right). Black line in the left-hand panel is the predicted fold-change assuming no parameters are modified. (B) Fold-change measurements at different temperatures plotted as a function of the repressor copy number (left) and free energy (right). Dashed-lines in left-hand plot show the predicted fold-change with a simple rescaling of the thermal energy. Solid lines are predicted fold-change upon inclusion of an entropic penalty. Points on right-hand plot were computed using parameters with an entropic penalty. All measurements and errors displayed are the mean and standard error of three to eight biological replicates. Light-grey points in right-hand panels are data from <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>, <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>, <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>, and <span class="citation" data-cites="chure2019">Chure et al. (<a href="#ref-chure2019" role="doc-biblioref">2019</a>)</span>, all of which were measured in glucose-supplemented media at 37<span class="math inline"><em></em><sup>∘</sup></span> C. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig11.py">Python code (<code>ch1_fig11.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>      The inclusion of a phenomenological entropic parameter is not by any means meant to shut the book on temperature effects in this model. Rather, it serves as a representation of what <em>may</em> help explain these effects and demands more focused theoretical and experimental work. To say that the current disagreement between theory and experiment embodies the “strategic failure” of thermodynamic models is, in my view, disingenuous. To say so would be to deem the initial failures of elasticity theory to properly predict the influence of impurities and temperature on the elastic constants of materials as a “strategic failure” in material science. Initial phenomenological models of the effects of impurities and temperature on elastic properties of solids <span class="citation" data-cites="friedel1974">(Friedel <a href="#ref-friedel1974" role="doc-biblioref">1974</a>)</span> led to several decades of focused theoretical and experimental work that resulted in a complete predictive and mechanistic description <span class="citation" data-cites="phillips2001">(Phillips <a href="#ref-phillips2001" role="doc-biblioref">2001</a>)</span>. Now is the time for a similar approach to biology in the context of temperature and the regulation of gene expression.</p>
<h2 id="on-facing-the-elements">On Facing the Elements</h2>
<p>The first four chapters of this work encompass myriad perspectives of adaptive processes at the level of transcription regulation. However, just as important as the regulation is the action of the gene that is ultimately expressed. While Monod’s work described in the preceding sections was focused on the expression of enzymes, we now turn to yet another level of physiological adaptation in bacteria – the regulation of turgor pressure.</p>
<p>     In the wild, microbes are constantly faced with an array of environmental insults ranging from changes in temperature, availability of oxygen for aerobic respiration, and even chemical warfare from neighboring microbial communities <span class="citation" data-cites="czaran2002">(Czaran, Hoekstra, and Pagie <a href="#ref-czaran2002" role="doc-biblioref">2002</a>)</span>. One such environmental challenge microbes often face is the variation in the osmolarity of their surroundings. Changes in ion concentrations can result in large volumes of water rushing through the cell membrane, leading to rupture of the membrane and ultimately cell death (Fig. 12 (A)). Unsurprisingly, all domains of life have evolved clever mechanisms to combat these osmotic shocks and regulate their internal turgor pressure.</p>
<p>     One such mechanism for osmoregulation in <em>E. coli</em> is through the action of mechanosensitive ion channels – large, transmembrane structures which sense tension in the cell membrane. Exposure to a hypo-osmotic shock (where water rushes across the cell membrane <em>into</em> the cell), a change in membrane tension is sensed by these mechanosensitive channels, triggering a conformational change which opens a pore in the membrane without rupture (Fig. 12 (B)). This acts as a pressure release valve, providing a means for turgor pressure to be relieved without a potentially fatal burst. This phenomenon represents yet another system in which adaptation can be found at the molecular, evolutionary, and physiological levels. In <strong>Chapters 5 and 9</strong> of this dissertation, we explore a fundamental question – how many mechanosensitive channels does a cell need to have an appreciable chance at surviving an osmotic shock?</p>
<p>     To approach this question, Heun Jin Lee and I collaborated on the experimental and data analysis components, respectively. This is a project that had been in preparation for several years before I had the privilege of joining the team in the summer of 2017. While the experimental techniques used to probe transcriptional regulation were far from simple, they pale in comparison to those employed by Heun Jin. This project required an enormous amount of molecular biology to generate the necessary strains in which the number of mechanosensitive channels could be tuned across three orders of magnitude and measured with precision. This process involved reworking classic techniques in molecular biology to remove the presence of osmotic shocks which would prove fatal for strains with few or no mechanosensitive channels. On top of the complex biochemistry, Heun Jin developed a clever microfluidic system where osmotic shocks could be imaged in real time at the single cell level. While the majority of Chapter 5 focuses on the analysis and interpretation of the data, none of it would have been possible without Heun Jin’s Herculean efforts.</p>
<figure>
<img src="ch1_fig12.png" id="fig:mscl_intro" data-short-caption="The connection between mechanosensitive channel number and probability of survival." alt="" /><figcaption>Figure 12: <strong>The connection between mechanosensitive channel copy number and probability of survival.</strong> (A) In the absence of mechanosensitive channels, water rushing across the membrane during a hypo-osmotic shock can lead to membrane rupture and large-scale release of intracellular components into the extracellular space, resulting in cell death. (B) In the presence of mechanosensitive channels (specifically, the major <em>E. coli</em> mechanosensitive channel MscL as shown in yellow), increased membrane tension results in a conformational change of the channel, resulting in the expulsion of water and some small constituents of the intracellular milieu. The inferred survival probability curves for slow and fast shock exchange rates are shown in (C) and (D), respectively. Different shaded purple regions correspond to different credible regions of the estimates. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig12.py">Python code (<code>ch1_fig12.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     While we leave the details of the inference to Chapter 5 and the supplemental Chapter 9, the survival probability as a function of the total mechanosensitive channel number is given for “slow” and “fast” osmotic shocks in Fig. 12 (C) and (D), respectively. The credible regions in this plot illustrate that for an <span class="math inline"> ≈ 80%</span> chance of surviving either a slow or a fast osmotic shock, at least <span class="math inline"> ≈ 500</span> channels are needed. This number is in agreement with recent proteomic measurements in <em>E. coli</em> <span class="citation" data-cites="li2014 schmidt2016 soufi2015">(Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Soufi et al. <a href="#ref-soufi2015" role="doc-biblioref">2015</a>)</span>, but are at odds with current theoretical models. While it is difficult to theoretically define a survival/death criterion, current physical models predict that only a few mechanosensitive channels (specifically MscL) are needed to relieve even large increases in membrane tension. These findings illustrate another avenue in which the disagreement between theory and careful, quantitative experiments reveal gaps in our understanding of fundamental biological phenomena.</p>
<h2 id="on-molecular-biophysics-and-evolutionary-dynamics">On Molecular Biophysics and Evolutionary Dynamics</h2>
<p>This thesis as a whole presents an attempt to understand how adaptive processes operate in biological systems at a mechanistic level beyond qualitative description. The thermodynamic model derived and explored in Chapter 2 presents a concrete theoretical framework through which we can understand how mutations and environmental perturbations influence the output of a simple genetic circuit with quantitative precision. While the work here specifically explores the <em>mean</em> level of gene expression of a population, I have had the privilege to be involved in several projects which explore the complete distribution of gene expression of various regulatory motifs using non-equilibrium models <span class="citation" data-cites="razo-mejia2020 laxhuber2020">(Razo-Mejia et al. <a href="#ref-razo-mejia2020" role="doc-biblioref">2020</a>; Laxhuber et al. <a href="#ref-laxhuber2020" role="doc-biblioref">2020</a>)</span>. Both equilibrium and non-equilibrium approaches, while differing in their fundamental assumptions of the system, can be used to understand how the regulation of gene expression occurs <em>in vivo</em> and should be viewed as complimentary rather than adversarial approaches.</p>
<p>     A combination of these types of approaches will be necessary to attack what I believe is the next great frontier of biological physics – predicting evolution. While this thesis is primarily focused on a single type of regulatory architecture regulating a single promoter via a single species of transcription factor, it is worth remembering that systems-level phenotypes are often complex and result from the concerted action of an array of biological processes. As was mentioned in our discussion on physiological adaptation, it has been known for nearly a century that the bacterial growth rate is directly correlated to the total protein content of the cell, with recent works illustrating rich phenomenology in the structure of the bacterial proteome as a whole <span class="citation" data-cites="li2014 schmidt2016 hui2015 scott2010 klumpp2014">(Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Hui et al. <a href="#ref-hui2015" role="doc-biblioref">2015</a>; Scott et al. <a href="#ref-scott2010" role="doc-biblioref">2010</a>; Klumpp and Hwa <a href="#ref-klumpp2014" role="doc-biblioref">2014</a>)</span>.</p>
<p>     In collaboration again with Nathan M. Belliveau, we have begun to explore how the composition of the bacterial proteome is structured at the single-protein level. Fig. 13 (A) shows data compiled from several independent proteomic data sets (using either quantitative mass spectrometry <span class="citation" data-cites="schmidt2016 peebo2015 valgepea2013">(Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Peebo et al. <a href="#ref-peebo2015" role="doc-biblioref">2015</a>; Valgepea et al. <a href="#ref-valgepea2013" role="doc-biblioref">2013</a><a href="#ref-valgepea2013" role="doc-biblioref">a</a>)</span> or ribosomal profiling <span class="citation" data-cites="li2014">(Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>)</span>) where the abundance of different molecular constituents of the bacterial proteome is plotted as a function of the growth rate. These components, broken down by their functional designation according to their Cluster of Orthologous Groups (COG) annotation <span class="citation" data-cites="galperin2015">(Galperin et al. <a href="#ref-galperin2015" role="doc-biblioref">2015</a>)</span>, reveal varied dependencies on the growth rate. Of note are the COG classes “cellular processes and signaling,” “metabolism,” and “information storage and processing” which all appear to have a correlation between the cellular growth rate and the total mass of that proteome sector. However, when plotted as the total <em>mass fraction</em> of the proteome instead of the total mass, a striking result is observed. Fig. 13 (B) reveals a very strong, negative correlation between the mass fraction of the proteome dedicated to information storage and processing (including ribosomal and transcriptional machinery) and the proteome fraction dedicated to metabolism. This direct competition for resources between the proteins involved in translation (ribosomes, elongation factors, etc.) and metabolic networks has been shown previously <span class="citation" data-cites="klumpp2008 scott2010 hui2015">(Klumpp and Hwa <a href="#ref-klumpp2008" role="doc-biblioref">2008</a>; Scott et al. <a href="#ref-scott2010" role="doc-biblioref">2010</a>; Hui et al. <a href="#ref-hui2015" role="doc-biblioref">2015</a>)</span> and suggests a strong evolutionary constraint on how resources can be optimally partitioned.</p>
<figure>
<img src="ch1_fig13.png" id="fig:proteomics" data-short-caption="Allocation of cellular resources induces compositional structure in the *E. coli* proteome." alt="" /><figcaption>Figure 13: <strong>Allocation of cellular resources induces compositional structure in the <em>E. coli</em> proteome.</strong> (A) The total proteome mass of the five major annotated COG categories is shown as a function of the experimental growth rate. Different marker shapes represent different data sets. (B) The fraction by mass of the proteome dedicated to metabolic machinery plotted as a function of the total proteome mass dedicated to the processes of the central dogma. Different shapes correspond to the different data sets shown in (A). Color indicates increasing growth rate from yellow to dark blue. Data shown in this figure come from <span class="citation" data-cites="peebo2015">Peebo et al. (<a href="#ref-peebo2015" role="doc-biblioref">2015</a>)</span> (inverted triangles), <span class="citation" data-cites="li2014">Li et al. (<a href="#ref-li2014" role="doc-biblioref">2014</a>)</span> (triangles), <span class="citation" data-cites="schmidt2016">Schmidt et al. (<a href="#ref-schmidt2016" role="doc-biblioref">2016</a>)</span> (circles), and <span class="citation" data-cites="valgepea2013">Valgepea et al. (<a href="#ref-valgepea2013" role="doc-biblioref">2013</a><a href="#ref-valgepea2013" role="doc-biblioref">a</a>)</span> (diamonds). The <a href="https://github.com/gchure/phd/blob/master/src/chapter_01/code/ch1_fig13.py">Python code (<code>ch1_fig13.py</code>)</a> used to generate this figure can be accessed via the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>      As of this writing, our understanding of the cellular resource allocation visible in Fig. 13 remains largely phenomenological <span class="citation" data-cites="scott2014">(Scott et al. <a href="#ref-scott2014" role="doc-biblioref">2014</a>)</span>. This is in part due to the tremendously high-dimensional nature of systems-level organization. Our understanding of systems with such huge numbers of degrees-of-freedom have classically benefited enormously from the application of statistical mechanics as this thesis shows in the context of transcriptional regulation. The quantitative framework derived and carefully dissected in this thesis, I believe, lays the groundwork to understand how phenomena such as that shown in Fig. 13 (B) arise, and perhaps more importantly, evolve. A recent work from Michael Lässig, Ville Mustonen, and Aleksandra Walczak entitled <em>Predicting evolution</em> <span class="citation" data-cites="lassig2017">(Lässig, Mustonen, and Walczak <a href="#ref-lassig2017" role="doc-biblioref">2017</a>)</span> describes what the future of evolutionary theory may look like given these types of models. Recent technological advancements in sequencing, microscopy, and computation coupled with theoretical advancements in the biophysics of gene regulation present an opportunity for a rich theoretical dialogue between molecular biophysics and evolutionary dynamics coupled with experimental dissection (Fig. 14).</p>
<p>     The first 20 years of the 21<span class="math inline"><em></em><sup>st</sup></span> century brought a paradigm shift in our understanding of noise in biological networks, illustrating how cross-disciplinary approaches to scientific discovery can solve and even create new fields of biological inquiry <span class="citation" data-cites="ciechonska2016 eldar2010b">(Ciechonska, Grob, and Isalan <a href="#ref-ciechonska2016" role="doc-biblioref">2016</a>; Eldar and Elowitz <a href="#ref-eldar2010b" role="doc-biblioref">2010</a>)</span>. I hope that some of the material described in the coming chapters can help contribute to a systems-biology approach to evolution.</p>
<figure>
<img src="ch1_fig14.png" id="fig:molbiophys_evoldynam" data-short-caption="The coming interplay between molecular biophysics and evolutionary dynamics." alt="" /><figcaption>Figure 14: <strong>The coming interplay between molecular biophysics and evolutionary dynamics.</strong> (A) Recent progress in our understanding of the structure and function of biological networks has resulted in many examples where high-dimensional biological phenomena can be boiled down to effective phenomena. Future work will draw from our understanding of these networks to place them in an evolutionary perspective (B) where the connection between perturbations at the level of nodes in biological networks can be drawn to fitness and evolutionary trajectories can be predicted.</figcaption>
</figure>
<h2 class="unnumbered" id="references" class="unnumbered">References</h2>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-auerbach2012">
<p>Auerbach, Anthony. 2012. “Thinking in Cycles: MWC Is a Good Model for Acetylcholine Receptor-Channels.” <em>The Journal of Physiology</em> 590 (1): 93–98. <a href="https://doi.org/10.1113/jphysiol.2011.214684">https://doi.org/10.1113/jphysiol.2011.214684</a>.</p>
</div>
<div id="ref-bintu2005">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Models.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 116–24. <a href="https://doi.org/10.1016/j.gde.2005.02.007">https://doi.org/10.1016/j.gde.2005.02.007</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates the Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-buchler2003">
<p>Buchler, N. E., U. Gerland, and T. Hwa. 2003. “On Schemes of Combinatorial Transcription Logic.” <em>Proceedings of the National Academy of Sciences</em> 100 (9): 5136–41. <a href="https://doi.org/10.1073/pnas.0930314100">https://doi.org/10.1073/pnas.0930314100</a>.</p>
</div>
<div id="ref-canals2012">
<p>Canals, Meritxell, J. Robert Lane, Adriel Wen, Peter J. Scammells, Patrick M. Sexton, and Arthur Christopoulos. 2012. “A Monod-Wyman-Changeux Mechanism Can Explain G Protein-Coupled Receptor (GPCR) Allosteric Modulation.” <em>Journal of Biological Chemistry</em> 287 (1): 650–59. <a href="https://doi.org/10.1074/jbc.M111.314278">https://doi.org/10.1074/jbc.M111.314278</a>.</p>
</div>
<div id="ref-chure2019">
<p>Chure, Griffin, Manuel Razo-Mejia, Nathan M Belliveau, Tal Einav, Zofii A Kaczmarek, Stephanie L Barnes, and Rob Phillips. 2019. “Predictive Shifts in Free Energy Couple Mutations to Their Phenotypic Consequences.” <em>Proceedings of the National Academy of Sciences</em> 116 (37). <a href="https://doi.org/10.1073/pnas.1907869116">https://doi.org/10.1073/pnas.1907869116</a>.</p>
</div>
<div id="ref-ciechonska2016">
<p>Ciechonska, Marta, Alice Grob, and Mark Isalan. 2016. “From Noise to Synthetic Nucleoli: Can Synthetic Biology Achieve New Insights?” <em>Integrative Biology</em> 8 (4): 383–93. <a href="https://doi.org/10.1039/c5ib00271k">https://doi.org/10.1039/c5ib00271k</a>.</p>
</div>
<div id="ref-czaran2002">
<p>Czaran, T. L., R. F. Hoekstra, and L. Pagie. 2002. “Chemical Warfare Between Microbes Promotes Biodiversity.” <em>Proceedings of the National Academy of Sciences</em> 99 (2): 786–90. <a href="https://doi.org/10.1073/pnas.012399899">https://doi.org/10.1073/pnas.012399899</a>.</p>
</div>
<div id="ref-dienert1900">
<p>Diénert, Frédéric. 1900. <em>Sur La Fermentation Du Galactose et Sur L’accoutumance Des Levures à Ce Sucre</em>. Sceaux: E. Charaire.</p>
</div>
<div id="ref-dill2010">
<p>Dill, Kenneth A., and Sarina Bromberg. 2010. <em>Molecular Driving Forces: Statistical Thermodynamics in Chemistry and Biology</em>. New York: Garland Science.</p>
</div>
<div id="ref-driessen2014">
<p>Driessen, Rosalie P. C., Gerrit Sitters, Niels Laurens, Geri F. Moolenaar, Gijs J. L. Wuite, Nora Goosen, and Remus Th. Dame. 2014. “Effect of Temperature on the Intrinsic Flexibility of DNA and Its Interaction with Architectural Proteins.” <em>Biochemistry</em> 53 (41): 6430–8. <a href="https://doi.org/10.1021/bi500344j">https://doi.org/10.1021/bi500344j</a>.</p>
</div>
<div id="ref-einav2016">
<p>Einav, Tal, Linas Mazutis, and Rob Phillips. 2016. “Statistical Mechanics of Allosteric Enzymes.” <em>The Journal of Physical Chemistry B</em> 120 (26): 6021–37. <a href="https://doi.org/10.1021/acs.jpcb.6b01911">https://doi.org/10.1021/acs.jpcb.6b01911</a>.</p>
</div>
<div id="ref-einav2017">
<p>Einav, Tal, and Rob Phillips. 2017. “Monod-Wyman-Changeux Analysis of Ligand-Gated Ion Channel Mutants.” <em>Journal of Physical Chemistry B</em> 121 (15): 3813–24. <a href="https://doi.org/10.1101/102194">https://doi.org/10.1101/102194</a>.</p>
</div>
<div id="ref-eldar2010b">
<p>Eldar, Avigdor, and Michael B. Elowitz. 2010. “Functional Roles for Noise in Genetic Circuits.” <em>Nature</em> 467 (7312): 167–73. <a href="https://doi.org/10.1038/nature09326">https://doi.org/10.1038/nature09326</a>.</p>
</div>
<div id="ref-elf2007">
<p>Elf, Johan, Gene-Wei Li, and X. Sunney Xie. 2007. “Probing Transcription Factor Dynamics at the Single-Molecule Level in a Living Cell.” <em>Science</em> 316 (5828): 1191–4. <a href="https://doi.org/10.1126/science.1141967">https://doi.org/10.1126/science.1141967</a>.</p>
</div>
<div id="ref-friedel1974">
<p>Friedel, J. 1974. “On the Stability of the Body Centred Cubic Phase in Metals at High Temperatures.” <em>Journal de Physique Lettres</em> 35 (4): 59–63. <a href="https://doi.org/10.1051/jphyslet:0197400350405900">https://doi.org/10.1051/jphyslet:0197400350405900</a>.</p>
</div>
<div id="ref-galperin2015">
<p>Galperin, Michael Y., Kira S. Makarova, Yuri I. Wolf, and Eugene V. Koonin. 2015. “Expanded Microbial Genome Coverage and Improved Protein Family Annotation in the COG Database.” <em>Nucleic Acids Research</em> 43 (D1): D261–D269. <a href="https://doi.org/10.1093/nar/gku1223">https://doi.org/10.1093/nar/gku1223</a>.</p>
</div>
<div id="ref-gama-castro2016">
<p>Gama-Castro, Socorro, Heladia Salgado, Alberto Santos-Zavaleta, Daniela Ledezma-Tejeida, Luis Muñiz-Rascado, Jair Santiago García-Sotelo, Kevin Alquicira-Hernández, et al. 2016. “RegulonDB Version 9.0: High-Level Integration of Gene Regulation, Coexpression, Motif Clustering and Beyond.” <em>Nucleic Acids Research</em> 44 (D1): D133–D143. <a href="https://doi.org/10.1093/nar/gkv1156">https://doi.org/10.1093/nar/gkv1156</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-goethe2015">
<p>Goethe, Martin, Ignacio Fita, and J. Miguel Rubi. 2015. “Vibrational Entropy of a Protein: Large Differences Between Distinct Conformations.” <em>Journal of Chemical Theory and Computation</em> 11 (1): 351–59. <a href="https://doi.org/10.1021/ct500696p">https://doi.org/10.1021/ct500696p</a>.</p>
</div>
<div id="ref-huang2018">
<p>Huang, Min, Kun Song, Xinyi Liu, Shaoyong Lu, Qiancheng Shen, Renxiao Wang, Jingze Gao, et al. 2018. “AlloFinder: A Strategy for Allosteric Modulator Discovery and Allosterome Analyses.” <em>Nucleic Acids Research</em> 46 (W1): W451–W458. <a href="https://doi.org/10.1093/nar/gky374">https://doi.org/10.1093/nar/gky374</a>.</p>
</div>
<div id="ref-hui2015">
<p>Hui, Sheng, Josh M. Silverman, Stephen S. Chen, David W. Erickson, Markus Basan, Jilong Wang, Terence Hwa, and James R. Williamson. 2015. “Quantitative Proteomic Analysis Reveals a Simple Strategy of Global Resource Allocation in Bacteria.” <em>Molecular Systems Biology</em> 11 (2). <a href="https://doi.org/10.15252/msb.20145697">https://doi.org/10.15252/msb.20145697</a>.</p>
</div>
<div id="ref-ireland2020">
<p>Ireland, William T., Suzannah M. Beeler, Emanuel Flores-Bautista, Nathan M. Belliveau, Michael J. Sweredoski, Annie Moradian, Justin B. Kinney, and Rob Phillips. 2020. “Deciphering the Regulatory Genome of <em>Escherichia Coli</em>, One Hundred Promoters at a Time.” <em>bioRxiv</em>, January. <a href="https://doi.org/10.1101/2020.01.18.910323">https://doi.org/10.1101/2020.01.18.910323</a>.</p>
</div>
<div id="ref-jablonski2001">
<p>Jablonski, David. 2001. “Lessons from the Past: Evolutionary Impacts of Mass Extinctions.” <em>Proceedings of the National Academy of Sciences</em> 98 (10): 5393–8. <a href="https://doi.org/10.1073/pnas.101092598">https://doi.org/10.1073/pnas.101092598</a>.</p>
</div>
<div id="ref-jun2018">
<p>Jun, Suckjoon, Fangwei Si, Rami Pugatch, and Matthew Scott. 2018. “Fundamental Principles in Bacterial Physiology - History, Recent Progress, and the Future with Focus on Cell Size Control: A Review.” <em>Reports on Progress in Physics</em> 81 (5): 056601. <a href="https://doi.org/10.1088/1361-6633/aaa628">https://doi.org/10.1088/1361-6633/aaa628</a>.</p>
</div>
<div id="ref-keymer2006">
<p>Keymer, Juan E., Robert G. Endres, Monica Skoge, Yigal Meir, and Ned S. Wingreen. 2006. “Chemosensing in <em>Escherichia Coli</em>: Two Regimes of Two-State Receptors.” <em>Proceedings of the National Academy of Sciences</em> 103 (6): 1786–91. <a href="https://doi.org/10.1073/pnas.0507438103">https://doi.org/10.1073/pnas.0507438103</a>.</p>
</div>
<div id="ref-klumpp2008">
<p>Klumpp, Stefan, and Terence Hwa. 2008. “Growth-Rate-Dependent Partitioning of RNA Polymerases in Bacteria.” <em>Proceedings of the National Academy of Sciences</em> 105 (51): 20245–50. <a href="https://doi.org/10.1073/pnas.0804953105">https://doi.org/10.1073/pnas.0804953105</a>.</p>
</div>
<div id="ref-klumpp2014">
<p>———. 2014. “Bacterial Growth: Global Effects on Gene Expression, Growth Feedback and Proteome Partition.” <em>Current Opinion in Biotechnology</em> 28 (August): 96–102. <a href="https://doi.org/10.1016/j.copbio.2014.01.001">https://doi.org/10.1016/j.copbio.2014.01.001</a>.</p>
</div>
<div id="ref-laxhuber2020">
<p>Laxhuber, Kathrin S, Muir Morrison, Griffin Chure, Nathan M Belliveau, Charlotte Strandkvist, Kyle Naughton, and Rob Phillips. 2020. “Theoretical Investigation of a Genetic Switch for Metabolic Adaptation.” <em>PLOS ONE</em> (in press).</p>
</div>
<div id="ref-lassig2017">
<p>Lässig, Michael, Ville Mustonen, and Aleksandra M. Walczak. 2017. “Predicting Evolution.” <em>Nature Ecology &amp; Evolution</em> 1 (3): 0077. <a href="https://doi.org/10.1038/s41559-017-0077">https://doi.org/10.1038/s41559-017-0077</a>.</p>
</div>
<div id="ref-li2014">
<p>Li, Gene-Wei, David Burkhardt, Carol Gross, and Jonathan S. Weissman. 2014. “Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources.” <em>Cell</em> 157 (3): 624–35. <a href="https://doi.org/10.1016/j.cell.2014.02.033">https://doi.org/10.1016/j.cell.2014.02.033</a>.</p>
</div>
<div id="ref-lindsley2006a">
<p>Lindsley, Janet E., and Jared Rutter. 2006. “Whence Cometh the Allosterome?” <em>Proceedings of the National Academy of Sciences</em> 103 (28): 10533–5. <a href="https://doi.org/10.1073/pnas.0604452103">https://doi.org/10.1073/pnas.0604452103</a>.</p>
</div>
<div id="ref-loison2013">
<p>Loison, Laurent. 2013. “Monod Before Monod: Enzymatic Adaptation, Lwoff, and the Legacy of General Biology.” <em>History and Philosophy of the Life Sciences</em> 35 (2): 167–92.</p>
</div>
<div id="ref-luo2018">
<p>Luo, Genming, Christopher K. Junium, Gareth Izon, Shuhei Ono, Nicolas J. Beukes, Thomas J. Algeo, Ying Cui, Shucheng Xie, and Roger E. Summons. 2018. “Nitrogen Fixation Sustained Productivity in the Wake of the Palaeoproterozoic Great Oxygenation Event.” <em>Nature Communications</em> 9 (1): 1–9. <a href="https://doi.org/10.1038/s41467-018-03361-2">https://doi.org/10.1038/s41467-018-03361-2</a>.</p>
</div>
<div id="ref-monod1941">
<p>Monod, Jacques. 1941. “Sur Un Phénomeene Nouveau de Croissance Complexe Dans Les Culures Bactériennes.” <em>Comptes Rendus Des Séances de L’Académie Des Sciences</em> 212: 934–39.</p>
</div>
<div id="ref-monod1947">
<p>———. 1947. “The Phenomenon of Enzymatic Adaptation and Its Bearings on Problems of Genetics and Cellular Differentiation.” <em>Growth Symposium</em> 9: 223–89.</p>
</div>
<div id="ref-monod1966">
<p>———. 1966. “From Enzymatic Adaptation to Allosteric Transitions.” <em>Science</em> 154 (3748): 475–83. <a href="https://doi.org/10.1126/science.154.3748.475">https://doi.org/10.1126/science.154.3748.475</a>.</p>
</div>
<div id="ref-monod1963">
<p>Monod, Jacques, Jean-Pierre Changeux, and François Jacob. 1963. “Allosteric Proteins and Cellular Control Systems.” <em>Journal of Molecular Biology</em> 6 (4): 306–29. <a href="https://doi.org/10.1016/S0022-2836(63)80091-1">https://doi.org/10.1016/S0022-2836(63)80091-1</a>.</p>
</div>
<div id="ref-monod1965">
<p>Monod, Jacque, Jeffries Wyman, and Jean-Pierre Changeux. 1965. “On the Nature of Allosteric Transitions: A Plausible Model.” <em>Journal of Molecular Biology</em> 12 (1): 88–118. <a href="https://doi.org/10.1016/S0022-2836(65)80285-6">https://doi.org/10.1016/S0022-2836(65)80285-6</a>.</p>
</div>
<div id="ref-nordstrom2006">
<p>Nordström, Kurt, and Santanu Dasgupta. 2006. “Copy-Number Control of the <em>Escherichia Coli</em> Chromosome: A Plasmidologist’s View.” <em>EMBO Reports</em> 7 (5): 484–89. <a href="https://doi.org/10.1038/sj.embor.7400681">https://doi.org/10.1038/sj.embor.7400681</a>.</p>
</div>
<div id="ref-peebo2015">
<p>Peebo, Karl, Kaspar Valgepea, Andres Maser, Ranno Nahku, Kaarel Adamberg, and Raivo Vilu. 2015. “Proteome Reallocation in <em>Escherichia Coli</em> with Increasing Specific Growth Rate.” <em>Molecular BioSystems</em> 11 (4): 1184–93. <a href="https://doi.org/10.1039/C4MB00721B">https://doi.org/10.1039/C4MB00721B</a>.</p>
</div>
<div id="ref-phillips2015">
<p>Phillips, Rob. 2015. “Napoleon Is in Equilibrium.” <em>Annual Review of Condensed Matter Physics</em> 6 (1): 85–111. <a href="https://doi.org/10.1146/annurev-conmatphys-031214-014558">https://doi.org/10.1146/annurev-conmatphys-031214-014558</a>.</p>
</div>
<div id="ref-phillips2001">
<p>———. 2001. <em>Crystals, Defects and Microstructures</em>. Cambridge University Press. <a href="https://doi.org/10.1017/CBO9780511606236">https://doi.org/10.1017/CBO9780511606236</a>.</p>
</div>
<div id="ref-phillips2019">
<p>Phillips, Rob, Nathan M. Belliveau, Griffin Chure, Hernan G. Garcia, Manuel Razo-Mejia, and Clarissa Scholes. 2019. “Figure 1 Theory Meets Figure 2 Experiments in the Study of Gene Expression.” <em>Annual Review of Biophysics</em> 48 (1): 121–63. <a href="https://doi.org/10.1146/annurev-biophys-052118-115525">https://doi.org/10.1146/annurev-biophys-052118-115525</a>.</p>
</div>
<div id="ref-razo-mejia2018">
<p>Razo-Mejia, Manuel, Stephanie L. Barnes, Nathan M. Belliveau, Griffin Chure, Tal Einav, Mitchell Lewis, and Rob Phillips. 2018. “Tuning Transcriptional Regulation Through Signaling: A Predictive Theory of Allosteric Induction.” <em>Cell Systems</em> 6 (4): 456–469.e10. <a href="https://doi.org/10.1016/j.cels.2018.02.004">https://doi.org/10.1016/j.cels.2018.02.004</a>.</p>
</div>
<div id="ref-razo-mejia2020">
<p>Razo-Mejia, Manuel, Sarah Marzen, Griffin Chure, Muir Morrison, Rachel Taubman, and Rob Phillips. 2020. “First-Principles Prediction of the Information Processing Capacity of a Simple Genetic Circuit.” <em>arXiv</em>.</p>
</div>
<div id="ref-schaechter1958">
<p>Schaechter, M., O. Maaløe, and N. O. Kjeldgaard. 1958. “Dependency on Medium and Temperature of Cell Size and Chemical Composition During Balanced Growth of <em>Salmonella Typhimurium</em>.” <em>Microbiology</em> 19 (3): 592–606. <a href="https://doi.org/10.1099/00221287-19-3-592">https://doi.org/10.1099/00221287-19-3-592</a>.</p>
</div>
<div id="ref-schmidt2016">
<p>Schmidt, Alexander, Karl Kochanowski, Silke Vedelaar, Erik Ahrné, Benjamin Volkmer, Luciano Callipo, Kèvin Knoops, Manuel Bauer, Ruedi Aebersold, and Matthias Heinemann. 2016. “The Quantitative and Condition-Dependent <em>Escherichia Coli</em> Proteome.” <em>Nature Biotechnology</em> 34 (1): 104–10. <a href="https://doi.org/10.1038/nbt.3418">https://doi.org/10.1038/nbt.3418</a>.</p>
</div>
<div id="ref-scott2010">
<p>Scott, Matthew, Carl W. Gunderson, Eduard M. Mateescu, Zhongge Zhang, and Terence Hwa. 2010. “Interdependence of Cell Growth and Gene Expression: Origins and Consequences.” <em>Science</em> 330 (6007): 1099–1102. <a href="https://doi.org/10.1126/science.1192588">https://doi.org/10.1126/science.1192588</a>.</p>
</div>
<div id="ref-scott2014">
<p>Scott, M., S. Klumpp, E. M. Mateescu, and T. Hwa. 2014. “Emergence of Robust Growth Laws from Optimal Regulation of Ribosome Synthesis.” <em>Molecular Systems Biology</em> 10 (8): 747–47. <a href="https://doi.org/10.15252/msb.20145379">https://doi.org/10.15252/msb.20145379</a>.</p>
</div>
<div id="ref-soufi2015">
<p>Soufi, Boumediene, Karsten Krug, Andreas Harst, and Boris Macek. 2015. “Characterization of the <em>E. Coli</em> Proteome and Its Modifications During Growth and Ethanol Stress.” <em>Frontiers in Microbiology</em> 6 (February). <a href="https://doi.org/10.3389/fmicb.2015.00103">https://doi.org/10.3389/fmicb.2015.00103</a>.</p>
</div>
<div id="ref-swem2008">
<p>Swem, Lee R., Danielle L. Swem, Ned S. Wingreen, and Bonnie L. Bassler. 2008. “Deducing Receptor Signaling Parameters from <em>in Vivo</em> Analysis: LuxN/AI-1 Quorum Sensing in <em>Vibrio Harveyi</em>.” <em>Cell</em> 134 (3): 461–73. <a href="https://doi.org/10.1016/j.cell.2008.06.023">https://doi.org/10.1016/j.cell.2008.06.023</a>.</p>
</div>
<div id="ref-taheri-araghi2015">
<p>Taheri-Araghi, Sattar, Serena Bradde, John T. Sauls, Norbert S. Hill, Petra A. Levin, Johan Paulsson, Massimo Vergassola, and Suckjoon Jun. 2015. “Cell-Size Control and Homeostasis in Bacteria.” <em>Current Biology</em> 25 (3): 385–91. <a href="https://doi.org/10.1016/j.cub.2014.12.009">https://doi.org/10.1016/j.cub.2014.12.009</a>.</p>
</div>
<div id="ref-ullmann2011">
<p>Ullmann, Agnes. 2011. “In Memoriam: Jacques Monod (1910-1976).” <em>Genome Biology and Evolution</em> 3 (January): 1025–33. <a href="https://doi.org/10.1093/gbe/evr024">https://doi.org/10.1093/gbe/evr024</a>.</p>
</div>
<div id="ref-valgepea2013">
<p>Valgepea, Kaspar, Kaarel Adamberg, Andrus Seiman, and Raivo Vilu. 2013a. “Escherichia Coli Achieves Faster Growth by Increasing Catalytic and Translation Rates of Proteins.” <em>Molecular BioSystems</em> 9 (9): 2344–58. <a href="https://doi.org/10.1039/C3MB70119K">https://doi.org/10.1039/C3MB70119K</a>.</p>
</div>
<div id="ref-valgepea2013">
<p>———. 2013a. “Escherichia Coli Achieves Faster Growth by Increasing Catalytic and Translation Rates of Proteins.” <em>Molecular BioSystems</em> 9 (9): 2344–58. <a href="https://doi.org/10.1039/C3MB70119K">https://doi.org/10.1039/C3MB70119K</a>.</p>
</div>
<div id="ref-vilar2003">
<p>Vilar, José M. G., and Stanislas Leibler. 2003. “DNA Looping and Physical Constraints on Transcription Regulation.” <em>Journal of Molecular Biology</em> 331 (5): 981–89. <a href="https://doi.org/10.1016/S0022-2836(03)00764-2">https://doi.org/10.1016/S0022-2836(03)00764-2</a>.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/abstract" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Abstract
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_02" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd//" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>


<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-109526846-4', 'auto');
	ga('send', 'pageview');
</script>



		</div><!-- /.js-footer-area -->
	</body>
</html>
