<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter I</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd/favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd/doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd/" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd/doks-theme/assets/images/aspens.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter I</h2>
								
								
									<h1 class="hero-subheader__title">Janus-Faced Molecules and Adaptation Across Biological Scales     
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/published_content" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_2" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
						<div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
							<div class="align-container" data-mh>
								<div class="align-inner">
									<div class="hero-subheader__author">
										<p class="hero-subheader__author-title">
												Summary
												<i class="icon icon--chevron-down"></i>
										</p><!-- /.hero-subheader__author-title -->
										<p>All forms of life are united in their obedience to the forces of evolution. The ability to adapt -- either at the level of molecules or at the level of populations -- is central to evolution. Blah Blah 
</p>
									</div><!-- /.hero-subheader__author -->
								</div><!-- /.align-inner -->
							</div><!-- /.align-container -->
						</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/published_content" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Published Content
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_2" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<h2 id="introduction">Introduction</h2>
<p>From archaea thriving in hydrothermal vents on the ocean floor to aspen trees dominating a Coloradan mountainside, all forms of life are unified in their obedience to the whims of their environment. Over the past 3.5 billion years of evolution, life has evolved myriad clever ways to combat (and exploit) environmental fluctuations to amplify reproductive success. The mechanisms behind this adaptation are diverse and traverse the biological scales ranging from nanosecond-scale conformational switching of proteins (Fig. 1(C)), to large-scale reconfiguration of metabolic networks (Fig. 1 (B)), to evolutionary trajectories that become detectable over many generations (Fig. 1 (C)). Thus, while “adaptation” is colloquialy associated <em>only</em> with organisms, one can use the same language to describe the microscopic operations of molecules.</p>
<figure>
<img src="ch1_fig1.png" id="fig:adaptation_levels" data-short-caption="The spatial, temporal, and mechanistic scale of adaptation." alt="" /><figcaption>Figure 1: <strong>The spatial, temporal, and mechanistic scale of adaptation.</strong> (A) Molecular adaptation in this work is defined through the lens of allostery where the activity of a protein complex is modulated the reversible binding of a small molecule. These binding and unbinding events lead to rapid changes in protein conformation whose behavior (both energetic and temporal) is comparable to that of thermal motion. (B) Physiological adaptation here is defined as the rewiring of biochemical reaction networks that lead to changes in cellular behavior (such as chemotaxis) or metabolic capacity (such as aerobic to fermentative metabolism). (C) Evolutionary adaptation is recorded in the variation in the genetic sequence of regulatory molecules. Variations in sequence influence the function of the proteins and RNAs they encode which ultimately define the cellular fitness.</figcaption>
</figure>
<p>     This idea of molecular adaption is not novel and demands a brief foray into the history of bacterial growth and the dawn of regulatory biology. In the late 1890’s, Emilé Duclaux and his graduate student Frédéric Diénert performed a series of experiments that revealed the phenomenon of <em>enzymatic adaptation</em> where “the production of diastases [enzymes] depends on the manner of nutrition” in which the cultures were grown <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. This is one of the the first observations of the fact that, while an organism may be able to digest a certain sugar, it may not <em>always</em> be able to do so. Rather, there seemed to be certain conditions in which the production or formation of these enzymes could occur. In his doctoral thesis in 1900, Diénert proposed two mechanisms for the origin of enzymatic adaptation observed for galactozymase <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. Either (a) the presence of galactose directly transformed enzymes already present in the cell (<em>S. cerevisiae</em>) into galactozymase or (b) that the galactose activated the production of galactozymase <em>de novo</em> <span class="citation" data-cites="dienert1900">(Diénert <a href="#ref-dienert1900" role="doc-biblioref">1900</a>, [@loison2013])</span>.</p>
<p>     Nearly half a century later, Jacques Monod would rediscover the phenomenon of enzymatic adaptation, this time in the context of bacterial growth. In his 1941 work, <em>Sur un phénoméne nouveau de croissance complexe dans les cultures bactériennes</em>, Monod for the first time reported on the phenomenon of diauxic growth, shown in Fig. 2 (A). He noted that for some mixtures of carbon sources, the culture grew “kinetically normal” meaning exponential growth to a saturating level blue points, 2 (A). However, some mixtures (such as sucrose and arabinose) led biphasic growth mode where the culture would grow exponentially, undergo a period where growth had ceased, followed by again by another round of exponential growth blue points, 2 (A). Additionally, Monod showed that the onset of this diauxic shift could be tuned by varying the relative concentrations of the carbon sources, indicating a controllable, chemical basis for the adaptive process(Fig. 2 (B)).</p>
<p>     Monod immediately recognized a connection between diauxic growth and enzymatic adaptation <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. Despite his work appearing 40 years after the pioneering work of Ducleaux and Diéner, there had been little progress towards a mechanistic, needless to say quantitative, explanation for the phenomenon of enzymatic adaptation. In fact, Monod was particularly disappointed by the teleological “explanations” for the phenomenon where the cells somehow controlled their behavior to perform only the chemical reactions that were needed <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. The teleological approach to much of biology during this time period, especially in the French scientific community, severely bothered Monod. To him, this kind of approach lacked the “postulate of objectivity” that other fields of science (particularly physics) had adopted and belonged to a pre-scientific era of biology <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. Near the middle of the 20th century, Monod published a remarkable treatise on the the phenomena of enzymatic adaptation with the level of quantitative rigor he thought it deserved <span class="citation" data-cites="monod1947">(Monod <a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. In this work, he set out to progressively deconstruct and invalidate a series of hypotheses for the phenomenon of enzymatic adaptation. In doing so, he laid the groundwork for what would become (in his opinion) his greatest contribution to science, the nature of allosteric transitions <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>, [@monod1963], [@monod1965])</span>, a topic that will feature prominently in the remainder of this thesis.</p>
<figure>
<img src="ch1_fig2.png" id="fig:diauxie_fig" data-short-caption="The phenomenon of enzymatic adaptation revealed in bacterial growth curves." alt="" /><figcaption>Figure 2: <strong>The phenomenon of enzymatic adaptation revealed in bacterial growth curves.</strong> (A) Optical density measurements of <em>Bacillus subtilis</em> cultures grown in a mixture of sucrose and either glucose (blue points) or arabinose (green points). Biphasic growth can be observed in the sucrose/arabinose mixture where the pause in growth (white vertical line) corresponds to enzymatic adaptation. Data digitized from <span class="citation" data-cites="monod1941">Monod (<a href="#ref-monod1941" role="doc-biblioref">1941</a>)</span>. (B) Diauxic growth curves of <em>Escherichia coli</em> cells grown on a mixture of glucose and sorbitol in different proportions. Data digitized from <span class="citation" data-cites="monod1947">Monod (<a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. Periods of enzymatic adaptation are highlighted by white vertical lines.</figcaption>
</figure>
<p>     The diauxic growth transitions shown in Fig. 2, in my opinion, illustrate adaptive processes across the biological scales, as were schematized in Fig. 1. While it was not known to Monod at the time, we now know that many cases of enzymatic adaptation are driven the regulation of gene expression. As the bacterial culture approaches the auxic shift, the presence or absence of the substrate is sensed by regulatory molecules that control whether the genes encoding the enzymes for metabolism of the substrate are expressed. This represents the level of <strong>molecular adaptation</strong> where, given binding or unbinding of the substrate molecule, the activity of the regulatory protein is modulated. The amino acid sequence of these proteinaceous regulators are the product of billions of years of <strong>evolutionary adaptation</strong> and define how the regulatory senses and responds to these signals. Finally, the precision with which these genes are regulated are determined by their sensitivity to <strong>physiological adaptation</strong>. For example, in Fig. 2 (B), the ability to digest glycerol is prohibited by a molecular regulator through two physiological states (consumption of glucose and sorbitol), which have different metabolic pathways and therefore represent different physiological states of the organism.</p>
<p>The central aim of this dissertation is to explore the biophysical mechanisms by which these levels of adaptation – molecular, physiological, and evolutionary, schematized in Fig. 1 – are interconnected. Furthermore, in the spirit of Monod, we seek to make our exploration quantitative and leverage the tools of statistical physics to provide precise predictions from pen-and-paper theory that can be rigorously tested through experiment. The remaining sections of this chapter will summarize this journey and present the key findings and will finish with a discussion of how these types of models can be used to explore the predictability of evolution.</p>
<h2 id="topic-i-the-janus-face-of-molecules">Topic I: The Janus Face of Molecules</h2>
<p>Monod famously referred to allostery as “the second secret of life” <span class="citation" data-cites="monod1965">(Monod, Wyman, and Changeux <a href="#ref-monod1965" role="doc-biblioref">1965</a>)</span>. This secret has been now been declassified as allosteric regulation can be found across all domains of life across myriad types of bioological processes. Allostery can be found governing the behavior of ion channels <span class="citation" data-cites="einav2017">(Einav and Phillips <a href="#ref-einav2017" role="doc-biblioref">2017</a>, [@auerbach2012a])</span>, enzymatic reactions <span class="citation" data-cites="einav2016">(Einav, Mazutis, and Phillips <a href="#ref-einav2016" role="doc-biblioref">2016</a>)</span>, chemotaxis <span class="citation" data-cites="keymer2006">(Keymer et al. <a href="#ref-keymer2006" role="doc-biblioref">2006</a>)</span>, G-protein coupled receptors <span class="citation" data-cites="canals2012">(Canals et al. <a href="#ref-canals2012" role="doc-biblioref">2012</a>)</span>, quorum sensing <span class="citation" data-cites="swem2008">(Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>)</span>, and transcriptional regulation <span class="citation" data-cites="huang2018">(Huang et al. <a href="#ref-huang2018" role="doc-biblioref">2018</a>, [@lindsley2006a])</span>, to name a few of many examples. Despite the objective complexity of the structures of all of these allosteric molecules, they can be frequently be reduced to simple cartoons where the details of conformational changes, substrate binding affinities, and more can be massaged into a small set of key details. Fig. 3 (A) shows the molecular structures of a variety of allosteric transcriptional regulators (top). While each has their own fascinating structure and continuum of conformational states, all can be coarse grained into a simple cartoon of an active (red) or inactive (purple) molecule, both of which states possess binding pockets (semicircular notches) for an inducer molecule (orange).</p>
<p>Much as we can reduce the complexity of allosteric molecules schematically, we can enumerate simple mathematical models that describe their behavior. Thermodynamic models built on an assumption of quasi-equilibrium are routinely used to describe complex biological phenomena despite the reality that being in thermodynamic equilibrium is synonymous with being dead. Despite this glaring assumption, such models have been shown to be exceptionally predictive for a variety of complex systems, especially in modeling molecular binding reactions <span class="citation" data-cites="dill2010">(Dill and Bromberg <a href="#ref-dill2010" role="doc-biblioref">2010</a>)</span> and allostery writ large <span class="citation" data-cites="swem2008 keymer2006">(Swem et al. <a href="#ref-swem2008" role="doc-biblioref">2008</a>; Keymer et al. <a href="#ref-keymer2006" role="doc-biblioref">2006</a>, [@einav2017], [@einav2016])</span>. As the timescales of binding and unbinding reactions are orders of magnitude smaller than that of many other processes in the cel, it is fair to make the approximation that molecular binding is in equilibrium. Under this assumption, we are granted the powerful mathematical privilege to say that the probability of any state of the system <span class="math inline"><em>P</em><sub>state</sub></span> follows a Boltzmann distribution, <span id="eq:boltzmann"><br /><span class="math display">$$
P_\text{state} \propto e^{-\frac{\epsilon_\text{state}}{k_BT}},
\qquad(1)$$</span><br /></span> where <span class="math inline"><em>ϵ</em><sub>state</sub></span> is the energy of that state, <span class="math inline"><em>k</em><sub><em>B</em></sub></span> is the Boltzmann constant, and <span class="math inline"><em>T</em></span> is the system temperature. Therefore, if we are interested in computing the probability of a given allosteric protein being in the active state, we merely have to enumerate all of the Boltzmann weights (given by Eq. 1) and compute <br /><span class="math display">$$
P_\text{active}  = \frac{\text{sum over all possible active states}}{\text{sum
over all possible states}}
$$</span><br />{#eq:verbal_pact}. This probability, defined as a function of the inducer concentration is shown schematically in Fig. 3(B). While we have passed over some of the more subtle details of this calculation, the plot in Fig. 3 (B) presents a <em>quantitative</em> prediction of how the activity of an allosteric molecule should scale as a function of the inducer (in this case, becoming less active as more inducer is present).</p>
<figure>
<img src="ch1_fig3.png" id="fig:allostery" data-short-caption="A Coarse grained representation of an allosteric molecule." alt="" /><figcaption>Figure 3: <strong>A Coarse grained representation of an allosteric molecule.</strong> (A) Crystal structures of a variety of allosteric transcription factors are shown at the top. In this thesis, we coarse grain away many of the details to a minimal model (bottom) where the protein can be represent as being either active (red), inactive (purple), both of which states can bind an inducer molecule (orange). (B) By making an assumption of quasi-equilibrium, we can trivially compute a mathematical description of the active probability of an allosteric protein as a function of the inducer concentration. In this particular case, the inactive state becomes stabilized relative to the active state at higher concentrations of inducer molecule</figcaption>
</figure>
<p>In <strong>Chapter 2</strong> of this dissertation, we use the Monod-Wyman-Changeux model of allosery <span class="citation" data-cites="monod1965">(Monod, Wyman, and Changeux <a href="#ref-monod1965" role="doc-biblioref">1965</a>)</span> to build a predictive model of gene expression changes in response to changing activity of an allosteric transcriptional repressor. Using the same tricks given by Eq. 1 and Eq. <strong>¿eq:verbal_pact?</strong>, we extend upon a previously characterized thermodynamic model of the simple repression motif. This motif, schematized in Fig. 4 (A) is not a convenient abstraction of a regulatory architecture. Rather, this motif is the most ubiquitous regulatory schemes in <em>E. coli</em> <span class="citation" data-cites="gama-castro2016">(Gama-Castro et al. <a href="#ref-gama-castro2016" role="doc-biblioref">2016</a>, [@ireland2020])</span>. This architecture has been the target of much theoretical and experimental dissection <span class="citation" data-cites="vilar2003 buchler2003 bintu2005 garcia2011 brewster2014 phillips2019">(Vilar and Leibler <a href="#ref-vilar2003" role="doc-biblioref">2003</a>; Buchler, Gerland, and Hwa <a href="#ref-buchler2003" role="doc-biblioref">2003</a>; Bintu et al. <a href="#ref-bintu2005" role="doc-biblioref">2005</a>; Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span>, though previous inclusion of allostery into a thermodynamic model of of this motif had left much to be desired.</p>
<p>At the beginning of 2016, Manuel Razo-Mejia, Stephanie L. Barnes, Nathan M. Belliveau, Tal Einav, and I joined forces and set out to build a complete theoretical model for allosteric transcriptional regulation coupled with a thorough experimental dissection. This was no small task and would have likely taken a full Ph.D.’s worth of effort for a single person to do. Yet, within a year of project inception we had submitted a version to the preprint servers where all of us were annotated as equal contributors. This experience defined how I view collaboration in scientific research and serves, at least in my mind, as a shining example of scientific socialism.</p>
<p>Together, we enumerated a complete thermodynamic model for the inducible simple repression motif and defined a succinct input-output function for the fold-change in gene expression schematized in 4 (B). This model, which is explored in depth in <strong>Chapter 2</strong> is defined by a minimal set of biophysical parameters, many of which can be directly measured using standard tricks of molecular biology and biochemistry. With a model in hand, we turned to a collection of 17 unique <em>E. coli</em> strains, each with different copy numbers of the repressor protein and different regulatory DNA sequences. Using our theoretical model, we inferred the lone two biophysical parameters which we did not know <em>a priori</em> from a single experimental strain white points in middle panel of 4 (C), and tested our predictions on all other experimental strains.</p>
<figure>
<img src="ch1_fig4.png" id="fig:induction_intro" data-short-caption="Experimental dissection of the inducible simple repression input-output function." alt="" /><figcaption>Figure 4: <strong>Experimental dissection of the inducible simple repression input-output function.</strong> (A) Schematic diagram of the inducible simple repression motif. (B) Schematic diagram of the input-output function as is derived in Chapter 2. (C) Experimental measurements of the fold-change in gene expression using the <em>lac</em> repressor from <em>E coli</em>. Different rows correspond to different operator sequences and therefore different values for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. Different colors correspond to different values for the average repressor copy number <span class="math inline"><em>R</em></span>. While filled points in the middle panel represent the experimental strain used to infer the values of the inducer dissociation constants. All points correspond to the mean of at least 10 biological replicates.</figcaption>
</figure>
<p>A key feature of this work is a derivation of thermodynamic state variable of this regulatory architecture which we term the “free energy”. This parameter provides an intuition for the effective free energy difference between states of the promoter in which thee repressor is bound relative those states in which the repressor is not bound to the promoter. This parameter accounts for all of the ways in which one can tune the parameter values and still achieve the same fold-change, as is diagrammed in Fig. 5 (A). While we leave the details of this derivation to <strong>Chapter 2</strong> and the corresponding supplement in <strong>Chapter 6</strong>, we emphasize that this formalism provides a means by which all of the experimental measurements plotted in Fig. 4 (C) can be collapsed onto a master curve defined <em>only</em> by the free energy, which is illustrated in Fig. 5 (B). This scaling, often referred to as “data collapse” in physics, concretely shows that one has identified the “natural variable” of the system. With this scaling function in hand, we are able to make a measurement of the biophysical parameters, compute the free energy, and make a concrete prediction of what the fold-change in gene expression will be. Or, as we will see in the following section, details of the biophysical parameters can be determined directly from an empirical measurement of the free energy.</p>
<figure>
<img src="ch1_fig5.png" id="fig:collapse_intro" data-short-caption="Collapse of individual induction profiles onto a simple scaling function." alt="" /><figcaption>Figure 5: <strong>Collapse of individual induction profiles onto a simple scaling function.</strong> (A) Many different combinations of parameter values yield the same value of the fold-change in gene expression, shown as red and blue horizontal planes. Any point on those planes corresponds to a single value of the free energy (middle) and will appear on the master curve. (B) Data presented in Fig. 4 (C) collapsed onto the master curve defined by the predicted value of the free energy.</figcaption>
</figure>
<h2 id="topic-ii-using-free-energy-to-examine-evolutionary-adaptation">Topic II: Using Free Energy to Examine Evolutionary Adaptation</h2>
<p>Allow us to briefly return to Monod and his biphasic growth curves in the mid 1940’s. At this point in scientific history, the French vision of biology had taken a strongly finalistic and vitalistic turn <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. In particular, a neo-Lamarckian view had been employed to explain the phenomenon of enzymatic adaptation where the formation of the appropriate genes could formed out the bacterial cytoplasm and inherited by the cell’s descendents, completely independent of genes. In general, this approach to biology deeply frustrated Monod and strongly influenced his desire to “physicalize” the science <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. One tool he knew was critical to this mission was the burgeoning field of genetics. In the mid 1930’s Monod undertook a short retreat to Thomas Hunt Morgan’s lab at Caltech where he was introduced to “biology’s first discipline”, genetics <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>. This visit had a profound impact on Monod, who remarked on it some three decades later:</p>
<blockquote>
<p><em>“Upon my return to France, I had again taken up the study of bacterial growth. But my mind remained full of the concepts of genetics and I was confident of its ability to analyze and convinced that one day these ideas would be applied to bacteria.”</em> <span class="citation" data-cites="monod1966">(Monod <a href="#ref-monod1966" role="doc-biblioref">1966</a>)</span></p>
</blockquote>
<p>     Once he returned to his study of bacterial growth and enzymatic adaptation, he was confronted with incorporating the role of genetic inheritance into his mechanistic explanations. In the mid 1940’s, Monod and his coworkers had begun experimenting with a strain of <em>E. coli</em> (isolated from the colon of André Lwoff, with whom Monod would share the Nobel prize with twenty years later) which was unable to digest lactose (termed <span class="math inline"><em>L</em>−</span>). When grown on a mixture of glucose and lactose, this strain would not display a diauxic shift and would only be able to consume the glucose in the medium (Fig. 6, black). However, Monod and Alice Audureau discovered a mutation in this strain which <em>enabled</em> the digestion of lactose, termed <span class="math inline"><em>L</em>+</span> <span class="citation" data-cites="monod1947">(Monod <a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>. This growth curve of this strain had the striking feature of diauxic growth. Rather than this mutation merely enabling the digestion of lactose, it did so in a non-constitutive manner and preserved the phenomenon of adaptation. This was an important step forward in Monod’s understanding of enzymatic adaptation <span class="citation" data-cites="loison2013">(Loison <a href="#ref-loison2013" role="doc-biblioref">2013</a>)</span>, revealing that it was a “truly genetic property <span class="citation" data-cites="monod1966">(Monod <a href="#ref-monod1966" role="doc-biblioref">1966</a>)</span>.”</p>
<figure>
<img src="ch1_fig6.png" id="fig:lacneg" data-short-caption="Growth curves of lactose-positive and lactose-negative *E. coli* strains." alt="" /><figcaption>Figure 6: <strong>Growth curves of lactose-positive and lactose-negative <em>E. coli</em> strains from Monod.</strong> Black curve shows the growth curve of an <em>E. coli</em> strain unable to digest lactose grown on a glucose/lactose mixed medium. Red curve shows a mutant of the same <em>E. coli</em> strain which is able to consume lactose. The latter displays a diauxic growth cycle with an adaptive period (highlighted in white), illustrating that enzymatic adaptation is a truly genetic property. Figure adapted from <span class="citation" data-cites="monod1947">Monod (<a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>.</figcaption>
</figure>
<p>     This finding illustrates the level of evolutionary adaptation operating at the level of molecules. While it is difficult to find any literature dissecting this particular “<span class="math inline"><em>L</em>+</span>” mutation, it is not difficult to imagine several different mechanisms by how it could be manifest. One such explanation is that this <span class="math inline"><em>L</em>+</span> mutation is within a transcriptional regulator itself where a deficiency in the ability to respond to the presence of lactose (and decreasing glucose) had been restored. Such mutations are the crux of <strong>Chapter 3</strong> and <strong>Chapter 7</strong> of this dissertation.</p>
<p>     As summarized in the previous section and discussed in depth in Chapter 2, Chapter 3 and the associated supplemental Chapter 7 of this dissertation focus on the influence of mutation within the allosteric transcription factor. Furthermore, Chapter 3 presents a generic mechanism by which shifts in the free energy can be mapped directly to changes in values of the biophysical parameters. This chapter, much like Chapter 2, was borne out of a wonderful collaboration with Manuel Razo-Mejia, Stephanie L. Barnes, Nathan M. Belliveau, Tal Einav, and Zofii A. Kaczmarek. Being able to launch another collaborative effort afforded us the opportunity to both develop a new theoretical interpretation for how mutations influence the free energy and acquire enough experimental data to thoroughly test it.</p>
<figure>
<img src="ch1_fig7.png" id="fig:pedagogical_delF_intro" data-short-caption="Mutations lead to predictive shifts in free energy." alt="" /><figcaption>Figure 7: <strong>Mutations lead to shifts in free energy, permitting prediction of double mutant phenotypes.</strong> Consider a wild-type bacterium [grey bug in (A)] which on, on average, exhibits a fold-change of <span class="math inline">≈</span> 0.3 and a free energy of <span class="math inline"> − 1 <em>k</em><sub><em>B</em></sub><em>T</em></span> [grey point in (B)]. We can consider that a single mutation (either orange or purple) changes the mean fold-change and therefore the free energy, translating the measurement about the master curve [black line in (B)]. Assuming there are no epistatic interactions between the two single mutations, a null hypothesis predicts that for the double mutant [blue bug in (A) and point in (B)], the net free energy is simply the sum of the individual free energy shifts.</figcaption>
</figure>
<p>     The primary conceptual development of Chapter 3 is illustrated in Fig. 7. Theoretically, we consider a bacterial strain with an allosteric repressor (which we term the “wild-type” repressor) that has been sufficiently characterized. Given enough parametric knowledge of the system, we can easily compute both its predicted average fold-change in gene expression along with the corresponding free energy. However, once a mutation has been introduced <em>into the repressor protein</em> (resulting in a non-synonymous amino acid change), we are once again ignorant <em>a priori</em> of what changes, if any, that mutation may have imparted on the system. In Fig. 7, we show two separate hypothetical mutations ,shown in purple and orange, which significantly change the character of the system by either increasing or decreasing the fold-change in gene expression, respectively. If we assume that these mutations do not change the underlying physics of the system, we are permitted to use the theoretical framework outlined in Chapter 2 and in Fig. 4 to characterize each mutation and determine what biophysical parameters have been changed. This permits us to calculate the new free energy of the system (<span class="math inline"><em>F</em><sub>mutation 1</sub></span>) as well as the shift in free energy, <span id="eq:intro_delF_definition"><br /><span class="math display"><em>Δ</em><em>F</em><sub>mutation 1</sub> = <em>F</em><sub>mutation 1</sub> − <em>F</em><sub>wt</sub>.   (2)</span><br /></span> As will be described in detail in Chapter 3 and the supplemental Chapter 7, the precise value of this free energy shift <span class="math inline"><em>Δ</em><em>F</em></span> can be directly computed given sufficient parametric knowledge.</p>
<p>     This formalism provides a mathematical hypothesis for how double mutants may behave. Given known values for <span class="math inline"><em>Δ</em><em>F</em></span> of each mutation in isolation, can we compute the shift in free energy of the pairwise double mutant <span class="math inline"><em>Δ</em><em>F</em><sub>mutations 1 &amp;2</sub></span>? Eq. 2 presents a mathematical null hypothesis that the net shift in the free energy is simply the sum of the individual shifts in free energy, <span id="eq:delF_null_intro"><br /><span class="math display"><em>Δ</em><em>F</em><sub>mutations 1 &amp;2</sub> = <em>Δ</em><em>F</em><sub>mutation 1</sub> + <em>Δ</em><em>F</em><sub>mutation 2</sub>,   (3)</span><br /></span> assuming there are no epistatic interactions between the mutations. Given the fact that we can compute the fold-change in gene expression given knowledge of the free energy, we can therefore predict the double mutant phenotype <em>a priori</em>, a prediction which had eluded us prior to this work.</p>
<p>     Over the course of two years (while this theory was in the works), the experimental cast of characters (Stephanie L. Barnes, Nathan M. Belliveau, Manuel Razo-Mejia, and Zofii A. Kaczmarek) and I made a series of mutations in the LacI repressor that we had characterized in the work presented in Chapter 2. These mutations included three point mutations in the DNA binding domain of the repressor, four mutations in the inducer binding domain, nine double mutants (one inducer binding and one DNA binding each), across four repressor copy numbers and three operator sequences. While this process of strain generation and data collection is not the primary focus of the work, it took <span class="math inline"> ≈ 80%</span> of the effort. Without them, this work would have remained an untested theoretical approach. While we leave many of the rich details of this prediction to the reader in Chapter 3, we showcase our experimental success in Fig. 8 (B) where the predicted induction profiles of nine double mutants (light blue shaded regions) are overlaid with their experimental measurements (points). The near 100% agreement between theory and experiment illustrates the utility of using free energy shifts as a means to predict new phenotypes.</p>
<figure>
<img src="ch1_fig8.png" id="fig:double_muts_intro" data-short-caption="Theoretical prediction and experimental validation of double mutant phenotypes." alt="" /><figcaption>Figure 8: <strong>Theoretical prediction and experimental validation of double mutant phenotypes.</strong> (A) Cartoon representation of the LacI repressor with mutations in the inducer binding domain and DNA binding domain represented by hats and socks, respectively. While the mutations have known chemical features, we characterize each mutation as potentially modifying four biophysical parameters, <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> for inducer binding mutants, or <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> for DNA binding mutants. (B) Predicted induction profiles for pairwise double mutants are shown as blue shaded regions representing the uncertainty in our predictions. Experimental measurements are shown as blue points (means of at least 10 biological replicates). Each row corresponds to a single DNA binding domain mutation and each column to a single inducer binding domain mutation.</figcaption>
</figure>
<h2 id="topic-iv-the-physiological-adaptability-of-transient-molecular-interactions">Topic IV: The Physiological Adaptability of Transient Molecular Interactions</h2>
<figure>
<img src="ch1_fig9.png" id="fig:triauxic_growth" data-short-caption="A metabolic hierarchy in a growth medium containing glucose, sorbitol, and glycerol." alt="" /><figcaption>Figure 9: <strong>A metabolic hierarchy in a three-component growth mixture of glucose, sorbitol, and glycerol.</strong> A “triauxic” growth curve illustrating a hierarchy of carbon source metabolism. An <em>E. coli</em> culture was grown in an medium with equal parts glucose, sorbitol and glycerol with utilization in that order. Auxic transitions are shown as black points and white vertical lines. Regions of the growth curve where glucose, sorbitol, and glycrol are primarily consumed are colored in blue, orange, and green, respectively. Data digitized from <span class="citation" data-cites="monod1947">Monod (<a href="#ref-monod1947" role="doc-biblioref">1947</a>)</span>.</figcaption>
</figure>
<h2 id="topic-v-adaptation-where-it-counts-and-facing-the-elements">Topic V: Adaptation Where It Counts and Facing the Elements</h2>
<h2 id="on-molecular-biophysics-evolutionary-dynamics-and-biophysical-darwinism">On Molecular Biophysics, Evolutionary Dynamics, and Biophysical Darwinism</h2>
<p>Blank placeholder</p>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-auerbach2012a">
<p>Auerbach, Anthony. 2012. “Thinking in Cycles: MWC Is a Good Model for Acetylcholine Receptor-Channels.” <em>The Journal of Physiology</em> 590 (1): 93–98. <a href="https://doi.org/10.1113/jphysiol.2011.214684">https://doi.org/10.1113/jphysiol.2011.214684</a>.</p>
</div>
<div id="ref-bintu2005">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Models.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 116–24. <a href="https://doi.org/10.1016/j.gde.2005.02.007">https://doi.org/10.1016/j.gde.2005.02.007</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-buchler2003">
<p>Buchler, N. E., U. Gerland, and T. Hwa. 2003. “On Schemes of Combinatorial Transcription Logic.” <em>Proceedings of the National Academy of Sciences</em> 100 (9): 5136–41. <a href="https://doi.org/10.1073/pnas.0930314100">https://doi.org/10.1073/pnas.0930314100</a>.</p>
</div>
<div id="ref-canals2012">
<p>Canals, Meritxell, J. Robert Lane, Adriel Wen, Peter J. Scammells, Patrick M. Sexton, and Arthur Christopoulos. 2012. “A Monod-Wyman-Changeux Mechanism Can Explain G Protein-Coupled Receptor (GPCR) Allosteric Modulation.” <em>Journal of Biological Chemistry</em> 287 (1): 650–59. <a href="https://doi.org/10.1074/jbc.M111.314278">https://doi.org/10.1074/jbc.M111.314278</a>.</p>
</div>
<div id="ref-dienert1900">
<p>Diénert, Frédéric. 1900. <em>Sur La Fermentation Du Galactose et Sur L’accoutumance Des Levures à Ce Sucre,</em> Sceaux.</p>
</div>
<div id="ref-dill2010">
<p>Dill, Kenneth A., and Sarina Bromberg. 2010. <em>Molecular Driving Forces: Statistical Thermodynamics in Chemistry and Biology</em>. New York: Garland Science.</p>
</div>
<div id="ref-einav2016">
<p>Einav, Tal, Linas Mazutis, and Rob Phillips. 2016. “Statistical Mechanics of Allosteric Enzymes.” <em>The Journal of Physical Chemistry B</em> 120 (26): 6021–37. <a href="https://doi.org/10.1021/acs.jpcb.6b01911">https://doi.org/10.1021/acs.jpcb.6b01911</a>.</p>
</div>
<div id="ref-einav2017">
<p>Einav, Tal, and Rob Phillips. 2017. “Monod-Wyman-Changeux Analysis of Ligand-Gated Ion Channel Mutants,” January, –. <a href="https://doi.org/10.1101/102194">https://doi.org/10.1101/102194</a>.</p>
</div>
<div id="ref-gama-castro2016">
<p>Gama-Castro, Socorro, Heladia Salgado, Alberto Santos-Zavaleta, Daniela Ledezma-Tejeida, Luis Muñiz-Rascado, Jair Santiago García-Sotelo, Kevin Alquicira-Hernández, et al. 2016. “RegulonDB Version 9.0: High-Level Integration of Gene Regulation, Coexpression, Motif Clustering and Beyond.” <em>Nucleic Acids Research</em> 44 (D1): D133–D143. <a href="https://doi.org/10.1093/nar/gkv1156">https://doi.org/10.1093/nar/gkv1156</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-huang2018">
<p>Huang, Min, Kun Song, Xinyi Liu, Shaoyong Lu, Qiancheng Shen, Renxiao Wang, Jingze Gao, et al. 2018. “AlloFinder: A Strategy for Allosteric Modulator Discovery and Allosterome Analyses.” <em>Nucleic Acids Research</em> 46 (W1): W451–W458. <a href="https://doi.org/10.1093/nar/gky374">https://doi.org/10.1093/nar/gky374</a>.</p>
</div>
<div id="ref-keymer2006">
<p>Keymer, Juan E., Robert G. Endres, Monica Skoge, Yigal Meir, and Ned S. Wingreen. 2006. “Chemosensing in Escherichia Coli: Two Regimes of Two-State Receptors.” <em>Proceedings of the National Academy of Sciences</em> 103 (6): 1786–91. <a href="https://doi.org/10.1073/pnas.0507438103">https://doi.org/10.1073/pnas.0507438103</a>.</p>
</div>
<div id="ref-lindsley2006a">
<p>Lindsley, Janet E., and Jared Rutter. 2006. “Whence Cometh the Allosterome?” <em>Proceedings of the National Academy of Sciences</em> 103 (28): 10533–5. <a href="https://doi.org/10.1073/pnas.0604452103">https://doi.org/10.1073/pnas.0604452103</a>.</p>
</div>
<div id="ref-loison2013">
<p>Loison, Laurent. 2013. “Monod Before Monod: Enzymatic Adaptation, Lwoff, and the Legacy of General Biology.” <em>History and Philosophy of the Life Sciences</em> 35 (2): 167–92.</p>
</div>
<div id="ref-monod1941">
<p>Monod, Jacques. 1941. “Sur Un Phénomeene Nouveau de Croissance Complexe Dans Les Culures Bactériennes.” <em>Comptes Rendus Des Séances de L’Académie Des Sciences</em> 212: 934–39.</p>
</div>
<div id="ref-monod1947">
<p>———. 1947. “The Phenomenon of Enzymatic Adaptation and Its Bearings on Problems of Genetics and Cellular Differentiation.” <em>Growth Symposium</em> 9: 223–89.</p>
</div>
<div id="ref-monod1966">
<p>———. 1966. “From Enzymatic Adaptation to Allosteric Transitions.” <em>Science</em> 154 (3748): 475–83. <a href="https://doi.org/10.1126/science.154.3748.475">https://doi.org/10.1126/science.154.3748.475</a>.</p>
</div>
<div id="ref-monod1963">
<p>Monod, Jacques, Jean-Pierre Changeux, and François Jacob. 1963. “Allosteric Proteins and Cellular Control Systems.” <em>Journal of Molecular Biology</em> 6 (4): 306–29. <a href="https://doi.org/10.1016/S0022-2836(63)80091-1">https://doi.org/10.1016/S0022-2836(63)80091-1</a>.</p>
</div>
<div id="ref-monod1965">
<p>Monod, Jacque, Jeffries Wyman, and Jean-Pierre Changeux. 1965. “On the Nature of Allosteric Transitions: A Plausible Model.” <em>Journal of Molecular Biology</em> 12 (1): 88–118. <a href="https://doi.org/10.1016/S0022-2836(65)80285-6">https://doi.org/10.1016/S0022-2836(65)80285-6</a>.</p>
</div>
<div id="ref-phillips2019">
<p>Phillips, Rob, Nathan M. Belliveau, Griffin Chure, Hernan G. Garcia, Manuel Razo-Mejia, and Clarissa Scholes. 2019. “Figure 1 Theory Meets Figure 2 Experiments in the Study of Gene Expression.” <em>Annual Review of Biophysics</em> 48 (1): 121–63. <a href="https://doi.org/10.1146/annurev-biophys-052118-115525">https://doi.org/10.1146/annurev-biophys-052118-115525</a>.</p>
</div>
<div id="ref-swem2008">
<p>Swem, Lee R., Danielle L. Swem, Ned S. Wingreen, and Bonnie L. Bassler. 2008. “Deducing Receptor Signaling Parameters from in Vivo Analysis: LuxN/AI-1 Quorum Sensing in Vibrio Harveyi.” <em>Cell</em> 134 (3): 461–73. <a href="https://doi.org/10.1016/j.cell.2008.06.023">https://doi.org/10.1016/j.cell.2008.06.023</a>.</p>
</div>
<div id="ref-vilar2003">
<p>Vilar, José M. G., and Stanislas Leibler. 2003. “DNA Looping and Physical Constraints on Transcription Regulation.” <em>Journal of Molecular Biology</em> 331 (5): 981–89. <a href="https://doi.org/10.1016/S0022-2836(03)00764-2">https://doi.org/10.1016/S0022-2836(03)00764-2</a>.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure_phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/published_content" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Published Content
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd{{site.doks.baseurl}}/chapter_2" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd/" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd/doks-theme/assets/js/scripts.min.js"></script>




		</div><!-- /.js-footer-area -->
	</body>
</html>
