Central to any definition of Life is the ability to sense changes in one's
environment and respond in kind. Adaptive phenomena can be found across the
biological scales ranging from the nanosecond-scale conformational changes of
proteins, to temporary rewiring of metabolic networks, to the 3.5 billion
years of evolution that produced the enormous biodiversity we see today. This
thesis presents a body of work which attempts to examine the overlap between
these three scales of adaptation through the quantitative lens of statistical
physics. Namely, we examine how molecular, physiological, and evolutionary
adaptation governs a feature common to all life -- the regulation of gene
expression.

We begin by examining the phenomenon of molecular adaptation in
the context of allostery, specifically in the context of allosteric
transcriptional repressors. Using simple tools of quasi-equilibrium
thermodynamics, we derive and experimentally dissect a quantitative model of
how such a repressor adapts to different concentrations of an extracellular
inducer molecule, modulating the repressors activity and thereby gene
expression. While the model is relatively simple, it is remarkably powerful
in its ability to draw concrete, quantitative predictions about not only the
level of gene expression at a given concentration of inducer, but details of
how the repressor responds to changes in the inducer concentration. With a
few lines of simple mathematics, we are able to use this model to derive a
state variable of the simple repression motif which we term the free energy
of the regulatory architecture. This permits us to collapse nearly 500
distinct measurements of the level of gene expression
onto a master curve defined by this free energy.

We leverage this feature of the model and use data collapse as a
method to identify the effects of mutation, a strong evolutionary force
responsible for much of the genetic diversity in bacteria. In Chapter 3, we examine how
mutations within the allosteric repressor itself can be mapped to changes in
the free energy. The precise value of these free energy shifts and their
dependence on the inducer concentration reveal different classes of mutations
with one class affecting only the DNA-repressor interaction and another class
governing the allosteric nature of the repressor. We test these pen-and-paper
predictions experimentally and illustrate that given sufficient knowledge of
how single mutants behave, the complete phenotypic response of pairwise
double mutants can be predicted with quantitative accuracy.

With this framework in hand, we turn to
exploring how changes in the physiological state of the cell influence the
molecular biophysics of the regulatory architecture. We hypothesize that
changes in the source of carbon in the growth medium or changes in the growth
temperature can be accounted for by the existing model without any
additional parameters. We experimentally show that the parameter values
determined in one physiological state are inherited when the available carbon
source is verified, but changes in the growth temperature require some
additional considerations. Chapter 4 as a whole reveals that, while there
remains work to be done both theoretically and experimentally when it comes
to temperature variation, thermodynamic models can remain powerful tools to
draw predictions of gene expression in different physiological contexts.

Finally, in Chapter 5, we explore physiological adaptation
and cellular decision making where it counts -- in the survival of cells to
environmental insults. We turn our focus beyond transcriptional regulation
and consider the relationship between osmotic shocks, the abundance of
mechanosensitive channels, and cellular survival with single cell resolution.
Using a combination of quantitative microscopy and tricks of statistical
inference, we infer how the probability of a cell surviving an osmotic shock
scales as a function of the cell's number of mechanosensitive channels.
