<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter VI</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd//favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd//doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd//doks-theme/assets/images/switchboard.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter VI</h2>
								
								
									<h1 class="hero-subheader__title">Supplemental Information For Chapter I: Signal Processing Via Allosteric Transcription Factors 
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phd/chapter_05" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phd/chapter_07" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/chapter_05" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter V
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_07" class="page-nav__item page-nav__item--next">
										Chapter VII
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<strong>Published as …</strong>
<hr/>
<p>A version of this chapter originally appeared as Razo-Mejia, M.<em>, Barnes, S.L.</em>, Belliveau, N.M.<em>, Chure, G.</em>, Einav, T.*, Lewis, M., and Phillips, R. (2018). Tuning transcriptional regulation through signaling: A predictive theory of allosteric induction. Cell Systems 6, 456-469.e10. DOI:https://doi.org/10.1016/j.cels.2018.02.004. M.R.M, S.L.B, N.M.B, G.C., and T.E. contributed equally to this work from the theoretical underpinnings to the experimental design and execution. M.R.M, S.L.B, N.M.B, G.C, T.E., and R.P. wrote the paper. M.L. provided extensive guidance and advice.</p>
<h2 id="inferring-allosteric-parameters-from-previous-data">Inferring Allosteric Parameters from Previous Data</h2>
<p>The fold-change function derived in Chapter 2 features three unknown parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. In this section, we explore different conceptual approaches to determining these parameters. We first discuss how the induction titration profile of the simple repression constructs used in this paper are not sufficient to determine all three MWC parameters simultaneously, since multiple degenerate sets of parameters can produce the same fold-change response. We then utilize an additional data set from <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> to determine the parameter <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>, after which the remaining parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> can be extracted from any induction profile with no further degeneracy.</p>
<h3 id="degenerate-parameter-values">Degenerate Parameter Values</h3>
<p>In this section, we discuss how multiple sets of parameters may yield identical fold-change profiles. More precisely, we show that if we try to fit the data into the fold-change and extract the three unknown parameters (<span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>), then multiple degenerate parameter sets would yield equally good fits. In other words, this data set alone is insufficient to uniquely determine the actual physical parameter values of the system. This problem persists even when fitting multiple data sets simultaneously as illustrated later in this chapter.</p>
<p>     In Fig. 1 we fit the <span class="math inline"><em>R</em> = 260</span> data by fixing <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> to the value shown on the <span class="math inline"><em>x</em></span>-axis and determine the parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> given this constraint. We use the fold-change function, but with <span class="math inline"><em>β</em><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> modified to the form <span class="math inline"><em>β</em><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span> to account for the underlying assumptions used when fitting previous data (as is defined in the following section).</p>
<p>     The best-fit curves for several different values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> are shown in Fig. 1 (B). Note that these fold-change curves are nearly overlapping, demonstrating that different sets of parameters can yield nearly equivalent responses. Without more data, the relationships between the parameter values represent the maximum information about the parameter values that can be extracted from the data. Additional experiments which independently measure any of these unknown parameters could resolve this degeneracy. For example, NMR measurements could be used to directly measure the fraction <span class="math inline">(1 + <em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></sup>)<sup> − 1</sup></span> of active repressors in the absence of IPTG <span class="citation" data-cites="gardino2003 boulton2016">(Gardino et al. <a href="#ref-gardino2003" role="doc-biblioref">2003</a>; Boulton and Melacini <a href="#ref-boulton2016" role="doc-biblioref">2016</a>)</span>.</p>
<figure>
<img src="ch6_figS1.png" id="fig:degenerate_epai" data-short-caption="Multiple sets of parameters yield identical fold-change responses." alt="" /><figcaption>Figure 1: <strong>Multiple sets of parameters yield identical fold-change responses.</strong> (A) The data for the O2 strain (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>) with <span class="math inline"><em>R</em> = 260</span> Fig. 2.5(C) was fit using Eq. 2.5 with <span class="math inline"><em>n</em> = 2</span>. The allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is forced to take on the value shown on the <span class="math inline"><em>x</em>−</span>axis, while <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are fit freely. (B) The resulting best-fit functions for several values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> yield nearly identical fold-change responses. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS1.py">Python code (<code>ch6_figS1.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="computing-deltavarepsilon_ai">Computing <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span></h3>
<p>As shown in the previous section, the fold-change response of a single strain is not sufficient to determine the three MWC parameters (<span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>), since degenerate sets of parameters yield nearly identical fold-change responses. To circumvent this degeneracy, we now turn to some previous data from the <em>lac</em> system in order to determine the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> for the induction of the Lac repressor. Specifically, we consider two previous sets of work from (i) <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span> and (ii) <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>, both of which measured fold-change with the same simple repression system in the absence of inducer (<span class="math inline"><em>c</em> = 0</span>), but at various repressor copy numbers <span class="math inline"><em>R</em></span>. The original analysis for both data sets assumed that in the absence of inducer, all of the Lac repressors were in the active state. As a result, the effective binding energies they extracted were a convolution of the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> between the Lac repressor’s active and inactive states. We refer to this convoluted energy value as <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span>. We first disentangle the relationship between these parameters in Garcia and Phillips and then use this relationship to extract the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> from <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>.</p>
<p>    Garcia and Phillips determined the total repressor copy numbers <span class="math inline"><em>R</em></span> of different strains using quantitative Western blots. Then they measured the fold-change at these repressor copy numbers for simple repression constructs carrying the O1, O2, O3, and Oid <em>lac</em> operators integrated into the chromosome. These data were then fit to the following thermodynamic model to determine the repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span> for each operator, <span id="eq:ind_si_simple_fc"><br /><span class="math display">$$
\text{fold-change}(c=0) = \left(
1+\frac{R}{N_{NS}}e^{-\beta \Delta\tilde{\varepsilon}_{RA}} \right)^{-1}.
\qquad(1)$$</span><br /></span> Note that this functional form does not exactly match our fold-change in the limit <span class="math inline"><em>c</em> = 0</span>, <span id="eq:ind_si_leakiness"><br /><span class="math display">$$
\text{fold-change}(c=0) = \left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI}}}\frac{R}{N_{NS}}e^{-\beta
\Delta\varepsilon_{RA}} \right)^{-1},
\qquad(2)$$</span><br /></span> since it is missing the factor <span class="math inline">$\frac{1}{1+e^{-\beta \Delta\varepsilon_{AI}}}$</span> which specifies what fraction of repressors are in the active state in the absence of inducer, <span id="eq:ind_si_r_active"><br /><span class="math display">$$
\frac{1}{1+e^{-\beta \Delta\varepsilon_{AI}}} = p_A(0). 
\qquad(3)$$</span><br /></span> In other words, <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span> assumed that in the absence of inducer, all repressors were active. In terms of our notation, the convoluted energy values <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span> extracted by Garcia and Phillips (namely, <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub> =  − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span> for O1 and <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span> for Oid) represent <span id="eq:ind_si_tilde_varepsilon"><br /><span class="math display">$$
\beta \Delta\tilde{\varepsilon}_{RA} = \beta \Delta\varepsilon_{RA} - \log
\left( \frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}}} \right).
\qquad(4)$$</span><br /></span> Note that if <span class="math inline"><em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></sup> ≪ 1</span>, then nearly all of the repressors are active in the absence of inducer so that <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub> ≈ <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. In simple repression systems where we definitively know the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>R</em></span>, we can use Eq. 4 to determine the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> by comparing with experimentally determined fold-change values. However, the binding energy values that we use from <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span> are effective parameters <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span>. In this case, we are faced with an undetermined system in which we have more variables than equations, and we are thus unable to determine the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. In order to obtain this parameter, we must turn to a more complex regulatory scenario which provides additional constraints that allow us to fit for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>.</p>
<p>     A variation on simple repression in which multiple copies of the promoter are available for repressor binding (for instance, when the simple repression construct is on plasmid) can be used to circumvent the problems that arise when using <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub></span>. This is because the behavior of the system is distinctly different when the number of active repressors <span class="math inline"><em>p</em><sub><em>A</em></sub>(0)<em>R</em></span> is less than or greater than the number of available promoters <span class="math inline"><em>N</em></span>. Repression data for plasmids with known copy number <span class="math inline"><em>N</em></span> allows us to perform a fit for the value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>.</p>
<p>    To obtain an expression for a system with multiple promoters <span class="math inline"><em>N</em></span>, we follow <span class="citation" data-cites="weinert2014">Weinert et al. (<a href="#ref-weinert2014" role="doc-biblioref">2014</a>)</span>, writing the fold-change in terms of the the grand canonical ensemble as <span id="eq:ind_si_fugacity"><br /><span class="math display">$$
\text{fold-change} = \frac{1}{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}},
\qquad(5)$$</span><br /></span> where <span class="math inline"><em>λ</em><sub><em>r</em></sub> = <em>e</em><sup><em>β</em><em>μ</em></sup></span> is the fugacity and <span class="math inline"><em>μ</em></span> is the chemical potential of the repressor. The fugacity will enable us to easily enumerate the possible states available to the repressor.</p>
<p>     To determine the value of <span class="math inline"><em>λ</em><sub><em>r</em></sub></span>, we first consider that the total number of repressors in the system, <span class="math inline"><em>R</em><sub>tot</sub></span>, is fixed and given by <span id="eq:ind_si_fugacity_rtot"><br /><span class="math display"><em>R</em><sub>tot</sub> = <em>R</em><sub><em>S</em></sub> + <em>R</em><sub><em>N</em><em>S</em></sub>,   (6)</span><br /></span></p>
<p>where <span class="math inline"><em>R</em><sub><em>S</em></sub></span> represents the number of repressors specifically bound to the promoter and <span class="math inline"><em>R</em><sub><em>N</em><em>S</em></sub></span> represents the number of repressors nonspecifically bound throughout the genome. The value of <span class="math inline"><em>R</em><sub><em>S</em></sub></span> is given by <span id="eq:ind_si_fugacity_rs"><br /><span class="math display">$$
R_S = N \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}{1 + \lambda_r
e^{-\beta \Delta \varepsilon_{RA}}},
\qquad(7)$$</span><br /></span></p>
<p>where <span class="math inline"><em>N</em></span> is the number of available promoters in the cell. Note that in counting <span class="math inline"><em>N</em></span>, we do not distinguish between promoters that are on plasmid or chromosomaly integrated provided that they both have the same repressor-operator binding energy <span class="citation" data-cites="weinert2014">(Weinert et al. <a href="#ref-weinert2014" role="doc-biblioref">2014</a>)</span>. The value of <span class="math inline"><em>R</em><sub><em>N</em><em>S</em></sub></span> is similarly given by <span id="eq:ind_si_fugacity_rns"><br /><span class="math display">$$
R_{NS} = N_{NS} \frac{\lambda_r}{1 + \lambda_r},
\qquad(8)$$</span><br /></span> where <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of non-specific sites in the cell (recall that we use <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub> = 4.6 × 10<sup>6</sup></span> for <em>E. coli</em>). Substitution yields the form <span id="eq:ind_si_fugacity_pa0"><br /><span class="math display">$$
p_A(0) R_{\text{tot}} = \frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}}} \left(
N \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}{1 + \lambda_r e^{-\beta
\Delta \varepsilon_{RA}}} + N_{NS} \frac{\lambda_r}{1 + \lambda_r} \right),
\qquad(9)$$</span><br /></span> where we recall from Eq. 4 that <span class="math inline">$\beta \Delta \varepsilon_{RA} = \beta \Delta \tilde\varepsilon_{RA} + \log{\left(\frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}}}\right)}.$</span> Numerically solving for <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> yields a fold-change function in which the only unknown parameter is <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>.</p>
<p>     With these calculations in hand, we can now determine the value of the <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> parameter. Fig. 2 shows how different values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> lead to significantly different fold-change response curves. Thus, analyzing the specific fold-change response of any strain with a known plasmid copy number <span class="math inline"><em>N</em></span> will fix <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. Interestingly, the inflection point occurs near <span class="math inline"><em>p</em><sub><em>A</em></sub>(0)<em>R</em><sub>tot</sub> = <em>N</em></span> (as shown by the triangles in Fig. 2), so that merely knowing where the fold-change response transitions from concave down to concave up is sufficient to obtain a rough value for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. We note, however, that for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> ≥ 5 <em>k</em><sub><em>B</em></sub><em>T</em></span>, increasing <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> further does not affect the fold-change because essentially every repressor will be in the active state in this regime. Thus, if the <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is in this regime, we can only bound it from below.</p>
<p>     We now analyze experimental induction data for different strains with known plasmid copy numbers to determine <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. Fig. 2 (B) shows experimental measurements of fold-change for two O1 promoters with <span class="math inline"><em>N</em> = 64</span> and <span class="math inline"><em>N</em> = 52</span> copy numbers and one Oid promoter with <span class="math inline"><em>N</em> = 10</span> from Brewster et al. (2014). By fitting these data to Eq. 5, we extracted the parameter value <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>. Substituting this value into Eq. 3 shows that 99% of the repressors are in the active state in the absence of inducer and <span class="math inline"><em>Δ</em><em>ε̃</em><sub><em>R</em><em>A</em></sub> ≈ <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, so that all of the previous energies and calculations made by <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span> and <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> were accurate.</p>
<figure>
<img src="ch6_figS2.png" id="fig:fugacity" data-short-caption="Fold-change in gene expression for multiple identical promoters." alt="" /><figcaption>Figure 2: <strong>Fold-change in gene expression for multiple identical promoters.</strong> (A) In the presence of <span class="math inline"><em>N</em> = 10</span> identical promoters, the fold-change depends strongly on the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> between the Lac repressor’s active and inactive states. The vertical dotted lines represent the number of repressors at which <span class="math inline"><em>R</em><sub><em>A</em></sub> = <em>N</em></span> for each value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. (B) Using fold–change measurements from <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> for the operators and gene copy numbers shown, we can determine the most likely value <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span> for LacI. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS2.py">Python code (<code>ch6_figS2.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="induction-of-simple-repression-with-multiple-promoters-or-competitor-sites">Induction of Simple Repression with Multiple Promoters or Competitor Sites</h2>
<p>We made the choice to perform all of our experiments using strains in which a single copy of our simple repression construct had been integrated into the chromosome. This stands in contrast with the methods used by a number of other studies <span class="citation" data-cites="oehler1994 setty2003 daber2009a daber2011 vilar2013 shis2014 sochor2014">(Oehler et al. <a href="#ref-oehler1994" role="doc-biblioref">1994</a>; Setty et al. <a href="#ref-setty2003" role="doc-biblioref">2003</a>; Daber, Sharp, and Lewis <a href="#ref-daber2009a" role="doc-biblioref">2009</a>; Daber, Sochor, and Lewis <a href="#ref-daber2011" role="doc-biblioref">2011</a>; Vilar and Saiz <a href="#ref-vilar2013" role="doc-biblioref">2013</a>; Shis et al. <a href="#ref-shis2014" role="doc-biblioref">2014</a>; Sochor <a href="#ref-sochor2014" role="doc-biblioref">2014</a>)</span>, in which reporter constructs are placed on plasmid, meaning that the number of constructs in the cell is not precisely known. It is also common to express repressor on plasmid to boost its copy number, which results in an uncertain value for repressor copy number. Here, we show that our treatment of the MWC model has broad predictive power beyond the single-promoter scenario we explore experimentally, and indeed can account for systems in which multiple promoters compete for the repressor of interest. Additionally, we demonstrate the importance of having precise control over these parameters, as they can have a significant effect on the induction profile.</p>
<h3 id="chemical-potential-formulation-to-calculate-fold-change">Chemical Potential Formulation to Calculate Fold-Change</h3>
<p>In this section, we discuss a simple repression construct which we generalize in two ways from the scenario discussed in Chapter 2. First, we will allow the repressor to bind to <span class="math inline"><em>N</em><sub><em>S</em></sub></span> identical specific promoters whose fold-change we are interested in measuring, with each promoter containing a single repressor binding site (<span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span> in Chapter 2). Second, we consider <span class="math inline"><em>N</em><sub><em>C</em></sub></span> identical competitor sites which do not regulate the promoter of interest, but whose binding energies are substantially stronger than non-specific binding (<span class="math inline"><em>N</em><sub><em>C</em></sub> = 0</span> in Chapter 2). As in Chapter 2, we assume that the rest of the genome contains <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> non-specific binding sites for the repressor. Using the formalism described in the previous section, we can write the fold-change in the grand canonical ensemble as <span id="eq:fc_fugacity"><br /><span class="math display">$$
\text{fold-change} = \frac{1}{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}},
\qquad(10)$$</span><br /></span> where <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> is the fugacity of the repressor and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> represents the energy difference between the repressor’s binding affinity to the specific operator of interest relative to the repressor’s non-specific binding affinity to the rest of the genome.</p>
<p>     We now expand our definition of the total number of repressors in the system, <span class="math inline"><em>R</em><sub>tot</sub></span>, so that it is given by <span id="eq:fugacity_rtot"><br /><span class="math display"><em>R</em><sub>tot</sub> = <em>R</em><sub><em>S</em></sub> + <em>R</em><sub><em>N</em><em>S</em></sub> + <em>R</em><sub><em>C</em></sub>,   (11)</span><br /></span> where <span class="math inline"><em>R</em><sub><em>S</em></sub></span>, <span class="math inline"><em>R</em><sub><em>N</em><em>S</em></sub></span>, and <span class="math inline"><em>R</em><sub><em>C</em></sub></span> represent the number of repressors bound to the specific promoter, a non-specific binding site, or to a competitor binding site, respectively. The value of <span class="math inline"><em>R</em><sub><em>S</em></sub></span> is given by <span id="eq:fugacity_rs"><br /><span class="math display">$$
R_S = N_S \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}},
\qquad(12)$$</span><br /></span> where <span class="math inline"><em>N</em><sub><em>S</em></sub></span> is the number of specific binding sites in the cell. The value of <span class="math inline"><em>R</em><sub><em>N</em><em>S</em></sub></span> is similarly given by <span id="eq:fugacity_rns"><br /><span class="math display">$$
R_{NS} = N_{NS} \frac{\lambda_r}{1 + \lambda_r},
\qquad(13)$$</span><br /></span> where <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of non-specific sites in the cell (recall that we use <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub> = 4.6 × 10<sup>6</sup></span> for <em>E. coli</em>), and <span class="math inline"><em>R</em><sub><em>C</em></sub></span> is given by <span id="eq:fugacity_rnc"><br /><span class="math display">$$
R_C = N_C \frac{\lambda_r e^{-\beta \Delta \varepsilon_C}}{1 + \lambda_r e^{-\beta \Delta \varepsilon_C}},
\qquad(14)$$</span><br /></span> where <span class="math inline"><em>N</em><sub><em>C</em></sub></span> is the number of competitor sites in the cell and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub></span> is the binding energy of the repressor to the competitor site relative to its non-specific binding energy to the rest of the genome.</p>
<p>      To account for the induction of the repressor, we replace the total number of repressors <span class="math inline"><em>R</em><sub>tot</sub></span> in Eq. 11 by the number of active repressors in the cell, <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)<em>R</em><sub>tot</sub></span>. Here, <span class="math inline"><em>p</em><sub>act</sub></span> denotes the probability that the repressor is in the active state (Eq. 2.4), <span id="eq:fugacity_pact"><br /><span class="math display">$$
p_\text{act}(c)=\frac{\left(1+\frac{c}{K_A}\right)^n}{\left(1+\frac{c}{K_A}\right)^n+e^{-\beta \Delta \varepsilon_{AI} }\left(1+\frac{c}{K_I}\right)^n}.
\qquad(15)$$</span><br /></span> Substituting Eq. 12 into the modified Eq. 11 yields <span id="eq:fugacity_reff"><br /><span class="math display">$$
p_\text{active}(c) R_{\text{tot}} = N_S \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}} + N_{NS} \frac{\lambda_r}{1 + \lambda_r} + N_C \frac{\lambda_r e^{-\beta \Delta \varepsilon_C}}{1 + \lambda_r e^{-\beta \Delta \varepsilon_C}}.
\qquad(16)$$</span><br /></span> For systems where the number of binding sites <span class="math inline"><em>N</em><sub><em>S</em></sub></span>, <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span>, and <span class="math inline"><em>N</em><sub><em>C</em></sub></span> are known, together with the binding affinities <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub></span>, we can solve numerically for <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> and substitute to obtain a fold-change at any concentration of inducer <span class="math inline"><em>c</em></span>. In the following sections, we will theoretically explore the induction curves given by Eq. 16 for a number of different combinations of simple repression binding sites, thereby predicting how the system would behave if additional specific or competitor binding sites were introduced.</p>
<h2 id="variable-repressor-copy-number-boldsymbolr-with-multiple-specific-binding-sites-boldsymboln_s-1">Variable Repressor Copy Number (<span class="math inline"><strong>R</strong></span>) with Multiple Specific Binding Sites (<span class="math inline"><strong>N</strong><sub><strong>S</strong></sub> <strong>&gt;</strong> <strong>1</strong></span>)</h2>
<p>In Chapter 2, we consider the induction profiles of strains with varying <span class="math inline"><em>R</em></span> but a single, specific binding site <span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span>. Here, we predict the induction profiles for similar strains in which <span class="math inline"><em>R</em></span> is varied, but <span class="math inline"><em>N</em><sub><em>S</em></sub> &gt; 1</span>, as shown in Fig. 3. The top row shows induction profiles in which <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span> and the bottom row shows profiles in which <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span>, assuming three different choices for the specific operator binding sites given by the O1, O2, and O3 operators. These values of <span class="math inline"><em>N</em><sub><em>S</em></sub></span> were chosen to mimic the common scenario in which a promoter construct is placed on either a low or high copy number plasmid. A few features stand out in these profiles. First, as the magnitude of <span class="math inline"><em>N</em><sub><em>S</em></sub></span> surpasses the number of repressors <span class="math inline"><em>R</em></span>, the leakiness begins to increase significantly, since there are no longer enough repressors to regulate all copies of the promoter of interest. Second, in the cases where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O1 operator or <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O2 operator, the profiles where <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span> are notably sharper than the profiles where <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span>, and it is possible to achieve dynamic ranges approaching 1. Finally, it is interesting to note that the profiles for the O3 operator where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span> are nearly indifferent to the value of <span class="math inline"><em>N</em><sub><em>S</em></sub></span>.</p>
<figure>
<img src="ch6_figS3.png" id="fig:fugacity_r" data-short-caption="Induction with variable $\boldsymbol{R}$ and multiple specific binding sites." alt="" /><figcaption>Figure 3: <strong>Induction with variable <span class="math inline"><strong>R</strong></span> and multiple specific binding sites.</strong> Induction profiles are shown for strains with variable R and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3</span>, <span class="math inline"> − 13.9</span>, or <span class="math inline"> − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The number of specific sites, <span class="math inline"><em>N</em><sub><em>S</em></sub></span>, is held constant at 10 as <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> are varied. <span class="math inline"><em>N</em><sub><em>S</em></sub></span> is held constant at 100 as <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> are varied. These situations mimic the common scenario in which a promoter construct is placed on either a low or high copy number plasmid. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS3.py">Python code (<code>ch6_figS3.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="variable-number-of-specific-binding-sites-boldsymboln_s-with-fixed-repressor-copy-number-boldsymbolr">Variable Number of Specific Binding Sites <span class="math inline"><strong>N</strong><sub><strong>S</strong></sub></span> with Fixed Repressor Copy Number (<span class="math inline"><strong>R</strong></span>)</h2>
<p>The second set of scenarios we consider is the case in which the repressor copy number <span class="math inline"><em>R</em> = 260</span> is held constant while the number of specific promoters <span class="math inline"><em>N</em><sub><em>S</em></sub></span> is varied (see Fig. 4). Again, we see that leakiness is increased significantly when <span class="math inline"><em>N</em><sub><em>S</em></sub> &gt; <em>R</em></span>, though all profiles for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span> exhibit high leakiness, making the effect less dramatic for this operator. Additionally, we find again that adjusting the number of specific sites can produce induction profiles with maximal dynamic ranges. In particular, the O1 and O2 profiles with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3</span> and <span class="math inline"> − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>, respectively, have dynamic ranges approaching 1 for <span class="math inline"><em>N</em><sub><em>S</em></sub> = 50</span> and 100.</p>
<figure>
<img src="ch6_figS4.png" id="fig:fugacity_Ns" data-short-caption="Induction with variable specific sites and fixed $\boldsymbol{R}$." alt="" /><figcaption>Figure 4: <strong>Induction with variable specific sites and fixed <span class="math inline"><strong>R</strong></span>.</strong> Induction profiles are shown for strains with <span class="math inline"><em>R</em> = 260</span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>, or <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The number of specific sites <span class="math inline"><em>N</em><sub><em>S</em></sub></span> is varied from 1 to 500. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS4.py">Python code (<code>ch6_figS4.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="competitor-binding-sites">Competitor Binding Sites</h2>
<p>An intriguing scenario is presented by the possibility of competitor sites elsewhere in the genome. This serves as a model for situations in which a promoter of interest is regulated by a transcription factor that has multiple targets. This is highly relevant, as the majority of transcription factors in <em>E. coli</em> have at least two known binding sites, with approximately 50 transcription factors having more than ten known binding sites <span class="citation" data-cites="rydenfelt2014 schmidt2016">(Rydenfelt et al. <a href="#ref-rydenfelt2014" role="doc-biblioref">2014</a>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>)</span>. If the number of competitor sites and their average binding energy is known, however, they can be accounted for in the model. Here, we predict the induction profiles for strains in which <span class="math inline"><em>R</em> = 260</span> and <span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span>, but there is a variable number of competitor sites <span class="math inline"><em>N</em><sub><em>C</em></sub></span> with a strong binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span>. In the presence of such a strong competitor when <span class="math inline"><em>N</em><sub><em>C</em></sub> &gt; <em>R</em></span>, the leakiness is greatly increased, as many repressors are siphoned into the pool of competitor sites. This is most dramatic for the case where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>, in which it appears that no repression occurs at all when <span class="math inline"><em>N</em><sub><em>C</em></sub> = 500</span>. Interestingly, when <span class="math inline"><em>N</em><sub><em>C</em></sub> &lt; <em>R</em></span>, the effects of the competitor are not especially notable.</p>
<figure>
<img src="ch6_figS5.png" id="fig:Nc" data-short-caption="Induction with variable competitor sites, a single specific site, and fixed $\boldsymbol{R}$." alt="" /><figcaption>Figure 5: <strong>Induction with variable competitor sites, a single specific site, and fixed <span class="math inline"><strong>R</strong></span>.</strong> Induction profiles are shown for strains with <span class="math inline"><em>R</em> = 260</span>, <span class="math inline"><em>N</em><sub><em>s</em></sub> = 1</span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O1 operator, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O2 operator, or <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span> for the O3 operator. The number of specific sites, <span class="math inline"><em>N</em><sub><em>C</em></sub></span>, is varied from 1 to 500. This mimics the common scenario in which a transcription factor has multiple binding sites in the genome. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS5.py">Python code (<code>ch6_figS5.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="properties-of-the-induction-response">Properties of the Induction Response</h2>
<p>As discussed in the main body of the paper, our treatment of the MWC model allows us to predict key properties of induction responses. Here, we consider the leakiness, saturation, and dynamic range (diagrammed in Fig. 2.1) by numerically solving Eq. 16 in the absence of inducer, <span class="math inline"><em>c</em> = 0</span>, and in the presence of saturating inducer <span class="math inline"><em>c</em> → ∞</span>. Using Eq. 15, the former case is given by <span id="eq:fugacity_leakiness"><br /><span class="math display">$$
R_\text{tot} \frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}}} = N_S \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}} + N_{NS} \frac{\lambda_r}{1 + \lambda_r} + N_C \frac{\lambda_r e^{-\beta \Delta \varepsilon_C}}{1 + \lambda_r e^{-\beta \Delta \varepsilon_C}}.
\qquad(17)$$</span><br /></span> Similarly, the limit of saturating inducer is found by determining <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> from the form <span id="eq:fugacity_saturation"><br /><span class="math display">$$
R_{\text{tot}} \frac{1}{1 + e^{-\beta \Delta \varepsilon_{AI}} \left(\frac{K_A}{K_I} \right)^2} = N_S \frac{\lambda_r e^{-\beta \Delta \varepsilon_{RA}}}{1 + \lambda_r e^{-\beta \Delta \varepsilon_{RA}}} + N_{NS} \frac{\lambda_r}{1 + \lambda_r} + N_C \frac{\lambda_r e^{-\beta \Delta \varepsilon_C}}{1 + \lambda_r e^{-\beta \Delta \varepsilon_C}}.
\qquad(18)$$</span><br /></span></p>
<p>In Fig. 6, we show how the leakiness, saturation, and dynamic range vary with <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> in systems with <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span> or <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span>. An inflection point occurs where <span class="math inline"><em>N</em><sub><em>S</em></sub> = <em>R</em></span>, with leakiness and dynamic range behaving differently when <span class="math inline"><em>R</em> &lt; <em>N</em><sub><em>S</em></sub></span> than when <span class="math inline"><em>R</em> &gt; <em>N</em><sub><em>S</em></sub></span>. This transition is more dramatic for <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span> than for <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span>. Interestingly, the saturation values consistently approach 1, indicating that full induction is easier to achieve when multiple specific sites are present. Moreover, dynamic range values for O1 and O2 strains with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 15.3</span> and <span class="math inline"> − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> approach 1 when <span class="math inline"><em>R</em> &gt; <em>N</em><sub><em>S</em></sub></span>, although when <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span>, there is a slight downward dip owing to saturation values of less than 1 at high repressor copy numbers.</p>
<figure>
<img src="ch6_figS6.png" id="fig:fugacity_properties_Ns" data-short-caption="Phenotypic properties of induction with multiple specific binding sites." alt="" /><figcaption>Figure 6: <strong>Phenotypic properties of induction with multiple specific binding sites.</strong> The leakiness, saturation, and dynamic range are shown for systems with number of specific binding sites <span class="math inline"><em>N</em><sub><em>S</em></sub> = 10</span> or <span class="math inline"><em>N</em><sub><em>S</em></sub> = 100</span>. The vertical white line indicates the point at which <span class="math inline"><em>N</em><sub><em>S</em></sub> = <em>R</em></span>. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS6.py">Python code (<code>ch6_figS6.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     In Fig. 7, we similarly show how the leakiness, saturation, and dynamic range vary with <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> in systems with <span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span> and multiple competitor sites <span class="math inline"><em>N</em><sub><em>C</em></sub> = 10</span> or <span class="math inline"><em>N</em><sub><em>C</em></sub> = 100</span>. Each of the competitor sites has a binding energy of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The phenotypic profiles are very similar to those for multiple specific sites with sharper transitions at <span class="math inline"><em>R</em> = <em>N</em><sub><em>C</em></sub></span> due to the greater binding strength of the competitor site. This indicates that introducing competitors has much the same effect on the induction phenotypes as introducing additional specific sites, as in either case the influence of the repressors is dampened when there are insufficient repressors to interact with all of the specific binding sites.</p>
<figure>
<img src="ch6_figS7.png" id="fig:fugacity_properties_Nc" data-short-caption="Phenotypic properties of induction with a single specific site and multiple competitor sites." alt="" /><figcaption>Figure 7: <strong>Phenotypic properties of induction with a single specific site and multiple competitor sites.</strong> The leakiness, saturation, and dynamic range are shown for systems with a single specific binding site <span class="math inline"><em>N</em><sub><em>S</em></sub> = 1</span> and a number of competitor sites <span class="math inline"><em>N</em><sub><em>C</em></sub> = 10</span> or <span class="math inline"><em>N</em><sub><em>C</em></sub> = 100</span>. All competitor sites have a binding energy of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>C</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span>. The vertical white line indicates the point at which <span class="math inline"><em>N</em><sub><em>C</em></sub> = <em>R</em></span>. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS7.py">Python code (<code>ch6_figS7.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     This section gives a quantitative analysis of the nuances imposed on induction response in the case of systems involving multiple gene copies as are found in the vast majority of studies on induction. In these cases, the intrinsic parameters of the MWC model get entangled with the parameters describing the gene copy number.</p>
<h2 id="flow-cytometry">Flow Cytometry</h2>
<p>In this section, we provide information regarding the equipment used to make experimental measurements of the fold-change in gene expression in the interests of transparency and reproducibility. We also provide a summary of our unsupervised method of gating the flow cytometry measurements for consistency between experimental runs.</p>
<h3 id="equipment">Equipment</h3>
<p>Due to past experience using the Miltenyi Biotec MACSQuant flow cytometer during the Physiology summer course at the Marine Biological Laboratory, we used the same flow cytometer for the formal measurements in this work graciously provided by the Pamela Björkman lab at Caltech. All measurements were made using an excitation wavelength of <span class="math inline">488 nm</span> with an emission filter set of 525/<span class="math inline">50 nm</span>. This excitation wavelength provides approximately 40% of the maximum YFP absorbance, and this was found to be sufficient for the purposes of these experiments. A useful feature of modern flow cytometry is the high-sensitivity signal detection through the use of photomultiplier tubes (PMT) whose response can be tuned by adjusting the voltage. Thus, the voltage for the forward-scatter (FSC), side-scatter (SSC), and gene expression measurements were tuned manually to maximize the dynamic range between autofluorescence signal and maximal expression without losing the details of the population distribution. Once these voltages were determined, they were used for all subsequent measurements. Extremely low signal producing particles were discarded before data storage by setting a basal voltage threshold, thus removing the majority of spurious events. The various instrument settings for data collection are given in Table 6.1.</p>
<table>
<caption>Instrument settings for data collection using the Miltenyi Biotec MACSQuant flow cytometer.</caption>
<thead>
<tr class="header">
<th style="text-align: left;"><strong>Laser</strong></th>
<th style="text-align: left;"><strong>Channel</strong></th>
<th style="text-align: center;"><strong>Sensor Voltage</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">488 nm</td>
<td style="text-align: left;">Forward-Scatter (FSC)</td>
<td style="text-align: center;">423 V</td>
</tr>
<tr class="even">
<td style="text-align: left;">488 nm</td>
<td style="text-align: left;">Side-Scatter (SSC)</td>
<td style="text-align: center;">537 V</td>
</tr>
<tr class="odd">
<td style="text-align: left;">488 nm</td>
<td style="text-align: left;">Intensity (B1 Filter, 525/50 nm)</td>
<td style="text-align: center;">790 V</td>
</tr>
<tr class="even">
<td style="text-align: left;">488 nm</td>
<td style="text-align: left;">Trigger (debris threshold)</td>
<td style="text-align: center;">24.5V</td>
</tr>
</tbody>
</table>
<h3 id="unsupervised-gating">Unsupervised Gating</h3>
<p>Flow cytometry data will frequently include a number of spurious events or other undesirable data points such as cell doublets and debris. The process of restricting the collected data set to those data determined to be “real” is commonly referred to as gating. These gates are typically drawn manually and restrict the data set to those points which display a high degree of linear correlation between their forward-scatter (FSC) and side-scatter (SSC). The development of unbiased and unsupervised methods of drawing these gates is an active area of research <span class="citation" data-cites="aghaeepour2013">(Aghaeepour et al. <a href="#ref-aghaeepour2013" role="doc-biblioref">2013</a>)</span>.</p>
<p>     For this study, we used an automatic unsupervised gating procedure to filter the flow cytometry data based on the front and side-scattering values returned by the MACSQuant flow cytometer. We assume that the region with highest density of points in these two channels corresponds to single-cell measurements. Everything extending outside of this region was discarded in order to exclude sources of error such as cell clustering, particulates, or other spurious events.</p>
<p>     In order to define the gated region, we fit a two-dimensional Gaussian function to the <span class="math inline">log<sub>10</sub></span> forward-scattering (FSC) and the <span class="math inline">log<sub>10</sub></span> side-scattering (SSC) data. We then kept a fraction <span class="math inline"><em>α</em> ∈ [0, 1]</span> of the data by defining an elliptical region given by <span id="eq:gauss_gate"><br /><span class="math display">(<strong>x</strong>−<strong>μ</strong>)<sup><em>T</em></sup><strong>Σ</strong><sup> − 1</sup>(<strong>x</strong>−<strong>μ</strong>) ≤ <em>χ</em><sub><em>α</em></sub><sup>2</sup>(<em>p</em>),   (19)</span><br /></span></p>
<p>where <span class="math inline"><strong>x</strong></span> is the <span class="math inline">2 × 1</span> vector containing the <span class="math inline">log (FSC)</span> and <span class="math inline">log (SSC)</span>, <span class="math inline"><strong>μ</strong></span> is the <span class="math inline">2 × 1</span> vector representing the mean values of <span class="math inline">log (FSC)</span> and <span class="math inline">log (SSC)</span> as obtained from fitting a two-dimensional Gaussian to the data, and <span class="math inline"><strong>Σ</strong></span> is the <span class="math inline">2 × 2</span> covariance matrix also obtained from the Gaussian fit. <span class="math inline"><em>χ</em><sub><em>α</em></sub><sup>2</sup>(<em>p</em>)</span> is the quantile function for probability <span class="math inline"><em>p</em></span> of the chi-squared distribution with two degrees of freedom. Fig. 8 shows an example of different gating contours that would arise from different values of <span class="math inline"><em>α</em></span> in Eq. 19. In this work, we chose <span class="math inline"><em>α</em> = 0.4</span> which we deemed was a sufficient constraint to minimize the noise in the data. The specific code where this gating is implemented can be found in <a href="https://github.com/RPGroup-PBoC/mwc_induction/blob/master/code/analysis/unsupervised_gating.ipynb">GitHub repository</a>.</p>
<figure>
<img src="ch6_figS8.png" id="fig:gate_contours" data-short-caption="Representative unsupervised gating contours of flow-cytometry data." alt="" /><figcaption>Figure 8: <strong>Representative unsupervised gating contours of flow-cytometry data.</strong> Points indicate individual flow cytometry measurements of forward scatter and side scatter. Colored contours indicate arbitrary gating contours ranging from 100% (<span class="math inline"><em>α</em> = 1</span>) to 5% (<span class="math inline"><em>α</em> = 0.05</span>). All measurements shown in Chapters 2 and 3 in this work were made by computing the mean fluorescence from the 40<span class="math inline"><em></em><sup>th</sup></span> percentile (<span class="math inline"><em>α</em> = 0.4)</span>. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS8.py">Python code (<code>ch6_figS8.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="comparison-of-flow-cytometry-with-other-methods">Comparison of Flow Cytometry with Other Methods</h3>
<p>Previous work from the Phillips’ lab experimentally determined fold-change for similar simple repression constructs using a variety of different measurement methods <span class="citation" data-cites="garcia2011b">(Garcia et al. <a href="#ref-garcia2011b" role="doc-biblioref">2011</a><a href="#ref-garcia2011b" role="doc-biblioref">a</a>)</span>. Garcia and Phillips used the same background strains as the ones used in this work, but gene expression was measured with Miller assays based on colorimetric enzymatic reactions with the LacZ protein <span class="citation" data-cites="garcia2011">(Garcia et al. <a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. The experiments in <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> (as well as in Chapter 4 of this dissertation) used a LacI dimer with the tetramerization region replaced with an mCherry tag, where the fold-change was measured as the ratio of the gene expression rate rather than a single snapshot of the gene output.</p>
<p>     Fig. 9 shows the comparison of these methods along with the flow cytometry method used in this work. The consistency of these three readouts validates the quantitative use of flow cytometry and unsupervised gating to determine the fold-change in gene expression. However, one important caveat revealed by this figure is that the sensitivity of flow cytometer measurements is not sufficient to accurately determine the fold-change for the high repressor copy number strains in O1 without induction. Instead, a method with a large dynamic range such as the Miller assay is needed to accurately resolve the fold-change at such low levels of expression.</p>
<figure>
<img src="ch6_figS9.png" id="fig:new_old_comparison" data-short-caption="Comparison of experimental methods to determine the fold-change in gene expression." alt="" /><figcaption>Figure 9: <strong>Comparison of experimental methods to determine the fold-change.</strong> The fold-change in gene expression for equivalent simple-repression constructs has been determined using three independent methods: flow cytometry (Chapter 2), colorimetric Miller assays (<span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>), and video microscopy (<span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>). All three methods give consistent results, although flow cytometry meeasurements lose accuracy for fold-change less than 0.01. Note that the repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> used for the theoretical predictions were determined in <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS9.py">Python code (<code>ch6_figS9.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="single-cell-microscopy">Single-Cell Microscopy</h2>
<p>In this section, we detail the procedures and results from single-cell microscopy verification of our flow cytometry measurements. Our previous measurements of fold-change in gene expression have been measured using bulk-scale Miller assays <span class="citation" data-cites="garcia2011">(Garcia et al. <a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span> or through single-cell microscopy <span class="citation" data-cites="brewster2014">(Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>. In this work, flow cytometry was an attractive method due to the ability to screen through many different strains at different concentrations of inducer in a short amount of time. To verify our results from flow cytometry, we examined two bacterial strains with different repressor-DNA binding energies (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>) of <span class="math inline"> − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> and <span class="math inline"> − 15.3 <em>k</em><sub><em>B</em></sub><em>T</em></span> with <span class="math inline"><em>R</em> = 260</span> repressors per cell using fluorescence microscopy, and estimated the values of the parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> for direct comparison between the two methods. For a detailed explanation of the Python code implementation of the processing steps described below, please see this paper’s <a href="https://rpgroup-pboc.github.io/mwc_induction/code/notebooks/unsupervised_gating.html">GitHub repository</a>.</p>
<h3 id="strains-and-growth-conditions">Strains and Growth Conditions</h3>
<p>Cells were grown in an identical manner to those used for measurement via flow cytometry (see Materials &amp; Methods of Chapter 2). Briefly, cells were grown to saturation in rich media broth (LB) with <span class="math inline">100 <em>μ</em>g ⋅ mL<sup> − 1</sup></span> spectinomycin in a deep-well 96-well plate at <span class="math inline">37<sup>∘</sup>C</span>. These cultures were then diluted 1000-fold into <span class="math inline">500 <em>μ</em>L</span> of M9 minimal medium supplemented with 0.5% glucose and the appropriate concentration of the inducer IPTG. Strains were allowed to grow at <span class="math inline">37<sup>∘</sup>C</span> with vigorous aeration for approximately 8 hours. Prior to mounting for microscopy, the cultures were diluted 10-fold into M9 glucose minimal medium in the absence of IPTG. Each construct was measured using the same range of inducer concentration values as was performed in the flow cytometry measurements (between <span class="math inline">100 nM</span> and <span class="math inline">5 mM</span> IPTG). Each condition was measured in triplicate in microscopy whereas approximately ten measurements were made using flow cytometry.</p>
<h3 id="imaging-procedure">Imaging Procedure</h3>
<p>During the last hour of cell growth, an agarose mounting substrate was prepared containing the appropriate concentration of the IPTG inducer. This mounting substrate was composed of M9 minimal medium supplemented with 0.5% glucose and 2% agarose (Life Technologies UltraPure Agarose, Cat. No. 16500100). This solution was heated in a microwave until molten followed by addition of the IPTG to the appropriate final concentration. This solution was then thoroughly mixed and a <span class="math inline">500 <em>μ</em>L</span> aliquot was sandwiched between two glass coverslips and was allowed to solidify.</p>
<p>     Once solid, the agarose substrates were cut into approximately <span class="math inline">10 mm × 10 mm</span> squares. An aliquot of one to two microliters of the diluted cell suspension was then added to each pad. For each concentration of inducer, a sample of the autofluorescence control, the <span class="math inline"><em>Δ</em><em>l</em><em>a</em><em>c</em><em>I</em></span> constitutive expression control, and the experimental strain was prepared yielding a total of thirty-six agarose mounts per experiment. These samples were then mounted onto two glass-bottom dishes (Ted Pella Wilco Dish, Cat. No. 14027-20) and sealed with parafilm.</p>
<p>     All imaging was performed on a Nikon Ti-Eclipse inverted fluorescent microscope outfitted with a custom-built laser illumination system and operated by the open-source MicroManager control software <span class="citation" data-cites="edelstein2014">(Edelstein et al. <a href="#ref-edelstein2014" role="doc-biblioref">2014</a>)</span>. The YFP fluorescence was imaged using a CrystaLaser <span class="math inline">514 nm</span> excitation laser coupled with a laser-optimized (Semrock Cat. No. LF514-C-000) emission filter.</p>
<p>     For each sample, between fifteen and twenty positions were imaged allowing for measurement of several hundred cells. At each position, a phase contrast image, an mCherry image, and a YFP image were collected in that order with exposures on a time scale of ten to twenty milliseconds. For each channel, the same exposure time was used across all samples in a given experiment. All images were collected and stored in <code>ome.tiff</code> format. All microscopy images are available on the CaltechDATA online repository under DOI: 10.22002/D1.229.</p>
<h3 id="image-processing">Image Processing</h3>
<h4 id="correcting-uneven-illumination">Correcting Uneven Illumination</h4>
<p>The excitation laser has a two-dimensional Gaussian profile. To minimize non-uniform illumination of a single field of view, the excitation beam was expanded to illuminate an area larger than that of the camera sensor. While this allowed for an entire field of view to be illuminated, there was still approximately a 10% difference in illumination across both dimensions. This nonuniformity was corrected for in post-processing by capturing twenty images of a homogeneously fluorescent plastic slide (Autofluorescent Plastic Slides, Chroma Cat. No. 920001) and averaging to generate a map of illumination intensity at any pixel <span class="math inline"><em>I</em><sub>YFP</sub></span>. To correct for shot noise in the camera (Andor iXon+ 897 EMCCD), twenty images were captured in the absence of illumination using the exposure time used for the experimental data. Averaging over these images produced a map of background noise at any pixel <span class="math inline"><em>I</em><sub>dark</sub></span>. To perform the correction, each fluorescent image in the experimental acquisition was renormalized with respect to these average maps as <span id="eq:flatfield_corr"><br /><span class="math display">$$
I_\text{flat} = \frac{I - I_\text{dark}}{I_\text{YFP} - I_\text{dark}}\langle
I_\text{YFP} - I_\text{dark} \rangle,
\qquad(20)$$</span><br /></span> where <span class="math inline"><em>I</em><sub>flat</sub></span> is the renormalized image and <span class="math inline"><em>I</em></span> is the original fluorescence image.</p>
<h3 id="cell-segmentation">Cell Segmentation</h3>
<p>Each bacterial strain constitutively expressed an mCherry fluorophore from a low copy-number plasmid. This served as a volume marker of cell mass allowing us to segment individual cells through edge detection in fluorescence. We used the Marr-Hildreth edge detector which identifies edges by taking the second derivative of a lightly Gaussian blurred image. Edges are identified as those regions which cross from highly negative to highly positive values or vice-versa within a specified neighborhood. Bacterial cells were defined as regions within an intact and closed identified edge. All segmented objects were then labeled and passed through a series of filtering steps.</p>
<p>     To ensure that primarily single cells were segmented, we imposed area and eccentricity bounds. We assumed that single cells projected into two dimensions are roughly <span class="math inline">2 <em>μ</em>m</span> long and <span class="math inline">1 <em>μ</em>m</span> wide, so that cells are likely to have an area between <span class="math inline">0.5 <em>μ</em>m<sup>2</sup></span> and <span class="math inline">6 <em>μ</em>m</span>. To determine the eccentricity bounds, we assumed that a single cell can be approximated by an ellipse with semi-major (<span class="math inline"><em>a</em></span>) and semi-minor (<span class="math inline"><em>b</em></span>) axis lengths of <span class="math inline">0.5 <em>μ</em>m</span> and <span class="math inline">0.25 <em>μ</em>m</span>, respectively. The eccentricity of this hypothetical cell can be computed as <span id="eq:eccentricity"><br /><span class="math display">$$
\text{eccentricity} = \sqrt{1 - \left(\frac{b}{a}\right)^2},
\qquad(21)$$</span><br /></span> yielding a value of approximately 0.8. Any objects with an eccentricity below this value were not considered to be single cells. After imposing both an area and eccentricity filter, the remaining objects were considered cells of interest and the mean fluorescence intensity of each cell was extracted.</p>
<h3 id="calculation-of-fold-change-and-empirical-comparison">Calculation of Fold-Change and Empirical Comparison</h3>
<p>Cells exhibited background fluorescence even in the absence of an expressed fluorophore. We corrected for this autofluorescence contribution to the fold-change calculation by subtracting the mean YFP fluorescence of cells expressing only the mCherry volume marker from each experimental measurement. The fold-change in gene expression was therefore calculated as <span id="eq:microscopy_fc"><br /><span class="math display">$$
\text{fold-change} = \frac{\langle I_{R &gt; 0} \rangle - \langle I_\text{auto}
\rangle}{\langle I_{R = 0} \rangle - \langle I_\text{auto} \rangle},
\qquad(22)$$</span><br /></span> where <span class="math inline">⟨<em>I</em><sub><em>R</em> &gt; 0</sub>⟩</span> is the mean fluorescence intensity of cells expressing LacI repressors, <span class="math inline">⟨<em>I</em><sub>auto</sub>⟩</span> is the mean intensity of cells expressing only the mCherry volume marker, and <span class="math inline">⟨<em>I</em><sub><em>R</em> = 0</sub>⟩</span> is the mean fluorescence intensity of cells in the absence of LacI.</p>
<p>     The agreement in the fold-change in gene expression between single-cell microscopy and flow cytometry can be seen in Fig. 10 (A) where the two methods have been plotted against each other. At this level, we see near perfect agreement between the methods when examining the mean level of gene expression. However, there is a distinct difference in higher moments of the gene expression distributions. Empirical cumulative distributions for a maximally-induced (5000 <span class="math inline"><em>μ</em></span>M IPTG, <span class="math inline"><em>R</em> = 160</span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>) sample are shown as purple and orange lines in Fig. 10 (B), respectively. To make the different methods directly comparable, the expressions distributions were normalized to range between 0 and 1, and then centered about the mean of the distribution. While the means agree between the methods, it is immediately obvious that the width of the distributions are different with microscopy yielding distributions with a higher variance. To compare the distributions more quantitatively, we computed the central moment values for the variance, skewness, and kurtosis of the distributions (Fig. 10 (C)). This quantitative comparison reveals that the value of the moments can differ by close to an order of magnitude between the methods with flow cytometry systematically lower than the same distribution measured via microscopy. These results show that in terms of measuring the mean level of gene expression, the two methods can be used interchangeably. However, if one is interested in the higher moments of the distribution, the choice of method does matter.</p>
<figure>
<img src="ch6_figS10.png" id="fig:empirical_comparison" data-short-caption="Empirical comparison of flow cytometry and single-cell microscopy." alt="" /><figcaption>Figure 10: <strong>Empirical comparison of flow cytometry and single-cell microscopy.</strong> (A) The observed fold-change in gene expression for the IPTG titration of a strain with <span class="math inline"><em>R</em> = 260</span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> using both microscopy (x-axis) and flow cytometry (y-axis). Points and errors represent the mean and standard error of 3 (microscopy) or 10 (flow cytometry) biological replicates. Black line indicates perfect agreement. (B) Empirical cumulative distributions of expression intensity for the strain used in (A) maximally induced with 5000 <span class="math inline"><em>μ</em></span>M IPTG. Purple and orange lines correspond to measurements with microscopy and flow cytometry, respectively. Fluorescence was normalized between 0 and 1 and centered about the observed mean. (C) Central moments of the distributions shown in (B) for microscopy and flow cytometry. Each point represents a single biological replicate. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS10.py">Python code (<code>ch6_figS10.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="fold-change-sensitivity-analysis">Fold-Change Sensitivity Analysis</h2>
<p>In Chapter 2,we found that the width of the credible regions varied widely depending on the repressor copy number <span class="math inline"><em>R</em></span> and repressor operator binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. More precisely, the credible regions were much narrower for low repressor copy numbers <span class="math inline"><em>R</em></span> and weak binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. In this section, we explain how this behavior comes about. We focus our attention on the maximum fold-change in the presence of saturating inducer. While it is straightforward to consider the width of the credible regions at any other inducer concentration, the predicted fold-change curves presented in Chapter 2 show that the credible regions are widest at saturation.</p>
<p>     The width of the credible regions corresponds to how sensitive the fold-change is to the fit values of the dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. To be quantitative, we define <span id="eq:foldchange_difference_ka"><br /><span class="math display"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub> ≡ fold-change(<em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub><sup>fit</sup>) − fold-change(<em>K</em><sub><em>A</em></sub><sup>fit</sup>, <em>K</em><sub><em>I</em></sub><sup>fit</sup>),   (23)</span><br /></span> the difference between the fold-change at a particular <span class="math inline"><em>K</em><sub><em>A</em></sub></span> value relative to the best-fit dissociation constant <span class="math inline"><em>K</em><sub><em>A</em></sub><sup>fit</sup> = 139 <em>μ</em>M</span>. For simplicity, we keep the inactive state dissociation constant fixed at its best-fit value <span class="math inline"><em>K</em><sub><em>I</em></sub><sup>fit</sup> = 0.53 <em>μ</em>M</span>. A larger difference <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub></span> implies a wider credible region. Similarly, we define the analogous quantity <span id="eq:foldchange_difference_ki"><br /><span class="math display"><em>Δ</em>fold-change<sub><em>K</em><sub><em>I</em></sub></sub> = fold-change(<em>K</em><sub><em>A</em></sub><sup>fit</sup>, <em>K</em><sub><em>I</em></sub>) − fold-change(<em>K</em><sub><em>A</em></sub><sup>fit</sup>, <em>K</em><sub><em>I</em></sub><sup>fit</sup>)   (24)</span><br /></span> to measure the sensitivity of the fold-change to <span class="math inline"><em>K</em><sub><em>I</em></sub></span> at a fixed <span class="math inline"><em>K</em><sub><em>A</em></sub><sup>fit</sup></span>. Fig. 11 shows both of these quantities in the limit <span class="math inline"><em>c</em> → ∞</span> for different repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and repressor copy numbers <span class="math inline"><em>R</em></span>.</p>
<p>     To understand how the width of the credible region scales with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>R</em></span>, we can Taylor-expand the difference in fold-change to first order, <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub> ≈ (∂fold-change/∂<em>K</em><sub><em>A</em></sub>)(<em>K</em><sub><em>A</em></sub>−<em>K</em><sub><em>A</em></sub><sup>fit</sup>)</span>, where the partial derivative has the form <span id="eq:foldchange_partial_ka"><br /><span class="math display">$$
\begin{aligned}
\frac{\partial \text{fold-change}}{\partial K_A} &amp;= \frac{e^{-\beta \Delta
\varepsilon_{AI}} \frac{n}{K_I}\left(\frac{K_A}{K_I}\right)^{n-1}}{\left( 1 +
e^{-\beta \Delta \varepsilon_{AI}} \left(\frac{K_A}{K_I}\right)^n
\right)^2}\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \times \\
&amp; \left(
    1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} }
    \left(\frac{K_A}{K_I}\right)^n }\frac{R}{N_{NS}}e^{-\beta
    \Delta\varepsilon_{RA}} \right)^{-2}.
\end{aligned}
\qquad(25)$$</span><br /></span> Similarly, the Taylor expansion <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>I</em></sub></sub> ≈ (∂fold-change/∂<em>K</em><sub><em>I</em></sub>)(<em>K</em><sub><em>I</em></sub>−<em>K</em><sub><em>I</em></sub><sup>fit</sup>)</span> features the partial derivative <span id="eq:foldchange_partial_ki"><br /><span class="math display">$$
\begin{aligned}
\frac{\partial \text{fold-change}}{\partial K_I} &amp;= -\frac{e^{-\beta \Delta
\varepsilon_{AI}} \frac{n}{K_I}\left(\frac{K_A}{K_I}\right)^{n}}{\left( 1 +
e^{-\beta \Delta \varepsilon_{AI}} \left(\frac{K_A}{K_I}\right)^n
\right)^2}\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \times \\
&amp; \left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} } \left(\frac{K_A}{K_I}\right)^n
}\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \right)^{-2}.
\end{aligned}
\qquad(26)$$</span><br /></span> From Eq. 25 and Eq. 26, we find that both <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub></span> and <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>I</em></sub></sub></span> increase in magnitude with <span class="math inline"><em>R</em></span> and decrease in magnitude with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. Accordingly, we expect that the O3 strains (with the least negative <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>) and the strains with the smallest repressor copy number will lead to partial derivatives with smaller magnitude and hence to tighter credible regions. Indeed, this prediction is carried out in Fig. 11.</p>
<p>     Lastly, we note that Eq. 25 and Eq. 26 enable us to quantify the scaling relationship between the width of the credible region and the two quantities <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. For example, for the O3 strains, where the fold-change at saturating inducer concentration is <span class="math inline"> ≈ 1</span>, the right-most term in both equations which equals the fold-change squared is roughly 1. Therefore, we find that both <span class="math inline">$\frac{\partial  \text{fold-change}}{\partial K_A}$</span> and <span class="math inline">∂fold-change/∂<em>K</em><sub><em>I</em></sub></span> scale linearly with <span class="math inline"><em>R</em></span> and <span class="math inline"><em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></sup></span>. Thus the width of the <span class="math inline"><em>R</em> = 22</span> strain will be roughly 1/1000 as large as that of the <span class="math inline"><em>R</em> = 1740</span> strain; similarly, the width of the O3 curves will be roughly 1/1000 the width of the O1 curves.</p>
<figure>
<img src="ch6_figS11.png" id="fig:kaki_sensitivity" data-short-caption="Sensitivity analysis of the fold-change function to $K_A$ and $K_I$ estimates." alt="" /><figcaption>Figure 11: <strong>Determining how sensitive the fold-change values are to the fit values of the dissociation constants.</strong> The difference <span class="math inline"><em>Δ</em>fold-change<sub><em>K</em><sub><em>A</em></sub></sub></span> in fold change when the dissociation constant <span class="math inline"><em>K</em><sub><em>A</em></sub></span> is slightly offset from its best-fit value <span class="math inline"><em>K</em><sub><em>A</em></sub> = 139<sub> − 22</sub><sup> + 29</sup><em>μ</em>M</span>. Fold-change is computed in the limit of saturating inducer concentration (<span class="math inline"><em>c</em> → ∞</span>, see ) where the credible regions are widest. The O3 strain (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 9.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>) is about 1/1000 as sensitive as the O1 operator to perturbations in the parameter values, and hence its credible region is roughly 1/1000 as wide. All curves were made using <span class="math inline"><em>R</em> = 260</span>. (B) The same analysis as shown in (A), but plotting the sensitivity of fold-change to the <span class="math inline"><em>K</em><sub><em>I</em></sub></span> parameter relative to the best-fit value <span class="math inline"><em>K</em><sub><em>I</em></sub> = 0.53<sub> − 0.04</sub><sup> + 0.04</sup><em>μ</em>M</span>. Note that only the magnitude, and not the sign, of this difference describes the sensitivity of each parameter. Hence, the O3 strain is again less sensitive than the O1 and O2 strains. (C) The same analysis as shown in (A), but showing how the fold-change sensitivity for different repressor copy numbers. The strains with lower repressor copy number are less sensitive to changes in the dissociation constants, and hence their corresponding curves have tighter credible regions. All curves were made using <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>. (D) The same analysis as shown in (C), the sensitivity of fold-change with respect to <span class="math inline"><em>K</em><sub><em>I</em></sub></span> is again smallest (in magnitude) for the low repressor copy number strains. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS11.py">Python code (<code>ch6_figS11.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="global-fit-of-all-parameters">Global Fit of All Parameters</h2>
<p>In Chapter 2, we used the repressor copy numbers <span class="math inline"><em>R</em></span> and repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> as reported by <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. However, any error in these previous measurements of <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> will necessarily propagate into our own fold-change predictions. In this section, we take an alternative approach to fitting the physical parameters of the system to that used in the main text. First, rather than fitting only a single strain, we fit the entire data set in along with microscopy data for the synthetic operator Oid. In addition, we also simultaneously fit the parameters <span class="math inline"><em>R</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> using the prior information given by the previous measurements. By using the entire data set and fitting all of the parameters, we obtain the best possible characterization of the statistical mechanical parameters of the system given our current state of knowledge.</p>
<p>    To fit all of the parameters simultaneously, we follow a similar approach to the one detailed in the Materials &amp; Methods of Chapter 2. Briefly, we perform a Bayesian parameter estimation of the dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, the six different repressor copy numbers <span class="math inline"><em>R</em></span> corresponding to the six <em>lacI</em> ribosomal binding sites used in our work, and the four different binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> characterizing the four distinct operators used to make the experimental strains. As in the main text, we fit the logarithms <span class="math inline">$\tilde{k}_A = -\log \frac{K_A}{1\,\text{M}}$</span> and <span class="math inline">$\tilde{k}_I = -\log \frac{K_I}{1\,\text{M}}$</span> of the dissociation constants which grants better numerical stability.</p>
<p>     We assume that deviations of the experimental fold-change from the theoretical predictions are normally distributed with mean zero and standard deviation <span class="math inline"><em>σ</em></span>. We begin by writing Bayes’ theorem, <span id="eq:bayes_global"><br /><span class="math display">$$
g(\tilde{k}_A, \tilde{k}_I, \mathbf{\textbf{\textit{R}}},
  \mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}, \sigma \mid D) =
  \frac{f(D \mid \tilde{k}_A, \tilde{k}_I, \mathbf{\textbf{\textit{R}}},
  \mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}, \sigma) g(\tilde{k}_A,
  \tilde{k}_I, \mathbf{\textbf{\textit{R}}},
  \mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}},
  \sigma)}{f(D)},
\qquad(27)$$</span><br /></span> where <span class="math inline">$\textbf{\textit{R}}$</span> is an array containing the six different repressor copy numbers to be fit, <span class="math inline">$\mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}$</span> is an array containing the four binding energies to be fit, and <span class="math inline"><em>D</em></span> is the experimental fold-change data. The term <span class="math inline">$P(\tilde{k}_A, \tilde{k}_I, \mathbf{\textbf{\textit{R}}}, \mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}, \sigma \mid D)$</span> gives the probability distributions of all of the parameters given the data. The prefixes <span class="math inline"><em>g</em></span> and <span class="math inline"><em>f</em></span> denote probability densities of parameters and data, respectively. The term <span class="math inline">$f(D \mid \tilde{k}_A, \tilde{k}_I, \mathbf{\textbf{\textit{R}}}, \mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}, \sigma)$</span> represents the likelihood of having observed our experimental data given some value for each parameter. <span class="math inline">$g(\tilde{k}_A, \tilde{k}_I, \mathbf{\textbf{\textit{R}}}, \mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}, \sigma)$</span> contains all the prior information on the values of these parameters. Lastly, <span class="math inline"><em>f</em>(<em>D</em>)</span> serves as a normalization constant and is neglected.</p>
<p>     Given <span class="math inline"><em>n</em></span> independent measurements of the fold-change, the first term in Eq. 27 can be written as <span id="eq:global_likelihood"><br /><span class="math display">$$
\begin{aligned}
    f(D \mid \tilde{k}_A, \tilde{k}_I, \mathbf{\textbf{\textit{R}}},
    \mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}, \sigma) =
    \frac{1}{(2\pi\sigma^2)^{\frac{n}{2}}}\prod\limits_{i=1}^n \exp
    \left[-\frac{(\text{fc}^{(i)}_{\exp} - \text{fc}(\tilde{k}_A, \tilde{k}_I,
    R^{(i)}, \Delta\varepsilon_{RA}^{(i)}, c^{(i)}))^2}{2\sigma^2}\right],
\end{aligned}
\qquad(28)$$</span><br /></span> where <span class="math inline">fc<sub>exp</sub><sup>(<em>i</em>)</sup></span> is the <span class="math inline"><em>i</em><sup>th</sup></span> experimental fold-change and <span class="math inline">fc( ⋅  ⋅  ⋅ )</span> is the theoretical prediction. Note that the standard deviation <span class="math inline"><em>σ</em></span> of this distribution is not known and hence needs to be included as a parameter to be fit.</p>
<p>    The second term in Eq. 27 represents the prior information of the parameter values. We assume that all parameters are independent of each other, such that <span class="math inline">$g(\tilde{k}_A, \tilde{k}_I, \mathbf{\textbf{\textit{R}}}, \mathbf{\Delta\boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}, \sigma)$</span> = <span class="math inline"><em>g</em>(<em>k̃</em><sub><em>A</em></sub>) ⋅ <em>P</em>(<em>k̃</em><sub><em>I</em></sub>) ⋅ ∏<sub><em>i</em></sub><em>P</em>(<em>R</em><sup>(<em>i</em>)</sup>) ⋅ ∏<sub><em>j</em></sub><em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>j</em>)</sup>) ⋅ <em>g</em>(<em>σ</em>),</span> where the superscript <span class="math inline">(<em>i</em>)</span> indicates the repressor copy number of index <span class="math inline"><em>i</em></span> and the superscript <span class="math inline">(<em>j</em>)</span> denotes the binding energy of index <span class="math inline"><em>j</em></span>. As above, we note that a prior must also be included for the unknown parameter <span class="math inline"><em>σ</em></span>.</p>
<p>     Because we know nothing about the values of <span class="math inline"><em>k̃</em><sub><em>A</em></sub></span>, <span class="math inline"><em>k̃</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>σ</em></span> before performing the experiment, we assign maximally uninformative priors to each of these parameters. More specifically, we assign uniform priors to <span class="math inline"><em>k̃</em><sub><em>A</em></sub></span> and <span class="math inline"><em>k̃</em><sub><em>I</em></sub></span> and a Jeffreys prior to <span class="math inline"><em>σ</em></span>, indicating that <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>σ</em></span> are scale parameters <span class="citation" data-cites="sivia2006">(Sivia and Skilling <a href="#ref-sivia2006" role="doc-biblioref">2006</a>)</span>. We do, however, have prior information for the repressor copy numbers and the repressor-DNA binding energies from <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. This prior knowledge is included within our model using an informative prior for these two parameters, which we assume to be Gaussian. Hence each of the <span class="math inline"><em>R</em><sup>(<em>i</em>)</sup></span> repressor copy numbers to be fit satisfies <span id="eq:global_prior_r"><br /><span class="math display">$$
g(R^{(i)}) = \frac{1}{\sqrt{2\pi\sigma_{R_i}^2}} \exp \left(-\frac{(R^{(i)} -
    \bar{R}^{(i)})^2}{2 \sigma_{R_i}^2} \right),
\qquad(29)$$</span><br /></span> where <span class="math inline"><em>R̄</em><sup>(<em>i</em>)</sup></span> is the mean repressor copy number and <span class="math inline"><em>σ</em><sub><em>R</em><sub><em>i</em></sub></sub></span> is the variability associated with this parameter as reported in <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. Note that we use the given value of <span class="math inline"><em>σ</em><sub><em>R</em><sub><em>i</em></sub></sub></span> from previous measurements rather than leaving this as a free parameter.</p>
<p>     Similarly, the binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>j</em>)</sup></span> are also assumed to have a Gaussian informative prior of the same form. We write it as <span id="eq:global_prior_epra"><br /><span class="math display">$$
g(\Delta\varepsilon_{RA}^{(j)}) = \frac{1}{\sqrt{2\pi\sigma_{\varepsilon_j}^2}}
\exp \left(- \frac{(\Delta\varepsilon_{RA}^{(j)} -
    \Delta\bar{\varepsilon}_{RA}^{(j)})^2}{2 \sigma_{\varepsilon_j}^2} \right),
\qquad(30)$$</span><br /></span> where <span class="math inline"><em>Δ</em><em>ε̄</em><sub><em>R</em><em>A</em></sub><sup>(<em>j</em>)</sup></span> is the binding energy and <span class="math inline"><em>σ</em><sub><em>ε</em><sub><em>j</em></sub></sub></span> is the variability associated with that parameter around the mean value as reported in <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>.</p>
<p>      The <span class="math inline"><em>σ</em><sub><em>R</em><sub><em>i</em></sub></sub></span> and <span class="math inline"><em>σ</em><sub><em>ε</em><sub><em>j</em></sub></sub></span> parameters will constrain the range of values for <span class="math inline"><em>R</em><sup>(<em>i</em>)</sup></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub><sup>(<em>j</em>)</sup></span> found from the fitting. For example, if for some <span class="math inline"><em>i</em></span> the standard deviation <span class="math inline"><em>σ</em><sub><em>R</em><sub><em>i</em></sub></sub></span> is very small, it implies a strong confidence in the previously reported value. Mathematically, the exponential in Eq. 29 will ensure that the best-fit <span class="math inline"><em>R</em><sup>(<em>i</em>)</sup></span> lies within a few standard deviations of <span class="math inline"><em>R̄</em><sup>(<em>i</em>)</sup></span>. Since we are interested in exploring which values could give the best fit, the errors are taken to be wide enough to allow the parameter estimation to freely explore parameter space in the vicinity of the best estimates. Putting all these terms together, we use Markov chain Monte Carlo to sample the posterior distribution <span class="math inline">$P(\tilde{k}_A, \tilde{k}_I, \mathbf{\textbf{\textit{R}}}, \mathbf{\Delta \boldsymbol{\varepsilon}_{\textbf{\textit{RA}}}}, \sigma \mid D)$</span>, enabling us to determine both the most likely value for each physical parameter as well as its associated credible region.</p>
<p>     Fig. 12 shows the result of this global fit. When compared with the results of Chapter 2, we can see that fitting for the binding energies and the repressor copy numbers improves the agreement between the theory and the data. Table 6.2 summarizes the values of the parameters as obtained with this MCMC parameter inference. We note that even though we allowed the repressor copy numbers and repressor-DNA binding energies to vary, the resulting fit values were very close to the previously reported values. The fit values of the repressor copy numbers were all within one standard deviation of the previous reported values provided in <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. And although some of the repressor-DNA binding energies differed by a few standard deviations from the reported values, the differences were always less than <span class="math inline">1 <em>k</em><sub><em>B</em></sub><em>T</em></span>, which represents a small change in the biological scales we are considering. The biggest discrepancy between our fit values and the previous measurements arose for the synthetic Oid operator, which we discuss in more detail in the following section of this chapter.</p>
<table>
<caption>Global parameter estimates and comparison to previously reported values.</caption>
<thead>
<tr class="header">
<th style="text-align: left;"><strong>Parameter</strong></th>
<th style="text-align: center;"><strong>Reported Values</strong> <span class="citation" data-cites="garcia2011">(Garcia et al. <a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span></th>
<th style="text-align: right;"><strong>Global Fit</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>K</em><sub><em>A</em></sub></span></td>
<td style="text-align: center;">-</td>
<td style="text-align: right;"><span class="math inline">205<sub> − 12</sub><sup> + 11</sup> <em>μ</em></span>M</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>K</em><sub><em>I</em></sub></span></td>
<td style="text-align: center;">-</td>
<td style="text-align: right;"><span class="math inline">0.73<sub> − 0.04</sub><sup>0.04</sup> <em>μ</em></span>M</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>R</em><sub>22</sub></span></td>
<td style="text-align: center;"><span class="math inline">22 ± 4</span> per cell</td>
<td style="text-align: right;"><span class="math inline">20<sub> − 1</sub><sup> + 1</sup></span> per cell</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>R</em><sub>60</sub></span></td>
<td style="text-align: center;"><span class="math inline">60 ± 20</span> per cell</td>
<td style="text-align: right;"><span class="math inline">74<sub> − 3</sub><sup> + 4</sup></span> per cell</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>R</em><sub>124</sub></span></td>
<td style="text-align: center;"><span class="math inline">124 ± 30</span> per cell</td>
<td style="text-align: right;"><span class="math inline">130<sub> − 6</sub><sup> + 6</sup></span> per cell</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>R</em><sub>260</sub></span></td>
<td style="text-align: center;"><span class="math inline">260 ± 40</span> per cell</td>
<td style="text-align: right;"><span class="math inline">257<sub> − 11</sub><sup> + 9</sup></span> per cell</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline"><em>R</em><sub>1220</sub></span></td>
<td style="text-align: center;"><span class="math inline">1220 ± 160</span> per cell</td>
<td style="text-align: right;"><span class="math inline">1191<sub> − 55</sub><sup> + 32</sup></span> per cell</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline"><em>R</em><sub>1740</sub></span></td>
<td style="text-align: center;"><span class="math inline">1740 ± 340</span> per cell</td>
<td style="text-align: right;"><span class="math inline">1599<sub> − 87</sub><sup> + 75</sup></span> per cell</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O1 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span></td>
<td style="text-align: center;"><span class="math inline"> − 15.3 ± 0.2 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: right;"><span class="math inline"> − 15.22<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
</tr>
<tr class="even">
<td style="text-align: left;">O2 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span></td>
<td style="text-align: center;"><span class="math inline"> − 13.9 ± 0.2 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: right;"><span class="math inline"> − 13.06<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
</tr>
<tr class="odd">
<td style="text-align: left;">O3 <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span></td>
<td style="text-align: center;"><span class="math inline"> − 9.7 ± 0.1 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: right;"><span class="math inline"> − 9.4<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
</tr>
<tr class="even">
<td style="text-align: left;">Oid <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span></td>
<td style="text-align: center;"><span class="math inline"> − 17.0 ± 0.2 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: right;"><span class="math inline"> − 17.7<sub> − 0.1</sub><sup> + 0.2</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
</tr>
</tbody>
</table>
<figure>
<img src="ch6_figS12.png" id="fig:induction_global_fit" data-short-caption="Induction curves using global parameter estimates." alt="" /><figcaption>Figure 12: <strong>Global fit of dissociation constants, repressor copy numbers, and binding energies.</strong> Theoretical prediction resulting from simultaneous estimation of the dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, the six repressor copy numbers <span class="math inline"><em>R</em></span>, and the four repressor-DNA binding energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> using the entire dataset. Points and errors represent the mean and standard error of ~10 biological replicates for O1, O2, O3, and 3 biological replicates for Oid. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS12.py">Python code (<code>ch6_fig12.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     Fig. 13 shows the same key properties as Fig. 2.7 , but uses the parameters obtained from this global fitting approach. We note that even by increasing the number of degrees of freedom in our fit, the result does not change substantially. or the O3 operator data, again, agreement between the predicted <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> and the effective Hill coefficient remains poor due to the theory being unable to capture the steepness of the response curves.</p>
<figure>
<img src="ch6_figS13.png" id="fig:properties_global_fit" data-short-caption="Key properties of induction profiles as predicted with a global fit using all data." alt="" /><figcaption>Figure 13: <strong>Key properties of induction profiles as predicted with a global fit using all data.</strong> Data for (A) leakiness, (B) saturation, and (C) dynamic range are computed directly from measured fold-change. Points and errors correspond to the mean and standard error of 10 - 11 biological replicates. Points in (D) and (E) for the [EC<span class="math inline"><em></em><sub>50</sub></span>] and the effective Hill coefficient, respectively, represent the estimated value using parameter estimates of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> for that particular strain. Errors represent the width of the 95% credible region. In all plots, curves represent the theoretical predictions given the parameter estimates conditioned on all data sets. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS13.py">Python code (<code>ch6_figS13.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="applicability-of-theory-to-the-oid-operator-sequence">Applicability of Theory to the Oid Operator Sequence</h2>
<p>In addition to the native operator sequences (O1, O2, and O3) considered in the main text, we were also interested in testing our model predictions against the synthetic Oid operator. In contrast to the other operators, Oid is one base pair shorter in length (20 bp), is fully symmetric, and is known to provide stronger repression than the native operator sequences considered so far. While the theory should be similarly applicable, measuring the lower fold-changes associated with this YFP construct was expected to be near the sensitivity limit for our flow cytometer, due to the especially strong binding energy of Oid (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span>) <span class="citation" data-cites="garcia2011">(Garcia et al. <a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. Accordingly, fluorescence data for Oid were obtained using microscopy, which is more sensitive than flow cytometry.</p>
<p>     We follow the approach of the main text and make fold-change predictions based on the parameter estimates from our strain with <span class="math inline"><em>R</em> = 260</span> and an O2 operator. These predictions are shown in Fig. 14, where we also plot data taken in triplicate for strains containing <span class="math inline"><em>R</em> = 22</span>, 60, and 124, obtained by single-cell microscopy. We find that the data are systematically below the theoretical predictions. We also considered our global fitting approach (see previous section) to see whether we might find better agreement with the observed data. Interestingly, we find that the majority of the parameters remain largely unchanged, but our estimate for the Oid binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is shifted to <span class="math inline"> − 17.7 <em>k</em><sub><em>B</em></sub><em>T</em></span> instead of the value <span class="math inline"> − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span> found in <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. In Fig. 14, we again plot the Oid fold-change data but with theoretical predictions, using the new estimate for the Oid binding energy from our global fit, and find substantially better agreement.</p>
<figure>
<img src="ch6_figS14.png" id="fig:oid_refit" data-short-caption="Predictions of fold-change for strains with an Oid binding sequence versus experimental measurements with different repressor copy numbers." alt="" /><figcaption>Figure 14: <strong>Predictions of fold-change for strains with an Oid binding sequence versus experimental measurements with different repressor copy numbers.</strong> Experimental data is plotted against the parameter-free predictions that are based on our fit to the O2 strain with <span class="math inline"><em>R</em> = 260</span>. Here we use the previously measured binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="garcia2011">(Garcia et al. <a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. The same experimental data is plotted against the best-fit parameters using the complete O1, O2, O3, and Oid data sets to infer <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, repressor copy numbers, and the binding energies of all operators. Here the major difference in the inferred parameters is a shift in the binding energy for Oid from <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.0 <em>k</em><sub><em>B</em></sub><em>T</em></span> to <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 17.7 <em>k</em><sub><em>B</em></sub><em>T</em></span>, which now shows agreement between the theoretical predictions and experimental data. Shaded regions from the theoretical curves denote the 95% credible region. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS14.py">Python code (<code>ch6_figS14.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="comparison-of-parameter-estimation-and-fold-change-predictions-across-strains">Comparison of Parameter Estimation and Fold-Change Predictions across Strains</h2>
<p>The inferred parameter values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> in Chapter 2 were determined by fitting to induction fold-change measurements from a single strain (<span class="math inline"><em>R</em> = 260</span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span>, <span class="math inline"><em>n</em> = 2</span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>). After determining these parameters, we were able to predict the fold-change of the remaining strains without any additional fitting. However, the theory should be independent of the specific strain used to estimate <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>; using any alternative strain to fit <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> should yield similar predictions. For the sake of completeness, here we discuss the values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> that are obtained by fitting to each of the induction data sets individually. These fit parameters are shown in Fig. 2.6 (D) of Chapter 2, where we find close agreement between strains, but with some deviation and poorer inferences observed with the O3 operator strains. Overall, we find that regardless of which strain is chosen to determine the unknown parameters, the predictions laid out by the theory closely match the experimental measurements. Here, we present a comparison of the strain specific predictions and measured fold-change data for each of the three operators considered.</p>
<p>     We follow the approach taken in Chapter 2 and infer values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> by fitting to each combination of binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and repressor copy number <span class="math inline"><em>R</em></span>. We then use these fitted parameters to predict the induction curves of all other strains. In Fig. 15, we plot these fold-change predictions along with experimental data for each of our strains that contains an O1 operator. To make sense of this plot, we consider the first row as an example. In the first row, <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> were estimated using data from the strain containing <span class="math inline"><em>R</em> = 22</span> and an O1 operator (top leftmost plot). The remaining plots in this row show the predicted fold-change using these values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. In each row, we then infer <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> using data from a strain containing a different repressor copy number (<span class="math inline"><em>R</em> = 60</span> in the second row, <span class="math inline"><em>R</em> = 124</span> in the third row, and so on). In Fig. 16 and Fig. 17, we similarly apply this inference to our strains with O2 and O3 operators, respectively. We note that the overwhelming majority of predictions closely match the experimental data.The notable exception is that using the <span class="math inline"><em>R</em> = 22</span> strain provides poor predictions for the strains with large copy numbers (especially <span class="math inline"><em>R</em> = 1220</span> and <span class="math inline"><em>R</em> = 1740</span>), though it should be noted that predictions made from the <span class="math inline"><em>R</em> = 22</span> strain have considerably broader credible regions. This loss in predictive power is due to the poorer estimates of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> for the <span class="math inline"><em>R</em> = 22</span> strain as shown in Fig. 2.6 (D).</p>
<figure>
<img src="ch6_figS15.png" id="fig:O1_comparison" data-short-caption="O1 strain fold-change predictions based on strain-specific parameter estimation of $K_A$ and $K_I$." alt="" /><figcaption>Figure 15: <strong>O1 strain fold-change predictions based on strain-specific parameter estimation of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>.</strong> Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O1 operator. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS15-17.py">Python code (<code>ch6_figS15-S17.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<figure>
<img src="ch6_figS16.png" id="fig:O2_comparison" data-short-caption="O2 strain fold-change predictions based on strain-specific parameter estimation of $K_A$ and $K_I$." alt="" /><figcaption>Figure 16: <strong>O2 strain fold-change predictions based on strain-specific parameter estimation of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>.</strong> Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O2 operator. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS15-17.py">Python code (<code>ch6_figS15-S17.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<figure>
<img src="ch6_figS17.png" id="fig:O3_comparison" data-short-caption="O3 strain fold-change predictions based on strain-specific parameter estimation of $K_A$ and $K_I$." alt="" /><figcaption>Figure 17: <strong>O3 strain fold-change predictions based on strain-specific parameter estimation of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>.</strong> Fold-change in expression is plotted as a function of IPTG concentration for all strains containing an O3 operator. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS15-17.py">Python code (<code>ch6_figS15-S17.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="properties-of-induction-titration-curves">Properties of Induction Titration Curves</h2>
<p>In this section, we expand on the phenotypic properties of the induction response that were explored in Chapter 2. We begin by expanding on our discussion of dynamic range and then show the analytic form of the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> for simple repression.</p>
<p>     As stated in the main text, the dynamic range is defined as the difference between the maximum and minimum system response, or equivalently, as the difference between the saturation and leakiness of the system. The dynamic range is therefore given by <span id="eq:induction_si_dyn_rng"><br /><span class="math display">$$
\begin{aligned}
\text{dynamic range} = &amp;\left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} } \left(\frac{K_A}{K_I}\right)^n
}\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \right)^{-1} -\\
&amp; \left(
1+\frac{1}{1+e^{-\beta \Delta \varepsilon_{AI} }}\frac{R}{N_{NS}}e^{-\beta
\Delta\varepsilon_{RA}} \right)^{-1}.
\end{aligned}
\qquad(31)$$</span><br /></span></p>
<p>     The dynamic range, along with saturation and leakiness were plotted with our experimental data in Fig. 2.7(A-C) as a function of repressor copy number. Fig. 18 shows how these properties are expected to vary as a function of the repressor-operator binding energy. Note that the resulting curves for all three properties have the same shape as in Fig. 2.7 (A-C), since the dependence of the fold-change upon the repressor copy number and repressor-operator binding energy are both contained in a single multiplicative term, <span class="math inline"><em>R</em><em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></sup></span>. Hence, increasing <span class="math inline"><em>R</em></span> on a logarithmic scale is equivalent to decreasing <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> on a linear scale.</p>
<p>     An interesting aspect of the dynamic range is that it exhibits a peak as a function of either the repressor copy number (or equivalently of the repressor-operator binding energy). Differentiating the dynamic range (Eq. 31) and setting it equal to zero, we find that this peak occurs at <span id="eq:loc_max_dyn_rng"><br /><span class="math display">$$
\frac{R^*}{N_{NS}} = e^{-\beta (\Delta\varepsilon_{AI} -
\Delta\varepsilon_{RA})} \sqrt{e^{\Delta\varepsilon_{AI}} + 1}
\sqrt{e^{\Delta\varepsilon_{AI}} + \left( \frac{K_A}{K_I} \right)^n}.
\qquad(32)$$</span><br /></span> The magnitude of the peak is given by <span id="eq:mag_max_dyn_rng"><br /><span class="math display">$$
\text{max dynamic range} = \frac{\left( \sqrt{e^{\Delta\varepsilon_{AI}} + 1} -
\sqrt{e^{\Delta\varepsilon_{AI}} + \left( \frac{K_A}{K_I} \right)^n}
\right)^2}{\left( \frac{K_A}{K_I} \right)^n - 1},
\qquad(33)$$</span><br /></span> which is independent of the repressor-operator binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> or <span class="math inline"><em>R</em></span>, and will only cause a shift in the location of the peak, but not its magnitude.</p>
<figure>
<img src="ch6_figS18.png" id="fig:properties_expanded" data-short-caption="Dependence of leakiness, saturation, and dynamic range on the operator binding energy and repressor copy number." alt="" /><figcaption>Figure 18: <strong>Dependence of leakiness, saturation, and dynamic range on the operator binding energy and repressor copy number.</strong> Increasing repressor copy number or decreasing the repressor-operator binding energy suppresses gene expression and decreases both the (A) leakiness and (B) saturation. (C) The dynamic range retains its shape, but shifts right as the repressor copy number increases. The peak in the dynamic range can be understood by considering the two extremes for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>: for small repressor-operator binding energies, the leakiness is small, but the saturation increases with <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, thereby decreasing the dynamic range. Repressor copy number does not affect the maximum dynamic range. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS18-S19.py">Python code (<code>ch6_figS18-S19.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     We now consider the two remaining properties, the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> and effective Hill coefficient, which determine the horizontal properties of a system - that is, they determine the range of inducer concentration in which the system’s response goes from its minimum to maximum values. The <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> denotes the inducer concentration required to generate fold-change halfway between its minimum and maximum value and was defined implicitly in Eq. 2.9. For the simple repression system, the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> is given by <span id="eq:ec50_explicit"><br /><span class="math display">$$
\frac{[EC_{50}]}{K_A} = \frac{\frac{K_A}{K_I} - 1}{\frac{K_A}{K_I} - \left(
\frac{\left( 1 + \frac{R}{N_{NS}} e^{-\beta \Delta\varepsilon_{RA}} \right) +
\left( \frac{K_A}{K_I} \right)^n \left( 2 e^{-\beta \Delta \varepsilon_{AI}} +
\left( 1 + \frac{R}{N_{NS}} e^{-\beta \Delta\varepsilon_{RA}} \right) \right)
}{ 2 \left( 1 + \frac{R}{N_{NS}} e^{-\beta \Delta\varepsilon_{RA}} \right) +
e^{-\beta \Delta \varepsilon_{AI}} + \left( \frac{K_A}{K_I} \right)^n e^{-\beta
\Delta \varepsilon_{AI}} } \right)^{\frac{1}{n}}} - 1.
\qquad(34)$$</span><br /></span> Using this expression, we can then find the effective Hill coefficient <span class="math inline"><em>h</em></span>, which equals twice the log-log slope of the normalized fold-change evaluated at <span class="math inline"><em>c</em> = [<em>E</em><em>C</em><sub>50</sub>]</span>. In Fig. 2.7 (D-E), we show how these two properties vary with repressor copy number, and in Fig. 19, we demonstrate how they depend on the repressor-operator binding energy. Both the <span class="math inline">[<em>E</em><em>C</em><sub>50</sub>]</span> and <span class="math inline"><em>h</em></span> vary significantly with repressor copy number for sufficiently strong operator binding energies. Interestingly, for weak operator binding energies on the order of the O3 operator, it is predicted that the effective Hill coefficient should not vary with repressor copy number. In addition, the maximum possible Hill coefficient is roughly 1.75, which stresses the point that the effective Hill coefficient should not be interpreted as the number of inducer binding sites, which is exactly 2.</p>
<figure>
<img src="ch6_figS19.png" id="fig:ec50_ehill" data-short-caption="[EC$_{50}$] and effective Hill coefficient depend strongly on repressor copy number and operator binding energy." alt="" /><figcaption>Figure 19: <strong>[EC<span class="math inline"><em></em><sub>50</sub></span>] and effective Hill coefficient depend strongly on repressor copy number and operator binding energy.</strong> (A) [EC<span class="math inline"><em></em><sub>50</sub></span>] values from very small and tightly clustered to relatively large and expanded for stronger operator binding energies. (B) The effective Hill coefficient generally decreases with increasing repressor copy number, indicating a flatter normalized response. The maximum possible Hill coefficient is roughly 1.75 for all repressor-operator binding energies. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS18-S19.py">Python code (<code>ch6_figS18-S19.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="applications-to-other-regulatory-architectures">Applications to Other Regulatory Architectures</h2>
<p>In this section, we discuss how the theoretical framework presented in this work is sufficiently general to include a variety of regulatory architectures outside of simple repression by LacI. We begin by noting that the exact same formula for fold-change given in Chapter 2 can also describe corepression. We then demonstrate how our model can be generalized to include other architectures, such as a coactivator binding to an activator to promote gene expression. In each case, we briefly describe the system and describe its corresponding theoretical description. For further details, we invite the interested reader to read <span class="citation" data-cites="bintu2005">Bintu et al. (<a href="#ref-bintu2005" role="doc-biblioref">2005</a>)</span> and <span class="citation" data-cites="marzen2013">Marzen, Garcia, and Phillips (<a href="#ref-marzen2013" role="doc-biblioref">2013</a>)</span>.</p>
<h3 id="corepression">Corepression</h3>
<p>Consider a regulatory architecture where binding of a transcriptional repressor occludes the binding of RNAP to the DNA. A corepressor molecule binds to the repressor and shifts its allosteric equilibrium towards the active state in which it binds more tightly to the DNA, thereby decreasing gene expression (in contrast, an inducer shifts the allosteric equilibrium towards the inactive state where the repressor binds more weakly to the DNA). As in the main text, we can enumerate the states and statistical weights of the promoter and the allosteric states of the repressor. We note that these states and weights exactly match those in Fig. 2.2 and yield the same fold-change equation, <span id="eq:corepression_fc"><br /><span class="math display">$$
\text{fold-change} \approx \left(1 + {\left(1 + {c \over K_A}\right)^n \over
\left(1 + {c \over K_A}\right)^n + e^{\beta\Delta\varepsilon_{AI} }\left(1 + {c
\over K_I}\right)^n} {R \over
N_{NS} }e^{-\beta\Delta\varepsilon_{RA} }\right)^{-1},
\qquad(35)$$</span><br /></span> where <span class="math inline"><em>c</em></span> now represents the concentration of the corepressor molecule. Mathematically, the difference between these two architectures can be seen in the relative sizes of the dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> between the inducer and repressor in the active and inactive states, respectively. The corepressor is defined by <span class="math inline"><em>K</em><sub><em>A</em></sub> &lt; <em>K</em><sub><em>I</em></sub></span>, since the corepressor favors binding to the repressor’s active state; an inducer must satisfy <span class="math inline"><em>K</em><sub><em>I</em></sub> &lt; <em>K</em><sub><em>A</em></sub></span>, as was found in Chapter 2. Much as was performed in Chapter 2, we can make some predictions regarding the response of a corepressor. In Fig. 20 (A), we show how varying the repressor copy number <span class="math inline"><em>R</em></span> and the repressor-DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> influence the response. We draw the reader’s attention to the decrease in fold-change as the concentration of effector is increased.</p>
<h3 id="activation">Activation</h3>
<p>We now turn to the case of activation. While this architecture was not studied in this work, we wish to demonstrate how the framework presented here can be extended to include transcription factors other than repressors. To that end, we consider a transcriptional activator which binds to DNA and aids in the binding of RNAP through energetic interaction term <span class="math inline"><em>ε</em><sub><em>A</em><em>P</em></sub></span>. Note that in this architecture, binding of the activator does not occlude binding of the polymerase. Binding of a coactivator molecule shifts its allosteric equilibrium towards the active state (<span class="math inline"><em>K</em><sub><em>A</em></sub> &lt; <em>K</em><sub><em>I</em></sub></span>), where the activator is more likely to be bound to the DNA and promote expression. Enumerating all of the states and statistical weights of this architecture and making the approximation that the promoter is weak generates a fold-change equation of the form <span id="eq:foldchange_activation"><br /><span class="math display">$$
\text{fold-change} = {1 + \frac{\left(1 + \frac{c}{K_A}\right)^n}{\left( 1 + \frac{c}{K_A}\right)^n + e^{\beta\Delta\varepsilon_{AI} }\left(1 + \frac{c}{K_I}\right)^n}\frac{A}{
N_{NS} }e^{-\beta\Delta\varepsilon_{AA} }e^{-\beta\varepsilon_{AP} } \over 1 +
{\left(1 + {c \over K_A}\right)^n \over \left(1 + {c \over K_A}\right)^n +
e^{\beta\Delta\varepsilon_{AI} }\left(1 + {c \over K_I}\right)^n}{A \over
N_{NS} }e^{-\beta\Delta\varepsilon_{AA} } },
\qquad(36)$$</span><br /></span> where <span class="math inline"><em>A</em></span> is the total number of activators per cell, <span class="math inline"><em>c</em></span> is the concentration of a coactivator molecule, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>A</em></sub></span> is the binding energy of the activator to the DNA in the active allosteric state, and <span class="math inline"><em>ε</em><sub><em>A</em><em>P</em></sub></span> is the interaction energy between the activator and the RNAP. Unlike in the cases of induction and corepression, the fold-change formula for activation includes terms from when the RNAP is bound by itself on the DNA as well as when both RNAP and the activator are simultaneously bound to the DNA. Fig. 20 (B) explores predictions of the fold-change in gene expression by manipulating the activator copy number, DNA binding energy, and the polymerase-activator interaction energy. Note that with this activation scheme, the fold-change must necessarily be greater than one. An interesting feature of these predictions is the observation that even small changes in the interaction energy (<span class="math inline"> &lt; 0.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>) can result in dramatic increase in fold-change.</p>
<p>     As in the case of induction, the approach towards inducible activation is straightforward to generalize. For example, the relative values of <span class="math inline"><em>K</em><sub><em>I</em></sub></span> and <span class="math inline"><em>K</em><sub><em>A</em></sub></span> can be switched such that <span class="math inline"><em>K</em><sub><em>I</em></sub> &lt; <em>K</em><sub><em>A</em></sub></span>, in which the secondary molecule drives the activator to assume the inactive state, represents induction of an activator. While these cases might be viewed as separate biological phenomena, mathematically they can all be described by the same underlying formalism.</p>
<figure>
<img src="ch6_figS20.png" id="fig:applications" data-short-caption="Representative fold-change predictions for allosteric corepression and activation." alt="" /><figcaption>Figure 20: <strong>Representative fold-change predictions for allosteric corepression and activation.</strong> (A) Contrary to the case of induction described in the main text, addition of a corepressor decreases fold-change in gene expression. The left and right panels demonstrate how varying the values of the repressor copy number <span class="math inline"><em>R</em></span> and repressor-DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, respectively, change the predicted response profiles. (B) In the case of inducible activation, binding of an effector molecule to an activator transcription factor increases the fold-change in gene expression. Note that for activation, the fold-change is greater than 1. The let and center panels show how changing the activator copy number <span class="math inline"><em>A</em></span> and activator-DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>A</em></sub></span> alter response, respectively. The right panel shows how varying the polymerase-activator interaction energy <span class="math inline"><em>ε</em><sub><em>A</em><em>P</em></sub></span> alters the fold-change. Relatively small perturbations to this energetic parameter drastically changes the level of activation and plays a major role in dictating the dynamic range of the system. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_06/code/ch6_figS20.py">Python code (<code>ch6_figS20.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="e.-coli-primer-and-strain-list"><em>E. coli</em> Primer and Strain List</h2>
<p>Here, we provide additional details about the genotypes of the strains used, as well as the primer sequences used to generate them. <em>E. coli</em> strains were derived from K12 MG1655. For those containing <span class="math inline"><em>R</em> = 22</span>, we used strain HG104 which additionally has the <em>lacYZA</em> operon deleted (positions 360,483 to 365,579), but still contains the native <em>lacI</em> locus. All other strains used strain HG105, where both the <em>lacYZA</em> and <em>lacI</em> operons have both been deleted (positions 360,483 to 366,637).</p>
<p>     All 25x+11-yfp expression constructs were integrated at the <em>galK</em> locus (between positions 1,504,078 and 1,505,112) while the 3*1x-lacI constructs were integrated at the <em>ybcN</em> locus (between positions 1,287,628 and 1,288,047). Integration was performed with <span class="math inline"><em>λ</em></span> Red recombineering <span class="citation" data-cites="sharan2009">(Sharan et al. <a href="#ref-sharan2009" role="doc-biblioref">2009</a>)</span> as described in <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span> using the primers listed in Table 6.3. We follow the notation of Lutz and Bujard <span class="citation" data-cites="lutz1997">(Lutz and Bujard <a href="#ref-lutz1997" role="doc-biblioref">1997</a>)</span> for the nomenclature of the different constructs used. Specifically, the first number refers to the antibiotic resistance cassette that is present for selection (2 = kanamycin, 3 = chloramphenicol, and 4 = spectinomycin) and the second number refers to the promoter used to drive expression of either YFP or LacI (1 = <span class="math inline"><em>P</em><sub><em>L</em><em>t</em><em>e</em><em>t</em><em>O</em> − 1</sub></span>, and 5 = <em>lacUV5</em>). Note that in 25x+11-yfp, x refers to the LacI operator used, which is centered at +11 (or alternatively, begins at the transcription start site). For the different LacI constructs, 3*1x-lacI, x refers to the different ribosomal binding site modifications that provide different repressor copy numbers and follows from <span class="citation" data-cites="garcia2011">Garcia et al. (<a href="#ref-garcia2011" role="doc-biblioref">2011</a><a href="#ref-garcia2011" role="doc-biblioref">a</a>)</span>. The asterisk refers to the presence of FLP recombinase sites flanking the chloramphenicol resistance gene that can be used to lose this resistance. However, we maintained the resistance gene in our constructs. A summary of the final genotypes of each strain is listed in Table 6.4. In addition, each strain also contained the plasmid pZS4*1-mCherry and provided constitutive expression of the mCherry fluorescent protein. This pZS plasmid is a low copy (SC101 origin of replication) where, like with 3*1x-lacI, mCherry is driven by a <span class="math inline"><em>P</em><sub><em>L</em><em>t</em><em>e</em><em>t</em><em>O</em> − 1</sub></span> promoter.</p>
<table>
<caption>Primers used in this work.</caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;"><strong>Primer</strong></th>
<th style="text-align: left;"><strong>Sequence (5’ <span class="math inline">→</span> 3’)</strong></th>
<th style="text-align: left;"><strong>Notes</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">pZSForwSeq2</td>
<td style="text-align: left;">TTCCCAACC TTACCAGAGG GC</td>
<td style="text-align: left;">Forward sequencing primer for 3*1x-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">251 F</td>
<td style="text-align: left;">CCTTTCGTCT TCACCTCGA</td>
<td style="text-align: left;">Forward sequencing primer for 25x+11-YFP</td>
</tr>
<tr class="odd">
<td style="text-align: left;">YFP1</td>
<td style="text-align: left;">ACTAGCAACAC CAGAACAGCCC</td>
<td style="text-align: left;">Reverse sequencing primer for 3*1x-lacI and 25x+11-YFP</td>
</tr>
<tr class="even">
<td style="text-align: left;">HG 6.1 (<em>galK</em>)</td>
<td style="text-align: left;">gtttgcgcgc agtcagcgat atccattttc gcgaatccg gagtgtaag aaACTAGCAAC ACCAGAACA GCC</td>
<td style="text-align: left;">Reverse primer for 25x+11-YFP integration in to the <em>galK</em> locus (lowercase).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">HG 6.3 (<em>galK</em>)</td>
<td style="text-align: left;">ttcatattgt tcagcgacag cttgctgtac ggcaggcac cagctcttc cgGGCTAATGC ACCCAGTAA GG</td>
<td style="text-align: left;">Forward integration primer for 25x+11-YFP with homology to the <em>galK</em> locus (lowercase.</td>
</tr>
<tr class="even">
<td style="text-align: left;">HG11.1 (<em>ybcN</em>)</td>
<td style="text-align: left;">acctctgcgg aggggaagcg tgaacctctc acaagacgg catcaaatt acACTAGCAAC ACCAGAACA GCC</td>
<td style="text-align: left;">Reverse integration primer for 3*1x-lacI with homology to the <em>ybcN</em> locus (lowercase).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">HG11.3 (<em>ybcN</em>)</td>
<td style="text-align: left;">ctgtagatgtg tccgttcatg acacgaataa gcggtgtag ccattacgc cGGCTAATGCA CCCAGTAAG G</td>
<td style="text-align: left;">Forward integration primer for 3*1x-lacI with homology to the <em>ybcN</em> locus (lowercase).</td>
</tr>
<tr class="even">
<td style="text-align: left;">ybcN-control-upstream-1</td>
<td style="text-align: left;">AGCGTTTGA CCTCTGCGGA</td>
<td style="text-align: left;">Sequencing primer to verify integration</td>
</tr>
<tr class="odd">
<td style="text-align: left;">ybcN-control-downstream-1</td>
<td style="text-align: left;">GCTCAGGTT TACGCTTAC GACG</td>
<td style="text-align: left;">Sequencing primer to verify integration</td>
</tr>
</tbody>
</table>
<table>
<caption><em>E. coli</em> strains used in this work.</caption>
<thead>
<tr class="header">
<th style="text-align: left;"><strong>Strain</strong></th>
<th style="text-align: left;"><strong>Genotype</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">O1, <span class="math inline"><em>R</em> = 0</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O1+11-YFP</td>
</tr>
<tr class="even">
<td style="text-align: left;">O1, <span class="math inline"><em>R</em> = 22</span></td>
<td style="text-align: left;">HG104::<em>galK</em>&lt;&gt;25O1+11-YFP</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O1, <span class="math inline"><em>R</em> = 60</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O1+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1147-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">O1, <span class="math inline"><em>R</em> = 124</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O1+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS446-lacI</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O1, <span class="math inline"><em>R</em> = 260</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O1+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1027-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">O1, <span class="math inline"><em>R</em> = 1220</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O1+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1-lacI</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O1, <span class="math inline"><em>R</em> = 1740</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O1+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1L-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">O2, <span class="math inline"><em>R</em> = 0</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O2+11-YFP</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O2, <span class="math inline"><em>R</em> = 22</span></td>
<td style="text-align: left;">HG104::<em>galK</em>&lt;&gt;25O2+11-YFP</td>
</tr>
<tr class="even">
<td style="text-align: left;">O2, <span class="math inline"><em>R</em> = 60</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O2+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1147-lacI</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O2, <span class="math inline"><em>R</em> = 124</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O2+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS446-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">O2, <span class="math inline"><em>R</em> = 260</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O2+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1027-lacI</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O2, <span class="math inline"><em>R</em> = 1220</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O2+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">O2, <span class="math inline"><em>R</em> = 1740</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O2+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1L-lacI</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O3, <span class="math inline"><em>R</em> = 0</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O3+11-YFP</td>
</tr>
<tr class="even">
<td style="text-align: left;">O3, <span class="math inline"><em>R</em> = 22</span></td>
<td style="text-align: left;">HG104::<em>galK</em>&lt;&gt;25O3+11-YFP</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O3, <span class="math inline"><em>R</em> = 60</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O3+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1147-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">O3, <span class="math inline"><em>R</em> = 124</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O3+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS446-lacI</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O3, <span class="math inline"><em>R</em> = 260</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O3+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1027-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">O3, <span class="math inline"><em>R</em> = 1220</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O3+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1-lacI</td>
</tr>
<tr class="odd">
<td style="text-align: left;">O3, <span class="math inline"><em>R</em> = 1740</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25O3+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1L-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">Oid, <span class="math inline"><em>R</em> = 22</span></td>
<td style="text-align: left;">HG104::<em>galK</em>&lt;&gt;25Oid+11-YFP</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Oid, <span class="math inline"><em>R</em> = 60</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25Oid+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS1147-lacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">Oid, <span class="math inline"><em>R</em> = 124</span></td>
<td style="text-align: left;">HG105::<em>galK</em>&lt;&gt;25Oid+11-YFP, <em>ybcN</em>&lt;&gt;3*1RBS446-lacI</td>
</tr>
</tbody>
</table>
<h2 class="unnumbered" id="references" class="unnumbered">References</h2>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-aghaeepour2013">
<p>Aghaeepour, Nima, Greg Finak, Holger Hoos, Tim R. Mosmann, Ryan Brinkman, Raphael Gottardo, and Richard H. Scheuermann. 2013. “Critical Assessment of Automated Flow Cytometry Data Analysis Techniques.” <em>Nature Methods</em> 10 (3): 228–38. <a href="https://doi.org/10.1038/nmeth.2365">https://doi.org/10.1038/nmeth.2365</a>.</p>
</div>
<div id="ref-bintu2005">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Models.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 116–24. <a href="https://doi.org/10.1016/j.gde.2005.02.007">https://doi.org/10.1016/j.gde.2005.02.007</a>.</p>
</div>
<div id="ref-boulton2016">
<p>Boulton, Stephen, and Giuseppe Melacini. 2016. “Advances in NMR Methods to Map Allosteric Sites: From Models to Translation.” <em>Chemical Reviews</em> 116 (11): 6267–6304. <a href="https://doi.org/10.1021/acs.chemrev.5b00718">https://doi.org/10.1021/acs.chemrev.5b00718</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates the Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-daber2009a">
<p>Daber, Robert, Kim Sharp, and Mitchell Lewis. 2009. “One Is Not Enough.” <em>Journal of Molecular Biology</em> 392 (5): 1133–44. <a href="https://doi.org/10.1016/j.jmb.2009.07.050">https://doi.org/10.1016/j.jmb.2009.07.050</a>.</p>
</div>
<div id="ref-daber2011">
<p>Daber, Robert, Matthew A. Sochor, and Mitchell Lewis. 2011. “Thermodynamic Analysis of Mutant Lac Repressors.” <em>Journal of Molecular Biology</em>, The Operon Model and its Impact on Modern Molecular Biology, 409 (1): 76–87. <a href="https://doi.org/10.1016/j.jmb.2011.03.057">https://doi.org/10.1016/j.jmb.2011.03.057</a>.</p>
</div>
<div id="ref-edelstein2014">
<p>Edelstein, Arthur D, Mark A Tsuchida, Nenad Amodaj, Henry Pinkard, Ronald D Vale, and Nico Stuurman. 2014. “Advanced Methods of Microscope Control Using <span class="math inline"><em>μ</em></span>Manager Software.” <em>Journal of Biological Methods</em> 1 (2): 10. <a href="https://doi.org/10.14440/jbm.2014.36">https://doi.org/10.14440/jbm.2014.36</a>.</p>
</div>
<div id="ref-garcia2011b">
<p>Garcia, Hernan G., Heun Jin Lee, James Q. Boedicker, and Rob Phillips. 2011a. “Comparison and Calibration of Different Reporters for Quantitative Analysis of Gene Expression.” <em>Biophysical Journal</em> 101 (3): 535–44. <a href="https://doi.org/10.1016/j.bpj.2011.06.026">https://doi.org/10.1016/j.bpj.2011.06.026</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011b. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-gardino2003">
<p>Gardino, Alexandra K., Brian F. Volkman, Ho S. Cho, Seok-Yong Lee, David E. Wemmer, and Dorothee Kern. 2003. “The NMR Solution Structure of BeF<sub>3</sub>–Activated Spo0F Reveals the Conformational Switch in a Phsophorelay System.” <em>Journal of Molecular Biology</em> 331 (1): 245–54. <a href="https://doi.org/10.1016/S0022-2836(03)00733-2">https://doi.org/10.1016/S0022-2836(03)00733-2</a>.</p>
</div>
<div id="ref-lutz1997">
<p>Lutz, Rolf, and Hermann Bujard. 1997. “Independent and Tight Regulation of Transcriptional Units in <em>Escherichia Coli</em> via the LacR/O, the TetR/O and AraC/I1-I2 Regulatory Elements.” <em>Nucleic Acids Research</em> 25 (6): 1203–10. <a href="https://doi.org/10.1093/nar/25.6.1203">https://doi.org/10.1093/nar/25.6.1203</a>.</p>
</div>
<div id="ref-marzen2013">
<p>Marzen, Sarah, Hernan G. Garcia, and Rob Phillips. 2013. “Statistical Mechanics of MonodWymanChangeux (MWC) Models.” <em>Journal of Molecular Biology</em>, Allosteric Interactions and Biological Regulation (Part I), 425 (9): 1433–60. <a href="https://doi.org/10.1016/j.jmb.2013.03.013">https://doi.org/10.1016/j.jmb.2013.03.013</a>.</p>
</div>
<div id="ref-oehler1994">
<p>Oehler, S., M. Amouyal, P. Kolkhof, B. von Wilcken-Bergmann, and B. Müller-Hill. 1994. “Quality and Position of the Three Lac Operators of <em>E. Coli</em> Define Efficiency of Repression.” <em>The EMBO Journal</em> 13 (14): 3348–55.</p>
</div>
<div id="ref-rydenfelt2014">
<p>Rydenfelt, Mattias, Hernan G. Garcia, Robert Sidney Cox, and Rob Phillips. 2014. “The Influence of Promoter Architectures and Regulatory Motifs on Gene Expression in <em>Escherichia Coli</em>.” Edited by Jordi Garcia-Ojalvo. <em>PLOS ONE</em> 9 (12): e114347. <a href="https://doi.org/10.1371/journal.pone.0114347">https://doi.org/10.1371/journal.pone.0114347</a>.</p>
</div>
<div id="ref-schmidt2016">
<p>Schmidt, Alexander, Karl Kochanowski, Silke Vedelaar, Erik Ahrné, Benjamin Volkmer, Luciano Callipo, Kèvin Knoops, Manuel Bauer, Ruedi Aebersold, and Matthias Heinemann. 2016. “The Quantitative and Condition-Dependent <em>Escherichia Coli</em> Proteome.” <em>Nature Biotechnology</em> 34 (1): 104–10. <a href="https://doi.org/10.1038/nbt.3418">https://doi.org/10.1038/nbt.3418</a>.</p>
</div>
<div id="ref-setty2003">
<p>Setty, Y., A. E. Mayo, M. G. Surette, and U. Alon. 2003. “Detailed Map of a Cis-Regulatory Input Function.” <em>Proceedings of the National Academy of Sciences</em> 100 (13): 7702–7. <a href="https://doi.org/10.1073/pnas.1230759100">https://doi.org/10.1073/pnas.1230759100</a>.</p>
</div>
<div id="ref-sharan2009">
<p>Sharan, S. K., L. C. Thomason, S. G. Kuznetsov, and D. L. Court. 2009. “Recombineering: A Homologous Recombination-Based Method of Genetic Engineering.” <em>Nature Protocols</em> 4: 206–23. <a href="https://doi.org/nprot.2008.227\\\%0020%5Bpii%5D\\\%002010.1038/nprot.2008.227">https://doi.org/nprot.2008.227\\\%0020[pii]\\\%002010.1038/nprot.2008.227</a>.</p>
</div>
<div id="ref-shis2014">
<p>Shis, David L., Faiza Hussain, Sarah Meinhardt, Liskin Swint-Kruse, and Matthew R. Bennett. 2014. “Modular, Multi-Input Transcriptional Logic Gating with Orthogonal LacI/GalR Family Chimeras.” <em>ACS Synthetic Biology</em> 3 (9): 645–51. <a href="https://doi.org/10.1021/sb500262f">https://doi.org/10.1021/sb500262f</a>.</p>
</div>
<div id="ref-sivia2006">
<p>Sivia, Devinderjit, and John Skilling. 2006. <em>Data Analysis: A Bayesian Tutorial</em>. OUP Oxford.</p>
</div>
<div id="ref-sochor2014">
<p>Sochor, Matthew Almond. 2014. “In Vitro Transcription Accurately Predicts Lac Repressor Phenotype <em>in Vivo</em> in <em>Escherichia Coli</em>.” <em>PeerJ</em> 2 (July): e498. <a href="https://doi.org/10.7717/peerj.498">https://doi.org/10.7717/peerj.498</a>.</p>
</div>
<div id="ref-vilar2013">
<p>Vilar, Jose M. G., and Leonor Saiz. 2013. “Reliable Prediction of Complex Phenotypes from a Modular Design in Free Energy Space: An Extensive Exploration of the Lac Operon.” <em>ACS Synthetic Biology</em> 2 (10): 576–86. <a href="https://doi.org/10.1021/sb400013w">https://doi.org/10.1021/sb400013w</a>.</p>
</div>
<div id="ref-weinert2014">
<p>Weinert, Franz M., Robert C. Brewster, Mattias Rydenfelt, Rob Phillips, and Willem K. Kegel. 2014. “Scaling of Gene Expression with Transcription-Factor Fugacity.” <em>Physical Review Letters</em> 113 (25). <a href="https://doi.org/10.1103/PhysRevLett.113.258101">https://doi.org/10.1103/PhysRevLett.113.258101</a>.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/chapter_05" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter V
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_07" class="page-nav__item page-nav__item--next">
										Chapter VII
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd//" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>


<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-109526846-4', 'auto');
	ga('send', 'pageview');
</script>



		</div><!-- /.js-footer-area -->
	</body>
</html>
