<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter IV</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd//favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd//doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd//doks-theme/assets/images/prismatic.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter IV</h2>
								
								
									<h1 class="hero-subheader__title">On The Physiological Adaptability of a Simple Genetic Circuit 
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phd/chapter_03" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phd/chapter_05" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
						<div class="col-md-5 col-md-offset-0 hidden-xs hidden-sm">
							<div class="align-container" data-mh>
								<div class="align-inner">
									<div class="hero-subheader__author">
										<p class="hero-subheader__author-title">
												Summary
												<i class="icon icon--chevron-down"></i>
										</p><!-- /.hero-subheader__author-title -->
										<p>Cells adapt to changing environmental conditions by repressing or activating gene expression from enormous fractions of their genome, drastically changing the molecular composition of the cell. This requires the concerted adaptation of transcription factors to the environmental signals, leading to binding or releasing of their cognate sequences. Here, we dissect a well characterized genetic circuit in a number of physiological states, make predictions of the response, and measure how the copy number of a regulator and its gene target are affected. We find the parameters defining the regulators behavior are remarkably robust to changes in the nutrient availability, but are  susceptible to temperature changes. We quantitatively explore these two effects and discuss how they challenge current models of transcriptional regulation.
</p>
									</div><!-- /.hero-subheader__author -->
								</div><!-- /.align-inner -->
							</div><!-- /.align-container -->
						</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/chapter_03" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter III
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_05" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<strong>Published as …</strong>
<hr/>
<p>A version of this chapter is currently under review. A preprint is released as Chure, G, Kaczmarek, Z. A., and Phillips, R. Physiological adaptability and parametric versatility in a simple genetic circuit. bioRxiv 2019. DOI: 10.1101/2019.12.19.878462. G.C. and R.P. designed experiments and developed theoretical models. G.C. and Z.A.K. collected and analyzed data. G.C. and R.P. wrote the paper.</p>
<h2 id="abstract">Abstract</h2>
<p>The intimate relationship between the environment and cellular growth rate has remained a major topic of inquiry in bacterial physiology for over a century. Now, as it becomes possible to understand how the growth rate dictates the wholesale reorganization of the intracellular molecular composition, we can interrogate the biophysical principles underlying this adaptive response. Regulation of gene expression drives this adaptation, with changes in growth rate tied to the activation or repression of genes covering enormous swaths of the genome. Here, we dissect how physiological perturbations alter the expression of a circuit which has been extensively characterized in a single physiological state. Given a complete thermodynamic model, we map changes in physiology directly to the biophysical parameters which define the expression. Controlling the growth rate via modulating the available carbon source or growth temperature, we measure the level of gene expression from a LacI-regulated promoter where the LacI copy number is directly measured in each condition, permitting parameter-free prediction of the expression level. The transcriptional output of this circuit is remarkably robust, with expression of the repressor being largely insensitive to the growth rate. The predicted gene expression quantitatively captures the observations under different carbon conditions, indicating that the biophysical parameters are indifferent to the physiology. Interestingly, temperature controls the expression level in ways that are inconsistent with the prediction, revealing temperature-dependent effects that challenge current models. This work exposes the strengths and weaknesses of thermodynamic models in fluctuating environments, posing novel challenges and utility in studying physiological adaptation.</p>
<h2 id="introduction">Introduction</h2>
<p>Cellular physiology is inextricably tied to the extracellular environment. Fluctuations in nutrient availability and variations in temperature, for example, can drastically modulate the cell’s growth rate, which is often used as a measure of the evolutionary fitness <span class="citation" data-cites="schaechter1958">(Schaechter, Maaløe, and Kjeldgaard <a href="#ref-schaechter1958" role="doc-biblioref">1958</a>)</span>. In response to such environmental insults, cells have evolved myriad clever mechanisms by which they can adapt to their changing surroundings, many of which involve restructuring their proteome such that critical processes (i.e. protein translation) are allocated the necessary resources. Recent work exploring this level of adaptation using mass spectrometry, ribosomal profiling, and RNA sequencing have revealed that various classes of genes (termed “sectors”) are tuned such that the protein mass fraction of the translational machinery is prioritized over the metabolic and catabolic machinery in nutrient replete environments <span class="citation" data-cites="scott2014 klumpp2014 hui2015 schmidt2016 li2014">(Scott et al. <a href="#ref-scott2014" role="doc-biblioref">2014</a>; Klumpp and Hwa <a href="#ref-klumpp2014" role="doc-biblioref">2014</a>; Hui et al. <a href="#ref-hui2015" role="doc-biblioref">2015</a>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>)</span>. This drastic reorganization is mediated by the regulation of gene expression, relying on the concerted action of myriad transcription factors. Notably, each gene in isolation is regulated by only one or a few components <span class="citation" data-cites="gama-castro2016">(Gama-Castro et al. <a href="#ref-gama-castro2016" role="doc-biblioref">2016</a>)</span>. The most common regulatory architecture in <em>Escherichia coli</em> is the simple repression motif in which a transcriptional repressor binds to a single site in the promoter region, occluding binding of an RNA polymerase <span class="citation" data-cites="rydenfelt2014 phillips2019">(Rydenfelt et al. <a href="#ref-rydenfelt2014" role="doc-biblioref">2014</a>; Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span>. The simple activation architecture, in which the simultaneous binding of an activator and an RNA polymerase amplifies gene expression, is another common mode of regulation. Combinatorial regulation such as dual repression, dual activation, or combined activation and repression can also be found throughout the genome, albeit with lower frequency <span class="citation" data-cites="phillips2019">(Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span>. The ubiquity of the simple repression and simple activation motifs illustrate that, for many genes, the complex systems-level response to a physiological perturbation boils down the binding and unbinding of a single regulator to its cognate binding sites.</p>
<p>     Despite our knowledge of these modes of regulation, there remains a large disconnect between concrete, physical models of their behavior and experimental validation. The simple repression motif is perhaps the most thoroughly explored theoretically and experimentally <span class="citation" data-cites="phillips2019">(Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span> where equilibrium thermodynamic <span class="citation" data-cites="garcia2011 garcia2012 brewster2014 razo-mejia2018 barnes2019">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Garcia et al. <a href="#ref-garcia2012" role="doc-biblioref">2012</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Barnes et al. <a href="#ref-barnes2019" role="doc-biblioref">2019</a>)</span> and kinetic <span class="citation" data-cites="jones2014 ko1991 kepler2001 michel2010">(Jones, Brewster, and Phillips <a href="#ref-jones2014" role="doc-biblioref">2014</a>; Ko <a href="#ref-ko1991" role="doc-biblioref">1991</a>; Kepler and Elston <a href="#ref-kepler2001" role="doc-biblioref">2001</a>; Michel <a href="#ref-michel2010" role="doc-biblioref">2010</a>)</span> models have been shown to accurately predict the level of gene expression in a variety of contexts. While these experiments involved variations of repressor copy number, operator sequence, concentration of an external inducer, and amino acid substitutions, none have explored how the physiological state of the cell as governed by external factors influences gene expression. This is arguably one of the most critical variables one can experimentally tune to understand the roles these regulatory architectures play in cellular physiology writ large.</p>
<p>     In this work, we interrogate the adaptability of a simple genetic circuit to various physiological stressors, namely carbon source quality and growth temperature. Following the aforementioned thermodynamic models, we build upon this theory-experiment dialogue by using environmental conditions as an experimentally tunable variable and determine their influence on various biophysical parameters. Specifically, we use physiological stressors to tune the growth rate. One mechanism by which we modulate the growth rate is by exchanging glucose in the growth medium for the poorer carbon sources glycerol and acetate, which decrease the growth rate by a factor of <span class="math inline">≈</span> 1.5 and <span class="math inline">≈</span> 4 compared to glucose, respectively. We hypothesize that different carbon sources should, if anything, only modulate the repressor copy number seeing as the relationship between growth rate and total protein content has been rigorously quantified <span class="citation" data-cites="schaechter1958 schmidt2016 li2014 jun2018">(Schaechter, Maaløe, and Kjeldgaard <a href="#ref-schaechter1958" role="doc-biblioref">1958</a>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>; Jun et al. <a href="#ref-jun2018" role="doc-biblioref">2018</a>)</span>. Using single-cell time-lapse fluorescence microscopy, we directly measure the copy number of the repressor in each condition. Under a simple hypothesis, all other parameters should be unperturbed, and we can thus rely on previously determined values to make parameter-free predictions of the fold-change in gene expression.</p>
<p>     Despite the decrease in growth rate, both the fold-change in gene expression and the repressor copy number remains largely unaffected. We confirm this is the case by examining how the effective free energy of the system changes between carbon sources, a method we have used previously to elucidate parametric changes due to mutations within a transcription factor <span class="citation" data-cites="chure2019">(Chure et al. <a href="#ref-chure2019" role="doc-biblioref">2019</a>)</span> and has been extensively discussed in Chapter 3. This illustrates that the energetic parameters defining the fraction of active repressors and their affinity for the DNA are ignorant of the carbon-dependent physiological states of the cell. Thus, in this context, the values of the biophysical parameters determined in one condition can be used to draw predictions in others.</p>
<p>     We then examine how variations in temperature influence the transcriptional output. Unlike in the case of carbon source variation, temperature dependence is explicit in our model: the repressor-DNA binding energy and the energetic difference between the active and inactive states of the repressor are scaled to the thermal energy of the system at 37<span class="math inline"><em></em><sup>∘</sup></span> C. This is defined via the Boltzmann distribution which states that the probability of a state <span class="math inline"><em>p</em><sub><em>s</em><em>t</em><em>a</em><em>t</em><em>e</em></sub></span> is related to the energy of that state <span class="math inline"><em>ε</em><sub><em>s</em><em>t</em><em>a</em><em>t</em><em>e</em></sub></span> as <span id="eq:boltz"><br /><span class="math display"><em>p</em><sub><em>s</em><em>t</em><em>a</em><em>t</em><em>e</em></sub> ∝ <em>e</em><sup> − <em>ε</em><sub><em>s</em><em>t</em><em>a</em><em>t</em><em>e</em></sub>/<em>k</em><sub><em>B</em></sub><em>T</em></sup>,   (1)</span><br /></span> where <span class="math inline"><em>k</em><sub><em>B</em></sub></span> is the Boltzmann constant and <span class="math inline"><em>T</em></span> is the temperature of the system. Given knowledge of <span class="math inline"><em>T</em></span> for a particular experiment, we can easily draw predictions of the fold-change in gene expression. However, we find that the fold-change in gene expression is inconsistent with this simple model, revealing an incomplete description of the energetics. We then examine how entropic effects neglected in the initial estimation of the energetic parameters may play an important role; a hypothesis that is supported when we examine the change in the effective free energy.</p>
<p>     The results presented here are, to our knowledge, the first attempts to systematically characterize the growth-dependent effects on biophysical parameters in thermodynamic models of transcription. While some parameters of our model are affected by changing the growth rate, they change in ways that are expected or fall close within our <em>a priori</em> predictions, suggesting that such modeling can still be powerful in understanding how adaptive processes influence physiology at the level of molecular interactions.</p>
<h2 id="results">Results</h2>
<h3 id="thermodynamic-model">Thermodynamic model</h3>
<p>This chapter builds off the theoretical details presented in Chapters 2-3 of this thesis. Here, we once again consider the simple repression motif in which expression of a target gene is regulated by the action of a single allosteric repressor. The key measurable quantity of the following work is the fold-change in expression of the regulated gene. Thermodynamic models described previously <span class="citation" data-cites="garcia2011 razo-mejia2018 phillips2019">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Phillips et al. <a href="#ref-phillips2019" role="doc-biblioref">2019</a>)</span> and in Chapters 2-3 of this work result in a succinct input-output function to quantitatively describe the fold-change in gene expression and is of the form <span id="eq:foldchange_simple"><br /><span class="math display">$$
\text{fold-change} = \left(1 + p_{act}(c)\frac{R}{N_{NS}}e^{-\Delta\varepsilon_{RA}
/ k_BT}\right)^{-1},
\qquad(2)$$</span><br /></span> where <span class="math inline"><em>R</em></span> is the total number of allosteric repressors per cell, <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of nonspecific binding sites for the repressor, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is the repressor-DNA binding energy, and <span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span> is the thermal energy of the system. The prefactor <span class="math inline"><em>p</em><sub><em>a</em><em>c</em><em>t</em></sub>(<em>c</em>)</span> defines the probability of the repressor being in the active state at a given concentration of inducer <span class="math inline"><em>c</em></span>. In the absence of inducer, <span class="math inline"><em>p</em><sub><em>a</em><em>c</em><em>t</em></sub>(<em>c</em> = 0)</span> can be written as <span id="eq:growth_pact"><br /><span class="math display"><em>p</em><sub><em>a</em><em>c</em><em>t</em></sub>(<em>c</em> = 0) = (1+<em>e</em><sup> − <em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub>/<em>k</em><sub><em>B</em></sub><em>T</em></sup>)<sup> − 1</sup>,   (3)</span><br /></span> where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the energy difference between the active and inactive states. Conditioned on only a handful of experimentally accessible parameters, this model has been verified using the well-characterized LacI repressor of <em>Escherichia coli</em> where parameters such as the repressor copy number and DNA binding affinity <span class="citation" data-cites="garcia2011">(Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>, copy number of the regulated promoter <span class="citation" data-cites="brewster2014">(Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>, and the concentration of an extracellular inducer <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>, Chapter 2)</span> can be tuned over orders of magnitude. Chapter 3 and the associated publication <span class="citation" data-cites="chure2019">(Chure et al. <a href="#ref-chure2019" role="doc-biblioref">2019</a>)</span> illustrated that this model permits the mapping of mutations within the repressor protein directly to biophysical parameters in a manner that permits accurate prediction of double mutant phenotypes. All of these applications, however, have been performed in a single physiological state where cells are grown in a glucose-supplemented minimal medium held at 37<span class="math inline"><em></em><sup>∘</sup></span> C with aeration. In this work, we challenge this model by changing the environmental conditions away from this gold-standard condition, perturbing the physiological state of the cell.</p>
<h3 id="experimental-setup">Experimental Setup</h3>
<p>Seminal studies from the burgeoning years of bacterial physiology have demonstrated a strong dependence of the total cellular protein content on the growth rate <span class="citation" data-cites="schaechter1958">(Schaechter, Maaløe, and Kjeldgaard <a href="#ref-schaechter1958" role="doc-biblioref">1958</a>)</span>, a relationship which has been rigorously quantified in recent years using mass spectrometry <span class="citation" data-cites="schmidt2016">(Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>)</span> and ribosomal profiling <span class="citation" data-cites="li2014">(Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>)</span>. Their combined results illustrate that modulation of the growth rate, either through controlling the available carbon source or the temperature of the growth medium, significantly alters the physiological state of the cell, triggering the reallocation of resources to prioritize expression of ribosome-associated genes. Eq. 2 has no explicit dependence on the available carbon source, but does depend on the temperature through the energetic parameters <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> which are defined relative to the thermal energy, <span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span>. With this parametric knowledge, we are able to draw quantitative predictions of the fold-change in gene expression in these physiologically distinct states.</p>
<p>     We modulated growth of <em>Escherichia coli</em> by varying either the quality of the available carbon source (differing ATP yield per C atom) or the temperature of the growth medium (Fig. 1 (A)). All experiments were performed in a defined M9 minimal medium supplemented with one of three carbon sources – glucose, glycerol, or acetate – at concentrations such that the total number of carbon atoms available to the culture remained the same. These carbon sources have been shown to drastically alter growth rate and gene expression profiles, indicating changes in the proteomic composition and distinct physiological states. These carbon sources yield an approximate four-fold modulation of the growth rate with doubling times ranging from <span class="math inline">≈</span> 220 minutes to <span class="math inline">≈</span> 65 minutes in an acetate or glucose supplemented medium, respectively Fig. 1 (B) and (C). While the growth temperature was varied over 10<span class="math inline"><em></em><sup>∘</sup></span> C, both 32<span class="math inline"><em></em><sup>∘</sup></span> and 42<span class="math inline"><em></em><sup>∘</sup></span> C result in approximately the same doubling time of <span class="math inline">≈</span> 90 min, which is 1.5 times slower than the optimal temperature of 37<span class="math inline"><em></em><sup>∘</sup></span> C (Fig. 1 (B) and (C)).</p>
<figure>
<img src="ch4_fig1.png" id="fig:growth_control" data-short-caption="Controlling physiological state of *E. coli* via growth rate modulation by environmental factors." alt="" /><figcaption>Figure 1: <strong>Control of growth rate via environmental factors.</strong> (A) Bacterial growth can be controlled by varying the available carbon source (top panel) or temperature (bottom panel). (B) Bulk bacterial growth curves under all conditions illustrated in (A). The <span class="math inline"><em>y</em></span>-axis is the optical density measurements at 600 nm relative to the initial value. Interval between points is <span class="math inline">≈</span> 6 min. Points and errors represent the mean and standard deviation of three to eight biological replicates. (C) Inferred maximum growth rate of each biological replicate for each condition. Points represent the doubling time computed from the maximum growth rate. Error bars correspond to the standard deviation of the inferred growth rate. Where not visible, error bars are smaller than the marker. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_04/code/ch4_fig1.py">Python code (<code>ch4_fig1.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     The growth rate dependence of the proteome composition suggests that changing physiological conditions could change the total repressor copy number of the cell. As emphasized in Chapter 2, it can be difficult to differentiate between a change in repressor copy number <span class="math inline"><em>R</em></span> and the allosteric energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> as there are many combinations of parameter values that yield the same fold-change. To combat this degeneracy, we used a genetically engineered strain of <em>E. coli</em> in which the expression of the repressor copy number and its regulated gene product (YFP) can be simultaneously measured. This strain, used previously to interrogate the transcription factor titration effect <span class="citation" data-cites="brewster2014">(Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>, is diagrammed in Fig. 2 (A). A dimeric form of the LacI repressor N-terminally tagged with an mCherry fluorophore is itself regulated through the action of the TetR repressor whose level of activity can be modulated through the addition of the allosteric effector anhydrous tetracycline (ATC). This dual repression genetic circuit allows for the expression of the LacI repressor to be tuned over several orders of magnitude. This is demonstrated in Fig. 2 (B) where a titration of ATC in the growth medium results in a steady increase in the expression of the LacI-mCherry gene product (red lines and points) which in turn represses expression of the YFP reporter (yellow lines and points).</p>
<p>     While the mCherry fluorescence is proportional to the repressor copy number, it is not a direct measurement as the fluorescence of a single LacI-mCherry dimer is unknown <em>a priori</em>. Using video microscopy, we measure the partitioning statistics of the fluorescence intensity into two sibling cells after division (Fig. 2 (C)). This method, described in detail in the Materials &amp; Methods as well as in <span class="citation" data-cites="rosenfeld2002">Rosenfeld, Elowitz, and Alon (<a href="#ref-rosenfeld2002" role="doc-biblioref">2002</a>)</span>, <span class="citation" data-cites="rosenfeld2005">Rosenfeld et al. (<a href="#ref-rosenfeld2005" role="doc-biblioref">2005</a>)</span>, and <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>, reveals a linear relationship between the variance in intensity between two sibling cells and the intensity of the parent cell, the slope of which is equal to the brightness of a single LacI repressor. Since this measurement is performed simultaneously with measurement of the expression of the YFP reporter, this calibration factor was determined for each unique experimental replicate. We direct the reader to the supplemental Chapter 8 for a more thorough discussion of this inference.</p>
<figure>
<img src="ch4_fig2.png" id="fig:dilution_circuit" data-short-caption="Control and quantification of repressor copy number through the binomial partitioning method." alt="" /><figcaption>Figure 2: <strong>Control and quantification of repressor copy number.</strong> (A) The dual repression expression system. The inducible repressor TetR (purple blob) is expressed from a low-copy-number plasmid in the cell and represses expression of the LacI-mCherry repressor by binding to its cognate operator (tetO). In the presence of anhydrous tetracycline (ATC, green sphere), the inactive state of TetR becomes energetically favorable, permitting expression of the LacI-mCherry construct (red). This in turn binds to the lacO operator sequence repressing the expression of the reporter Yellow Fluorescent Protein (YFP, yellow lightbulb). (B) An ATC titration curve showing anticorrelated YFP (yellow) and mCherry (red) intensities. Reported values are scaled to the maximum mean fluorescence for each channel. Points and errors correspond to the mean and standard error of eight biological replicates. (C) Determination of a fluorescence calibration factor. After cessation of LacI-mCherry expression, cells are allowed to divide, partitioning the fluorescently tagged LacI repressors into the two daughter cells (left panel). The total intensity of the parent cell is equivalent to the summed intensities of the daughters. The squared fluctuations in intensity of the two sibling cells is linearly related to the parent cell with a slope <span class="math inline"><em>α</em></span>, which is the fluorescence signal measured per partitioned repressor (right panel). Black points represent single divisions and red points are the means of 50 division events. Line corresponds to linear fit to the black points with a slope of <span class="math inline"><em>α</em> = 740 ± 40</span> a.u. per LacI. The Python codes <a href="https://github.com/gchure/phd/blob/master/src/chapter_04/code/ch4_fig2b.py">(<code>ch4_fig2b.py</code>)</a> and <a href="https://github.com/gchure/phd/blob/master/src/chapter_04/code/ch4_fig2c.py">(<code>ch4_fig2c.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="scaling-of-gene-expression-with-growth-rate">Scaling of Gene Expression With Growth Rate</h3>
<p>Given the single-cell resolution of our experimental method, we examined how the cell volume and repressor copy number scaled across the different growth conditions at different levels of ATC induction. In agreement with the literature <span class="citation" data-cites="schaechter1958 jun2018 shehata1975">(Schaechter, Maaløe, and Kjeldgaard <a href="#ref-schaechter1958" role="doc-biblioref">1958</a>; Jun et al. <a href="#ref-jun2018" role="doc-biblioref">2018</a>; Shehata and Marr <a href="#ref-shehata1975" role="doc-biblioref">1975</a>)</span>, our measurement reveals a strong linear dependence of the cell volume on the choice of carbon source, but no significant dependence on temperature (Fig. 3 (A) and (B)). Additionally, these findings are consistent across different ATC induction regimes. Together, these observations confirm that the particular details of our experimental system does not introduce unintended physiological consequences.</p>
<p>     Using a fluorescence calibration factor determined for each experimental replicate (see Fig. 2 (C) and the Materials &amp; Methods), we estimated the number of repressors per cell from snapshots of the mCherry signal intensity of each induction condition. Fig. 3 (C) reveals a remarkable insensitivity of the repressor copy number on the growth rate under different carbon sources. Despite the change in cellular volume, the mean number of repressors expressed at a given induction condition is within error between all carbon sources. Previous work using mass spectrometry, a higher resolution method, has shown that there is a slight dependence of LacI copy number on growth rate expressed from its native promoter <span class="citation" data-cites="schmidt2016">(Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>)</span>. It is possible that such a dependence exists in our experimental setup, but is not detectable with our lower resolution method. We also observe an insensitivity of copy number to growth rate when the temperature of the system is tuned (Fig. 3 (D)), though two aberrant points with large error obfuscate the presence of a growth rate dependence at high concentrations of ATC. For concentrations below 7 ng /mL, however, the repressor copy number remains constant across conditions. With no significant change in the repressor copy number and thus no dependence on the carbon source in our theoretical model, we can immediately draw predictions of the fold-change in gene expression in different growth media.</p>
<figure>
<img src="ch4_fig3.png" id="fig:scaling" data-short-caption="Scaling of cel size and repressor expression as a function of maximum growth rate" alt="" /><figcaption>Figure 3: <strong>Scaling of cell size and repressor expression as a function of maximum growth rate.</strong> Dependence of cell volume on maximum growth rate under varying (A) carbon sources and (B) temperatures. Points and errors correspond to the mean and standard errors of five to eight biological replicates. The cell volume was calculated by approximating each cell as a spherocylinder and using measurements of the short-and long axis lengths of each segmentation mask. Measured volumes are from snapshots of a non-synchronously growing culture. The measured repressor copy number for each ATC induction condition (colored lines) as a function of the growth rate for various (C) carbon sources and (D) temperatures. Points and errors represent the mean and standard error of five to eight biological replicates. Colors correspond to the ATC induction concentration ranging from 10 ng/mL (yellow) to 0.1 ng/mL (black).</figcaption>
</figure>
<h2 id="fold-change-dependence-on-temperature-variation">Fold-change Dependence on Temperature Variation</h2>
<p>Unlike the identity of the carbon source, the temperature of the system is explicitly stated in Eq. 2 and Eq. 3 where <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> are defined relative to the thermal energy of the system in which they were determined. This scaling is mathematically quantified as <span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span> dividing the exponentiated terms in Eq. 2 and Eq. 3. As all biophysical parameters were determined at a reference temperature of 37<span class="math inline"><em></em><sup>∘</sup></span> C, any change in the growth temperature must be included as a correction factor. The simplest approach is to rescale the energy by the relative change in temperature. This is a simple multiplicative factor of <span class="math inline"><em>ϕ</em><sub><em>T</em></sub> = <em>k</em><sub><em>B</em></sub><em>T</em><sub><em>r</em><em>e</em><em>f</em></sub>/<em>k</em><sub><em>B</em></sub><em>T</em><sub><em>e</em><em>x</em><em>p</em></sub></span> where <span class="math inline"><em>T</em><sub><em>r</em><em>e</em><em>f</em></sub></span> is the reference temperature of 37<span class="math inline"><em></em><sup>∘</sup></span> C and <span class="math inline"><em>T</em><sub><em>e</em><em>x</em><em>p</em></sub></span> is the experimental temperature. This is an intuitive result since an increase in temperature relative to the reference results in <span class="math inline"><em>ϕ</em><sub><em>T</em></sub> &lt; 1</span>, weakening the binding. Similarly, decreasing the temperature scales <span class="math inline"><em>ϕ</em><sub><em>T</em></sub> &gt; 1</span>, strengthening the binding relative to that of the reference temperature.</p>
<p>     Fig. 4 (A) shows the measured fold-change in gene expression (points) plotted against the theoretical prediction with this correction factor (orange line). It is immediately evident that a simple rescaling of the energetic parameters is not sufficient for the 32<span class="math inline"><em></em><sup>∘</sup></span> C condition and slightly underestimates the fold-change in the 42<span class="math inline"><em></em><sup>∘</sup></span> C condition. To identify the source of this disagreement, we can again examine the free energy shift <span class="math inline"><em>Δ</em><em>F</em></span>. As both <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> are scaled to the thermal energy, <span class="math inline"><em>Δ</em><em>F</em></span> defined as <span class="math inline"><em>F</em><sub><em>T</em></sub> − <em>F</em><sub><em>r</em><em>e</em><em>f</em></sub></span> can be directly calculated as <span id="eq:delF_temp"><br /><span class="math display">$$
\Delta F = k_BT_{exp} \left[ -\log{1 + e^{-{\Delta\varepsilon_{AI} \over
 k_BT_{ref}}} \over 1 + e^{-{\phi_T\Delta\varepsilon_{AI} \over
 k_BT_{exp}}}}- \log{R_{T} \over R_{ref}}\right] +
 \Delta\varepsilon_{R}\left(1 -\phi_T\right).
\qquad(4)$$</span><br /></span> This prediction along with the empirically determined <span class="math inline"><em>Δ</em><em>F</em></span> is shown in Fig. 4 (B). Again, we see that this simple correction factor significantly undershoots or overshoots the observed <span class="math inline"><em>Δ</em><em>F</em></span> for 32<span class="math inline"><em></em><sup>∘</sup></span> C and 42<span class="math inline"><em></em><sup>∘</sup></span> C, respectively, indicating that there are temperature dependent effects that are not accounted for in the simplest null model of temperature dependence.</p>
<p>     The model described by Eq. 2 and Eq. 3 subsumes the myriad rich dynamical processes underlying protein binding and conformational changes into two effective energies, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. By no means is this done to undercut the importance of these details in transcriptional regulation. Rather, it reduces the degrees of freedom in this objectively complex system to the set of the details critical to particular conditions in which we want to draw predictions. All prior dissections of this thermodynamic model have been performed at a single temperature, abrogating the need to consider temperature dependent effects. As we now vary temperature, we must consider details that are swept into the effective energies.</p>
<p>     The model presented here only considers entropy by enumerating the multiplicity of states in which the repressor can bind to the DNA nonspecifically, resulting in terms of the form <span class="math inline"><em>R</em>/<em>N</em><sub><em>N</em><em>S</em></sub></span>. However, there are many other temperature-dependent entropic contributions to the effective energies such as the fraction of repressors bound to DNA versus in solution <span class="citation" data-cites="elf2007 kao-huang1977">(Elf, Li, and Xie <a href="#ref-elf2007" role="doc-biblioref">2007</a>; Kao-Huang et al. <a href="#ref-kao-huang1977" role="doc-biblioref">1977</a>)</span>, the vibrational entropy of the repressor <span class="citation" data-cites="goethe2015">(Goethe, Fita, and Rubi <a href="#ref-goethe2015" role="doc-biblioref">2015</a>)</span>, or conformational entropy of the genome <span class="citation" data-cites="driessen2014 mondal2011">(Driessen et al. <a href="#ref-driessen2014" role="doc-biblioref">2014</a>; Mondal et al. <a href="#ref-mondal2011" role="doc-biblioref">2011</a>)</span>. We can consider the effective energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> as having generic temperature dependent-entropic components <span class="math inline"><em>Δ</em><em>S</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>S</em><sub><em>A</em><em>I</em></sub></span>, <span id="eq:epra_entropy"><br /><span class="math display"><em>Δ</em><em>ε</em><sub><em>R</em></sub> = <em>Δ</em><em>H</em><sub><em>R</em></sub> − <em>T</em><em>Δ</em><em>S</em><sub><em>R</em></sub>,   (5)</span><br /></span> and <span id="eq:epai_entropy"><br /><span class="math display"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = <em>Δ</em><em>H</em><sub><em>A</em><em>I</em></sub> − <em>T</em><em>Δ</em><em>S</em><sub><em>A</em><em>I</em></sub>,   (6)</span><br /></span> where <span class="math inline"><em>Δ</em><em>H</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>H</em><sub><em>A</em><em>I</em></sub></span> is the enthalpic contribution to the energies <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, respectively. Given the fold-change measurements at 32<span class="math inline"><em></em><sup>∘</sup></span> C and 42<span class="math inline"><em></em><sup>∘</sup></span> C, we estimated the entropic parameters <span class="math inline"><em>Δ</em><em>S</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>S</em><sub><em>A</em><em>I</em></sub></span> under the constraints that at 37<span class="math inline"><em></em><sup>∘</sup></span> C, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub> =  − 13.9 <em>k</em><sub><em>B</em></sub><em>T</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span> (see supplemental Chapter 8 for more discussion on this parameter estimation). The grey shaded lines in Fig. 4 show the result of this fit where the width represents the 95% credible region of the prediction given the estimated values of <span class="math inline"><em>Δ</em><em>S</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>S</em><sub><em>A</em><em>I</em></sub></span>. Including this phenomenological treatment of the entropy improves the prediction of the fold-change in gene expression (Fig. 4 (A)) as well as shift in free energy (Fig. 4 (B)). This phenomenological description suggests that even small shifts in temperature can drastically alter the expression of a genetic circuit simply by tuning hidden entropic effects rather than scaling the difference in affinity between specific and nonspecific binding.</p>
<figure>
<img src="ch4_fig5.png" id="fig:deltaF_temp" data-short-caption="Temperature effects on the fold-change in gene expression and free energy." alt="" /><figcaption>Figure 4: <strong>Temperature effects on the fold-change in gene expression and free energy.</strong> (A) The fold-change in gene expression for growth in glucose supplemented medium at 32<span class="math inline"><em></em><sup>∘</sup></span> C (left) and 42<span class="math inline"><em></em><sup>∘</sup></span> C (right). Points and errors correspond to the mean and standard error of five biological replicates. Predictions of the fold-change are shown without correcting for temperature (purple), with multiplicative scaling (orange), and with an entropic penalty (grey). The width of the prediction of the entropic penalty is the 95% credible region. (B) Predicted and observed shifts in the free energy for growth in glucose medium at 32<span class="math inline"><em></em><sup>∘</sup></span> C (left) and 42<span class="math inline"><em></em><sup>∘</sup></span> C (right). Points correspond to the median of the inferred shift in free energy. Vertical error bars indicate the bounds of the 95% credible region. Horizontal position and error corresponds to the mean and standard error for the repressor count over five to eight biological replicates. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_04/code/ch4_fig5.py">Python code (<code>ch4_fig5.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<p>The past century of work in bacterial physiology has revealed a rich molecular complexity that drives cellular growth rate <span class="citation" data-cites="jun2018">(Jun et al. <a href="#ref-jun2018" role="doc-biblioref">2018</a>)</span>. A key finding of this body of work is that the composition of the proteome is highly dependent on the specific growth condition, with entire classes of genes being up- or down-regulated to ensure that enough resources are allocated towards maintaining a pool of active ribosomes <span class="citation" data-cites="hui2015 scott2014">(Hui et al. <a href="#ref-hui2015" role="doc-biblioref">2015</a>; Scott et al. <a href="#ref-scott2014" role="doc-biblioref">2014</a>)</span>. These studies have led to a coarse-grained view of global gene expression where physiological perturbations substantially change the molecular composition of the cell, obfuscating the utility of using thermodynamic models of individual regulatory elements across physiological states. In this work, we rigorously examine how robust the values of the various biophysical parameters are to changes in cellular physiology.</p>
<p>     We first examined how nutrient fluctuations dictate the output of this architecture. We took three carbon sources with distinct metabolic pathways and varying quality and measured the level of gene expression, hypothesizing that the values of the biophysical parameters to be independent of the growth medium. We found that even when the growth rate is varied across a wide range (220 minute doubling time in acetate to 60 minute doubling time in glucose supported medium), there is no significant change to the fold-change in gene expression or in the expression of the transcription factor itself, within the resolution of our experiments. Given numerous quantitative studies of the proteomic composition reveal a dependence on protein content with growth rate <span class="citation" data-cites="hui2015 schmidt2016 li2014">(Hui et al. <a href="#ref-hui2015" role="doc-biblioref">2015</a>; Schmidt et al. <a href="#ref-schmidt2016" role="doc-biblioref">2016</a>; Li et al. <a href="#ref-li2014" role="doc-biblioref">2014</a>)</span>, we find this robustness to be striking.</p>
<p>     <span class="citation" data-cites="schmidt2016">Schmidt et al. (<a href="#ref-schmidt2016" role="doc-biblioref">2016</a>)</span> found that the native expression of LacI has a weak positive correlation with the growth rate. The native LacI promoter region is solely regulated by activation via the cAMP Receptor Protein (CRP), a broadly acting dual regulator in <em>E. coli</em> <span class="citation" data-cites="gama-castro2016">(Gama-Castro et al. <a href="#ref-gama-castro2016" role="doc-biblioref">2016</a>)</span>. This is in contrast to the LacI expression system used in the present work where the promoter is negatively regulated by the TetR repressor, itself expressed from a low-copy number plasmid. Furthermore, the expression of LacI in this work is tuned by the addition of the allosteric effector of TetR, ATC, adding yet another layer of allosteric regulation on LacI expression. The significant difference in the regulatory mechanisms between the native and synthetic circuit used in this work makes the two findings difficult to directly compare. Regardless, our finding that the fold-change in gene expression is unaltered from one carbon source to another illustrates that the values of the biophysical parameters <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> remain unperturbed, permitting quantitative prediction of gene expression across numerous physiological states.</p>
<p>     However, in varying the temperature, we find that the predictive utility of the biophysical parameter values determined at 37<span class="math inline"><em></em><sup>∘</sup></span> C is diminished, indicating that there are hidden effects not explicitly accounted for in our thermodynamic model. The measurements of the fold-change in gene expression are under- or over-estimated when the temperature is increased or decreased, respectively, when one simply rescales the energetic terms by the relative change in temperature. There are many features of transcriptional regulation that are not explicitly considered in our coarse-graining of the architecture into a two-state model. Recently, it has been suggested that the phenomenon of allostery writ large should be framed in the language of an ensemble of states rather than a simple active/inactive distinction <span class="citation" data-cites="motlagh2014">(Motlagh et al. <a href="#ref-motlagh2014" role="doc-biblioref">2014</a>)</span>. While our recent work illustrates that a two-state rendering of an allosteric repressor is highly predictive in a variety of situations <span class="citation" data-cites="razo-mejia2018 chure2019">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>; Chure et al. <a href="#ref-chure2019" role="doc-biblioref">2019</a>)</span>, we must now consider details which are dependent on the temperature of the system. In Fig. 4, we demonstrate that incorporating a temperature-dependent entropic cost to the energetic terms significantly improves the description of the experimental data. This is not to say, however, that this is now an open-and-closed case for what precisely defines this entropic cost. Rather, we conclude that the phenomenology of the temperature dependence can be better described by the inclusion of a correction factor that is linearly dependent on the system temperature. Biology is replete with phenomena which can introduce such an effect, including changes to the material properties of the repressor and DNA <span class="citation" data-cites="goethe2015 mondal2011">(Goethe, Fita, and Rubi <a href="#ref-goethe2015" role="doc-biblioref">2015</a>; Mondal et al. <a href="#ref-mondal2011" role="doc-biblioref">2011</a>)</span>, excluded volume effects <span class="citation" data-cites="driessen2014">(Driessen et al. <a href="#ref-driessen2014" role="doc-biblioref">2014</a>)</span>, and solubilities <span class="citation" data-cites="elf2007 kao-huang1977 yakovchuk2006">(Elf, Li, and Xie <a href="#ref-elf2007" role="doc-biblioref">2007</a>; Kao-Huang et al. <a href="#ref-kao-huang1977" role="doc-biblioref">1977</a>; Yakovchuk, Protozanova, and Frank-Kamenetskii <a href="#ref-yakovchuk2006" role="doc-biblioref">2006</a>)</span>. Understanding the mechanistic underpinnings of temperature dependence in elasticity theory was borne out of similar phenomenological characterization <span class="citation" data-cites="friedel1974">(Friedel <a href="#ref-friedel1974" role="doc-biblioref">1974</a>)</span> and required a significant dialogue between theory and experiment <span class="citation" data-cites="phillips2001">(Phillips <a href="#ref-phillips2001" role="doc-biblioref">2001</a>)</span>. Further work is now needed to develop a theory of temperature effects in the regulation of gene expression.</p>
<p>     The effective free energy <span class="math inline"><em>F</em></span>, as defined in Eq. <strong>¿eq:growth_bohr_parameter?</strong>, is a state variable of the simple repression regulatory architecture. This is illustrated in Fig. 5 where fold-change measurements from a wide array of conditions (and measurement techniques) can be collapsed onto the same theoretical description. Evolutionary perturbations (such as mutations in the operator or repressor sequence), physiological changes (such as modulations of the growth rate), or changes in the level of activity of the repressor (due to changes in inducer concentration) do not change the fundamental physics of the system and can all be described by changes in the free energy relative to one another. While such a statement is not “surprising,” we can now say it with quantitative confidence and use this principle to probe the degree to which physiological perturbations influence the biophysical parameters writ large. With such a framework in hand, we are in the auspicious position to take a predictive approach towards understanding how this regulatory architecture evolves in experimental settings, shedding light on the interplay between biophysical parameters, organismal fitness, and the fundamental forces of evolution.</p>
<figure>
<img src="ch4_fig6.png" id="fig:collapse" data-short-caption="A singular theoretical description for the molecular biophysics of physiological and evolutionary adaptation in the simple repression motif." alt="" /><figcaption>Figure 5: <strong>A singular theoretical description for the molecular biophysics of physiological and evolutionary adaptation in the simple repression regulatory architecture.</strong> Measurements of the fold-change in gene expression varying the sequence of the operator site <span class="citation" data-cites="garcia2011">(orange pentagon, Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>, concentration of the extracellular inducer <span class="citation" data-cites="razo-mejia2018">(green squares, Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a> and Chapter 2)</span>, amino-acid sequence of the repressor <span class="citation" data-cites="chure2019">(blue points, Chure et al. <a href="#ref-chure2019" role="doc-biblioref">2019</a> and Chapter 3)</span>, and the various growth conditions queried in this work can be collapsed as a function of the effective free energy. Error bars correspond to the standard error of five to 10 biological replicates. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_04/code/ch4_fig6.py">Python code (<code>ch4_fig6.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="materials-methods">Materials &amp; Methods</h2>
<h3 id="bacterial-strains-and-growth-media">Bacterial Strains and Growth Media</h3>
<p>Three genotypes were primarily used in this work, all in the genetic background of <em>Escherichia coli</em> MG1655-K12 and all derived from those used in <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>. For each experiment, an autofluorescence control was used which contained no fluorescent reporters <span class="citation" data-cites="brewster2014">(except for a CFP volume marker used for segmentation in Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> which had the <em>lacI</em> and <em>lacZYA</em> genes deleted from the chromosome. The constitutive expression strain (<span class="math inline"><em>Δ</em><em>l</em><em>a</em><em>c</em><em>I</em>; <em>Δ</em><em>l</em><em>a</em><em>c</em><em>Z</em><em>Y</em><em>A</em></span>) included a YFP reporter gene integrated into the <em>galK</em> locus of the chromosome along with a kanamycin resistance cassette. The experimental strains in which LacI expression was controlled contained a <em>lacI-mCherry</em> fluorescent fusion integrated into the <em>ybcN</em> locus of the chromosome along with a chloramphenicol resistance cassette. This cassette was later deleted from the chromosome using FLP/FRT recombination <span class="citation" data-cites="schlake1994 zhu1995">(Schlake and Bode <a href="#ref-schlake1994" role="doc-biblioref">1994</a>; Zhu and Sadowski <a href="#ref-zhu1995" role="doc-biblioref">1995</a>)</span>. The strain was then transformed with plasmid <span class="citation" data-cites="lutz1997">(pZS3-pN25-tetR following notation described in Lutz and Bujard <a href="#ref-lutz1997" role="doc-biblioref">1997</a>)</span> constitutively expressing the TetR repressor along with a chloramphenicol resistance cassette. All bacterial strains and plasmids used in this work are reported in the supplemental Chapter 8.</p>
<h3 id="bacterial-growth-curves">Bacterial Growth Curves</h3>
<p>Bacterial growth curves were measured in a multi-well plate reader (BioTek Cytation5) generously provided by the David Van Valen lab at Caltech. Cells constitutively expressing YFP were grown overnight to saturation in LB broth (BD Medical) at 37<span class="math inline"><em></em><sup>∘</sup></span> C with aeration. Once saturated, cells were diluted 1000 fold into 50 mL of the desired growth medium and were allowed to grow at the appropriate experimental temperature with aeration for several hours until an OD<span class="math inline"><em></em><sub>600<em>n</em><em>m</em></sub> ≈ 0.3</span> was reached. Cells were then diluted 1:10 into the desired growth media at the desired temperature. After being thoroughly mixed, 500 <span class="math inline"><em>μ</em></span>L aliquots were transferred to a black-walled 96-well plate (Brooks Automation Incorporated, Cat No. MGB096-1-2-LG-L), leaving two rows and two columns of wells on each side of the plate filled with sterile growth medium to serve as blanks and buffer against temperature variation. The plate was then transferred to the pre-warmed plate reader. OD<span class="math inline"><em></em><sub>600<em>n</em><em>m</em></sub></span> measurements were made every five minutes for 12 to 24 hours until cultures had saturated. In between measurements, the plate incubated at the appropriate temperature with a linear shaking mode. We found that double-orbital shaking modes led to the formation of cell aggregates which gave inconsistent measurements.</p>
<h3 id="estimation-of-bacterial-growth-rate">Estimation of Bacterial Growth Rate</h3>
<p>Non-parametric estimation of the maximum growth rate was performed using the FitDeriv Python package as described in <span class="citation" data-cites="swain2016">Swain et al. (<a href="#ref-swain2016" role="doc-biblioref">2016</a>)</span>. Using this approach, the bacterial growth curve is modeled as a Gaussian process in which the measured growth at a given time point is modeled as a Gaussian distribution whose mean is dependent on the mean of the neighboring time points. This allows for smooth interpolation between adjacent measurements and calculation of second derivatives without an underlying parametric model. The reported growth rates are the maximum value inferred from the exponential phase of the experimental growth curve.</p>
<h3 id="growth-conditions">Growth Conditions</h3>
<p>Parent strains (autofluorescence control, <span class="math inline"><em>Δ</em><em>l</em><em>a</em><em>c</em><em>I</em></span> constitutive control, and the ATC-inducible LacI-mCherry strain) were grown in LB Miller broth (B.D. Medical, Cat. No. 244620 ) at 37<span class="math inline"><em></em><sup>∘</sup></span> C with vigorous aeration until saturated. Cells were then diluted between 1000 and 5000 fold into 3 mL of M9 minimal medium (B.D. Medical, Cat. No. 248510). The bacterial strain expressing the tetracycline-inducible LacI-mCherry was diluted into six 3 mL cultures with differing concentrations of ATC (Chemodex, Cat. No. CDX-A0197-T78) ranging from 0.1 to 10 ng / mL to induce expression of the transcription factor. These concentrations were reached by dilution from 1 <span class="math inline"><em>μ</em></span>g / mL stock in 50% ethanol. All cultures were shielded from ambient light using either aluminum foil or via an enclosure and were grown at the appropriate experimental temperature with aeration until an OD<span class="math inline"><em></em><sub>600<em>n</em><em>m</em></sub></span> of approximately <span class="math inline">0.25 − 0.35</span>. Due to pipetting errors, cultures reached OD<span class="math inline"><em></em><sub>600<em>n</em><em>m</em></sub> ≈ 0.3</span> at slightly different points in time. To ensure that strains could be directly compared, all strains were diluted by several fold until equivalent. When all samples reached the appropriate OD<span class="math inline"><em></em><sub>600<em>n</em><em>m</em></sub></span>, the cells were harvested for imaging.</p>
<h3 id="imaging-sample-preparation">Imaging Sample Preparation</h3>
<p>Prior to the preparation of cell cultures for imaging, a 2% (w/v) agarose substrate (UltraPure, Thermo Scientific) was prepared and allowed to reach room temperature. For experiments conducted at 42<span class="math inline"><em></em><sup>∘</sup><em>C</em></span>, 4% (w/v) agarose substrates were prepared. Briefly, the agarose was mixed with the appropriate growth medium in a 50 mL conical polystyrene tube and then microwaved until molten. A 300 to 500 <span class="math inline"><em>μ</em></span>L aliquot was then sandwiched between two glass coverslips to ensure a flat imaging surface. Once solidified, the agarose pads were cut into squares approximately <span class="math inline">0.5</span> cm per side.</p>
<p>     To determine the calibration factor between fluorescence and protein copy number, production the fluorophore in question must be halted such that all differences in intensity between two daughter cells result from binomial partitioning of the fluorophores and not from continuing expression. This was achieved by removing the anhydrous tetracycline inducer from the growth medium through several washing steps as outlined in <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>. Aliquots of 100 <span class="math inline"><em>μ</em></span>L from each ATC-induced culture were combined and pelleted at 13000<span class="math inline"> × <em>g</em></span> for 2 minutes. The supernatant (containing ATC) was aspirated and replaced with 1 mL of M9 growth medium without ATC. The pellet was resuspended and pelleted at 13000<span class="math inline"> × <em>g</em></span>. This wash step was repeated three times to ensure that residual ATC had been removed and LacI-mCherry production was halted. After the final wash, the cell pellet was resuspended in 1 mL of M9 medium and diluted ten-fold. A 1 <span class="math inline"><em>μ</em></span>L aliquot of the diluted mixture was then spotted onto an agarose substrate containing the appropriate carbon source. The remaining bacterial cultures (autofluorescence control, constitutive expression control, and the ATC-induced samples) were diluted ten-fold into sterile M9 medium. This dilution was thoroughly mixed and 1 <span class="math inline"><em>μ</em></span>L aliquots were spotted onto agarose substrates lacking the carbon source.</p>
<p>     Once the spotted cells had dried onto the agarose substrates (about 5 to 10 minutes after deposition), the agarose pads were inverted and pressed onto a glass bottom dish (Electron Microscopy Sciences, Cat. No. 70674-52) and sealed with parafilm. Strips of double stick tape were added to the edge of the dish to help immobilize the sample on the microscope stage and minimize drift.</p>
<h3 id="microscopy">Microscopy</h3>
<p>All imaging was performed on a Nikon Ti-Eclipse inverted microscope outfitted with a SOLA LED fluorescence illumination system. All images were acquired on a Andor Zyla 5.5 sCMOS camera (Oxford Instruments Group). The microscope body and stage was enclosed in a plexiglass incubation chamber (Haison, approximately 1<span class="math inline"><em></em><sup>∘</sup></span> C regulation control) connected to an external heater. Temperature of the stage was measured via a thermometer which controlled heating of the system.</p>
<p>     All static images (i.e. images from which fold-change and repressor counts were calculated) were measured in an identical manner. Ten to fifteen fields of view containing on average 25 cells were imaged using phase contrast and fluorescence excitation. Fluorescence exposures were each 5 seconds while the phase contrast exposure time was between 75 ms and 150 ms. This procedure was repeated for each unique strain and ATC induction concentration.</p>
<p>     To compute the calibration factor for that day of imaging, time-lapse images were taken of a separate agarose pad covered in cells containing various levels of LacI-mCherry. Fifteen to twenty positions were marked, choosing fields of view containing 20 to 50 cells. Cells were allowed to grow for a period of 90 to 120 minutes (depending on the medium-dependent growth rate) with phase contrast images taken every 5 to 10 minutes. At the final time-point, both phase contrast and fluorescence images were acquired using the same settings for the snapshots. Once the experiment was completed, images were exported to <code>.tif</code> format and transferred to cold storage and a computational cluster for analysis.</p>
<h3 id="lineage-tracking">Lineage Tracking</h3>
<p>Cells were segmented and lineages reconstructed using the deep-learning-based bacterial segmentation software SuperSegger <code>v1.1</code> <span class="citation" data-cites="stylianidou2016 cass2017">(Stylianidou et al. <a href="#ref-stylianidou2016" role="doc-biblioref">2016</a>; Cass et al. <a href="#ref-cass2017" role="doc-biblioref">2017</a>)</span> operated through MATLAB (Mathworks, <code>v2017b</code>). We found that the pre-trained network constants <code>100XEcM9</code> (packaged with the SuperSegger software) worked well for all growth conditions tested in this work. The generated files (<code>clist.mat</code>) associated with each sample and position were parsed using bespoke Python scripts to reconstruct lineages and apply appropriate filtering steps before calculating the fluorescence calibration factor. We direct the reader to the SI text for details of our data validation procedure to ensure proper lineage tracking.</p>
<h3 id="calculation-of-the-calibration-factor">Calculation of the Calibration Factor</h3>
<p>To estimate the calibration factor <span class="math inline"><em>α</em></span>, we used a Bayesian definition of probability to define a posterior distribution of <span class="math inline"><em>α</em></span> conditioned on intensity measurements of sibling cells after division. We direct the reader to supplemental Chapter 8 for a detailed discussion of the inferential procedures and estimation of systematic error. We give a brief description of the inference below.</p>
<p>     We are interested in determining the fluorescence of a single LacI-mCherry repressor dimer given a set of intensity measurements of sibling cells, <span class="math inline">[<em>I</em><sub>1</sub>, <em>I</em><sub>2</sub>]</span>. The intensity of a given cell <span class="math inline"><em>I</em></span> is related to the number of LacI-mCherry dimers it is expressing by a multiplicative factor <span class="math inline"><em>α</em></span> which can be enumerated mathematically as <span id="eq:growth_ian"><br /><span class="math display"><em>I</em> = <em>α</em><em>N</em>,   (7)</span><br /></span> where <span class="math inline"><em>N</em></span> is the total number of LacI-mCherry dimers. We can define the posterior probability distribution of <span class="math inline"><em>α</em></span> conditioned on the intensity measurements using Bayes’ theorem as <span id="eq:growth_bayes_generic"><br /><span class="math display">$$
g(\alpha\,\vert\, [I_1, I_2]) = \frac{f([I_1, I_2]\,\vert\,
\alpha)g(\alpha)}{f([I_1, I_2])},
\qquad(8)$$</span><br /></span> where we have used <span class="math inline"><em>g</em></span> and <span class="math inline"><em>f</em></span> as probability densities over parameters and data, respectively. The denominator of this expression (the evidence) is equivalent to the first term of the numerator (the likelihood) marginalized over <span class="math inline"><em>α</em></span>. In this work, this term serves as normalization factor and can be neglected.</p>
<p>     Assuming that no more LacI-mCherry dimers are produced during cell division, the number of repressors that each sibling cell receives after division of the parent cell is binomially distributed with a probability <span class="math inline"><em>p</em></span>. We can make the approximation that partitioning of the repressors is even such that <span class="math inline"><em>p</em> = 1/2</span>. The validity of this approximation is discussed in detail in Chapter 8. Using Eq. 7 and the change-of-variables formula, we can define the likelihood <span class="math inline"><em>g</em>([<em>I</em><sub>1</sub>, <em>I</em><sub>2</sub>] | <em>α</em>)</span> as <span id="eq:growth_binom_like"><br /><span class="math display">$$
g([I_1, I_2]\,\vert\,\alpha) =
\frac{1}{\alpha^k}\prod_i^k\frac{\Gamma\left(\frac{ {I_1}_i + {I_2}_i}{\alpha} +
1\right)}{\Gamma\left(\frac{ {I_1}_i}{\alpha} +
1\right)\Gamma\left(\frac{ {I_2}_i}{\alpha} + 1\right)}2^{-\frac{ {I_1}_i +
{I_2}_i}{\alpha}},
\qquad(9)$$</span><br /></span> where <span class="math inline"><em>k</em></span> is the total number of sibling pairs measured.</p>
<p>     With a likelihood defined, we must now define a functional form for <span class="math inline"><em>g</em>(<em>α</em>)</span> which describes all prior information known about the calibration factor knowing nothing about the actual measurements. Knowing that we design the experiments such that only <span class="math inline"> ≈ 2/3</span> of the dynamic range of the camera is used and <span class="math inline"><em>α</em></span> cannot be less than or equal to zero, we can define a half-normal distribution with a standard deviation of <span class="math inline"><em>σ</em></span> as <span id="eq:alpha_prior"><br /><span class="math display">$$
g(\alpha) = \sqrt{2 \over \pi\sigma^2}\exp\left[{-\alpha^2 \over
2\sigma^2}\right]\,;\, \forall \alpha &gt; 0,
\qquad(10)$$</span><br /></span> where the standard deviation is large, for example, <span class="math inline"><em>σ</em> = 500</span> a.u. / fluorophore. We evaluated the posterior distribution using Markov chain Monte Carlo (MCMC) as is implemented in the Stan probabilistic programming language <span class="citation" data-cites="carpenter2017">(Carpenter et al. <a href="#ref-carpenter2017" role="doc-biblioref">2017</a>)</span>. The <code>.stan</code> file associated with this model along with the Python code used to execute it can be accessed on the <a href="https://rpgroup.caltech.edu/mwc_growth">paper website</a>.</p>
<h3 id="counting-repressors">Counting Repressors</h3>
<p>Given an estimation for <span class="math inline"><em>α</em></span> for each experiment, we calculate the total number of repressors per cell from <span id="eq:nia"><br /><span class="math display">$$
R = \frac{I_{mCherry}}{\alpha}.
\qquad(11)$$</span><br /></span> However, as discussed in detail in Chapter 8, a systematic error in the repressor count is introduced due to division in the asynchronous culture between the cessation of LacI-mCherry production and the actual imaging. The entire sample preparation procedure is <span class="math inline"> ≈ 30 − 60</span> min, during which time some cells complete a cell division, thereby diluting the total repressor count. To ensure that the measured number of repressors corresponded to the measured YFP intensity, we restricted our dataset for all experiments to cells that had a pole-to-pole length <span class="math inline">ℓ ≥ 3.5 <em>μ</em></span>m, indicating that they had likely not undergone a division during the sample preparation.</p>
<h3 id="code-and-data-availability">Code and Data Availability</h3>
<p>All code used in this work is available on the <a href="https://github.com/rpgroup-pboc/mwc_growth">paper website</a> and <a href="https://www.github.com/rpgroup-pboc/mwc_growth">associated GitHub repository</a>(DOI: 0.5281/zenodo.3560369). This work also used the open-source software tools <a href="http://mtshasta.phys.washington.edu/website/SuperSegger.php">SuperSegger</a> <code>v.1.1</code><span class="citation" data-cites="stylianidou2016 cass2017">(Stylianidou et al. <a href="#ref-stylianidou2016" role="doc-biblioref">2016</a>; Cass et al. <a href="#ref-cass2017" role="doc-biblioref">2017</a>)</span> for lineage tracking and <a href="http://swainlab.bio.ed.ac.uk/software/fitderiv/">FitDeriv</a> <code>v.1.03</code> <span class="citation" data-cites="swain2016">(Swain et al. <a href="#ref-swain2016" role="doc-biblioref">2016</a>)</span> for the nonparametric estimation of growth rates. Raw image files are large (1.8 Tb) and are therefore available upon request. The clist.mat files used to calculate fold-change and to assign sibling cells can be accessed via the associated <a href="https://github.com/rpgroup-pboc/mwc_growth">GitHub repository</a> via (<a href="http://doi.org/10.5281/zenodo.3560369">DOI: 0.5281/zenodo.3560369</a>) or through Caltech DATA under the DOI: 10.22002/D1.1315.</p>
<h2 class="unnumbered" id="references" class="unnumbered">References</h2>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-barnes2019">
<p>Barnes, Stephanie L., Nathan M. Belliveau, William T. Ireland, Justin B. Kinney, and Rob Phillips. 2019. “Mapping DNA Sequence to Transcription Factor Binding Energy <em>in Vivo</em>.” Edited by Gary D. Stormo. <em>PLOS Computational Biology</em> 15 (2). <a href="https://doi.org/10.1371/journal.pcbi.1006226">https://doi.org/10.1371/journal.pcbi.1006226</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates the Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-carpenter2017">
<p>Carpenter, Bob, Andrew Gelman, Matthew D. Hoffman, Daniel Lee, Ben Goodrich, Michael Betancourt, Marcus Brubaker, Jiqiang Guo, Peter Li, and Allen Riddell. 2017. “Stan: A Probabilistic Programming Language.” <em>Journal of Statistical Software</em> 76 (1): 1–32. <a href="https://doi.org/10.18637/jss.v076.i01">https://doi.org/10.18637/jss.v076.i01</a>.</p>
</div>
<div id="ref-cass2017">
<p>Cass, Julie A., Stella Stylianidou, Nathan J. Kuwada, Beth Traxler, and Paul A. Wiggins. 2017. “Probing Bacterial Cell Biology Using Image Cytometry.” <em>Molecular Microbiology</em> 103 (5): 818–28. <a href="https://doi.org/10.1111/mmi.13591">https://doi.org/10.1111/mmi.13591</a>.</p>
</div>
<div id="ref-chure2019">
<p>Chure, Griffin, Manuel Razo-Mejia, Nathan M Belliveau, Tal Einav, Zofii A Kaczmarek, Stephanie L Barnes, and Rob Phillips. 2019. “Predictive Shifts in Free Energy Couple Mutations to Their Phenotypic Consequences.” <em>Proceedings of the National Academy of Sciences</em> 116 (37). <a href="https://doi.org/10.1073/pnas.1907869116">https://doi.org/10.1073/pnas.1907869116</a>.</p>
</div>
<div id="ref-driessen2014">
<p>Driessen, Rosalie P. C., Gerrit Sitters, Niels Laurens, Geri F. Moolenaar, Gijs J. L. Wuite, Nora Goosen, and Remus Th. Dame. 2014. “Effect of Temperature on the Intrinsic Flexibility of DNA and Its Interaction with Architectural Proteins.” <em>Biochemistry</em> 53 (41): 6430–8. <a href="https://doi.org/10.1021/bi500344j">https://doi.org/10.1021/bi500344j</a>.</p>
</div>
<div id="ref-elf2007">
<p>Elf, Johan, Gene-Wei Li, and X. Sunney Xie. 2007. “Probing Transcription Factor Dynamics at the Single-Molecule Level in a Living Cell.” <em>Science</em> 316 (5828): 1191–4. <a href="https://doi.org/10.1126/science.1141967">https://doi.org/10.1126/science.1141967</a>.</p>
</div>
<div id="ref-friedel1974">
<p>Friedel, J. 1974. “On the Stability of the Body Centred Cubic Phase in Metals at High Temperatures.” <em>Journal de Physique Lettres</em> 35 (4): 59–63. <a href="https://doi.org/10.1051/jphyslet:0197400350405900">https://doi.org/10.1051/jphyslet:0197400350405900</a>.</p>
</div>
<div id="ref-gama-castro2016">
<p>Gama-Castro, Socorro, Heladia Salgado, Alberto Santos-Zavaleta, Daniela Ledezma-Tejeida, Luis Muñiz-Rascado, Jair Santiago García-Sotelo, Kevin Alquicira-Hernández, et al. 2016. “RegulonDB Version 9.0: High-Level Integration of Gene Regulation, Coexpression, Motif Clustering and Beyond.” <em>Nucleic Acids Research</em> 44 (D1): D133–D143. <a href="https://doi.org/10.1093/nar/gkv1156">https://doi.org/10.1093/nar/gkv1156</a>.</p>
</div>
<div id="ref-garcia2012">
<p>Garcia, Hernan G., Alvaro Sanchez, James Q. Boedicker, Melisa Osborne, Jeff Gelles, Jane Kondev, and Rob Phillips. 2012. “Operator Sequence Alters Gene Expression Independently of Transcription Factor Occupancy in Bacteria.” <em>Cell Reports</em> 2 (1): 150–61. <a href="https://doi.org/10.1016/j.celrep.2012.06.004">https://doi.org/10.1016/j.celrep.2012.06.004</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-goethe2015">
<p>Goethe, Martin, Ignacio Fita, and J. Miguel Rubi. 2015. “Vibrational Entropy of a Protein: Large Differences Between Distinct Conformations.” <em>Journal of Chemical Theory and Computation</em> 11 (1): 351–59. <a href="https://doi.org/10.1021/ct500696p">https://doi.org/10.1021/ct500696p</a>.</p>
</div>
<div id="ref-hui2015">
<p>Hui, Sheng, Josh M. Silverman, Stephen S. Chen, David W. Erickson, Markus Basan, Jilong Wang, Terence Hwa, and James R. Williamson. 2015. “Quantitative Proteomic Analysis Reveals a Simple Strategy of Global Resource Allocation in Bacteria.” <em>Molecular Systems Biology</em> 11 (2). <a href="https://doi.org/10.15252/msb.20145697">https://doi.org/10.15252/msb.20145697</a>.</p>
</div>
<div id="ref-jones2014">
<p>Jones, Daniel L., Robert C. Brewster, and Rob Phillips. 2014. “Promoter Architecture Dictates Cell-to-Cell Variability in Gene Expression.” <em>Science</em> 346 (6216): 1533–6. <a href="https://doi.org/10.1126/science.1255301">https://doi.org/10.1126/science.1255301</a>.</p>
</div>
<div id="ref-jun2018">
<p>Jun, Suckjoon, Fangwei Si, Rami Pugatch, and Matthew Scott. 2018. “Fundamental Principles in Bacterial Physiology - History, Recent Progress, and the Future with Focus on Cell Size Control: A Review.” <em>Reports on Progress in Physics</em> 81 (5): 056601. <a href="https://doi.org/10.1088/1361-6633/aaa628">https://doi.org/10.1088/1361-6633/aaa628</a>.</p>
</div>
<div id="ref-kao-huang1977">
<p>Kao-Huang, Ying, Arnold Revzin, Andrew P. Butler, Pamela O’Conner, Daniel W. Noble, and Peter H. Von Hippel. 1977. “Nonspecific DNA Binding of Genome-Regulating Proteins as a Biological Control Mechanism: Measurement of DNA-Bound <em>Escherichia Coli</em> Lac Repressor <em>in Vivo</em>.” <em>Proceedings of the National Academy of Sciences</em> 74 (10): 4228–32. <a href="https://doi.org/10.1073/pnas.74.10.4228">https://doi.org/10.1073/pnas.74.10.4228</a>.</p>
</div>
<div id="ref-kepler2001">
<p>Kepler, Thomas B., and Timothy C. Elston. 2001. “Stochasticity on Transcriptional Regulation: Origins, Consequences, and Mathematical Representations.” <em>Biophysical Journal</em> 81 (6): 3116–36. <a href="https://doi.org/10.1016/S0006-3495(01)75949-8">https://doi.org/10.1016/S0006-3495(01)75949-8</a>.</p>
</div>
<div id="ref-klumpp2014">
<p>Klumpp, Stefan, and Terence Hwa. 2014. “Bacterial Growth: Global Effects on Gene Expression, Growth Feedback and Proteome Partition.” <em>Current Opinion in Biotechnology</em> 28 (August): 96–102. <a href="https://doi.org/10.1016/j.copbio.2014.01.001">https://doi.org/10.1016/j.copbio.2014.01.001</a>.</p>
</div>
<div id="ref-ko1991">
<p>Ko, Minoru S. H. 1991. “A Stochastic Model for Gene Induction.” <em>Journal of Theoretical Biology</em> 153 (2): 181–94. <a href="https://doi.org/10.1016/S0022-5193(05)80421-7">https://doi.org/10.1016/S0022-5193(05)80421-7</a>.</p>
</div>
<div id="ref-li2014">
<p>Li, Gene-Wei, David Burkhardt, Carol Gross, and Jonathan S. Weissman. 2014. “Quantifying Absolute Protein Synthesis Rates Reveals Principles Underlying Allocation of Cellular Resources.” <em>Cell</em> 157 (3): 624–35. <a href="https://doi.org/10.1016/j.cell.2014.02.033">https://doi.org/10.1016/j.cell.2014.02.033</a>.</p>
</div>
<div id="ref-lutz1997">
<p>Lutz, Rolf, and Hermann Bujard. 1997. “Independent and Tight Regulation of Transcriptional Units in <em>Escherichia Coli</em> via the LacR/O, the TetR/O and AraC/I1-I2 Regulatory Elements.” <em>Nucleic Acids Research</em> 25 (6): 1203–10. <a href="https://doi.org/10.1093/nar/25.6.1203">https://doi.org/10.1093/nar/25.6.1203</a>.</p>
</div>
<div id="ref-michel2010">
<p>Michel, Denis. 2010. “How Transcription Factors Can Adjust the Gene Expression Floodgates.” <em>Progress in Biophysics and Molecular Biology</em> 102 (1): 16–37. <a href="https://doi.org/10.1016/j.pbiomolbio.2009.12.007">https://doi.org/10.1016/j.pbiomolbio.2009.12.007</a>.</p>
</div>
<div id="ref-mondal2011">
<p>Mondal, Jagannath, Benjamin P. Bratton, Yijie Li, Arun Yethiraj, and James C. Weisshaar. 2011. “Entropy-Based Mechanism of Ribosome-Nucleoid Segregation in <em>E. coli</em> Cells.” <em>Biophysical Journal</em> 100 (11): 2605–13. <a href="https://doi.org/10.1016/j.bpj.2011.04.030">https://doi.org/10.1016/j.bpj.2011.04.030</a>.</p>
</div>
<div id="ref-motlagh2014">
<p>Motlagh, Hesam N., James O. Wrabl, Jing Li, and Vincent J. Hilser. 2014. “The Ensemble Nature of Allostery.” <em>Nature</em> 508 (7496): 331–39. <a href="https://doi.org/10.1038/nature13001">https://doi.org/10.1038/nature13001</a>.</p>
</div>
<div id="ref-phillips2001">
<p>Phillips, Rob. 2001. <em>Crystals, Defects and Microstructures</em>. Cambridge University Press. <a href="https://doi.org/10.1017/CBO9780511606236">https://doi.org/10.1017/CBO9780511606236</a>.</p>
</div>
<div id="ref-phillips2019">
<p>Phillips, Rob, Nathan M. Belliveau, Griffin Chure, Hernan G. Garcia, Manuel Razo-Mejia, and Clarissa Scholes. 2019. “Figure 1 Theory Meets Figure 2 Experiments in the Study of Gene Expression.” <em>Annual Review of Biophysics</em> 48 (1): 121–63. <a href="https://doi.org/10.1146/annurev-biophys-052118-115525">https://doi.org/10.1146/annurev-biophys-052118-115525</a>.</p>
</div>
<div id="ref-razo-mejia2018">
<p>Razo-Mejia, Manuel, Stephanie L. Barnes, Nathan M. Belliveau, Griffin Chure, Tal Einav, Mitchell Lewis, and Rob Phillips. 2018. “Tuning Transcriptional Regulation Through Signaling: A Predictive Theory of Allosteric Induction.” <em>Cell Systems</em> 6 (4): 456–469.e10. <a href="https://doi.org/10.1016/j.cels.2018.02.004">https://doi.org/10.1016/j.cels.2018.02.004</a>.</p>
</div>
<div id="ref-rosenfeld2002">
<p>Rosenfeld, N., M. B. Elowitz, and U. Alon. 2002. “Negative Autoregulation Speeds the Response Times of Transcription Networks.” <em>Journal of Molecular Biology</em> 323 (November): 785–93. <a href="https://doi.org/S0022283602009944\\\%0020%5Bpii%5D">https://doi.org/S0022283602009944\\\%0020[pii]</a>.</p>
</div>
<div id="ref-rosenfeld2005">
<p>Rosenfeld, Nitzan, Jonathan W. Young, Uri Alon, Peter S. Swain, and Michael B. Elowitz. 2005. “Gene Regulation at the Single-Cell Level.” <em>Science</em> 307 (5717): 1962–5. <a href="https://doi.org/10.1126/science.1106914">https://doi.org/10.1126/science.1106914</a>.</p>
</div>
<div id="ref-rydenfelt2014">
<p>Rydenfelt, Mattias, Hernan G. Garcia, Robert Sidney Cox, and Rob Phillips. 2014. “The Influence of Promoter Architectures and Regulatory Motifs on Gene Expression in <em>Escherichia Coli</em>.” Edited by Jordi Garcia-Ojalvo. <em>PLOS ONE</em> 9 (12): e114347. <a href="https://doi.org/10.1371/journal.pone.0114347">https://doi.org/10.1371/journal.pone.0114347</a>.</p>
</div>
<div id="ref-schaechter1958">
<p>Schaechter, M., O. Maaløe, and N. O. Kjeldgaard. 1958. “Dependency on Medium and Temperature of Cell Size and Chemical Composition During Balanced Growth of <em>Salmonella Typhimurium</em>.” <em>Microbiology</em> 19 (3): 592–606. <a href="https://doi.org/10.1099/00221287-19-3-592">https://doi.org/10.1099/00221287-19-3-592</a>.</p>
</div>
<div id="ref-schlake1994">
<p>Schlake, Thomas, and Juergen Bode. 1994. “Use of Mutated FLP Recognition Target (FRT) Sites for the Exchange of Expression Cassettes at Defined Chromosomal Loci.” <em>Biochemistry</em> 33 (43): 12746–51. <a href="https://doi.org/10.1021/bi00209a003">https://doi.org/10.1021/bi00209a003</a>.</p>
</div>
<div id="ref-schmidt2016">
<p>Schmidt, Alexander, Karl Kochanowski, Silke Vedelaar, Erik Ahrné, Benjamin Volkmer, Luciano Callipo, Kèvin Knoops, Manuel Bauer, Ruedi Aebersold, and Matthias Heinemann. 2016. “The Quantitative and Condition-Dependent <em>Escherichia Coli</em> Proteome.” <em>Nature Biotechnology</em> 34 (1): 104–10. <a href="https://doi.org/10.1038/nbt.3418">https://doi.org/10.1038/nbt.3418</a>.</p>
</div>
<div id="ref-scott2014">
<p>Scott, M., S. Klumpp, E. M. Mateescu, and T. Hwa. 2014. “Emergence of Robust Growth Laws from Optimal Regulation of Ribosome Synthesis.” <em>Molecular Systems Biology</em> 10 (8): 747–47. <a href="https://doi.org/10.15252/msb.20145379">https://doi.org/10.15252/msb.20145379</a>.</p>
</div>
<div id="ref-shehata1975">
<p>Shehata, T E, and A G Marr. 1975. “Effect of Temperature on the Size of <em>Escherichia Coli Cells</em>.” <em>Journal of Bacteriology</em> 124 (2): 857–62.</p>
</div>
<div id="ref-stylianidou2016">
<p>Stylianidou, Stella, Connor Brennan, Silas B. Nissen, Nathan J. Kuwada, and Paul A. Wiggins. 2016. “SuperSegger: Robust Image Segmentation, Analysis and Lineage Tracking of Bacterial Cells.” <em>Molecular Microbiology</em> 102 (4): 690–700. <a href="https://doi.org/10.1111/mmi.13486">https://doi.org/10.1111/mmi.13486</a>.</p>
</div>
<div id="ref-swain2016">
<p>Swain, Peter S., Keiran Stevenson, Allen Leary, Luis F. Montano-Gutierrez, Ivan B. N. Clark, Jackie Vogel, and Teuta Pilizota. 2016. “Inferring Time Derivatives Including Cell Growth Rates Using Gaussian Processes.” <em>Nature Communications</em> 7 (December): 13766. <a href="https://doi.org/10.1038/ncomms13766">https://doi.org/10.1038/ncomms13766</a>.</p>
</div>
<div id="ref-yakovchuk2006">
<p>Yakovchuk, Peter, Ekaterina Protozanova, and Maxim D. Frank-Kamenetskii. 2006. “Base-Stacking and Base-Pairing Contributions into Thermal Stability of the DNA Double Helix.” <em>Nucleic Acids Research</em> 34 (2): 564–74. <a href="https://doi.org/10.1093/nar/gkj454">https://doi.org/10.1093/nar/gkj454</a>.</p>
</div>
<div id="ref-zhu1995">
<p>Zhu, Xu-Dong, and Paul D. Sadowski. 1995. “Cleavage-Dependent Ligation by the FLP Recombinase: Characterization of a Mutant Protein with an Alteration in a Catalytic Amino Acid.” <em>Journal of Biological Chemistry</em> 270 (39): 23044–54. <a href="https://doi.org/10.1074/jbc.270.39.23044">https://doi.org/10.1074/jbc.270.39.23044</a>.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/chapter_03" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter III
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_05" class="page-nav__item page-nav__item--next">
										Chapter 2
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd//" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>


<script>
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

	ga('create', 'UA-109526846-4', 'auto');
	ga('send', 'pageview');
</script>



		</div><!-- /.js-footer-area -->
	</body>
</html>
