<h2 id="theoretical-model">Theoretical Model</h2>
<h3 id="inducible-transcriptional-repression-via-the-mwc-model-of-allostery">Inducible Transcriptional Repression Via The MWC Model of Allostery</h3>
<p>We begin by considering a simple repression genetic architecture in which the binding of an allosteric repressor occludes the binding of RNA polymerase (RNAP) to the DNA <span class="citation" data-cites="ackers1982 buchler2003">(Ackers and Johnson <a href="#ref-ackers1982" role="doc-biblioref">1982</a>; Buchler, Gerland, and Hwa <a href="#ref-buchler2003" role="doc-biblioref">2003</a>)</span>. When an effector (hereafter referred to as an “inducer” for the case of induction) binds to the repressor, it shifts the repressor’s allosteric equilibrium towards the inactive state as specified by the MWC model <span class="citation" data-cites="monod1965">(Monod, Wyman, and Changeux <a href="#ref-monod1965" role="doc-biblioref">1965</a>)</span>. This causes the repressor to bind more weakly to the operator, which increases gene expression. Simple repression motifs in the absence of inducer have been previously characterized by an equilibrium model where the probability of each state of repressor and RNAP promoter occupancy is dictated by the Boltzmann distribution <span class="citation" data-cites="ackers1982 buchler2003 vilar2003 bintu2005a garcia2011 brewster2014">(Ackers and Johnson <a href="#ref-ackers1982" role="doc-biblioref">1982</a>; Buchler, Gerland, and Hwa <a href="#ref-buchler2003" role="doc-biblioref">2003</a>; Vilar and Leibler <a href="#ref-vilar2003" role="doc-biblioref">2003</a>; Bintu, Buchler, Garcia, Gerland, Hwa, Kondev, Kuhlman, et al. <a href="#ref-bintu2005a" role="doc-biblioref">2005</a>; Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> [we note that non-equilibrium models of simple repression have been shown to have the same functional form that we derive below <span class="citation" data-cites="phillips2015">(Phillips <a href="#ref-phillips2015" role="doc-biblioref">2015</a>)</span>]. We extend these models to consider allostery by accounting for the equilibrium state of the repressor through the MWC model.</p>
<p>Thermodynamic models of gene expression begin by enumerating all possible states of the promoter and their corresponding statistical weights. As shown in Fig. <a href="#fig:states_weights">1</a> (A), the promoter can either be empty, occupied by RNAP, or occupied by either an active or inactive repressor. The probability of binding to the promoter will be affected by the protein copy number, which we denote as <span class="math inline"><em>P</em></span> for RNAP, <span class="math inline"><em>R</em><sub><em>A</em></sub></span> for active repressor, and <span class="math inline"><em>R</em><sub><em>I</em></sub></span> for inactive repressor. We note that repressors fluctuate between the active and inactive conformation in thermodynamic equilibrium, such that <span class="math inline"><em>R</em><sub><em>A</em></sub></span> and <span class="math inline"><em>R</em><sub><em>I</em></sub></span> will remain constant for a given inducer concentration <span class="citation" data-cites="monod1965">(Monod, Wyman, and Changeux <a href="#ref-monod1965" role="doc-biblioref">1965</a>)</span>. We assign the repressor a different DNA binding affinity in the active and inactive state. In addition to the specific binding sites at the promoter, we assume that there are <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> non-specific binding sites elsewhere (i.e. on parts of the genome outside the simple repression architecture) where the RNAP or the repressor can bind. All specific binding energies are measured relative to the average non-specific binding energy. Thus, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>P</em></sub></span> represents the energy difference between the specific and non-specific binding for RNAP to the DNA. Likewise, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>I</em></sub></span> represent the difference in specific and non-specific binding energies for repressor in the active or inactive state, respectively.</p>
<div id="fig:states_weights" class="fignos">
<figure>
<img src="ch2_fig2.png" data-short-caption="States and statistical weights for the simple repression motif." alt="" /><figcaption><span>Figure 1:</span> States and weights for the simple repression motif. (A) Occupancy states of the promoter. RNAP (light blue) and a repressor compete for binding to a promoter of interest. There are <span class="math inline"><em>R</em><sub><em>A</em></sub></span> repressors in the active state (red) and <span class="math inline"><em>R</em><sub><em>I</em></sub></span> repressors in the inactive state (purple). The difference in energy between a repressor bound to the promoter of interest versus another non-specific site elsewhere on the DNA equals <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> in the active state and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>I</em></sub></span> in the inactive state; the <span class="math inline"><em>P</em></span> RNAP have a corresponding energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>P</em></sub></span> relative to non-specific binding on the DNA. <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> represents the number of non-specific binding sites for both RNAP and repressor. (B) Allosteric states of the repressor. A repressor has an active conformation (red, left column) and an inactive conformation (purple, right column), with the energy difference between these two states given by <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. The inducer (orange circle) at concentration <span class="math inline"><em>c</em></span> is capable of binding to the repressor with dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> in the active state and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> in the inactive state. The eight states for a dimer with <span class="math inline"><em>n</em> = 2</span> inducer binding sites are shown along with the sums of the active and inactive states.</figcaption>
</figure>
</div>
<p>Thermodynamic models of transcription <span class="citation" data-cites="ackers1982 buchler2003 vilar2003 bintu2005 bintu2005a kuhlman2007 daber2011a garcia2011 brewster2014 weinert2014">(Ackers and Johnson <a href="#ref-ackers1982" role="doc-biblioref">1982</a>; Buchler, Gerland, and Hwa <a href="#ref-buchler2003" role="doc-biblioref">2003</a>; Vilar and Leibler <a href="#ref-vilar2003" role="doc-biblioref">2003</a>; Bintu, Buchler, Garcia, Gerland, Hwa, Kondev, and Phillips <a href="#ref-bintu2005" role="doc-biblioref">2005</a>; Bintu, Buchler, Garcia, Gerland, Hwa, Kondev, Kuhlman, et al. <a href="#ref-bintu2005a" role="doc-biblioref">2005</a>; Kuhlman et al. <a href="#ref-kuhlman2007" role="doc-biblioref">2007</a>; Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>; Garcia and Phillips <a href="#ref-garcia2011" role="doc-biblioref">2011</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Weinert et al. <a href="#ref-weinert2014" role="doc-biblioref">2014</a>)</span> posit that gene expression is proportional to the probability that the RNAP is bound to the promoter <span class="math inline"><em>p</em><sub>bound</sub></span>, which is given by</p>
<p><span id="eq:pbound_def" class="eqnos"><br /><span class="math display">$$
p_\text{bound} = \frac{\frac{P}{N_{NS}}e^{-\beta
\Delta\varepsilon_{P}}}{1+\frac{R_A}{N_{NS}}e^{-\beta
\Delta\varepsilon_{RA}}+\frac{R_I}{N_{NS}}e^{-\beta
\Delta\varepsilon_{RI}}+\frac{P}{N_{NS}}e^{-\beta\Delta\varepsilon_{P}}},
$$</span><br /><span class="eqnos-number">(1)</span></span></p>
<p>with <span class="math inline"><em>β</em> = 1/<em>k</em><sub><em>B</em></sub><em>T</em></span> where <span class="math inline"><em>k</em><sub><em>B</em></sub></span> is the Boltzmann constant and <span class="math inline"><em>T</em></span> is the temperature of the system. As <span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span> is the natural unit of energy at the molecular length scale, we treat the products <span class="math inline"><em>β</em><em>Δ</em><em>ε</em><sub><em>j</em></sub></span> as single parameters within our model. Measuring <span class="math inline"><em>p</em><sub>bound</sub></span> directly is fraught with experimental difficulties, as determining the exact proportionality between expression and <span class="math inline"><em>p</em><sub>bound</sub></span> is not straightforward. Instead, we measure the fold-change in gene expression due to the presence of the repressor. We define fold-change as the ratio of gene expression in the presence of repressor relative to expression in the absence of repressor (i.e. constitutive expression), namely,</p>
<p><span id="eq:fold_change_definition" class="eqnos"><br /><span class="math display">$$
\text{fold-change} \equiv \frac{p_\text{bound}(R &gt; 0)}{p_\text{bound}(R = 0)}.
$$</span><br /><span class="eqnos-number">(2)</span></span></p>
<p>We can simplify this expression using two well-justified approximations: firstly, <span class="math inline">(<em>P</em>/<em>N</em><sub><em>N</em><em>S</em></sub>)<em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>P</em></sub></sup>≪</span> 1 implying that the RNAP binds weakly to the promoter (<span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub> = 4.6 × 10<sup>6</sup></span>, <span class="math inline"><em>P</em> ≈ 10<sup>3</sup></span> <span class="citation" data-cites="klumpp2008">(Klumpp and Hwa <a href="#ref-klumpp2008" role="doc-biblioref">2008</a>)</span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>P</em></sub> ≈  − 2  to  − 5 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="brewster2012">(Brewster, Jones, and Phillips <a href="#ref-brewster2012" role="doc-biblioref">2012</a>)</span>, so that <span class="math inline">(<em>P</em>/<em>N</em><sub><em>N</em><em>S</em></sub>)<em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>P</em></sub></sup> ≈ 0.01</span>) and (2) <span class="math inline">(<em>R</em><sub><em>I</em></sub>/<em>N</em><sub><em>N</em><em>S</em></sub>)<em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>R</em><em>I</em></sub></sup> ≪ 1 + (<em>R</em><sub><em>A</em></sub>/<em>N</em><sub><em>N</em><em>S</em></sub>)<em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></sup></span> which reflects our assumption that the inactive repressor binds weakly to the promoter of interest. Using these approximations, the fold-change reduces to the form</p>
<p><span id="eq:fold_change_RA" class="eqnos"><br /><span class="math display">$$
\text{fold-change} \approx \left(1+\frac{R_A}{N_{NS}}e^{-\beta
\Delta\varepsilon_{RA}}\right)^{-1} \equiv \left( 1+p_\text{act}(c)
\frac{R}{N_{NS}}e^{-\beta\Delta\varepsilon_{RA}} \right)^{-1},
$$</span><br /><span class="eqnos-number">(3)</span></span></p>
<p>where in the last step we have introduced the fraction <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> of repressors in the active state given a concentration <span class="math inline"><em>c</em></span> of inducer, such that <span class="math inline"><em>R</em><sub><em>A</em></sub>(<em>c</em>) = <em>p</em><sub>act</sub>(<em>c</em>)<em>R</em></span>. Since inducer binding shifts the repressors from the active to the inactive state, <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> grows smaller as <span class="math inline"><em>c</em></span> increases .</p>
<p>We use the MWC model to compute the probability <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> that a repressor with <span class="math inline"><em>n</em></span> inducer binding sites will be active. The value of <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> is given by the sum of the weights of the active repressor states divided by the sum of the weights of all possible repressor states [see Fig. <a href="#fig:states_weights">1</a> (B)], namely,</p>
<p><span id="eq:eq_pactive" class="eqnos"><br /><span class="math display">$$
p_\text{act}(c)=\frac{\left(1+\frac{c}{K_A}\right)^n}{\left(1+\frac{c}{K_A}\right)^n+e^{-\beta
\Delta \varepsilon_{AI} }\left(1+\frac{c}{K_I}\right)^n},
$$</span><br /><span class="eqnos-number">(4)</span></span></p>
<p>where <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> represent the dissociation constant between the inducer and repressor in the active and inactive states, respectively, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = <em>ε</em><sub><em>I</em></sub> − <em>ε</em><sub><em>A</em></sub></span> is the free energy difference between a repressor in the inactive and active state [the quantity <span class="math inline"><em>e</em><sup> − <em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></sup></span> is sometimes denoted by <span class="math inline"><em>L</em></span> <span class="citation" data-cites="monod1965 marzen2013">(Monod, Wyman, and Changeux <a href="#ref-monod1965" role="doc-biblioref">1965</a>; Marzen, Garcia, and Phillips <a href="#ref-marzen2013" role="doc-biblioref">2013</a>)</span> or <span class="math inline"><em>K</em><sub>RR*</sub></span> <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span>]. In this equation, <span class="math inline"><em>c</em>/<em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>c</em>/<em>K</em><sub><em>I</em></sub></span> represent the change in free energy when an inducer binds to a repressor in the active or inactive state, respectively, while <span class="math inline"><em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></sup></span> represents the change in free energy when the repressor changes from the active to inactive state in the absence of inducer. Thus, a repressor which favors the active state in the absence of inducer (<span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> &gt; 0</span>) will be driven towards the inactive state upon inducer binding when <span class="math inline"><em>K</em><sub><em>I</em></sub> &lt; <em>K</em><sub><em>A</em></sub></span>. The specific case of a repressor dimer with <span class="math inline"><em>n</em> = 2</span> inducer binding sites is shown in Fig. <a href="#fig:states_weights">1</a> (B).</p>
<p>Substituting <span class="math inline"><em>p</em><sub>act</sub>(<em>c</em>)</span> from into yields the general formula for induction of a simple repression regulatory architecture <span class="citation" data-cites="phillips2015">(Phillips <a href="#ref-phillips2015" role="doc-biblioref">2015</a>)</span>, namely, <span id="eq:fold_change_full" class="eqnos"><br /><span class="math display">$$
\text{fold-change} =
\left(1+\frac{\left(1+\frac{c}{K_A}\right)^n}{\left(1+\frac{c}{K_A}\right)^n+e^{-\beta
\Delta \varepsilon_{AI}
}\left(1+\frac{c}{K_I}\right)^n}\frac{R}{N_{NS}}e^{-\beta
\Delta\varepsilon_{RA}} \right)^{-1}.
$$</span><br /><span class="eqnos-number">(5)</span></span></p>
<p>While we have used the specific case of simple repression with induction to craft this model, the same mathematics describe the case of corepression in which binding of an allosteric effector stabilizes the active state of the repressor and decreases gene expression (see Fig. ??). Interestingly, we shift from induction (governed by <span class="math inline"><em>K</em><sub><em>I</em></sub> &lt; <em>K</em><sub><em>A</em></sub></span>) to corepression (<span class="math inline"><em>K</em><sub><em>I</em></sub> &gt; <em>K</em><sub><em>A</em></sub></span>) as the ligand transitions from preferentially binding to the inactive repressor state to stabilizing the active state. Furthermore, this general approach can be used to describe a variety of other motifs such as activation, multiple repressor binding sites, and combinations of activator and repressor binding sites <span class="citation" data-cites="bintu2005a brewster2014 weinert2014">(Bintu, Buchler, Garcia, Gerland, Hwa, Kondev, Kuhlman, et al. <a href="#ref-bintu2005a" role="doc-biblioref">2005</a>; Brewster et al. <a href="#ref-brewster2014" role="doc-biblioref">2014</a>; Weinert et al. <a href="#ref-weinert2014" role="doc-biblioref">2014</a>)</span>.</p>
<p>The formula presented in Eq. <a href="#eq:fold_change_full">5</a> enables us to make precise quantitative statements about induction profiles. Motivated by the broad range of predictions implied by Eq. <a href="#eq:fold_change_full">5</a>, we designed a series of experiments using the <em>lac</em> system in <em>E. coli</em> to tune the control parameters for a simple repression genetic circuit. As discussed in Fig. ?? (C), previous studies from our lab have provided well-characterized values for many of the parameters in our experimental system, leaving only the values of the the MWC parameters (<span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>) to be determined. We note that while previous studies have obtained values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>L</em> = <em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></sup></span> <span class="citation" data-cites="ogorman1980 daber2011a">(O’Gorman et al. <a href="#ref-ogorman1980" role="doc-biblioref">1980</a>; Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span>, they were either based upon biochemical experiments or <em>in vivo</em> conditions involving poorly characterized transcription factor copy numbers and gene copy numbers. These differences relative to our experimental conditions and fitting techniques led us to believe that it was important to perform our own analysis of these parameters. After inferring these three MWC parameters (see , Section “” for details regarding the inference of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, which was fitted separately from <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>), we were able to predict the input/output response of the system under a broad range of experimental conditions. For example, this framework can predict the response of the system at different repressor copy numbers <span class="math inline"><em>R</em></span>, repressor-operator affinities <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, inducer concentrations <span class="math inline"><em>c</em></span>, and gene copy numbers (see Chapter 7.2).</p>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-ackers1982">
<p>Ackers, Gary K, and Alexander D Johnson. 1982. “Quantitative Model for Gene Regulation by A Phage Repressor.” <em>Proc. Natl. Acad. Sci. USA</em>, 5.</p>
</div>
<div id="ref-bintu2005a">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, Thomas Kuhlman, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Applications.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 125–35. <a href="https://doi.org/10.1016/j.gde.2005.02.006">https://doi.org/10.1016/j.gde.2005.02.006</a>.</p>
</div>
<div id="ref-bintu2005">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Models.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 116–24. <a href="https://doi.org/10.1016/j.gde.2005.02.007">https://doi.org/10.1016/j.gde.2005.02.007</a>.</p>
</div>
<div id="ref-brewster2012">
<p>Brewster, Robert C., Daniel L. Jones, and Rob Phillips. 2012. “Tuning Promoter Strength Through RNA Polymerase Binding Site Design in Escherichia Coli.” Edited by Erik van Nimwegen. <em>PLoS Computational Biology</em> 8 (12): e1002811. <a href="https://doi.org/10.1371/journal.pcbi.1002811">https://doi.org/10.1371/journal.pcbi.1002811</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-buchler2003">
<p>Buchler, N. E., U. Gerland, and T. Hwa. 2003. “On Schemes of Combinatorial Transcription Logic.” <em>Proceedings of the National Academy of Sciences</em> 100 (9): 5136–41. <a href="https://doi.org/10.1073/pnas.0930314100">https://doi.org/10.1073/pnas.0930314100</a>.</p>
</div>
<div id="ref-daber2011a">
<p>Daber, Robert, Matthew A. Sochor, and Mitchell Lewis. 2011. “Thermodynamic Analysis of Mutant Lac Repressors.” <em>Journal of Molecular Biology</em>, The Operon Model and its Impact on Modern Molecular Biology, 409 (1): 76–87. <a href="https://doi.org/10.1016/j.jmb.2011.03.057">https://doi.org/10.1016/j.jmb.2011.03.057</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-klumpp2008">
<p>Klumpp, Stefan, and Terence Hwa. 2008. “Growth-Rate-Dependent Partitioning of RNA Polymerases in Bacteria.” <em>Proceedings of the National Academy of Sciences</em> 105 (51): 20245–50. <a href="https://doi.org/10.1073/pnas.0804953105">https://doi.org/10.1073/pnas.0804953105</a>.</p>
</div>
<div id="ref-kuhlman2007">
<p>Kuhlman, T., Z. Zhang, M. H. Saier, and T. Hwa. 2007. “Combinatorial Transcriptional Control of the Lactose Operon of Escherichia Coli.” <em>Proceedings of the National Academy of Sciences</em> 104 (14): 6043–8. <a href="https://doi.org/10.1073/pnas.0606717104">https://doi.org/10.1073/pnas.0606717104</a>.</p>
</div>
<div id="ref-marzen2013">
<p>Marzen, Sarah, Hernan G. Garcia, and Rob Phillips. 2013. “Statistical Mechanics of MonodWymanChangeux (MWC) Models.” <em>Journal of Molecular Biology</em>, Allosteric Interactions and Biological Regulation (Part I), 425 (9): 1433–60. <a href="https://doi.org/10.1016/j.jmb.2013.03.013">https://doi.org/10.1016/j.jmb.2013.03.013</a>.</p>
</div>
<div id="ref-monod1965">
<p>Monod, Jacque, Jeffries Wyman, and Jean-Pierre Changeux. 1965. “On the Nature of Allosteric Transitions: A Plausible Model.” <em>Journal of Molecular Biology</em> 12 (1): 88–118. <a href="https://doi.org/10.1016/S0022-2836(65)80285-6">https://doi.org/10.1016/S0022-2836(65)80285-6</a>.</p>
</div>
<div id="ref-ogorman1980">
<p>O’Gorman, R. B., J. M. Rosenberg, O. B. Kallai, R. E. Dickerson, K. Itakura, A. D. Riggs, and K. S. Matthews. 1980. “Equilibrium Binding of Inducer to Lac Repressor Operator DNA Complex.” <em>Journal of Biological Chemistry</em> 255 (21): 10107–14.</p>
</div>
<div id="ref-phillips2015">
<p>Phillips, Rob. 2015. “Napoleon Is in Equilibrium.” <em>Annual Review of Condensed Matter Physics</em> 6 (1): 85–111. <a href="https://doi.org/10.1146/annurev-conmatphys-031214-014558">https://doi.org/10.1146/annurev-conmatphys-031214-014558</a>.</p>
</div>
<div id="ref-vilar2003">
<p>Vilar, José M. G., and Stanislas Leibler. 2003. “DNA Looping and Physical Constraints on Transcription Regulation.” <em>Journal of Molecular Biology</em> 331 (5): 981–89. <a href="https://doi.org/10.1016/S0022-2836(03)00764-2">https://doi.org/10.1016/S0022-2836(03)00764-2</a>.</p>
</div>
<div id="ref-weinert2014">
<p>Weinert, Franz M., Robert C. Brewster, Mattias Rydenfelt, Rob Phillips, and Willem K. Kegel. 2014. “Scaling of Gene Expression with Transcription-Factor Fugacity.” <em>Physical Review Letters</em> 113 (25). <a href="https://doi.org/10.1103/PhysRevLett.113.258101">https://doi.org/10.1103/PhysRevLett.113.258101</a>.</p>
</div>
</div>