<h2 id="introduction">Introduction</h2>
<p>Understanding how organisms sense and respond to changes in their environment has long been a central theme of biological inquiry. At the cellular level, this interaction is mediated by a diverse collection of molecular signaling pathways. A pervasive mechanism of signaling in these pathways is allosteric regulation, in which the binding of a ligand induces a conformational change in some target molecule, triggering a signaling cascade <span class="citation" data-cites="lindsley2006">(<span class="citeproc-not-found" data-reference-id="lindsley2006"><strong>???</strong></span>)</span>. One of the most important examples of such signaling is offered by transcriptional regulation, where a transcription factor’s propensity to bind to DNA will be altered upon binding to an allosteric effector.</p>
<p>Despite allostery’s ubiquity, we lack a formal, rigorous, and generalizable framework for studying its effects across the broad variety of contexts in which it appears. A key example of this is transcriptional regulation, in which allosteric transcription factors can be induced or corepressed by binding to a ligand. An allosteric transcription factor can adopt multiple conformational states, each of which has its own affinity for the ligand and for its DNA target site. <em>In vitro</em> studies have rigorously quantified the equilibria of different conformational states for allosteric transcription factors and measured the affinities of these states to the ligand <span class="citation" data-cites="harman2001 lanfranco2017">(Harman <a href="#ref-harman2001">2001</a>; Lanfranco et al. <a href="#ref-lanfranco2017">2017</a>)</span>. In spite of these experimental observations, the lack of a coherent quantitative model for allosteric transcriptional regulation has made it impossible to predict the behavior of even a simple genetic circuit across a range of regulatory parameters.</p>
<p>The ability to predict circuit behavior robustly— that is, across both broad ranges of parameters and regulatory architectures —is important for multiple reasons. First, in the context of a specific gene, accurate prediction demonstrates that all components relevant to the gene’s behavior have been identified and characterized to sufficient quantitative precision. Second, in the context of genetic circuits in general, robust prediction validates the model that generated the prediction. Possessing a validated model also has implications for future work. For example, when we have sufficient confidence in the model, a single data set can be used to accurately extrapolate a system’s behavior in other conditions. Moreover, there is an essential distinction between a predictive model, which is used to predict a system’s behavior given a set of input variables, and a retroactive model, which is used to describe the behavior of data that has already been obtained. We note that even some of the most careful and rigorous analysis of transcriptional regulation often entails only a retroactive reflection on a single experiment. This raises the fear that each regulatory architecture may require a unique analysis that cannot carry over to other systems, a worry that is exacerbated by the prevalent use of phenomenological functions (e.g. Hill functions or ratios of polynomials) that can analyze a single data set but cannot be used to extrapolate a system’s behavior in other conditions <span class="citation" data-cites="setty2003 poelwijk2011 rogers2015 rohlhill2017">(Setty et al. <a href="#ref-setty2003">2003</a>; <span class="citeproc-not-found" data-reference-id="poelwijk2011"><strong>???</strong></span>; Rogers et al. <a href="#ref-rogers2015">2015</a>; Rohlhill, Sandoval, and Papoutsakis <a href="#ref-rohlhill2017">2017</a>)</span>.</p>
<p>This work explores what happens when theory takes center stage, namely, we first write down the equations governing a system and describe its expected behavior across a wide array of experimental conditions, and only then do we set out to experimentally confirm these results. Building upon previous work <span class="citation" data-cites="garcia2011 brewster2014 weinert204">(H. G. Garcia and Phillips <a href="#ref-garcia2011">2011</a>; Brewster et al. <a href="#ref-brewster2014">2014</a>; <span class="citeproc-not-found" data-reference-id="weinert204"><strong>???</strong></span>)</span> and the work of Monod, Wyman, and Changeux <span class="citation" data-cites="monod1965">(Monod, Wyman, and Changeux <a href="#ref-monod1965">1965</a>)</span>, we present a statistical mechanical rendering of allostery in the context of induction and corepression (shown schematically in Fig. [<a href="#fig:ind_corep_prop">1</a>] (A) and henceforth referred to as the MWC model) and use it as the basis of parameter-free predictions which we then test experimentally. More specifically, we study the simple repression motif – a widespread bacterial genetic regulatory architecture in which binding of a transcription factor occludes binding of an RNA polymerase, thereby inhibiting transcription initiation. The MWC model stipulates that an allosteric protein fluctuates between two distinct conformations – an active and inactive state – in thermodynamic equilibrium <span class="citation" data-cites="monod1965">(Monod, Wyman, and Changeux <a href="#ref-monod1965">1965</a>)</span>. During induction, for example, effector binding increases the probability that a repressor will be in the inactive state, weakening its ability to bind to the promoter and resulting in increased expression. To test the predictions of our model across a wide range of operator binding strengths and repressor copy numbers, we design an <em>E. coli</em> genetic construct in which the binding probability of a repressor regulates gene expression of a fluorescent reporter.</p>
<p>In total, the work presented here demonstrates that one extremely compact set of parameters can be applied self-consistently and predictively to different regulatory situations including simple repression on the chromosome, cases in which decoy binding sites for repressor are put on plasmids, cases in which multiple genes compete for the same regulatory machinery, cases involving multiple binding sites for repressor leading to DNA looping, and induction by signaling <span class="citation" data-cites="garcia2011 garcia2011b brewster2012 boedicker2013 boedicker2013b">(H. G. Garcia and Phillips <a href="#ref-garcia2011">2011</a>; Garcia et al. <a href="#ref-garcia2011b">2011</a>; <span class="citeproc-not-found" data-reference-id="brewster2012"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="boedicker2013b"><strong>???</strong></span>; Boedicker, Garcia, and Phillips <a href="#ref-boedicker2013">2013</a>)</span>. Thus, rather than viewing the behavior of each circuit as giving rise to its own unique input-output response, the MWC model provides a means to characterize these seemingly diverse behaviors using a single unified framework governed by a small set of parameters.</p>
<div id="fig:ind_corep_prop" class="fignos">
<figure>
<img src="ch2_fig1.png" alt="Figure 1: (A) We consider a promoter regulated solely by an allosteric repressor. When bound, the repressor prevents RNAP from binding and initiating transcription. Induction is characterized by the addition of an effector which binds to the repressor and stabilizes the inactive state (defined as the state which has a low affinity for DNA), thereby increasing gene expression. In corepression, the effector stabilizes the repressor’s active state and thus further reduces gene expression. We list several characterized examples of induction and corepression that support different physiological roles in E. coli . (B) A schematic regulatory response of the two architectures shown in Panel (A) plotting the fold-change in gene expression as a function of effector concentration, where fold-change is defined as the ratio of gene expression in the presence versus the absence of repressor. We consider the following key phenotypic properties that describe each response curve: the minimum response (leakiness), the maximum response (saturation), the difference between the maximum and minimum response (dynamic range), the concentration of ligand which generates a fold-change halfway between the minimal and maximal response (EC_{50}), and the log-log slope at the midpoint of the response (effective Hill coefficient). (C) Over time we have refined our understanding of simple repression architectures. A first round of experiments used colorimetric assays and quantitative Western blots to investigate how single-site repression is modified by the repressor copy number and repressor-DNA binding energy . A second round of experiments used video microscopy to probe how the copy number of the promoter and presence of competing repressor binding sites affect gene expression, and we use this data set to determine the free energy difference between the repressor’s inactive and active conformations . Here we used flow cytometry to determine the inducer-repressor dissociation constants and demonstrate that with these parameters we can predict a priori the behavior of the system for any repressor copy number, DNA binding energy, gene copy number, and inducer concentration." data-short-caption="This is the short cap" /><figcaption><span>Figure 1:</span> (A) We consider a promoter regulated solely by an allosteric repressor. When bound, the repressor prevents RNAP from binding and initiating transcription. Induction is characterized by the addition of an effector which binds to the repressor and stabilizes the inactive state (defined as the state which has a low affinity for DNA), thereby increasing gene expression. In corepression, the effector stabilizes the repressor’s active state and thus further reduces gene expression. We list several characterized examples of induction and corepression that support different physiological roles in <em>E. coli</em> . (B) A schematic regulatory response of the two architectures shown in Panel (A) plotting the fold-change in gene expression as a function of effector concentration, where fold-change is defined as the ratio of gene expression in the presence versus the absence of repressor. We consider the following key phenotypic properties that describe each response curve: the minimum response (leakiness), the maximum response (saturation), the difference between the maximum and minimum response (dynamic range), the concentration of ligand which generates a fold-change halfway between the minimal and maximal response (<span class="math inline"><em>E</em><em>C</em><sub>50</sub></span>), and the log-log slope at the midpoint of the response (effective Hill coefficient). (C) Over time we have refined our understanding of simple repression architectures. A first round of experiments used colorimetric assays and quantitative Western blots to investigate how single-site repression is modified by the repressor copy number and repressor-DNA binding energy . A second round of experiments used video microscopy to probe how the copy number of the promoter and presence of competing repressor binding sites affect gene expression, and we use this data set to determine the free energy difference between the repressor’s inactive and active conformations . Here we used flow cytometry to determine the inducer-repressor dissociation constants and demonstrate that with these parameters we can predict <em>a priori</em> the behavior of the system for any repressor copy number, DNA binding energy, gene copy number, and inducer concentration.</figcaption>
</figure>
</div>
<div id="refs" class="references">
<div id="ref-boedicker2013">
<p>Boedicker, James Q., Hernan G. Garcia, and Rob Phillips. 2013. “Theoretical and Experimental Dissection of DNA Loop-Mediated Repression.” <em>Physical Review Letters</em> 110 (1): 018101. <a href="https://doi.org/10.1103/PhysRevLett.110.018101" class="uri">https://doi.org/10.1103/PhysRevLett.110.018101</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022" class="uri">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-garcia2011b">
<p>Garcia, Hernan G., Heun Jin Lee, James Q. Boedicker, and Rob Phillips. 2011. “Comparison and Calibration of Different Reporters for Quantitative Analysis of Gene Expression.” <em>Biophysical Journal</em> 101 (3): 535–44. <a href="https://doi.org/10.1016/j.bpj.2011.06.026" class="uri">https://doi.org/10.1016/j.bpj.2011.06.026</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108" class="uri">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-harman2001">
<p>Harman, James G. 2001. “Allosteric Regulation of the cAMP Receptor Protein.” <em>Biochimica et Biophysica Acta (BBA) - Protein Structure and Molecular Enzymology</em> 1547 (1): 1–17. <a href="https://doi.org/10.1016/S0167-4838(01)00187-X" class="uri">https://doi.org/10.1016/S0167-4838(01)00187-X</a>.</p>
</div>
<div id="ref-lanfranco2017">
<p>Lanfranco, Maria Fe, Fernanda Gárate, Ashton J. Engdahl, and Rodrigo A. Maillard. 2017. “Asymmetric Configurations in a Reengineered Homodimer Reveal Multiple Subunit Communication Pathways in Protein Allostery.” <em>Journal of Biological Chemistry</em> 292 (15): 6086–93. <a href="https://doi.org/10.1074/jbc.M117.776047" class="uri">https://doi.org/10.1074/jbc.M117.776047</a>.</p>
</div>
<div id="ref-monod1965">
<p>Monod, Jacque, Jeffries Wyman, and Jean-Pierre Changeux. 1965. “On the Nature of Allosteric Transitions: A Plausible Model.” <em>Journal of Molecular Biology</em> 12 (1): 88–118. <a href="https://doi.org/10.1016/S0022-2836(65)80285-6" class="uri">https://doi.org/10.1016/S0022-2836(65)80285-6</a>.</p>
</div>
<div id="ref-rogers2015">
<p>Rogers, Jameson K., Christopher D. Guzman, Noah D. Taylor, Srivatsan Raman, Kelley Anderson, and George M. Church. 2015. “Synthetic Biosensors for Precise Gene Control and Real-Time Monitoring of Metabolites.” <em>Nucleic Acids Research</em> 43 (15): 7648–60. <a href="https://doi.org/10.1093/nar/gkv616" class="uri">https://doi.org/10.1093/nar/gkv616</a>.</p>
</div>
<div id="ref-rohlhill2017">
<p>Rohlhill, Julia, Nicholas R. Sandoval, and Eleftherios T. Papoutsakis. 2017. “Sort-Seq Approach to Engineering a Formaldehyde-Inducible Promoter for Dynamically Regulated Escherichia Coli Growth on Methanol.” <em>ACS Synthetic Biology</em> 6 (8): 1584–95. <a href="https://doi.org/10.1021/acssynbio.7b00114" class="uri">https://doi.org/10.1021/acssynbio.7b00114</a>.</p>
</div>
<div id="ref-setty2003">
<p>Setty, Y., A. E. Mayo, M. G. Surette, and U. Alon. 2003. “Detailed Map of a Cis-Regulatory Input Function.” <em>Proceedings of the National Academy of Sciences</em> 100 (13): 7702–7. <a href="https://doi.org/10.1073/pnas.1230759100" class="uri">https://doi.org/10.1073/pnas.1230759100</a>.</p>
</div>
</div>