<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta name="viewport" content="width=device-width, minimum-scale=1.0">
<title>Chapter VII</title>


<script type="text/x-mathjax-config">
	MathJax.Hub.Config({
    "HTML-CSS" : {
        availableFonts : ["STIX"],
        preferredFont : "Gyre Pagella",
        webFont : "STIX-Web",
        imageFont : null
    },
	  TeX: {
		equationNumbers: {
		  autoNumber: "AMS"
		}
	  },
	  tex2jax: {
		inlineMath: [['$','$']], 
		displayMath: [['$$','$$']],
		processEscapes: true,
	  }
	});
  </script>
  <style>
	.eqnos { display: inline-block; position: relative; width: 100%; }
	.eqnos br { display: none; }
	.eqnos-number { position: absolute; right: 0em; top: 50%; line-height: 0; }
  </style>
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>
  

<link rel="icon" href="https://gchure.github.io/phd//favicon.ico" type="image/x-icon">
<link href="https://fonts.googleapis.com/css?family=Nanum+Myeongjo&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Alegreya|Lora&display=swap" rel="stylesheet">

<link rel="stylesheet" href="https://gchure.github.io/phd//doks-theme/assets/css/style.css">

	</head>
	<body class="grey" data-spy="scroll" data-target=".js-scrollspy">
		


	<header class="site-header">
		<div class="container">
			<div class="row">
				<div class="col-xs-12">
					
						<a href="https://gchure.github.io/phd//" class="site-header__logo"><i class="icon icon--chevron-left"></i> Home</a>
					
					
				</div><!-- /.col -->
			</div><!-- /.row -->
		</div><!-- /.container -->
	</header><!-- /.site-header -->


		<div class="hero-subheader" style="background-image: url(https://gchure.github.io/phd//doks-theme/assets/images/islands.jpg); background-position: center; object-fit: cover; width:100%;">

			<div class="container" style="background-color: rgba(0,0,0, 0.75); border:0.75px solid white;">
				<div class="row">
					<div class="col-md-7">
						<div class="align-container" data-mh>
							<div class="align-inner">
								
									<h2 class="hero-subheader__title">Chapter VII</h2>
								
								
									<h1 class="hero-subheader__title">Supplemental Information For Chapter III: Predictive Shifts in Free Energy Couple Mutations to Their Phenotypic Consequences
</h1>
								
								
								<!-- 
									
										<a href="https://gchure.github.io/phd/chapter_06" class="btn btn--dark btn--rounded btn--w-icon btn--w-icon-left">
											<i class="icon icon--arrow-left"></i>	
										</a>
									
									
										<a href="https://gchure.github.io/phd/chapter_08" class="btn btn--dark btn--rounded btn--w-icon">
											<i class="icon icon--arrow-right"></i>
										</a>
									
								 -->
							</div><!-- /.align-inner -->
						</div><!-- /.align-container -->
					</div><!-- /.col -->
					
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.hero-subheader -->
		
				<nav class="page-nav">
					<div class="container" >
						<div class="row" >
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/chapter_06" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter VI
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_08" class="page-nav__item page-nav__item--next">
										Chapter VIII
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			

		<div class="section">
			<div class="container">
				<div class="row">
					<div class="col-md-7">
						<div class="content">
							<strong>Published as …</strong>
<hr/>
<p>A version of this chapter originally appeared as Chure, G; Razo-Mejia, M., Belliveau, N.M.; Kaczmarek, Zofii A.; Einav, T.; Barnes, Stephanie L.; Lewis, M., and Phillips, R. (2019). <em>Predictive Shifts in Free Energy Couple Mutations to Their Phenotypic Consequences.</em> PNAS 116(37) DOI: https://doi.org/10.1073/pnas.1907869116. G.C., M.R.M, N.M.B., Z.A.K., and S.L.B designed the experiments and collected and analyzed data. G.C. developed theoretical treatment of free energy shifts. G.C., M.R.M, N.M.B., Z.A.K., T.E., S.L.B., and R.P. designed the research project. G.C. and R.P. wrote the paper. M.L. provided guidance and advice.</p>
<h2 id="non-monotonic-behavior-of-delta-f-under-changing-k_a-and-k_i">Non-Monotonic Behavior of <span class="math inline"><em>Δ</em><em>F</em></span> Under Changing <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span></h2>
<p>In Chapter 3, we illustrated that perturbations only to the allosteric parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> relative to the wild-type values can result in a non-monotonic dependence of <span class="math inline"><em>Δ</em><em>F</em></span> on the inducer concentration <span class="math inline"><em>c</em></span>. In this section, we prove that when the ratio of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> to <span class="math inline"><em>K</em><sub><em>I</em></sub></span> is the same between the mutant and wild-type proteins, the function must be monotonic. This section is paired with an interactive figure available on the <a href="https://www.rpgroup.caltech.edu/mwc_mutants">paper website</a> which illustrates how scaling <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> relative to the wild-type value results in non-monotonic behavior.</p>
<p>     We define a monotonic function as a continuous function whose derivative does not change sign across the domain upon which it is defined. To show that <span class="math inline"><em>Δ</em><em>F</em></span> is non-monotonic when <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are perturbed, we can compute the derivative of <span class="math inline"><em>Δ</em><em>F</em></span> with respect to the inducer concentration <span class="math inline"><em>c</em></span> and evaluate the sign of the derivative at the limits of inducer concentration. If the sign of the derivative is different at the limits of <span class="math inline"><em>c</em> = 0</span> and <span class="math inline"><em>c</em> ≫ 0</span>, we can see that the function is non-monotonic. However, if the sign is the same in both limits, we can not say conclusively if it is non-monotonic and must consider other diagnostics.</p>
<p>     The free energy difference between a mutant and wild-type repressor when all parameters other than <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are unperturbed can be written as <span id="eq:delF_pact"><br /><span class="math display">$$
\beta\Delta F(c) = -\log\left({\left[1 + e^{-\beta\Delta\varepsilon_{AI} }\left({1
+ {c \over K_I^\text{(mut)} } \over 1 + {c \over
K_A^\text{(mut)} }}\right)^2\right]^{-1} \over \left[1 +
e^{-\beta\Delta\varepsilon_{AI} }\left({1 + {c \over K_I^\text{(wt)} } \over 1
+ {c \over K_A^\text{(wt)} }}\right)^2\right]^{-1} }\right),
\qquad(1)$$</span><br /></span> in which <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is the energy difference between the active and inactive states of the repressor, <span class="math inline"><em>c</em></span> is the inducer concentration, and <span class="math inline"><em>β</em> = 1/<em>k</em><sub><em>B</em></sub><em>T</em></span> where <span class="math inline"><em>k</em><sub><em>B</em></sub></span> is the Boltzmann constant and <span class="math inline"><em>T</em></span> is the temperature. The derivative with respect to <span class="math inline"><em>c</em></span>, which we determined using Mathematica’s (Wolfram Research, version 11.2) symbolic computing ability, is given as <span id="eq:partial_explicit"><br /><span class="math display">$$
\begin{gathered}
{\partial \beta\Delta F(c) \over \partial c} =
2e^{-\beta\Delta\varepsilon_{AI} }\left(\frac{ {K_A^\text{(mut)} }^2 \left(K_A^\text{(mut)}
- K_I^\text{(mut)}\right)\left(c + K_I^\text{(mut)}\right)}{\left(c +
K_A^\text{(mut)}\right) \left[\left(c +
K_A^\text{(mut)}\right)^2{K_I^\text{(mut)} }^2 +
e^{-\beta\Delta\varepsilon_{AI} }{K_A^\text{(mut)} }^2\left(c +
K_I^\text{(mut)}\right)^2\right]} \right. \\ - \left.
\frac{ {K_A^\text{(wt)} }^2\left(K_A^\text{(wt)} - K_I^\text{(wt)}\right)\left(c +
K_I^\text{(wt)}\right)}{\left(c + K_A^\text{(wt)}\right) \left[\left(c +
K_A^\text{(wt)}\right)^2{K_I^\text{(wt)} }^2 +
e^{-\beta\Delta\varepsilon_{AI} }{K_A^\text{(wt)} }^2\left(c +
K_I^\text{(wt)}\right)^2\right]} \right).
\end{gathered}
\qquad(2)$$</span><br /></span> This unwieldy expression can be simplified by defining the values of <span class="math inline"><em>K</em><sub><em>A</em></sub><sup>(mut)</sup> = <em>θ</em><em>K</em><sub><em>A</em></sub><sup>(wt)</sup></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub><sup>(mut)</sup> = <em>θ</em><em>K</em><sub><em>I</em></sub><sup>(wt)</sup></span> as relative changes to the wild-type values where <span class="math inline"><em>θ</em></span> is a scaling parameter. While we can permit <span class="math inline"><em>K</em><sub><em>A</em></sub><sup>(mut)</sup></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub><sup>(mut)</sup></span> to vary by different degrees, we will consider the case in which they are equally perturbed such that the ratio of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> to <span class="math inline"><em>K</em><sub><em>I</em></sub></span> is the same between the mutant and wild-type versions of the repressor. While the equations become more cumbersome when one permits the dissociation constants to vary by different amounts (i.e. <span class="math inline"><em>θ</em><sub><em>K</em><sub><em>A</em></sub></sub>, <em>θ</em><sub><em>K</em><sub><em>I</em></sub></sub></span>), one arrives at the same conclusion. This definition allows us to rewrite Eq. 2 in the form of <span id="eq:partial_theta"><br /><span class="math display">$$
\begin{gathered}
{\partial \beta\Delta F(c) \over \partial c} = 2{K_A^\text{(wt)} }e^{-\beta\Delta\varepsilon_{AI} }\left({\theta^3 \left({K_A^\text{(wt)} }- {K_I^\text{(wt)} }\right)\left(c + \theta {K_I^\text{(wt)} }\right) \over \left(c + \theta {K_A^\text{(wt)} }\right)\left[\theta^2 {K_I^\text{(wt)} }^2\left(c + \theta {K_A^\text{(wt)} }\right)^2 + e^{-\beta\Delta\varepsilon_{AI} }\theta^2{K_A^\text{(wt)} }^2\left(c + \theta {K_I^\text{(wt)} }\right)^2\right]} \right. \\ - \left. {\left({K_A^\text{(wt)} } - {K_I^\text{(wt)} }\right)\left(c + {K_I^\text{(wt)} }\right) 
\over \left(c + {K_A^\text{(wt)} }\right) \left[\left(c +
{K_A^\text{(wt)} }\right)^2{K_I^\text{(wt)} }^2 +
e^{-\beta\Delta\varepsilon_{AI} }{K_A^\text{(wt)} }^2\left(c +
{K_I^\text{(wt)} }\right)^2\right]} \right).
\end{gathered}
\qquad(3)$$</span><br /></span>      With this derivative in hand, we can examine the limits of inducer concentration. As discussed in the main text, the free energy difference between the mutant and wild-type repressors when <span class="math inline"><em>c</em> = 0</span> should be equal to <span class="math inline">0</span>. However, the derivative at <span class="math inline"><em>c</em> = 0</span> will be different between the wild-type and the mutant. In this limit, Eq. 3 simplifies to <span id="eq:partial_c0"><br /><span class="math display">$$
{\partial \beta\Delta F(c) \over \partial c}\bigg\vert_{c = 0} =
{2e^{-\beta\Delta\varepsilon_{AI} }\left({K_A^\text{(wt)} } - {K_I^\text{(wt)} }\right) \over {K_A^\text{(wt)} }{K_I^\text{(wt)} }\left(1
+ e^{-\beta\Delta\varepsilon_{AI} }\right)}\left({1 \over \theta} - 1\right).
\qquad(4)$$</span><br /></span></p>
<p>     When <span class="math inline"><em>θ</em> &lt; 1</span>, meaning that the affinity of the active and inactive states of the repressor to the inducer is increased relative to wild-type, the derivative is positive. Thus, the repressor bound state of the promoter becomes less energetically favorable than the repressor bound state. Similarly, if <span class="math inline"><em>θ</em> &gt; 1</span>, binding of the inducer to the mutant repressor is weaker than the wild-type repressor, making <span class="math inline">∂<em>β</em><em>Δ</em><em>F</em>(<em>c</em>)/∂<em>c</em> &lt; 0</span>, meaning the repressor bound state becomes more energetically favorable than the repressor unbound state of the promoter.</p>
<p>With an intuition for the sign of the derivative when <span class="math inline"><em>c</em> = 0</span>, we can compute the derivative at another extreme where <span class="math inline"><em>c</em> ≫ 0</span>. Here, Eq. 3 reduces to <span id="eq:partial_cinf"><br /><span class="math display">$$
{\partial \beta\Delta F(c) \over \partial c}\bigg\vert_{c \gg 0} \approx
{2e^{-\beta\Delta\varepsilon_{AI} }{K_A^\text{(wt)} }^2({K_A^\text{(wt)} } -
{K_I^\text{(wt)} }) \over c^2\left({K_I^\text{(wt)} }^2 +
e^{-\beta\Delta\varepsilon_{AI} }{K_A^\text{(wt)} }^2\right)} \left(\theta -
1\right).
\qquad(5)$$</span><br /></span>      When <span class="math inline"><em>θ</em> &gt; 1</span>, Eq. 5 is positive. This is the opposite sign of the derivative when <span class="math inline"><em>c</em> = 0</span> when <span class="math inline"><em>θ</em> &gt; 1</span>. When <span class="math inline"><em>θ</em> &lt; 1</span>, Eq. 5 becomes negative whereas Eq. 4 is positive. As the derivative of <span class="math inline"><em>Δ</em><em>F</em></span> with respect to <span class="math inline"><em>c</em></span> changes signs across the defined range of inducer concentrations, we can say the function is non-monotonic.</p>
<p>     Fig. 1 shows the non-monotonic behavior of <span class="math inline"><em>Δ</em><em>F</em></span> when <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> change by the same factor <span class="math inline"><em>θ</em></span> (maintaining the wild-type ratio, Fig. 1 (A)) and when <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> change by different factors (Fig. 1 (B)). In both cases, non-monotonic behavior is observed with the peak difference in the free energy covering several <span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span>. We have hosted an interactive figure similar to Fig. 1 on the <a href="https://rpgroup.caltech.edu/mwc_mutants">paper website</a> where the reader can modify how <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are affected by a mutation and examine how the active probability, free energy difference, and <span class="math inline">∂<em>β</em><em>Δ</em><em>F</em>/∂<em>c</em></span> are tuned.</p>
<figure>
<img src="ch7_figS1.png" id="fig:nonmono" data-short-caption="Non-monotonic behavior of $\Delta F$ with changes in $K_A$ and $K_I$." alt="" /><figcaption>Figure 1: <strong>Non-monotonic behavior of <span class="math inline"><em>Δ</em><em>F</em></span> with changes in <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>.</strong> Middle column shows the allosteric contribution of free energy <span class="math inline"><em>F</em></span> plotted as a function of the inducer concentration. Right column shows the free energy difference <span class="math inline"><em>Δ</em><em>F</em></span> as a function of inducer concentration, revealing non-monotonicity. (A) Behavior of <span class="math inline"><em>F</em></span> and <span class="math inline"><em>Δ</em><em>F</em></span> when the values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> change relative to wild-type, but maintain the same ratio. <span class="math inline"><em>θ</em></span> is the scaling factor for both inducer dissociation constants. (B) Behavior of <span class="math inline"><em>F</em></span> and <span class="math inline"><em>Δ</em><em>F</em></span> when the values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> change relative to the wild-type, but by different factors. In both panels, the wild-type parameter values were taken to be <span class="math inline"><em>K</em><sub><em>A</em></sub> = 200 <em>μ</em></span>M, <span class="math inline"><em>K</em><sub><em>I</em></sub> = 1 <em>μ</em></span>M and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>. An interactive version of this figure is available on the <a href="http://rpgroup.caltech.edu/mwc_mutants">paper website.</a> The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS1.py">Python code (<code>ch7_figS1.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="bayesian-parameter-estimation-for-dna-binding-mutants">Bayesian Parameter Estimation For DNA Binding Mutants</h2>
<p>In this section, we outline the statistical model used in this work to estimate the DNA binding energy for a given mutation in the DNA binding domain. The methodology presented here is similar to that performed in Chapter 2 and outlined in accompanying Chapter 6. In the following text, we take a very detailed approach to vetting the robustness of our statistical inference machinery as determination of parameter values is critical to assessing the effects of mutations. Similarly to what is presented in Chapter 6, we begin with a derivation of our statistical model using Bayes’ theorem and then perform a series of principled steps to validate our choices of priors, ensure computational feasibility, and assess the validity of the model given the collected data. This work follows the analysis pipeline outlined by Michael Betancourt in his case-study entitled <a href="https://betanalpha.github.io/assets/case_studies/principled_bayesian_workflow.html">“Towards A Principled Bayesian Workflow.”</a></p>
<p>     The second subsection <em>Building a Generative Statistical Model</em> lays out the statistical model used in this work to estimate the DNA binding energy and the error term <span class="math inline"><em>σ</em></span>. The subsequent subsections – <em>Prior Predictive Checks</em>, <em>Simulation Based Calibration</em>, and <em>Posterior Predictive Checks</em> – define and summarize a series of tests that ensure that the parameters of the statistical model can be identified and are computationally tractable. To understand how we defined our statistical model, only the second subsection is needed.</p>
<h3 id="calculation-of-the-fold-change-in-gene-expression">Calculation of the Fold-Change in Gene Expression</h3>
<p>     We appreciate the subtleties of the efficiency of photon detection in the flow cytometer, fluorophore maturation and folding, and autofluorescence correction, and we understand the importance in modeling the effects that these processes have on the reported value of the fold-change. However, in order to be consistent with the methods used in the literature, we took a more simplistic approach to calculate the fold-change. Given a set of fluorescence measurements of the constitutive expression control (<span class="math inline"><em>R</em> = 0</span>), an autofluorescence control (no YFP), and the experimental strain (<span class="math inline"><em>R</em> &gt; 0</span>), we calculate the fold-change as <span id="eq:experimental_foldchange"><br /><span class="math display">$$
\text{fold-change} = {\langle I_\text{cell}(R &gt; 0)\rangle - \langle I_\text{autofluorescence}\rangle \over \langle I_\text{cell}(R = 0) \rangle - \langle I_\text{autofluorescence}\rangle}.
\qquad(6)$$</span><br /></span> It is important to note here that for a given biological replicate, we consider only a point estimate of the mean fluorescence for each sample and perform a simple subtraction to adjust for background fluorescence. For the analysis going forward, all mentions of measured fold-change are determined by this calculation.</p>
<h3 id="building-a-generative-statistical-model">Building a Generative Statistical Model</h3>
<p>To identify the minimal parameter set affected by a mutation, we assume that mutations in the DNA binding domain of the repressor alters only the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, while the other parameters of the repressor are left unperturbed from their wild-type values. As a first approach, we can assume that all of the other parameters are known without error and can be taken as constants in our physical model. Ultimately, we want to know how probable a particular value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is given a set of experimental measurements <span class="math inline"><em>y</em></span>. Bayes’ theorem computes this distribution, termed the <em>posterior distribution</em> as <span id="eq:simple_bayes_DNA"><br /><span class="math display">$$
g(\Delta\varepsilon_{RA}\,\vert\, y) = {f(y\,\vert\,
 \Delta\varepsilon_{RA}) g(\Delta\varepsilon_{RA}) \over f(y)}
\qquad(7)$$</span><br /></span> where we have used <span class="math inline"><em>g</em></span> and <span class="math inline"><em>f</em></span> to represent probability densities over parameters and data, respectively. The expression <span class="math inline"><em>f</em>(<em>y</em> |<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>)</span> captures the likelihood of observing our data set <span class="math inline"><em>y</em></span> given a value for the DNA binding energy under our physical model. All knowledge we have of what the DNA binding energy <em>could</em> be, while remaining completely ignorant of the experimental measurements, is defined in <span class="math inline"><em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>)</span>, referred to as the <em>prior distribution</em>. Finally, the likelihood that we would observe the data set <span class="math inline"><em>y</em></span> while being ignorant of our physical model is defined by the denominator <span class="math inline"><em>f</em>(<em>y</em>)</span>. In this work, this term serves only as a normalization factor and as a result will be treated as a constant. We can therefore say that the posterior distribution of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is proportional to the joint distribution between the likelihood and the prior, <span id="eq:epRA_post_prop"><br /><span class="math display"><em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub> | <em>y</em>) ∝ <em>f</em>(<em>y</em> | <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>)<em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>).   (8)</span><br /></span></p>
<p>     We are now tasked with translating this generic notation into a concrete functional form. Our physical model derived in Chapter 2 given by computes the average fold-change in gene expression. Speaking practically, we make several replicate measurements of the fold-change to reduce the effects of random errors. As each replicate is independent of the others, it is reasonable to expect that these measurements will be normally distributed about the theoretical value of the fold-change <span class="math inline"><em>μ</em></span>, computed for a given <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. We can write this mathematically for each measurement as <span id="eq:likelihood_DNA_explicit"><br /><span class="math display">$$
f(y\,\vert\, \Delta\varepsilon_{RA}) = {1 \over
    (2\pi\sigma^2)^{N/2}}\prod\limits_{i}^N \exp\left[-(y_i -
    \mu(\Delta\varepsilon_{RA}))^2 \over 2\sigma^2\right],
\qquad(9)$$</span><br /></span> where <span class="math inline"><em>N</em></span> is the number of measurements in <span class="math inline"><em>y</em></span> and <span class="math inline"><em>y</em><sub><em>i</em></sub></span> is the <span class="math inline"><em>i</em><sup>th</sup></span> experimental fold-change measurement. We can write this likelihood in shorthand as <span id="eq:likelihood_DNA_short"><br /><span class="math display"><em>f</em>(<em>y</em> | <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>) = Normal{<em>μ</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>), <em>σ</em>}    (10)</span><br /></span> which we will use for the remainder of this section.</p>
<p>     Using a normal distribution for our likelihood has introduced a new parameter <span class="math inline"><em>σ</em></span> which describes the spread of our measurements about the true value. We must therefore include it in our parameter estimation and assign an appropriate prior distribution such that the posterior distribution becomes <span id="eq:full_post_sig_prior"><br /><span class="math display"><em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>, <em>σ</em> |<em>y</em>) ∝ <em>f</em>(<em>y</em> | <em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>, <em>σ</em>)<em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>)<em>g</em>(<em>σ</em>).   (11)</span><br /></span> We are now tasked with assigning functional forms to the priors <span class="math inline"><em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>)</span> and <span class="math inline"><em>g</em>(<em>σ</em>)</span>. Though one hopes that the result of the inference is not too dependent on the choice of prior, it is important to choose one that is in agreement with our physical and physiological intuition of the system.</p>
<p>     We can impose physically reasonable bounds on the possible values of the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. We can say that it is unlikely that any given mutation in the DNA binding domain will result in an affinity greater than that of biotin to streptavidin (<span class="math inline">1 <em>f</em><em>M</em> ≈  − 35 <em>k</em><sub><em>B</em></sub><em>T</em></span>, BNID 107139 <span class="citation" data-cites="milo2010">(Milo et al. <a href="#ref-milo2010" role="doc-biblioref">2010</a>)</span>), one of the strongest known non-covalent bonds. Similarly, it’s unlikely that a given mutation will result in a large, positive binding energy, indicating non-specific binding is preferable to specific binding (<span class="math inline"> ∼ 1</span> to <span class="math inline">10 <em>k</em><sub><em>B</em></sub><em>T</em></span>). While it is unlikely for the DNA binding energy to exceed these bounds, it’s not impossible, meaning we should not impose these limits as hard boundaries. Rather, we can define a weakly informative prior as a normal distribution with a mean and standard deviation as the average of these bounds, <span id="eq:epRA_prior"><br /><span class="math display"><em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>) ∼ Normal{ − 12, 12}   (12)</span><br /></span> whose probability density function in shown in Fig. 2 (A).</p>
<p>     By definition, fold-change is restricted to the bounds <span class="math inline">[0, 1]</span>. Measurement noise and fluctuations in autofluorescence background subtraction means that experimental measurements of fold-change can extend beyond these bounds, though not substantially. By definition, the scale parameter <span class="math inline"><em>σ</em></span> must be positive and greater than zero. We also know that for the measurements to be of any use, the error should be less than the available range of fold-change, <span class="math inline">1.0</span>. We can choose such a prior as a half normal distribution <span id="eq:sig_hn_explicit"><br /><span class="math display">$$
g(\sigma) = {1 \over \phi}\sqrt{2 \over \pi}\exp\left[-{\sigma^2 \over
2\phi^2}\right];\, \forall\, \sigma \geq 0
\qquad(13)$$</span><br /></span> where <span class="math inline"><em>ϕ</em></span> is the standard deviation. By choosing <span class="math inline"><em>ϕ</em> = 0.1</span>, it is unlikely that <span class="math inline"><em>σ</em> ≥ 1</span> yet not impossible, permitting the occasional measurement significantly outside of the theoretical bounds. The probability density function for this prior is shown in Fig. 2 (B).</p>
<p>     While these choices for the priors seem reasonable, we can check their appropriateness by using them to simulate a data set and checking that the hypothetical fold-change measurements obey our physical and physiological intuition.</p>
<figure>
<img src="ch7_figS2.png" id="fig:epRA_prior_pred" data-short-caption="Prior distributions and prior predictive check for estimation of the DNA binding energy." alt="" /><figcaption>Figure 2: <strong>Prior distributions and prior predictive check for estimation of the DNA binding energy.</strong> (A) Prior probability density function for DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> as <span class="math inline">∼</span> Normal(<span class="math inline"> − 12, 12</span>). (B) Prior probability density function for the standard deviation in measurement noise <span class="math inline"><em>σ</em>∼</span> HalfNormal(<span class="math inline">0, 0.1</span>). (C) Percentiles of values drawn from the likelihood distribution given draws from prior distributions given <span class="math inline"><em>R</em> = 260</span>, <span class="math inline"><em>K</em><sub><em>A</em></sub> = 139 <em>μ</em></span>M, <span class="math inline"><em>K</em><sub><em>I</em></sub> = 0.53 <em>μ</em></span>M, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub> = 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>, which match the parameters used in <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>. Black points at top of (A) and (B) represent draws used to generate fold-change measurements from the likelihood distribution. Percentiles in (C) generated from 800 draws from the prior distributions. For each draw from the prior distributions, a data set of 70 measurements over 12 IPTG concentrations (ranging from <span class="math inline">0</span> to <span class="math inline">5000 <em>μ</em></span>M) were generated from the likelihood. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS2.py">Python code (<code>ch7_figS2.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="prior-predictive-checks">Prior Predictive Checks</h3>
<p>If our choice of prior distribution for each parameter is appropriate, we should be able to simulate data sets using these priors that match our expectations. In essence, we would hope that these prior choices would generate some data sets with fold-change measurements above 1 or below zero, but they should be infrequent. If we end up getting primarily negative values for fold-change, for example, then we can surmise that there is something wrong in our definition of the prior distribution. This method, coined a <em>prior predictive check</em>, was first put forward in <span class="citation" data-cites="good1950">Good (<a href="#ref-good1950" role="doc-biblioref">1950</a>)</span> and has received newfound attention in computational statistics.</p>
<p>     We perform the simulation in the following manner. We first draw a random value for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> out of its prior distribution stated in Eq. 12 and calculate what the mean fold-change should be given our physical model. With this in hand, we draw a random value for <span class="math inline"><em>σ</em></span> from its prior distribution, specified in Eq. 13. We then generate a simulated data set by drawing <span class="math inline">≈</span> 70 fold-change values across twelve inducer concentrations from the likelihood distribution which we defined in Eq. 10. This roughly matches the number of measurements made for each mutant in this work. We repeat this procedure for 800 draws from the prior distributions, which is enough to observe the occasional extreme fold-change value from the likelihood. As the DNA binding energy is the only parameter of our physical model that we are estimating, we had to choose values for the others. We kept the values of the inducer binding constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> the same as the wild-type repressor (<span class="math inline">139 <em>μ</em></span>M and <span class="math inline">0.53 <em>μ</em></span>M, respectively). We chose to use <span class="math inline"><em>R</em> = 260</span> repressors per cell as this is the repressor copy number we used in the main text to estimate the DNA binding energies of the three mutants.</p>
<p>     The draws from the priors are shown in Fig. 2 (A) and (B) as black points above the corresponding distribution. To display the results, we computed the percentiles of the simulated data sets at each inducer concentration. These percentiles are shown as red shaded regions in Fig. 2 (C). The 5<span class="math inline">th</span> percentile (dark purple band) has the characteristic profile of an induction curve. Given that the prior distribution for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> is centered at <span class="math inline"> − 12 <em>k</em><sub><em>B</em></sub><em>T</em></span> and we chose <span class="math inline"><em>R</em> = 260</span>, we expect the generated data sets to cluster about the induction profile defined by these values. More importantly, approximately 95% of the generated data sets fall between fold-change values of -0.1 and 1.1, which is within the realm of possibility given the systematic and biological noise in our experiments. The 99<span class="math inline"><em></em><sup>th</sup></span> percentile maximum is approximately <span class="math inline">1.3</span> and the minimum approximately <span class="math inline"> − 0.3</span>. While we could tune our choice of prior further to minimize draws this far from the theoretical bounds, we err on the side of caution and accept these values as it is possible that fold-change measurements this high or low can be observed, albeit rarely. Through these prior predictive checks, we feel confident that these choices of priors are appropriate for the parameters we wish to estimate. We can now move forward and make sure that the statistical model as a whole is valid and computationally tractable.</p>
<h3 id="sensitivity-analysis-and-simulation-based-calibration">Sensitivity Analysis and Simulation Based Calibration</h3>
<p>Satisfied with our choice of prior distributions, we can proceed to check other properties of the statistical model and root out any pathologies lurking in our model assumptions.</p>
<p>     To build trust in our model, we could generate a data set <span class="math inline"><em>ỹ</em></span> with a <em>known</em> value for <span class="math inline"><em>σ</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, estimate the posterior distribution <span class="math inline"><em>g</em>(<em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub>, <em>σ</em> | <em>ỹ</em>)</span>, and determine how well we were able to retrieve the true value of the parameters. However, running this once or twice for handpicked values of <span class="math inline"><em>σ</em></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> won’t reveal edge-cases in which the inference fails, some of which may exist in our data. Rather than performing this operation once, we can run this process over a variety of data sets where the ground truth parameter value is drawn from the prior distribution (as we did for the prior predictive checks). For an arbitrary parameter <span class="math inline"><em>θ</em></span>, the joint distribution between the ground truth value <span class="math inline"><em>θ̃</em></span>, the inferred value <span class="math inline"><em>θ</em></span>, and the simulated data set <span class="math inline"><em>ỹ</em></span> can be written as <span id="eq:bayesian_joint_dist"><br /><span class="math display"><em>π</em>(<em>θ</em>, <em>ỹ</em>, <em>θ̃</em>) = <em>g</em>(<em>θ</em> | <em>ỹ</em>)<em>f</em>(<em>ỹ</em> | <em>θ̃</em>)<em>g</em>(<em>θ̃</em>).   (14)</span><br /></span> If this process is run for a large number of simulations, Eq. 14 can be marginalized over all data sets <span class="math inline"><em>ỹ</em></span> and all ground truth values <span class="math inline"><em>θ̃</em></span> to yield the original prior distribution, <span id="eq:self_consistency_relation"><br /><span class="math display">∫<em>d</em><em>θ̃</em>∫<em>d</em><em>ỹ</em><em>π</em>(<em>θ</em>, <em>ỹ</em>, <em>θ̃</em>) = <em>g</em>(<em>θ</em>).   (15)</span><br /></span></p>
<p>     This result, described by <span class="citation" data-cites="talts2018">Talts et al. (<a href="#ref-talts2018" role="doc-biblioref">2018</a>)</span>, holds true for <em>any</em> statistical model and is a natural self consistency property of Bayesian inference. Any deviation between the distribution of our inferred values for <span class="math inline"><em>θ</em></span> and the original prior distribution <span class="math inline"><em>g</em>(<em>θ</em>)</span> indicates that either our statistical model is malformed or the computational method is not behaving as expected. There are a variety of ways we can ensure that this condition is satisfied, which we outline below.</p>
<p>      Using the data set generated for the prior predictive checks (shown in Fig. 2 (C)), we sampled the posterior distribution and compute <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>σ</em></span> for each simulation and checked that they matched the original prior distribution. To perform the inference, we use Markov chain Monte Carlo (MCMC) to sample the posterior distribution. Specifically, we use the Hamiltonian Monte Carlo algorithm implemented in the Stan probabilistic programming language <span class="citation" data-cites="carpenter2017">(Carpenter et al. <a href="#ref-carpenter2017" role="doc-biblioref">2017</a>)</span>. The specific code files can be accessed through the <a href="http://rpgroup.caltech.edu/mwc_mutants">paper website</a> or the associated <a href="https://github.com/rpgroup-pboc/mwc_mutants">GitHub repository</a>. The original prior distribution and the distribution of inferred parameter values can be seen in Fig. 3 (A) and (B). For both <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>σ</em></span>, we can accurately recover the ground truth distribution (purple) via sampling with MCMC (orange). For <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, there appears to be an upper and lower limit past which we are unable to accurately infer the binding energy. This can be seen in both the histogram Fig. 3(A) and the empirical cumulative distribution Fig. 3 (B) as deviations from the ground truth when DNA binding is below <span class="math inline"> ≈  − 25 <em>k</em><sub><em>B</em></sub><em>T</em></span> or above <span class="math inline"> ≈  − 5 <em>k</em><sub><em>B</em></sub><em>T</em></span>. These limits hinder our ability to comment on exceptionally strong or weak binding affinities. However, as all mutants queried in this work exhibited binding energies between these limits, we surmise that the inferential scheme permits us to draw conclusions about the inferred DNA binding strengths.</p>
<figure>
<img src="ch7_figS3.png" id="fig:epRA_sbc" data-short-caption="Comparison of averaged posterior and prior distributions for DNA binding energy and homoscedastic error." alt="" /><figcaption>Figure 3: <strong>Comparison of averaged posterior and prior distributions for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>σ</em></span>.</strong> (A) Distribution of the average values for the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> (orange) overlaid with the ground truth distribution (purple). (B) Data averaged posterior (orange) for the standard deviation of fold-change measurements overlaid with the ground truth distribution (purple). Top and bottom show the same data with different visualizations. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS3.py">Python code (<code>ch7_figS3.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     Rather than examining the agreement of the data-averaged posterior and the ground truth prior distribution solely by eye, we can compute summary statistics using the mean <span class="math inline"><em>μ</em></span> and standard deviation <span class="math inline"><em>σ</em></span> of the posterior and prior distributions which permit easier identification of pathologies in the inference. One such quantity is the posterior <span class="math inline"><em>z</em></span>-score, which is defined as <span id="eq:zscore"><br /><span class="math display">$$
z = {\mu_\text{posterior} - \tilde\theta \over \sigma_\text{posterior}}.
\qquad(16)$$</span><br /></span> This statistic summarizes how accurately the posterior recovers the ground truth value beyond simply reporting the mean, median, or mode of the posterior distribution. <span class="math inline"><em>Z</em></span>-scores around <span class="math inline">0</span> indicate that the posterior is concentrating tightly about the true value of the parameter whereas large values (either positive or negative) indicate that the posterior is concentrating elsewhere. A useful feature of this metric is that the width of the posterior is also considered. It is possible that the posterior could have a mean very close to the ground truth value, but have an incredibly narrow distribution/spread such that it does not overlap with the ground-truth. Only comparing the mean value to the ground truth would suggest that the inference “worked”. However with a small standard deviation generates a very large <span class="math inline"><em>z</em></span>-score, telling us that something has gone awry.</p>
<p>     If our inferential model is behaving properly, the width of the posterior distribution should be significantly smaller than the width of the prior, meaning that the posterior is being informed by the data. The level to which the posterior is being informed by the data can be easily calculated given knowledge of both the prior and posterior distribution. This quantity, aptly named the shrinkage <span class="math inline"><em>s</em></span>, can be computed as <span id="eq:shrinkage"><br /><span class="math display">$$
s = 1 - {\sigma^2_\text{posterior} \over \sigma^2_\text{prior}}.
\qquad(17)$$</span><br /></span> When the shrinkage is close to zero, the variance of the posterior is approximately the same as the variance of the prior, model is not being properly informed by the data. When <span class="math inline"><em>s</em> ≈ 1</span>, the variance of the posterior is much smaller than the variance of the prior, indicating that the it is being highly informed by the data. A shrinkage less than 0 indicates that the posterior is wider than the prior distribution, revealing a severe pathology in either the model itself or the implementation.</p>
<p>     In Fig. 4, we compute these summary statistics for each parameter. For both <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>σ</em></span>, we see clustering of the <span class="math inline"><em>z</em></span>-score about 0 with the extrema reaching <span class="math inline"> ≈  ± 3</span>. This suggests that for the vast majority of our simulated data sets, the posterior distribution concentrated about the ground truth value. We also see that for both parameters, the posterior shrinkage <span class="math inline"><em>s</em></span> is <span class="math inline"> ≈ 1</span>, indicating that the posterior is being highly informed by the data. There is a second distribution centered <span class="math inline">≈</span> 0.8 for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>, indicating that for a subset of the data sets, the posterior is only <span class="math inline"> ≈ 80</span>% narrower than the prior distribution. These samples are those that were drawn outside of the limits of <span class="math inline"> ≈  − 25</span> to <span class="math inline"> − 5 <em>k</em><sub><em>B</em></sub><em>T</em></span> where the inferential power is limited. Nevertheless, the posterior still significantly shrank, indicating that the data strongly informs the posterior.</p>
<figure>
<img src="ch7_figS4.png" id="fig:epRA_sbc_sensitivity" data-short-caption="Inferential sensitivity for estimation of the DNA binding energy and homoscedastic error." alt="" /><figcaption>Figure 4: <strong>Inferential sensitivity for estimation of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>σ</em></span></strong>. The posterior <span class="math inline"><em>z</em></span>-score for each posterior distribution inferred from a simulated data set is plotted against the shrinkage for (A) the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and (B) the standard deviation of fold-change measurements <span class="math inline"><em>σ</em></span>. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS4.py">Python code (<code>ch7_figS4.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     The general self-consistency condition given by Eq. 15 provides another route to ensure that the model is computationally tractable. Say that we draw a value for the DNA binding energy from the prior distribution, simulate a data set, and sample the posterior using MCMC. The result of this sampling is a collection of <span class="math inline"><em>N</em></span> values of the parameter which may be above, below, or equal to the ground-truth value. From this set of values, we select <span class="math inline"><em>L</em></span> of them and rank order them by their value. <span class="citation" data-cites="talts2018">Talts et al. (<a href="#ref-talts2018" role="doc-biblioref">2018</a>)</span> derived a general theorem which states that the number of samples less than the ground truth value of the parameter (termed the rank statistic) is uniformly distributed over the interval <span class="math inline">[0, <em>L</em>]</span>. As Eq. 15 <em>must</em> hold true for any statistical model, deviations from uniformity signal that there is a problem in the implementation of the statistical model. How the distribution deviates is also informative as different types of failures result in different distributions. The nature of these deviations, along with a more formal proof of the uniform distribution of rank statistics can be found in <span class="citation" data-cites="talts2018">Talts et al. (<a href="#ref-talts2018" role="doc-biblioref">2018</a>)</span> where it was originally derived.</p>
<p>     Given the sampling statistics for each of the simulated data sets, we took 800 of the MCMC samples of the posterior distribution for each of the 800 simulated data sets and computed the rank statistic. The distributions are shown in Fig. 5 as both histograms and ECDFs for the DNA binding energy and standard deviation. The distribution of rank statistics for both parameters appears to be uniform. The purple band overlaying the histograms (top row) as well as the purple envelopes overlaying the ECDFs (bottom row) represent the <span class="math inline">99<sup>th</sup></span> percentile expected from a true uniform distribution. The uniformity of this distribution, along with the well-behaved <span class="math inline"><em>z</em></span>-scores and shrinkage for each parameter, tells us that there are no underlying pathologies in our statistical model and that it is computationally tractable. However, this does not mean that it is correct. Whether this model is valid for the actual observed data is the topic of the next section.</p>
<figure>
<img src="ch7_figS5.png" id="fig:epRA_sbc_rank" data-short-caption="Rank distribution of the posterior samples from simulated data for the DNA binding energy and homoscedastic error." alt="" /><figcaption>Figure 5: <strong>Rank distribution of the posterior samples from simulated data.</strong> Top row shows a histogram of the rank distribution with <span class="math inline"><em>n</em> = 20</span> bins. Bottom row is the cumulative distribution for the same data. Purple bands correspond to the 99th percentile of expected variation from a uniform distribution. (A) Distribution for the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and (B) for the standard deviation <span class="math inline"><em>σ</em></span>. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS5.py">Python code (<code>ch7_figS5.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="parameter-estimation-and-posterior-predictive-checks">Parameter Estimation and Posterior Predictive Checks</h3>
<p>We now turn to applying our vetted statistical model to experimental measurements. While the same statistical model was applied to all three DNA binding mutants, here we only focus on the mutant Q18M for brevity.</p>
<p>     Using a single induction profile, we sampled the posterior distribution over both the DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and the standard deviation <span class="math inline"><em>σ</em></span> using MCMC implemented in the Stan programming language. The output of this process is a set of 4000 samples of both parameters along with the value of their log posterior probabilities, which serves as an approximate measure of the probability of each value. The individual samples are shown in Fig. 6. The joint distribution between <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>σ</em></span> is shown in the lower left hand corner, and the marginal distributions for each parameter are shown above and to the right of the joint distribution, respectively. The joint distribution is color coded by the value of the log posterior, with bright orange and dark purple corresponding to high and low probability, respectively. The symmetric shape of the joint distribution is a telling sign that there is no correlation between two parameters. The marginal distributions for each parameter are also relatively narrow, with the DNA binding energy covering a range of <span class="math inline"> ≈ 0.6 <em>k</em><sub><em>B</em></sub><em>T</em></span> and <span class="math inline"><em>σ</em></span> spanning <span class="math inline"> ≈ 0.02</span>. To more precisely quantify the uncertainty, we computed the shortest interval of the marginal distribution for each parameter contains 95% of the probability. The bounds of this interval, coined the Bayesian credible region, can accommodate asymmetry in the marginal distribution since the upper and lower bounds of the estimate are reported. In the main text, we reported the DNA binding energy estimated from these data to be <span class="math inline">15.43<sub> − 0.06</sub><sup> + 0.06</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span>, where the first value is the median of the distribution and the super- and subscripts correspond to the upper and lower bounds of the credible region, respectively.</p>
<p>     While looking at the shape of the posterior distribution can be illuminating, it is not enough to tell us if the parameter values extracted make sense or accurately describe the data on which they were conditioned. To assess the validity of the statistical model in describing actual data, we again turn to simulation, this time using the posterior distributions for each parameter rather than the prior distributions. The likelihood of our statistical model assumes that across the entire induction profile, the observed fold-change is normally distributed about the theoretical prediction with a standard deviation <span class="math inline"><em>σ</em></span>. If this is an accurate depiction of the generative process, we should be able to draw values from the likelihood using the sampled values for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>σ</em></span> that are indistinguishable from the actual experimental measurements. This process is known as a <em>posterior predictive check</em> and is a Bayesian method of assessing goodness-of-fit.</p>
<p>     For each sample from the posterior, we computed the theoretical mean fold-change given the sampled value for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span>. With this mean in hand, we used the corresponding sample for <span class="math inline"><em>σ</em></span> and drew a data set from the likelihood distribution the same size as the real data set used for the inference. As we did this for every sample of our MCMC output (a total of <span class="math inline"> ≈ 4000</span>), it is more instructive to compute the percentiles of the generated data than to show the entire output. In Fig. 6 (B), the percentiles of the generated data sets are shown overlaid with the data used for the inference. We see that all of the data points fall within the 99<span class="math inline"><em></em><sup>th</sup></span> percentile of simulated data sets with the <span class="math inline">5<sup>th</sup></span> percentile tracking the mean of the data at each inducer concentration. As there are no systematic deviations or experimental observations that fall far outside those generated from the statistical model, we can safely say that the statistical model derived here accurately describes the observed data.</p>
<figure>
<img src="ch7_figS6.png" id="fig:epRA_ppc" data-short-caption="Markov chain Monte Carlo samples and posterior predictive check for DNA binding mutant Q18M." alt="" /><figcaption>Figure 6: <strong>Markov Chain Monte Carlo (MCMC) samples and posterior predictive check for DNA binding mutant Q18M.</strong> (A) Marginal and joint sampling distributions for DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and <span class="math inline"><em>σ</em></span>. Each point in the joint distribution is a single sample. Marginal distributions for each parameter are shown adjacent to joint distribution. Color in the joint distribution corresponds to the value of the log posterior with the progression of dark purple to bright orange corresponding to increasing probability. (B) The posterior predictive check of model. The measurements of the fold-change in gene expression are shown as black open-faced circles. The percentiles are shown as colored bands and indicate the fraction of simulated data drawn from the likelihood that fall within the shaded region. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS6.py">Python code (<code>ch7_figS6.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="inferring-the-free-energy-from-fold-change-measurements">Inferring the Free Energy From Fold-Change Measurements</h2>
<p>In this section, we describe the statistical model to infer the free energy <span class="math inline"><em>F</em></span> from a set of fold-change measurements. We follow the same principled workflow as described previously for the DNA binding estimation, including declaration of the generative model, prior predictive checks, simulation based calibration, and posterior predictive checks. Finally, we determine an empirical limit in our ability to infer the free energy and define a heuristic which can be used to identify measurements that are likely inaccurate. To understand the statistical model and the empirical limits of detection, only the subsections <em>Building A Generative Model</em> and <em>Sensitivity Limits and Systematic Errors in Inference</em> are necessary.</p>
<h3 id="building-a-generative-model">Building A Generative Model</h3>
<p>In Chapter 2, we showed that the fold-change equation can be rewritten in the form of a Fermi function, <span id="eq:mut_si_fermi"><br /><span class="math display">$$
\text{fold-change} = {1 \over 1 + e^{-F / k_BT}},
\qquad(18)$$</span><br /></span> where <span class="math inline"><em>F</em></span> corresponds to the free energy difference between the repressor bound and unbound states of the promoter. While the theory prescribes a way for us to calculate the free energy based on our knowledge of the biophysical parameters, we can directly calculate the free energy of a measurement of fold-change by simply rearranging Eq. 18 as <span id="eq:empirical_F"><br /><span class="math display">$$
F = -k_BT\log\left({1 \over \text{fold-change}} - 1\right).
\qquad(19)$$</span><br /></span> With perfect measurement of the fold-change in gene expression (assuming no experimental or measurement noise), the free energy can be directly calculated. However, actual measurements of the fold-change in gene expression can extend beyond the theoretical bounds of <span class="math inline">0</span> and <span class="math inline">1</span>, for which the free energy is mathematically undefined.</p>
<p>     As the fold-change measurements between biological replicates are independent, it is reasonable to assume that they are normally distributed about a mean value <span class="math inline"><em>μ</em></span> with a standard deviation <span class="math inline"><em>σ</em></span>. While the mean value is restricted to the bounds of [<span class="math inline">0, 1</span>], fold-change measurements outside of these bounds are still possible given that they are distributed about the mean with a scale of <span class="math inline"><em>σ</em></span>. Thus, if we have knowledge of the mean fold-change in gene expression about which the observed fold-change is distributed, we can calculate the mean free energy as <span id="eq:empirical_F_mu"><br /><span class="math display">$$
F = -k_BT\log\left({1 \over \mu} - 1\right).
\qquad(20)$$</span><br /></span> For a given set of fold-change measurements <span class="math inline"><em>y</em></span>, we wish to infer the posterior probability distribution for <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span>, given by Bayes’ theorem as <span id="eq:fc_mu_bayes"><br /><span class="math display"><em>g</em>(<em>μ</em>, <em>σ</em> | <em>y</em>) ∝ <em>f</em>(<em>y</em> | <em>μ</em>, <em>σ</em>)<em>g</em>(<em>μ</em>)<em>g</em>(<em>σ</em>),   (21)</span><br /></span> where we have dropped the normalization constant <span class="math inline"><em>f</em>(<em>y</em>)</span> and assigned a proportionality between the posterior and joint probability distribution. Given that the measurements are independent, we define the likelihood <span class="math inline"><em>f</em>(<em>y</em> | <em>μ</em>, <em>σ</em>)</span> as a normal distribution, <span id="eq:fc_mu_likelihood"><br /><span class="math display"><em>f</em>(<em>y</em> | <em>μ</em> <em>σ</em>) ∼ Normal{<em>μ</em>, <em>σ</em>}.   (22)</span><br /></span> While the mean <span class="math inline"><em>μ</em></span> is restricted to the interval [<span class="math inline">0, 1</span>], there is no reason <em>a priori</em> to think that it is more likely to be closer to either bound. To remain uninformative and be as permissive as possible, we define a prior distribution for <span class="math inline"><em>μ</em></span> as a Uniform distribution between 0 and 1, <span id="eq:fc_mu_prior"><br /><span class="math display">$$
g(\mu)=\begin{cases}{1\over \mu_\text{max} - \mu_\text{min}} &amp; \mu_\text{min} &lt; \mu &lt; \mu_\text{max}\\
0 &amp; \text{otherwise}\end{cases}.
\qquad(23)$$</span><br /></span> Here, <span class="math inline"><em>μ</em><sub>min</sub> = 0</span> and <span class="math inline"><em>μ</em><sub>max</sub> = 1</span>, reducing <span class="math inline"><em>g</em>(<em>μ</em>)</span> to 1. For <span class="math inline"><em>σ</em></span>, we can again assume a half-normal distribution with a standard deviation of 0.1 as was used for estimating the DNA binding energy Eq. 13, <span id="eq:fc_sigma_prior"><br /><span class="math display"><em>g</em>(<em>σ</em>) = HalfNormal{0, 0.1}.   (24)</span><br /></span></p>
<p>     With a full generative model defined, we can now use prior predictive checks to ensure that our choices of prior are appropriate for the inference.</p>
<h3 id="prior-predictive-checks-1">Prior Predictive Checks</h3>
<p>     To check the validity of the chosen priors, we pulled 1000 combinations of <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span> from their respective distributions (Fig. 7 (A)) and subsequently drew a set of 10 fold-change values (a number comparable to the number of biological replicates used in this work) from a normal distribution defined by <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span>. To visualize the range of values generated from these checks, we computed the percentiles of the empirical cumulative distributions of the fold-change values, as can be seen in Fig. 7 (C). Approximately 95% of the the generated fold-change measurements were between the theoretical bounds of [<span class="math inline">0, 1</span>] whereas 5% of the data sets fell outside with the maximum and minimum values extending to <span class="math inline"> ≈ 1.2</span> and <span class="math inline"> − 0.2</span>, respectively. Given our familiarity with these experimental strains and the detection sensitivity of the flow cytometer, these excursions beyond the theoretical bounds agree with our intuition. Satisfied with our choice of prior distributions, we can proceed to check the sensitivity and computational tractability of our model through simulation based calibration.</p>
<figure>
<img src="ch7_figS7.png" id="fig:empirical_F_prior_pred" data-short-caption="Prior predictive checks for inference of the mean fold-change" alt="" /><figcaption>Figure 7: <strong>Prior predictive checks for inference of the mean fold-change.</strong> (A) The prior distributions for <span class="math inline"><em>μ</em></span> (left) and <span class="math inline"><em>σ</em></span> (right). The vertical axis is proportional to the probability of the value. Black points above distributions correspond to the values used to perform the prior predictive checks. (B) Percentiles of the data generated for each draw from the prior distributions shown as a cumulative distribution. Percentiles were calculated for 1000 generated data sets, each with 10 fold-change measurements drawn from the likelihood given the drawn values of <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span>. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS7.py">Python code (<code>ch7_figS7.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="simulation-based-calibration">Simulation Based Calibration</h3>
<p>     To ensure that the parameters can be estimated with confidence, we sampled the posterior distribution of <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span> for each data set generated from the prior predictive checks. For each inference, we computed the <span class="math inline"><em>z</em></span>-score and shrinkage for each parameter, shown in Fig. 8(A). For both parameters, the <span class="math inline"><em>z</em></span>-scores are approximately centered about zero, indicating that the posteriors concentrate about the ground truth value of the parameter. The <span class="math inline"><em>z</em></span>-scores for <span class="math inline"><em>σ</em></span> green points in Fig. 8) (A) appear to be slightly off centered with more negative values than positive. This suggests that <span class="math inline"><em>σ</em></span> is more likely to be slightly overestimated in some cases. The shrinkage parameter for <span class="math inline"><em>μ</em></span> (red points) is very tightly distributed about <span class="math inline">1.0</span>, indicating that the prior is being strongly informed by the data. The shrinkage is more broadly distributed for for <span class="math inline"><em>σ</em></span> with a minimum value of <span class="math inline"> ≈ 0.5</span>. However, the median shrinkage for <span class="math inline"><em>σ</em></span> is <span class="math inline"> ≈ 0.9</span>, indicating that half of the inferences shrank the prior distribution by at least 90%. While we could revisit the model to try and improve the shrinkage values, we are more concerned with <span class="math inline"><em>μ</em></span> which shows high shrinkage and zero-centered <span class="math inline"><em>z</em></span>-scores.</p>
<p>     To ensure that the model is computationally tractable, we computed the rank statistic of each parameter for each inference. The empirical cumulative distributions for <span class="math inline"><em>μ</em></span> (black) and <span class="math inline"><em>σ</em></span> (red) can be seen in Fig. 8 (B). Both distributions appear to be uniform, falling within the 99<span class="math inline"><em></em><sup>th</sup></span> percentile of the variation expected from a true uniform distribution. This indicates that the self-consistency relation defined by Eq. 15. holds for this statistical model. With a computationally tractable model in hand, we can now apply the statistical model to our data and verify that data sets drawn from the data-conditioned posterior are indistinguishable from the experimental measurements.</p>
<figure>
<img src="ch7_figS8.png" id="fig:empirical_F_sensitivity" data-short-caption="Sensitivity measurements and rank statistic distribution of the statistical model estimating $\mu$ and $\sigma$." alt="" /><figcaption>Figure 8: <strong>Sensitivity measurements and rank statistic distribution of the statistical model estimating <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span>.</strong> (A) Posterior <span class="math inline"><em>z</em></span>-score of each inference plotted against the posterior shrinkage factor for the parameters <span class="math inline"><em>μ</em></span> (blue points) and <span class="math inline"><em>σ</em></span> (green points). (B) Distribution of rank statistics for <span class="math inline"><em>μ</em></span> (red) and <span class="math inline"><em>σ</em></span> (black). Purple envelope represents the 99<span class="math inline"><em></em><sup>th</sup></span> percentile of a true uniform distribution. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS8.py">Python code (<code>ch7_figS8.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="posterior-predictive-checks">Posterior Predictive Checks</h3>
<p>The same statistical model was applied to every unique set of fold-change measurements used in this work. Here, we focus only on the set of fold-change measurements for the double mutant Y17I-Q291V at 50 <span class="math inline"><em>μ</em></span>M IPTG. The samples from the posterior distribution conditioned on this dataset can be seen in Fig. 9 (A). The joint distribution, shown in the lower left-hand corner, appears fairly symmetric, indicating that <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span> are independent. There is a slight asymmetry in the sampling of <span class="math inline"><em>σ</em></span>, which can be more clearly seen in the corresponding marginal distribution to the right of the joint distribution.</p>
<p>     For each MCMC sample of <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span>, we drew 10 samples from a normal distribution defined by these parameters. From this collection of data sets, we computed the percentiles of the empirical cumulative distribution and plotted them over the data, as can be seen in Fig. 9 (B). We find that the observed data falls within the 99<span class="math inline"><em></em><sup>th</sup></span> percentile of the generated data sets. This illustrates that the model can produce data which is identically distributed to the actual experimental measurements, validating our choice of statistical model.</p>
<figure>
<img src="ch7_figS9.png" id="fig:empirical_F_post_pred" data-short-caption="MCMC sampling output and posterior predictive checks of the statistical model for the mean fold-change and standard deviation." alt="" /><figcaption>Figure 9: <strong>MCMC sampling output and posterior predictive checks of the statistical model for the mean fold-change <span class="math inline"><em>μ</em></span> and standard deviation <span class="math inline"><em>σ</em></span>.</strong> (A) Corner plot of sampling output. The joint distribution between <span class="math inline"><em>σ</em></span> and <span class="math inline"><em>μ</em></span> is shown in the lower left hand corner. Each point is an individual sample. Points are colored by the value of the log posterior with increasing probability corresponding to transitions from purple to orange. Marginal distributions for each parameter are shown adjacent to the joint distribution. (B) Percentiles of the cumulative distributions from the posterior predictive checks are shown as shaded bars. Data on which the posterior was conditioned are shown as white orange circles and lines. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS9.py">Python code (<code>ch7_figS9.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="sensitivity-limits-and-systematic-errors-in-inference">Sensitivity Limits and Systematic Errors in Inference</h3>
<p>Considering the results from the prior predictive checks, simulation based calibration, and posterior predictive checks, we can say that the statistical model for inferring <span class="math inline"><em>μ</em></span> and <span class="math inline"><em>σ</em></span> fold-change from a collection of noisy fold-change measurements is valid and computationally tractable. Upon applying this model to the experimental data of the wild-type strain (where the free energy is theoretically known), we observed that systematic errors arise when the fold-change is exceptionally high or low, making the resulting inference of the free energy inaccurate.</p>
<p>     To elucidate the source of this systematic error, we return to a simulation based approach in which the true free energy is known (black points in Fig. 10 (A)). For a range of free energies, we computed the theoretical fold-change prescribed by Eq. 18. For each free energy value, we pulled a value for <span class="math inline"><em>σ</em></span> from the prior distribution defined in Eq. 13 and generated a data set of 10 measurements by drawing values from a normal distribution defined by the true fold-change and the drawn value of <span class="math inline"><em>σ</em></span> (purple points in Fig. 10 (A)). We then sampled the statistical model over these data and inferred the mean fold-change <span class="math inline"><em>μ</em></span> (orange points in Fig. 10 (A)). By eye, the inferred points appear to collapse onto the master curve, in many cases overlapping the true values. However, the points with a free energy less than <span class="math inline"> ≈  − 2 <em>k</em><sub><em>B</em></sub><em>T</em></span> and greater than <span class="math inline"> ≈ 2 <em>k</em><sub><em>B</em></sub><em>T</em></span> are slightly above or below the master curve, respectively. This becomes more obvious when the inferred free energy is plotted as a function of the true free energy, shown in Fig. 10 (B). Points in which the difference between <span class="math inline"><em>μ</em></span> and the nearest boundary (<span class="math inline">0</span> or <span class="math inline">1</span>) is less than the value of <span class="math inline"><em>σ</em></span> are shown as purple or green. When this condition is met, the inferred mean free energy strays from the true value, introducing a systematic error. This suggests that the spread of the fold-change measurements sets the detection limit of fold-change close to either boundary. Thus, the narrower the spread in the fold-change the better the estimate of the fold-change near the boundaries.</p>
<p>     These systematic errors can be seen in experimental measurements of the wild-type repressor. Data from <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> in which the IPTG titration profiles of seventeen different bacterial strains were measured is shown collapsed onto the master curve in Fig. 10 (C) as red points. Here, each point corresponds to a single biological replicate. The inferred mean fold-change <span class="math inline"><em>μ</em></span> and 95% credible regions are shown as purple, blue, or green points. The color of these points correspond to the relative value of <span class="math inline"><em>μ</em></span> or <span class="math inline">1 − <em>μ</em></span> to <span class="math inline"><em>σ</em></span>. The discrepancy between the predicted and inferred free energy of each measurement set can be seen in Fig. 10 (D). The significant deviation from the predicted and inferred free energy occurs past the detection limit set by <span class="math inline"><em>σ</em></span>. In this work, we therefore opted to not display inferred free energies at the extrema where the inferred fold-change was closer to the boundaries than the corresponding standard deviation, as it reflects limitations in our measurement rather than a deviation from the theoretical predictions.</p>
<figure>
<img src="ch7_figS10.png" id="fig:empirical_F_error" data-short-caption="Identification of systematic error in simulated and real data when considering the free energy." alt="" /><figcaption>Figure 10: <strong>Identification of systematic error in simulated and real data when considering the free energy.</strong> (A) The true fold-change (black points), simulated fold-change distribution (purple points), and inferred mean fold-change (orange) is plotted as a function of the true free energy. Error bars on inferred fold-change correspond to the 95% credible region of the mean fold-change <span class="math inline"><em>μ</em></span>. (B) Inferred free energy plotted as a function of the true free energy. Black line indicates perfect agreement between the ground truth free energy and inferred free energy. Blue points correspond to the inferred free energy where the median values of the parameters satisfy the condition <span class="math inline"><em>μ</em> &gt; <em>σ</em></span> and <span class="math inline">1 − <em>μ</em> &gt; <em>σ</em></span>. Purple points correspond to the inferred mean fold-change <span class="math inline"><em>μ</em> &lt; <em>σ</em></span>. Green points correspond to those where the inferred mean fold-change <span class="math inline">1 − <em>μ</em> &lt; <em>σ</em></span>. Error bars correspond to the bounds of the 95% credible region. (C) Biological replicate data from <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> (red points) plotted as a function of the theoretical free energy. Inferred mean fold-change <span class="math inline"><em>μ</em></span> and the 95% credible region are shown as blue points. Purple and green points are colored by the same conditions as in (B). (D) Inferred free energy as a function of the predicted free energy colored by the satisfied condition. Error bars are the bounds of the 95% credible region. All inferred values in (A – D) are the median values of the posterior distribution. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS10.py">Python code (<code>ch7_figS10.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="additional-characterization-of-dna-binding-mutants">Additional Characterization of DNA Binding Mutants</h2>
<p>In Chapter 3, we estimated the DNA binding energy o f each mutant using the mutant strains that had approximately 260 repressors per cell. In this section, we examine the effect of the choice of fit strain on the predictions of both the induction profiles and <span class="math inline"><em>Δ</em><em>F</em></span> for each DNA binding domain mutant.</p>
<p>     We applied the statistical model derived in Section 2 of this chapter for each unique strain of the DNA binding mutants and estimated the DNA binding energy. The median of the posterior distribution along with the upper and lower bounds of the 95% credible region are reported in Table 7.1. We found that the choice of fitting strain did not strongly influence the estimate of the DNA binding energy. The largest deviations appear for the weakest binding mutants paired with the lowest repressor copy number. In these cases, such as for Q18A, the difference in binding energy between the repressor copy numbers is <span class="math inline"> ≈ 1 <em>k</em><sub><em>B</em></sub><em>T</em></span> which is small compared to the overall DNA binding energy. Using these energies, we computed the predicted induction profiles of each mutant with different repressor copy numbers, shown in Fig. 11. In this plot, the rows correspond to the repressor copy number of the strain used to estimate the DNA binding energy. The columns correspond to the repressor copy number of the predicted strains. The diagonals, shaded in grey, show the induction profile of the fit strain along with the corresponding data. In all cases, we find that the predicted profiles are relatively accurate with the largest deviations resulting from using the lowest repressor copy number as the fit strain.</p>
<table>
<caption>Estimated DNA binding energy for DNA binding domain mutants with different repressor copy numbers. Reported values are the median of the posterior distribution with the upper and lower bounds of the 95% credible regions.</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Mutant</strong></th>
<th style="text-align: center;"><strong>Repressors</strong></th>
<th style="text-align: center;"><strong>DNA Binding Energy</strong> [<span class="math inline"><strong>k</strong><sub><strong>B</strong></sub><strong>T</strong></span>]</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Q18A</td>
<td style="text-align: center;">60</td>
<td style="text-align: center;"><span class="math inline"> − 9.8<sub> − 0.2</sub><sup> + 0.2</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">124</td>
<td style="text-align: center;"><span class="math inline"> − 10.3<sub> − 0.1</sub><sup> + 0.1</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;">260</td>
<td style="text-align: center;"><span class="math inline"> − 11.0<sub> − 0.1</sub><sup> + 0.1</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">1220</td>
<td style="text-align: center;"><span class="math inline"> − 11.3<sub> − 0.1</sub><sup> + 0.1</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Q18M</td>
<td style="text-align: center;">60</td>
<td style="text-align: center;"><span class="math inline"> − 15.83<sub> − 0.08</sub><sup> + 0.08</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">124</td>
<td style="text-align: center;"><span class="math inline"> − 15.7<sub> − 0.1</sub><sup> + 0.1</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;">260</td>
<td style="text-align: center;"><span class="math inline"> − 15.43<sub> − 0.06</sub><sup> + 0.07</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">1220</td>
<td style="text-align: center;"><span class="math inline"> − 15.27<sub> − 0.07</sub><sup><em></em><sub>+</sub>0.07</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Y17I</td>
<td style="text-align: center;">60</td>
<td style="text-align: center;"><span class="math inline"> − 9.4<sub> − 0.3</sub><sup> + 0.3</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">124</td>
<td style="text-align: center;"><span class="math inline"> − 9.5<sub> − 0.1</sub><sup> + 0.1</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;">260</td>
<td style="text-align: center;"><span class="math inline"> − 9.9<sub> − 0.1</sub><sup> + 0.1</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">1220</td>
<td style="text-align: center;"><span class="math inline"> − 10.1<sub> − 0.2</sub><sup> + 0.2</sup></span></td>
</tr>
</tbody>
</table>
<p>     The predicted change in free energy <span class="math inline"><em>Δ</em><em>F</em></span> using each fit strain can be seen in Fig. 12 . In this figure, the rows represent the repressor copy number of the strain to which the DNA binding energy was fit whereas the columns correspond to each mutant. In each plot, we have shown the data for all repressor copy numbers with the fit strain represented by white filled circles. Much as for the induction profiles, we see little difference in the predicted <span class="math inline"><em>Δ</em><em>F</em></span> for each strain, all of which accurately describe the inferred free energies. The ability to accurately predict the majority of the induction profiles of each mutant with repressor copy numbers ranging over two orders of magnitude strengthens our assessment that for these DNA binding domain mutations, only the DNA binding energy is modified.</p>
<figure>
<img src="ch7_figS11.png" id="fig:DNA_profile_pairwise_comparisons" data-short-caption="Pairwise comparisons of DNA binding mutant estimated induction profiles." alt="" /><figcaption>Figure 11: <strong>Pairwise comparisons of DNA binding mutant induction profiles.</strong> Rows correspond to the repressor copy number of the strain used to estimate the DNA binding energy for each mutant. Columns correspond to the repressor copy number of the strains that are predicted. Diagonals in which the data used to estimate the DNA binding energy are shown with a gray background. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS11.py">Python code (<code>ch7_figS11.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<figure>
<img src="ch7_figS12.png" id="fig:DNA_delF_pairwise_comparison" data-short-caption="Dependence of fitting strain on $\Delta F$ predictions of DNA binding domain mutants." alt="" /><figcaption>Figure 12: <strong>Dependence of fitting strain on <span class="math inline"><em>Δ</em><em>F</em></span> predictions of DNA binding domain mutants.</strong> Rows correspond to the repressor copy number used to estimate the DNA binding energy. Columns correspond to the particular mutant. Colored lines are the bounds of the 95% credible region of the predicted <span class="math inline"><em>Δ</em><em>F</em></span>. Open face points indicate the strain to which the DNA binding energy was fit. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS12.py">Python code (<code>ch7_figS12.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="bayesian-parameter-estimation-for-inducer-binding-domain-mutants">Bayesian Parameter Estimation for Inducer Binding Domain Mutants</h2>
<p>In Chapter 3, we put forward two naïve hypotheses for which parameters of our fold-change equation are affected by mutations in the inducer binding domain of the repressor. The first hypothesis was that only the inducer dissociation constants, <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, were perturbed from their wild-type values. Another hypothesis was that the inducer dissociation constants were affected in addition to the energetic difference between the active and inactive states of the repressor, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>.</p>
<p>     In this section, we first derive the statistical model for each hypothesis and then perform a series of diagnostic tests that expose the inferential limitations of each model. With well calibrated statistical models, we then apply each to an induction profile of the inducer binding mutant Q291K and assess the validity of each hypothesis. To understand the statistical models for each hypothesis, only the subsection <em>Building A Generative Statistical Model</em> is necessary.</p>
<h3 id="building-a-generative-statistical-model-1">Building a Generative Statistical Model</h3>
<p>For both hypotheses, we assume that the underlying physical model is the same while a subset of the parameters are modified. As the fold-change measurements for each biological replicate are statistically independent, we can assume that they are normally distributed about the theoretical fold-change value. Thus, for each model, we must include a parameter <span class="math inline"><em>σ</em></span> which is the standard deviation of the distribution of fold-change measurements. For the first hypothesis, in which only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are changed, we are interested in sampling the posterior distribution <span id="eq:hyp1_generic_bayes"><br /><span class="math display"><em>g</em>(<em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>, <em>σ</em> | <em>y</em>) ∝ <em>f</em>(<em>y</em> | <em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>, <em>σ</em>)<em>g</em>(<em>K</em><sub><em>A</em></sub>)<em>g</em>(<em>K</em><sub><em>I</em></sub>)<em>g</em>(<em>σ</em>),   (25)</span><br /></span> where <span class="math inline"><em>y</em></span> corresponds to the set of fold-change measurements. In the above model, we have assumed that the priors for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are independent. It is possible that it is more appropriate to assume that they are dependent and that a single prior distribution captures both parameters, <span class="math inline"><em>g</em>(<em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>)</span>. However, assigning this prior is more difficult and requires strong knowledge <em>a priori</em> about the relationship between them. Therefore, we continue under the assumption that the priors are independent.</p>
<p>     The generic posterior given in Eq. 25 can be extended to evaluate the second hypothesis in which <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is also modified, <span id="eq:hyp2_generic_bayes"><br /><span class="math display"><em>g</em>(<em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>, <em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub>, <em>σ</em> | <em>y</em>) ∝ <em>f</em>(<em>y</em> | <em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>, <em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub>, <em>σ</em>)<em>g</em>(<em>K</em><sub><em>A</em></sub>)<em>g</em>(<em>K</em><sub><em>I</em></sub>)<em>g</em>(<em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub>)<em>g</em>(<em>σ</em>)   (26)</span><br /></span> where we have included <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> as an estimated parameter and assigned a prior distribution.</p>
<p>     As we have assumed that the fold-change measurements across replicates are independent and normally distributed, the likelihoods for each hypothesis can be written as <span id="eq:hyp1_likelihood"><br /><span class="math display"><em>f</em>(<em>y</em> | <em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>, <em>σ</em>) ∼ Normal{<em>μ</em>(<em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>), <em>σ</em>},   (27)</span><br /></span> for the first hypothesis and <span id="eq:hyp2_likelihood"><br /><span class="math display"><em>f</em>(<em>y</em> | <em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>, <em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub>, <em>σ</em>) ∼ Normal{<em>μ</em>(<em>K</em><sub><em>A</em></sub>, <em>K</em><sub><em>I</em></sub>, <em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub>), <em>σ</em>},   (28)</span><br /></span> for the second. Here, we have assigned <span class="math inline"><em>μ</em>(…)</span> as the mean of the normal distribution as a function of the parameters defined by our fold-change equation.</p>
<p>     With a likelihood distribution in hand, we now turn toward assigning functional forms to each prior distribution. As we have used in the previous sections of this chapter, we can assign a half-normal prior for <span class="math inline"><em>σ</em></span> with a standard deviation of 0.1, namely, <span id="eq:IND_halfnorm_sigma"><br /><span class="math display"><em>g</em>(<em>σ</em>) ∼ HalfNormal{0, 0.1}.   (29)</span><br /></span> It is important to note that the inducer dissociation constants <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are scale invariant, meaning that a change from <span class="math inline">0.1 <em>μ</em></span>M to <span class="math inline">1 <em>μ</em></span>M yields a decrease in affinity equal to a change from <span class="math inline">10 <em>μ</em></span>M to <span class="math inline">100 <em>μ</em></span>M. As such, it is better to sample the dissociation constants on a logarithmic scale. We can assign a log normal prior for each dissociation constant as <span id="eq:lognormal_explicit"><br /><span class="math display">$$
g(K_A) = {1 \over K_A\sqrt{2\pi\phi^2}}\exp\left[-{(\log {K_A \over 1\,\mu\text{M}} - \mu_{K_A})^2 \over 2\phi^2}\right],
\qquad(30)$$</span><br /></span> or with the short-hand notion of <span id="eq:lognormal_shorthand"><br /><span class="math display"><em>g</em>(<em>K</em><sub><em>A</em></sub>) ∼ LogNormal{<em>μ</em><sub><em>K</em><sub><em>A</em></sub></sub>, <em>ϕ</em>}   (31)</span><br /></span> For <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, we assigned a mean <span class="math inline"><em>μ</em><sub><em>K</em><sub><em>A</em></sub></sub> = 2</span> and a standard deviation <span class="math inline"><em>ϕ</em> = 2</span>. For <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, we chose a mean of <span class="math inline"><em>μ</em><sub><em>K</em><sub><em>I</em></sub></sub> = 0</span> and <span class="math inline"><em>ϕ</em> = 2</span>, capturing our prior knowledge that <span class="math inline"><em>K</em><sub><em>A</em></sub> &gt; <em>K</em><sub><em>I</em></sub></span> for the wild-type LacI. While the prior distributions are centered differently, they both show extensive overlap, permitting mutations in which <span class="math inline"><em>K</em><sub><em>A</em></sub> &lt; <em>K</em><sub><em>I</em></sub></span>. For <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, we assign a normal distribution of the prior centered at 0 with a standard deviation of <span class="math inline">5 <em>k</em><sub><em>B</em></sub><em>T</em></span>, <span id="eq:epAI_prior"><br /><span class="math display"><em>g</em>(<em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub>) ∼ Normal{0, 5}.   (32)</span><br /></span> This permits values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> that are above or below zero, meaning that the inactive state of the repressor can be either more or less energetically favorable to the active state. A standard deviation of <span class="math inline">5 <em>k</em><sub><em>B</em></sub><em>T</em></span> permits a wide range of energies with <span class="math inline"> + 5 <em>k</em><sub><em>B</em></sub><em>T</em></span> and <span class="math inline"> − 5 <em>k</em><sub><em>B</em></sub><em>T</em></span> corresponding to <span class="math inline"> ≈ 99.5%</span> and <span class="math inline"> ≈ 0.5%</span> of the repressors being active in the absence of inducer, respectively.</p>
<h3 id="prior-predictive-checks-2">Prior predictive checks</h3>
<p>To ensure that these choices of prior distributions are appropriate, we performed prior predictive checks for each hypothesis as previously described in the second section of this chapter. We drew 1000 values from the prior distributions shown in Fig. 13 (A) for <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. Using the draws from the <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> priors alone, we generated data sets of <span class="math inline"> ≈ 70</span> measurements. The percentiles of the fold-change values drawn for the 1000 simulations is shown in the top panel of Fig. 13 (B).</p>
<p>     It can be seen that in the absence of inducer, the fold-change values are close to zero and are with distributed about the leakiness value due to <span class="math inline"><em>σ</em></span>. This is in contrast to the data sets generated when <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is permitted to vary along with <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. In the bottom panel of Fig. 13 (B), the fold-change when <span class="math inline"><em>c</em> = 0</span> can extend above 1.0 which is possible only when <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is included, which sets what fraction of the repressors is active. Under both hypotheses, the 99<span class="math inline"><em></em><sup>th</sup></span> percentile of the fold-change extends to just above 1 or just below 0, which matches our intuition of how the data should behave. Given these results, we are satisfied with these choices of priors and continue onto the next level of calibration of our model.</p>
<figure>
<img src="ch7_figS13.png" id="fig:ind_prior_predictive" data-short-caption="Prior predictive checks for two hypotheses of inducer binding domain mutants." alt="" /><figcaption>Figure 13: <strong>Prior predictive checks for two hypotheses of inducer binding domain mutants.</strong> (A) Probability density functions for <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, and <span class="math inline"><em>σ</em></span>. Black points correspond to draws from the distributions used for prior predictive checks. (B) Percentiles of the simulated data sets using draws from the <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> distributions only (top, purple bands) and using draws from <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> (bottom, orange bands). The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS13.py">Python code (<code>ch7_fig13.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="simulation-based-calibration-1">Simulation Based Calibration</h3>
<p>With an appropriate choice of priors, we turn to simulation based calibration to root out any pathologies lurking in the model itself or the implementation through MCMC. For each parameter under each model, we compute the <span class="math inline"><em>z</em></span>-score and shrinkage of each inference, shown in Fig. 14. Under the first hypothesis in which <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are the only perturbed parameters Fig. 14(A), we see all parameters have <span class="math inline"><em>z</em>−</span>scores clustered around 0, indicating that the value of the ground-truth is being accurately estimated through the inference. While the shrinkage for <span class="math inline"><em>σ</em></span> is close to 1 (indicating the prior is being informed by the data), the shrinkage for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> is heavily tailed with some values approaching zero. This is true for both statistical models, indicating that for some values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, the parameters are difficult to pin down with high certainty. In the application of these models to data, this will be revealed as large credible regions in the reported parameters. Under the second hypothesis in which all allosteric parameters are allowed to change, we see moderate shrinkage for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> purple points in 14(B) with the minimum shrinkage being around 0.5. The samples resulting in low shrinkage correspond to values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> that are highly positive or highly negative, in which small changes in the active fraction of repressors cannot be accurately measured through our model. However, the median shrinkage for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is approximately 0.92, meaning that the the data highly informed the prior distributions for the majority of the inferences. The rank distributions for all parameters under each model appear to be highly uniform, indicating that both statistical models are computationally tractable.</p>
<p>      With knowledge of the caveats of estimating <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> for both models, we proceed with our analysis and examine how accurately these models can capture the phenomenology of the data.</p>
<figure>
<img src="ch7_figS14.png" id="fig:ind_sbc" data-short-caption="Simulation based calibration of statistical models for inducer binding domain mutants." alt="" /><figcaption>Figure 14: <strong>Simulation based calibration of statistical models for inducer binding domain mutants.</strong> (A) Sensitivity statistics and rank distribution for a statistical model in which <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are the only parameters permitted to vary. (B) Sensitivity statistics and rank distribution for a model in which all allosteric parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> are allowed to be modified by the mutation. Gray envelope in the bottom plots correspond to the 99<span class="math inline"><em></em><sup>th</sup></span> percentile of variation expected from a true uniform distribution. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS14.py">Python code (<code>ch7_figS14.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h3 id="posterior-predictive-checks-1">Posterior Predictive Checks</h3>
<p>With a properly calibrated statistical model for each hypothesis, we now apply it to a representative dataset. While each model was applied to each inducer binding domain mutant, we only show the application to the mutant Q291K with 260 repressors per cell paired with the native <em>lac</em> operator O2.</p>
<p>     The results from applying the statistical model in which only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> can change is shown in Fig. 15. The joint and marginal distributions for each parameter (Fig. 15 (A)) reveal a strong correlation between <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> whereas all other parameters are symmetric and independent. While the joint and marginal distributions look well behaved, the percentiles of the posterior predictive checks (Fig. 15 (B)) are more suspect. While all data falls within the 95<span class="math inline"><em></em><sup>th</sup></span> percentile, the overall trend of the data is not well predicted. Furthermore, the percentiles expand far below zero, indicating that the sampling of <span class="math inline"><em>σ</em></span> is compensating for the leakiness in the data being larger than it should be if only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> were the changing parameters.</p>
<p>     We see significant improvement when <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is permitted to vary in addition to <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>. Fig. 16 (A) shows the joint and marginal distributions between all parameters from the MCMC sampling. We still see correlation between <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, although it is not as strong as in the case where they are the only parameters allowed to change due to the mutation. We also see that the marginal distribution for <span class="math inline"><em>σ</em></span> has shrunk significantly compared to the marginal distribution in Fig. 15 (A). The percentiles of the posterior predictive checks, shown in Fig. 16 (B) are much more in line with the experimental measurements, with the 5<span class="math inline">th</span> percentile following the data for the entire induction profile.</p>
<p>     In this section we have presented two hypotheses for the minimal parameter set needed to describe the inducer binding mutations, derived a statistical model for each, thoroughly calibrated its behavior, and applied it to a representative data set. The posterior predictive checks (Fig. 15 and Fig. 16) help us understand which hypothesis is more appropriate for that particular mutant. The incredibly wide percentiles and significant change in the leakiness that result from a model in which only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are perturbed suggests that more than those two parameters should be changing. We see significant improvement in the description of the data when <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is altered, indicating that it is the more appropriate hypothesis of the two.</p>
<figure>
<img src="ch7_figS15.png" id="fig:kaki_post_predictive" data-short-caption="Posterior predictive checks for inducer binding domain mutants where only $K_A$ and $K_I$ are changed." alt="" /><figcaption>Figure 15: <strong>Posterior predictive checks for inducer binding domain mutants where only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are changed.</strong> (A) MCMC sampling output for each parameter. Joint distributions are colored by the value of the log posterior with increasing probability corresponding to transition from blue to yellow. (B) Percentiles of the data generated from the likelihood distribution for each sample of <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>σ</em></span>. Overlaid points are the experimentally observed measurements. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS15-S16.py">Python code (<code>ch7_figS15-S16.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<figure>
<img src="ch7_figS16.png" id="fig:kaki_epai_post_predictive" data-short-caption="Posterior predictive checks for inducer binding domain mutants where all allosteric parameter can change." alt="" /><figcaption>Figure 16: <strong>Posterior predictive checks for inducer binding domain mutants where all allosteric parameters can change.</strong> (A) MCMC sampling output for all parameters. Joint distributions are colored by the value of the log posterior with increasing probability corresponding to the transition from blue to yellow. Marginal distributions are shown adjacent to each joint distribution. (B) Percentiles of the data generated from the likelihood for each sample of <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, and <span class="math inline"><em>σ</em></span>. The corresponding experimental data for Q291K are shown as black open-faced circles. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS15-S16.py">Python code (<code>ch7_figS15-S16.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<h2 id="additional-characterization-of-inducer-binding-domain-mutants">Additional Characterization of Inducer Binding Domain Mutants</h2>
<p>To predict the induction profiles of the inducer binding mutants, we used only the induction profile of each mutant paired with the native O2 <em>lac</em> operator to infer the parameters. Here, we examine the influence the choice of fit strain has on the predictions of the induction profiles and <span class="math inline"><em>Δ</em><em>F</em></span> for each mutant.</p>
<p>     In Chapter 3, we dismissed the hypothesis that only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> were changing due to the mutation and based the fit to a single induction profile. In Fig. 17, the fits and predictions for each mutant paired with each operator sequence queried. Here, the rows correspond to the operator sequence of the fit strain while the columns correspond to the operator sequence of the predicted strain. The diagonals show the fit induction profiles and the corresponding data. Regardless of the choice of fit strain, the predicted induction profiles of the repressor paired with the O3 operator are poor, with the leakiness in each case being significantly underestimated. We also see that fitting to O3 results in poor predictions with incredibly wide credible regions for the other two operators. In <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>, we also found that fitting <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> to the induction profile of O3 generally resulted in poor predictions of the other strains with comparably wide credible regions.</p>
<p>     When <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> is included as a parameter, however, the predictive power is improved for all three operators, as can be seen in Fig. 18. While the credible regions are still wide when fit to the O3 operator, they are much narrower than under the first hypothesis. We emphasize that we are able to accurately predict the leakiness of nearly every strain by redetermining <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> whereas the leakiness was not predicted when only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> were considered. Thus, we conclude that all three allosteric parameters <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> are modified for these four inducer binding domain mutations. The values of the inferred parameters are reported in Table 7.3.</p>
<p>      We also examined the effect the choice of fit strain has on the predicted <span class="math inline"><em>Δ</em><em>F</em></span>, shown in Fig. 19. We find that the predictions agree with the data regardless of the choice of fit strain. One exception is the prediction of the Q291K <span class="math inline"><em>Δ</em><em>F</em></span> when the parameters fit to the O3 induction profile are used. As the induction profile for Q291K paired with O3 is effectively flat at a fold-change of 1, it is difficult to properly estimate the parameters of our sigmoidal function. We note all measurements of <span class="math inline"><em>Δ</em><em>F</em></span> for Q291K are described by using either the parameters fit to either O1 or O3 induction profiles, suggesting that the choice of fit strain makes little difference.</p>
<figure>
<img src="ch7_figS17.png" id="fig:KaKi_only_pairwise_comparison" data-short-caption="Pairwise comparison of fit strain versus predictions assuming only $K_A$ and $K_I$ are influenced by mutations in the inducer binding domain." alt="" /><figcaption>Figure 17: <strong>Pairwise comparison of fit strain versus predictions assuming only <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> are influenced by the mutation.</strong> Rows correspond to the operator sequence of the strain used for the parameter inference. Columns correspond to the operator sequence of the predicted strain. Colors identify the mutation. Diagonal positions show the induction fit strain and profiles. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS17-S18.py">Python code<br />
(<code>ch7_figS17-S18.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<figure>
<img src="ch7_figS18.png" id="fig:KaKi_epAI_pairwise_comparison" data-short-caption="Pairwise comparison of fit strain versus predictions assuming all allosteric parameters are affected by the mutation in the inducer binding domain." alt="" /><figcaption>Figure 18: <strong>Pairwise comparison of fit strain versus predictions assuming all allosteric parameters are affected by the mutation.</strong> Rows correspond to the operator of the strain used to fit the parameters. Columns correspond to the operator of the strains whose induction profile is predicted. Mutants are identified by color. Diagonals (gray background) show the induction profiles of the strain to which the parameters were fit. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS17-S18.py">Python code<br />
(<code>ch7_figS17-S18.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<figure>
<img src="ch7_figS19.png" id="fig:IND_deltaF_comparison" data-short-caption="Comparison of choice of fit strain on predicted $\Delta F$ profiles for inducer binding domain mutants." alt="" /><figcaption>Figure 19: <strong>Comparison of choice of fit strain on predicted <span class="math inline"><em>Δ</em><em>F</em></span> profiles.</strong> Rows correspond to the operator of the strain to which the parameters were fit. Columns correspond to mutations. Points are colored by their operator sequence. The data corresponding to the operator of the fit strain are shown as white-faced points. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS19.py">Python code<br />
(<code>ch7_figS19.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<table>
<caption>Inferred values of <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> for inducer binding domain mutants. Values reported are the mean of the posterior distribution with the upper and lower bounds of the 95% credible region.</caption>
<colgroup>
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Mutant</strong></th>
<th style="text-align: center;"><strong>Operator</strong></th>
<th style="text-align: center;"><span class="math inline"><strong>K</strong><sub><strong>A</strong></sub></span> <strong>[<span class="math inline"><em>μ</em></span>M]</strong></th>
<th style="text-align: center;"><span class="math inline"><strong>K</strong><sub><strong>I</strong></sub></span> <strong>[<span class="math inline"><em>μ</em></span>M]</strong></th>
<th style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> <strong>[<span class="math inline"><strong>k</strong><sub><strong>B</strong></sub><strong>T</strong></span>]</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">F164T</td>
<td style="text-align: center;">O1</td>
<td style="text-align: center;"><span class="math inline">290<sub> − 56</sub><sup> + 60</sup></span></td>
<td style="text-align: center;"><span class="math inline">1<sub> − 0.98</sub><sup> + 4</sup></span></td>
<td style="text-align: center;"><span class="math inline">4<sub> − 3</sub><sup> + 5</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">O2</td>
<td style="text-align: center;"><span class="math inline">165<sub> − 65</sub><sup> + 90</sup></span></td>
<td style="text-align: center;"><span class="math inline">3<sub> − 3</sub><sup> + 6</sup></span></td>
<td style="text-align: center;"><span class="math inline">1<sub> − 2</sub><sup> + 5</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;">O3</td>
<td style="text-align: center;"><span class="math inline">110<sub> − 105</sub><sup> + 700</sup></span></td>
<td style="text-align: center;"><span class="math inline">7<sub> − 4</sub><sup> + 5</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 0.9<sub> − 0.3</sub><sup> + 0.4</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;">Q291K</td>
<td style="text-align: center;">O1</td>
<td style="text-align: center;"><span class="math inline"> &gt; 1000</span></td>
<td style="text-align: center;"><span class="math inline">410<sub> − 100</sub><sup> + 150</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 3.2<sub> − 0.1</sub><sup> + 0.1</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;">O2</td>
<td style="text-align: center;"><span class="math inline"> &gt; 1000</span></td>
<td style="text-align: center;"><span class="math inline">310<sub> − 60</sub><sup> + 70</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 3.11<sub> − 0.07</sub><sup> + 0.07</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">O3</td>
<td style="text-align: center;"><span class="math inline">10<sub> − 10</sub><sup> + 200</sup></span></td>
<td style="text-align: center;"><span class="math inline">1<sub> − 1</sub><sup> + 9</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 7<sub> − 5</sub><sup> + 3</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Q291R</td>
<td style="text-align: center;">O1</td>
<td style="text-align: center;"><span class="math inline">3<sub> − 3</sub><sup> + 27</sup></span></td>
<td style="text-align: center;"><span class="math inline">2<sub> − 2</sub><sup> + 20</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 1.9<sub> − 0.3</sub><sup> + 0.4</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">O2</td>
<td style="text-align: center;"><span class="math inline">9<sub> − 9</sub><sup> + 20</sup></span></td>
<td style="text-align: center;"><span class="math inline">8<sub> − 8</sub><sup> + 20</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 2.32<sub> − 0.09</sub><sup> + 0.01</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;">O3</td>
<td style="text-align: center;"><span class="math inline">6<sub> − 6</sub><sup> + 24</sup></span></td>
<td style="text-align: center;"><span class="math inline">9<sub> − 9</sub><sup> + 30</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 2.6<sub> − 0.5</sub><sup> + 0.4</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;">Q291V</td>
<td style="text-align: center;">O1</td>
<td style="text-align: center;"><span class="math inline"> &gt; 1000</span></td>
<td style="text-align: center;"><span class="math inline">3<sub> − 3</sub><sup> + 13</sup></span></td>
<td style="text-align: center;"><span class="math inline">6<sub> − 4</sub><sup> + 4</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;">O2</td>
<td style="text-align: center;"><span class="math inline">650<sub> − 250</sub><sup> + 450</sup></span></td>
<td style="text-align: center;"><span class="math inline">8<sub> − 8</sub><sup> + 8</sup></span></td>
<td style="text-align: center;"><span class="math inline">3<sub> − 3</sub><sup> + 6</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;">O3</td>
<td style="text-align: center;"><span class="math inline">100<sub> − 90</sub><sup> + 400</sup></span></td>
<td style="text-align: center;"><span class="math inline">22<sub> − 18</sub><sup> + 33</sup></span></td>
<td style="text-align: center;"><span class="math inline">0.1<sub> − 0.6</sub><sup> + 0.8</sup></span></td>
</tr>
</tbody>
</table>
<h2 id="comparing-parameter-values-to-the-literature">Comparing Parameter Values To The Literature</h2>
<p>In this section, we compare and contrast the biophysical parameter values we use to characterize the wild-type Lac repressor with the rich literature that consists of <em>in vitro</em> and <em>in vivo</em> experiments. This section has an accompanying interactive figure available on the <a href="http://rpgroup.caltech.edu/mwc_mutants">paper website</a> which allows the reader to examine different combinations of parameter values and their agreement or disagreement with data taken from <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>; <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span> and <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>.</p>
<p>     While the mutations used in this work and those in <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> are the same, we report significantly different values for the inducer binding, DNA binding parameters, and the relative energy difference between active and inactive states of the mutant repressors. The apparent disagreement of parameter values between the present work and those presented in <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> in part stem from different treatments of the values for the wild-type Lac repressor. Since its isolation by Gilbert and Müller-Hill in the 1960’s <span class="citation" data-cites="gilbert1966">Gilbert and Müller-Hill (<a href="#ref-gilbert1966" role="doc-biblioref">1966</a>)</span>, the Lac repressor has been the subject of intense biochemical and structural study. Many measurements of the inducer and DNA binding kinetics of the repressor <em>in vitro</em> <span class="citation" data-cites="ogorman1980">(such as O’Gorman et al. <a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span> and their values have informed the fitting of other parameters from measurements <em>in vivo</em> (such as <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> and <span class="citation" data-cites="daber2009a">Daber, Sharp, and Lewis (<a href="#ref-daber2009a" role="doc-biblioref">2009</a>)</span>). All of these measurements, however, do not <em>directly</em> measure the DNA- or inducer-binding kinetics nor the equilibrium constant between the active and inactive states of the repressor. To properly estimate the parameters, one must either have direct measurement of a subset of the parameters or make assumptions regarding the states of the system. Examples of the estimated allosteric parameter values of the wild-type LacI repressor from our previous work <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>, that of <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span>, and <em>in vitro</em> measurements from <span class="citation" data-cites="ogorman1980">O’Gorman et al. (<a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span> are given in Table 7.3 The theoretical predictions for the fold-change in gene expression, along with the values reported in <span class="citation" data-cites="daber2009a">Daber, Sharp, and Lewis (<a href="#ref-daber2009a" role="doc-biblioref">2009</a>)</span>, can be seen using the interactive figure on the <a href="https://rpgroup.caltech.edu/mwc_mutants">paper website</a>, where the reader can also enter their own parameter values and independently assess the agreement or lack thereof with the data.</p>
<p>      It is notable that differences between the various references shown in Table 7.3 can be drastic, in some cases differing by almost an order of magnitude. Of particular note is the disagreement in the energy difference between the active and inactive states of the repressor, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>. <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> determines a negative value of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> meaning that the inactive state of the repressor is energetically favorable to the active state. In stark contrast is the value reported in our previous work of <span class="math inline"> + 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>, implying that the active state is significantly more stable than the inactive state. The now seminal <em>in vitro</em> measurements reported in <span class="citation" data-cites="ogorman1980">O’Gorman et al. (<a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span> suggest that the two states are nearly energetically equivalent.</p>
<p>     The wide range of these reported values demonstrate that such thermodynamic models are highly degenerate, meaning that many combinations of parameter values can result in nearly equally good descriptions of the data. To illustrate this point, we estimated <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and DNA binding energy <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> for each operator to the data reported in <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>; <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>; <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>, using the three values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> shown in Table 7.4. The resulting fits can be seen in Fig. 20. Despite the drastically different values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> it is possible to generate adequate fits by modulating the other parameters. The parameter values of these fits, reported in Table 7.3 further illustrate this point as they differ significantly from one another.</p>
<p>     The theoretically predicted fold-change in the presence of multiple promoters, shown in Fig. 20 (E) is perhaps the most informative test of the parameter values. The theoretical advancements made in <span class="citation" data-cites="weinert2014">Weinert et al. (<a href="#ref-weinert2014" role="doc-biblioref">2014</a>)</span> provide a means to mathematically grasp the intricacies of plasmid-borne expression through calculation of the chemical potential. This formalism transforms the intimidating combinatorics of <span class="math inline"><em>R</em></span> repressors and <span class="math inline"><em>N</em></span> specific binding sites (i.e. plasmid reporter genes) into a two-state system where one can compute an <em>effective</em> repressor copy number regulating a single promoter. Using this formalism, the input-output function for the fold-change in gene expression can be written as <span id="eq:fc_fug"><br /><span class="math display">$$
\text{fold-change} = {1\over 1 +
    \lambda_{R}(c)e^{-\beta\Delta\varepsilon_{RA}}},
\qquad(33)$$</span><br /></span> where the effective repressor copy number (termed the fugacity) is denoted as <span class="math inline"><em>λ</em><sub><em>R</em></sub>(<em>c</em>)</span>. In Chapter 6 of this thesis, we show that the inflection point of Eq. 33, whose curves shown in the bottom right-hand plot of Fig. 20, is located where the number of specific binding sites for the repressor is approximately equal to the number of repressors in the active state. Using this key feature, one can infer <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> given prior knowledge of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> and the total number of repressors per cell. As shown in Fig. 20, using values of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> from <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>; <span class="citation" data-cites="ogorman1980">O’Gorman et al. (<a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span> approximately agree with the measurements whereas the predicted curves using <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> from <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> overestimates the fold-change, even though these values accurately describe the simple-repression data shown in the other panels of Fig. 20.</p>
<figure>
<img src="ch7_figS20.png" id="fig:lit_degeneracy" data-short-caption="Degenerate fits of data using parameter values from the literature." alt="" /><figcaption>Figure 20: <strong>Degenerate fits of data from <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>; <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>; <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span> using different values for active/inactive state energy difference <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span></strong>. In all plots, the solid, dashed, and dotted lines correspond to the best-fit curves conditioned on the data using parameter values for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> of <span class="math inline">4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span>, <span class="math inline"> − 1.75 <em>k</em><sub><em>B</em></sub><em>T</em></span>, and <span class="math inline">0.35 <em>k</em><sub><em>B</em></sub><em>T</em></span>, respectively. Induction profiles from <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> for operators O1 (A), O2 (B), and O3 (C) are shown as points and errors which correspond to the mean and standard error of at least 10 biological replicates. (D) Leakiness measurements of the simple repression motif with one unique regulated reporter gene. Data shown are from <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>; <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>. (E) The transcription factor titration effect. For gene expression measurements on plasmids, the fold-change as a function of repressor copy number exhibits strong nonlinearities. We used this effect as a way to independently infer the parameter <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> and it can be seen that this breaks the degeneracy between different parameters. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS20.py">Python code (<code>ch7_figS20.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<p>     Without some direct <em>in vivo</em> measurements of these parameters, one must make assumptions about the system to make any quantitative progress. We chose to use the parameter values defined in our laboratory as the repressor fugacity provides us with an independent, albeit indirect, measurement of <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> which other works such as <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> and <span class="citation" data-cites="ogorman1980">O’Gorman et al. (<a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span> do not. Both of these works determine all of parameter values simultaneously by fitting to a single set of measurements. While these measurements accurately describe their data, their parameter values are less successful in accounting for data from <span class="citation" data-cites="brewster2014">Brewster et al. (<a href="#ref-brewster2014" role="doc-biblioref">2014</a>)</span>; <span class="citation" data-cites="garcia2011">Garcia and Phillips (<a href="#ref-garcia2011" role="doc-biblioref">2011</a>)</span>; <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span>. In the context of this work, we emphasize that we make many of the same qualitative conclusions as in <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> with respect to the effects of the mutations using our particular set of parameter values.</p>
<p>     While we use different values for <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>, the qualitative results between this work and that of <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> are in agreement. For example, both works determine that mutations in the DNA binding domain alter only the DNA binding affinity whereas the mutations in the inducer binding pocket affect only the allosteric parameters. Because the biological variables such as repressor and reporter gene copy number are tightly controlled in our system, we are able to more precisely measure features of the induction profiles such as the leakiness in gene expression. This ability allows us to detect changes in the active/inactive equilibrium which were masked in <span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span> by the experimental design.While the precise parameter values may be different between publications, the exploration of free energy differences resulting from mutations are parameter-value independent and any parameter disagreements do not change our theoretical model. Fig. 2 of the main text is presented with no knowledge of parameter values – it simply shows the mathematics of the model. While the value of <span class="math inline"><em>Δ</em><em>F</em></span> will ultimately depend on the parameter values, the formalism of this work remains agnostic to the parameter values and can be a useful tool for classifying mutations and couple the sequence-level variation to the systems-level response.</p>
<table>
<caption>Thermodynamic parameter values of wild-type LacI from the literature.</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Parameter</strong></th>
<th style="text-align: center;"><strong>Value</strong></th>
<th style="text-align: center;"><strong>Reference</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span></td>
<td style="text-align: center;"><span class="math inline"> ≥ 4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 1.7 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">0.35 <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="citation" data-cites="ogorman1980">O’Gorman et al. (<a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"><span class="math inline"><em>K</em><sub><em>A</em></sub></span></td>
<td style="text-align: center;"><span class="math inline">139<sub> − 20</sub><sup> + 22</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">16 <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">133 <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="citation" data-cites="ogorman1980">O’Gorman et al. (<a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span class="math inline"><em>K</em><sub><em>I</em></sub></span></td>
<td style="text-align: center;"><span class="math inline">0.53<sub> − 0.01</sub><sup> + 0.01</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">2 <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="citation" data-cites="daber2011a">Daber, Sochor, and Lewis (<a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">4 <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="citation" data-cites="ogorman1980">O’Gorman et al. (<a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
</tbody>
</table>
<table>
<caption>Estimated parameters from global fits of data from literature.</caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Parameter</strong></th>
<th style="text-align: center;"><strong>Value</strong></th>
<th style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> <strong>Reference Value</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> (O1 operator)</td>
<td style="text-align: center;"><span class="math inline"> − 15.1<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline">4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 17.1<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline"> − 1.75 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 15.7<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline">0.35 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="ogorman1980">(O’Gorman et al. <a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> (O1 operator)</td>
<td style="text-align: center;"><span class="math inline"> − 15.1<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline">4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 17.1<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline"> − 1.75 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 15.7<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline">0.35 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="ogorman1980">(O’Gorman et al. <a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> (O2 operator)</td>
<td style="text-align: center;"><span class="math inline"> − 13.4<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline">4.5<em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 15.4<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline"> − 1.75 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 14.0<sub> − 0.1</sub><sup> + 0.1</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline">0.35 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="ogorman1980">(O’Gorman et al. <a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> (O3 operator)</td>
<td style="text-align: center;"><span class="math inline"> − 9.21<sub> − 0.06</sub><sup> + 0.06</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline">4.5<em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 11.29<sub> − 0.06</sub><sup> + 0.06</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline"> − 1.75 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline"> − 9.85<sub> − 0.05</sub><sup> + 0.06</sup> <em>k</em><sub><em>B</em></sub><em>T</em></span></td>
<td style="text-align: center;"><span class="math inline">0.35 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="ogorman1980">(O’Gorman et al. <a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"><span class="math inline"><em>K</em><sub><em>A</em></sub></span></td>
<td style="text-align: center;"><span class="math inline">225<sub> − 10</sub><sup> + 10</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">290<sub> − 20</sub><sup> + 20</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline"> − 1.75 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">270<sub> − 20</sub><sup> + 20</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">0.35 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="ogorman1980">(O’Gorman et al. <a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"><span class="math inline"><em>K</em><sub><em>I</em></sub></span></td>
<td style="text-align: center;"><span class="math inline">0.81<sub> − 0.05</sub><sup> + 0.05</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">4.5 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="razo-mejia2018">(Razo-Mejia et al. <a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span></td>
</tr>
<tr class="odd">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">8.2<sub> − 0.5</sub><sup> + 0.5</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline"> − 1.75 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="daber2011a">(Daber, Sochor, and Lewis <a href="#ref-daber2011a" role="doc-biblioref">2011</a>)</span></td>
</tr>
<tr class="even">
<td style="text-align: center;"></td>
<td style="text-align: center;"><span class="math inline">5.5<sub> − 0.3</sub><sup> + 0.5</sup> <em>μ</em></span>M</td>
<td style="text-align: center;"><span class="math inline">0.35 <em>k</em><sub><em>B</em></sub><em>T</em></span> <span class="citation" data-cites="ogorman1980">(O’Gorman et al. <a href="#ref-ogorman1980" role="doc-biblioref">1980</a>)</span></td>
</tr>
</tbody>
</table>
<h2 id="parameter-estimation-using-all-induction-profiles">Parameter Estimation Using All Induction Profiles</h2>
<p>In Chapter 3 and Sec. 7.1 and 7.2 of this chapter, we have laid out our strategy for inferring the the various parameters of our model to a single induction profile and using the resulting values to predict the free energy and induction profiles of other strains. In this section, we estimate the parameters using all induction profiles of a single mutant and using the estimated values to predict the free energy profiles.</p>
<p>      The inferred DNA binding energies considering induction profiles of all repressor copy numbers for the three DNA binding mutants are reported in Table 7.5. These parameters are close to those reported in Table 7.1 for each repressor copy number with Q18A showing the largest differences. The resulting induction profiles and predicted change in free energy for these mutants can be seen in Fig. 21. Overall, the induction profiles match the data to an appreciable agree. We acknowledge that even when using <em>all</em> repressor copy numbers, the fit to Q18A remains imperfect. However we contend that this disagreement is comparable to that observed in <span class="citation" data-cites="razo-mejia2018">Razo-Mejia et al. (<a href="#ref-razo-mejia2018" role="doc-biblioref">2018</a>)</span> which described the induction profile of the wild-type repressor. We find that the predicted change in free energy (bottom row in Fig. 21 (B)) narrows compared to that in Fig. 12 and Fig. 3.3 of Chapter 3, confirming that considering all induction profiles improves our inference of the most-likely DNA binding energy. There appears to be a very slight trend in the <span class="math inline"><em>Δ</em><em>F</em></span> for Q18A at higher inducer concentrations, though the overall change in free energy from 0 to 5000 <span class="math inline"><em>μ</em></span>M IPTG is small.</p>
<figure>
<img src="ch7_figS21.png" id="fig:global_DNA_profiles" data-short-caption="Induction profiles and predicted change in free energy using parameters estimated from the complete data sets." alt="" /><figcaption>Figure 21: <strong>Induction profiles and predicted change in free energy using parameters estimated from the complete data sets.</strong> Top row shows fold-change measurements (points) as mean and standard error with ten to fifteen biological replicates. Shaded lines correspond to the 95% credible regions of the induction profiles using the estimated values of the DNA binding energies reported in Table 7.5. Bottom row shows the 95% credible regions of the predicted change in free energy (shaded lines) along with the inferred free energy of data shown in the top row. In all plots, the inducer concentration is shown on a symmetric log scale with linear scaling between 0 and <span class="math inline">10<sup> − 2</sup> <em>μ</em></span>M and log scaling elsewhere. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS21.py">Python code (<code>ch7_figS21.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<table>
<caption>Estimated DNA binding energies for each DNA binding domain mutant using all repressor copy numbers</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Mutant</strong></th>
<th style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em><em>A</em></sub></span> [<span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span>]</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Y17I</td>
<td style="text-align: center;"><span class="math inline"> − 9.81<sub> − 0.08</sub><sup> + 0.04</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;">Q18A</td>
<td style="text-align: center;"><span class="math inline"> − 10.60<sub> − 0.07</sub><sup> + 0.07</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Q18M</td>
<td style="text-align: center;"><span class="math inline"> − 15.61<sub> − 0.05</sub><sup> + 0.05</sup></span></td>
</tr>
</tbody>
</table>
<p>     We also estimated the allosteric parameters (<span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span>) for all inducer binding domain mutations using the induction profiles of all three operator sequences. The values, reported in Table 7.6 are very similar to those estimated from a single induction profile (Table 7.3). We note that for Q291R, it is difficult to properly estimate the values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span> and <span class="math inline"><em>K</em><sub><em>I</em></sub></span> as the observed induction profile is approximately flat. The induction profiles and predicted change in free energy for each inducer binding mutant is shown in Fig. 22. We see notable improvement in the agreement between the induction profiles and the observed data, indicating that considering all data significantly shrinks the uncertainty of each parameter. The predicted change in free energy is also improved compared to that shown in Fig. 19. We emphasize that the observed free energy difference for each point assumes no knowledge of the underlying parameters and comes directly from measurements. The remarkable agreement between the predicted free energy and the observations illustrates that redetermining the allosteric parameters is sufficient to describe how the free energy changes as a result of the mutation.</p>
<figure>
<img src="ch7_figS22.png" id="fig:global_IND_profiles" data-short-caption="Induction profiles and predicted change in free energy using parameters estimated from the complete data sets for inducer binding domain mutants." alt="" /><figcaption>Figure 22: <strong>Induction profiles and predicted change in free energy using parameters estimated from the complete data sets for inducer binding domain mutants.</strong> Top row shows fold-change measurements (points) as mean and standard error with ten to fifteen biological replicates. Shaded lines correspond to the 95% credible regions of the induction profiles using the estimated values of the allosteric parameters reported in Table 7.6. Bottom row shows the 95% credible regions of the predicted change in free energy (shaded lines) along with the inferred free energy of data shown in the top row. In all plots, the inducer concentration is shown on a symmetric log scale with linear scaling between 0 and 10<span class="math inline"><em></em><sup> − 2</sup> <em>m</em><em>u</em></span>M and log scaling elsewhere. The <a href="https://github.com/gchure/phd/blob/master/src/chapter_07/code/ch7_figS22.py">Python code (<code>ch7_figS22.py</code>)</a> used to generate this figure can be found on the thesis <a href="https://github.com/gchure/phd">GitHub repository</a>.</figcaption>
</figure>
<table>
<caption>Estimated values for <span class="math inline"><em>K</em><sub><em>A</em></sub></span>, <span class="math inline"><em>K</em><sub><em>I</em></sub></span>, and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> for inducer binding domain mutations using induction profiles of all operator sequences.</caption>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Mutant</strong></th>
<th style="text-align: center;"><span class="math inline"><em>K</em><sub><em>A</em></sub></span> [<span class="math inline"><em>μ</em></span>M]</th>
<th style="text-align: center;"><span class="math inline"><em>K</em><sub><em>I</em></sub></span> [<span class="math inline"><em>μ</em></span>M]</th>
<th style="text-align: center;"><span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em><em>I</em></sub></span> [<span class="math inline"><em>k</em><sub><em>B</em></sub><em>T</em></span>]</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">F161T</td>
<td style="text-align: center;"><span class="math inline">300<sub> − 60</sub><sup> + 60</sup></span></td>
<td style="text-align: center;"><span class="math inline">12.7<sub> − 0.1</sub><sup> + 0.1</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 0.9<sub> − 0.3</sub><sup> + 0.3</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;">Q291K</td>
<td style="text-align: center;"><span class="math inline"> &gt; 1</span> mM</td>
<td style="text-align: center;"><span class="math inline">330<sub> − 70</sub><sup> + 60</sup></span></td>
<td style="text-align: center;"><span class="math inline"> − 3.17<sub> − 0.07</sub><sup> + 0.07</sup></span></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Q291R</td>
<td style="text-align: center;"><span class="math inline"> &gt; 1</span> mM</td>
<td style="text-align: center;"><span class="math inline"> &gt; 1</span> mM</td>
<td style="text-align: center;"><span class="math inline"> − 2.4<sub> − 0.2</sub><sup> + 0.2</sup></span></td>
</tr>
<tr class="even">
<td style="text-align: center;">Q291V</td>
<td style="text-align: center;"><span class="math inline"> &gt; 1</span> mM</td>
<td style="text-align: center;"><span class="math inline">53<sub> − 13</sub><sup> + 17</sup></span></td>
<td style="text-align: center;"><span class="math inline">0<sub> − 0.3</sub><sup> + 0.3</sup></span></td>
</tr>
</tbody>
</table>
<h2 id="generalizability-of-data-collapse-to-other-regulatory-architectures">Generalizability of Data Collapse To Other Regulatory Architectures</h2>
<p>In Chapters 2, 3, and 4, we have stated that the input-output function for the fold-change in gene expression can be rewritten in the form of a Fermi function. However, this result can be derived directly by coarse graining the transcription factor bound and unbound states of the systems’ partition function. In this section, we show how coarse graining of promoter occupancy states results in a general Fermi function so long as the transcription factor bound states and transcriptionally active states do not overlap.</p>
<p>     As shown in Chapter 2, the partition function <span class="math inline">𝒵</span> for the simple repression motif (ignoring allosteric control) can be enumerated as <span id="eq:z_simple_rep"><br /><span class="math display">$$
\mathcal{Z} = \overbrace{1 + {P \over
N_{NS}}e^{-\beta\Delta\varepsilon_P}}^\text{repressor unbound states} +
\underbracee^{-\beta\Delta\varepsilon_{R}}}_\text{repressor
bound state} = \mathcal{Z}_{\neg r} + \mathcal{Z}_{r},
\qquad(34)$$</span><br /></span> where <span class="math inline"><em>R</em></span> is the number of repressors, <span class="math inline"><em>P</em></span> is the number of polymerases, <span class="math inline"><em>N</em><sub><em>N</em><em>S</em></sub></span> is the number of nonspecific binding sites and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>P</em></sub></span> and <span class="math inline"><em>Δ</em><em>ε</em><sub><em>R</em></sub></span> are the binding energies of the polymerase and repressor to the DNA, respectively. The states can be grouped into either repressor bound states (right-hand terms) or repressor unbound states (left hand terms), denoted as <span class="math inline">𝒵<sub><em>r</em></sub></span> and <span class="math inline">𝒵<sub>¬<em>r</em></sub></span>, respectively. The probability of the repressor <em>not</em> being bound to the promoter can be computed as <span id="eq:p_notr"><br /><span class="math display">$$
P(\neg r) = {\mathcal{Z}_{\neg r} \over \mathcal{Z}_{\neg r} + \mathcal{Z}_r}.
\qquad(35)$$</span><br /></span></p>
<p>    In this coarse-grained description, the transcriptionally active states are separate from the repressor bound states. Thus, to calculate the fold-change in gene expression, we can compute the ratio of <span class="math inline"><em>P</em>(¬<em>r</em>|<em>R</em> &gt; 0)</span> when repressor is present to <span class="math inline"><em>P</em>(¬<em>r</em> | <em>R</em> = 0)</span> when repressor is absent. As the latter term is equal to 1, the fold-change in gene expression is equivalent to Eq. 35. We can compute an <em>effective</em> free energy <span class="math inline"><em>F̃</em></span> of each coarse-grained state as <span id="eq:eff_energy"><br /><span class="math display"><em>F̃</em><sub>¬<em>r</em></sub> =  − <em>k</em><sub><em>B</em></sub><em>T</em>log 𝒵<sub>¬<em>r</em></sub> ; <em>F̃</em><sub><em>r</em></sub> =  − <em>k</em><sub><em>B</em></sub><em>T</em>log 𝒵<sub><em>r</em></sub>.   (36)</span><br /></span> With an effective energy in hand, we can write Eq. 35 (which is equivalent to the fold-change) in terms of the Boltzmann weights of these two coarse-grained states as <span id="eq:fc_boltz"><br /><span class="math display">$$
\text{fold-change} = {e^{-\beta \tilde{F}_{\neg r}} \over e^{-\beta \tilde{F}_{\neg r}} + e^{-\beta\tilde{F}_r}}.
\qquad(37)$$</span><br /></span> A simple rearrangement of this result produces a Fermi function <span id="eq:fermi_fn"><br /><span class="math display">$$
\text{fold-change} = {1 \over 1 + e^{-\beta \tilde{F}}},
\qquad(38)$$</span><br /></span> where <span class="math inline"><em>F̃</em></span> is the effective free energy of the system defined as <span id="eq:f_def"><br /><span class="math display"><em>F̃</em> =  − <em>k</em><sub><em>B</em></sub><em>T</em>(log𝒵<sub><em>r</em></sub>−log𝒵<sub>¬<em>r</em></sub>).   (39)</span><br /></span></p>
<p>     In the case of the simple repression motif, we apply the weak promoter approximation which is based on the fact that <span class="math inline">(<em>P</em>/<em>N</em><sub><em>N</em><em>S</em></sub>)<em>e</em><sup> − <em>β</em><em>Δ</em><em>ε</em><sub><em>P</em></sub></sup> ≪ 1</span>, reducing <span class="math inline">𝒵<sub>¬<em>r</em></sub> = 1</span>. With this approximation, the effective free energy <span class="math inline"><em>F̃</em></span> defined in Eq. 39 can be interpreted as the free energy between the repressed and transcriptionally active states. In general, this coarse-graining approach can be applied to any regulatory architecture in which the transcription factor bound states and transcriptionally active states share no overlapping substates. Fig. 23 shows various repression based architectures along with the coarse graining of states and the effective free energy.</p>
<p>      This approach cannot be applied to architectures in which the transcription factor bound and transcriptionally active states cannot be separated. One such example is the case of simple activation in which the binding of an activator increases gene expression. For this architecture, the total partition function <span class="citation" data-cites="bintu2005">(as derived in Bintu et al. <a href="#ref-bintu2005" role="doc-biblioref">2005</a>)</span> is <span id="eq:act_zed"><br /><span class="math display">$$
\mathcal{Z} = \overbrace{1 + {P \over N_{NS}}e^{-\beta\Delta\varepsilon_{P}}}^\text{activator unbound states} +
              \underbracee^{-\beta\Delta\varepsilon_A} + {A \over N_{NS}}{P \over N_{NS}}e^{-\beta(\Delta\varepsilon_{A} + 
              \Delta\varepsilon_{P} + \varepsilon_\text{interaction})}}_\text{activator bound states} = \mathcal{Z}_{\neg a} + \mathcal{Z}_a.
\qquad(40)$$</span><br /></span> Here we’ve used <span class="math inline"><em>A</em></span> to denote the number of activators, <span class="math inline"><em>Δ</em><em>ε</em><sub><em>A</em></sub></span> is the binding energy of the activator to its specific binding site, and <span class="math inline"><em>ε</em><sub>interaction</sub></span> is the interaction energy between the activator and polymerase which makes the activator-polymerase-DNA state more energetically favorable to the polymerase-DNA state. Unlike in the case of repression, the transcriptionally active states are present in the <span class="math inline">𝒵<sub><em>a</em></sub></span>. While we can compute the probability of the activator not being bound <span class="math inline"><em>P</em>(¬<em>a</em>)</span>, this is not equivalent to the fold-change in gene expression as was the case for simple repression. Rather, the fold-change in gene expression can be written as <span id="eq:unlabbed"><br /><span class="math display">$$
\text{fold-change} = {\mathcal{Z}_a -1 + \mathcal{Z}_{\neg a} - {A \over N_{NS}}e^{-\beta\Delta\varepsilon_{A}} \over \mathcal{Z}_{a} + \mathcal{Z}_{\neg a}}{\mathcal{Z}_a \over \mathcal{Z}_a - 1}.
\qquad(41)$$</span><br /></span> This is a more cumbersome expression than in the case for simple repression, and cannot be massaged into a one-parameter description of the fold-change. Thus, the analysis presented here is only applicable to cases in which the occupancy of the promoter by either the polymerase or the transcription factor are mutually exclusive.</p>
<figure>
<img src="ch7_figS23.png" id="fig:twostate_archs" data-short-caption="Various repression-based regulatory architectures and their coarse-grained states." alt="" /><figcaption>Figure 23: <strong>Various repression-based regulatory architectures and their coarse-grained states.</strong> The left column shows a schematic of the regulatory architecture. The middle column shows the equilibrium states of the system coarse grained into repressor bound (orange boxes) and repressor unbound (purple boxes) states. States in which the polymerase is bound is shaded in white and are neglected by the weak promoter approximation. The right column illustrates that the formulation of the effective free energy is the same for all architectures, although the formulation of <span class="math inline">𝒵<sub>¬<em>r</em></sub></span> and <span class="math inline">𝒵<sub><em>r</em></sub></span> are different between the examples. The bottom row illustrates the coarse graining of a simple repression motif in which the same repressor regulates multiple copies of the same gene. Rather than considering all states, the gene expression can calculated by considering one promoter and an effective copy number, described by the repressor fugacity <span class="math inline"><em>λ</em><sub><em>r</em></sub></span> and is derived in <span class="citation" data-cites="weinert2014">Weinert et al. (<a href="#ref-weinert2014" role="doc-biblioref">2014</a>)</span>. Grayed-out states illustrate this further level of coarse graining.</figcaption>
</figure>
<h2 id="strain-and-oligonucleotide-information">Strain and Oligonucleotide Information</h2>
<table>
<caption><em>Escherichia coli</em> strains used in this work</caption>
<colgroup>
<col style="width: 17%" />
<col style="width: 29%" />
<col style="width: 17%" />
<col style="width: 17%" />
<col style="width: 17%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;"><strong>Class</strong></th>
<th style="text-align: left;"><strong>LacI Mutant</strong></th>
<th style="text-align: left;"><strong>Operator</strong></th>
<th style="text-align: left;"><strong>Rep. per Cell</strong></th>
<th style="text-align: left;"><strong>Genotype</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">–</td>
<td style="text-align: left;">–</td>
<td style="text-align: left;">–</td>
<td style="text-align: left;">22</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacZYA</em></td>
</tr>
<tr class="even">
<td style="text-align: left;">–</td>
<td style="text-align: left;">–</td>
<td style="text-align: left;">O1</td>
<td style="text-align: left;">0</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O1+11-YFP</td>
</tr>
<tr class="odd">
<td style="text-align: left;">–</td>
<td style="text-align: left;">–</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">0</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP</td>
</tr>
<tr class="even">
<td style="text-align: left;">–</td>
<td style="text-align: left;">–</td>
<td style="text-align: left;">O3</td>
<td style="text-align: left;">0</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O3+11-YFP</td>
</tr>
<tr class="odd">
<td style="text-align: left;">WT</td>
<td style="text-align: left;">WT</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI</td>
</tr>
<tr class="even">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Y17I</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">60</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1147LacI(Y17I)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Y17I</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">124</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS446LacI(Y17I)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Y17I</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Y17I)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Y17I</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">1220</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1LacI(Y17I)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Q18A</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">60</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1147LacI(Q18A)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Q18A</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">124</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS446LacI(Q18A)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Q18A</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q18A)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Q18A</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">1220</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1LacI(Q18A)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Q18M</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">60</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1147LacI(Q18M)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Q18M</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">124</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS446LacI(Q18M)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Q18M</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q18M)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DNA</td>
<td style="text-align: left;">Q18M</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">1220</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1LacI(Q18M)</td>
</tr>
<tr class="even">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">F161T</td>
<td style="text-align: left;">O1</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O1+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(F161T)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">F161T</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(F161T)</td>
</tr>
<tr class="even">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">F161T</td>
<td style="text-align: left;">O3</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O3+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(F161T)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291V</td>
<td style="text-align: left;">O1</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O1+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291V)</td>
</tr>
<tr class="even">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291V</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291V)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291V</td>
<td style="text-align: left;">O3</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O3+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291V)</td>
</tr>
<tr class="even">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291K</td>
<td style="text-align: left;">O1</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O1+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291K)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291K</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291K)</td>
</tr>
<tr class="even">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291K</td>
<td style="text-align: left;">O3</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O3+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291K)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291R</td>
<td style="text-align: left;">O1</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O1+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291R)</td>
</tr>
<tr class="even">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291R</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291R)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IND</td>
<td style="text-align: left;">Q291R</td>
<td style="text-align: left;">O3</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O3+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q291R)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Y17I-F161T</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Y17IF161T)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Y17I-Q291V</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Y17IQ291V)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Y17I-Q291K</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Y17IQ291K)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Q18A-F161T</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q18AF161T)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Q18A-Q291V</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q18AQ291V)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Q18A-Q291K</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q18AQ291K)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Q18M-F161T</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q18MF161T)</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Q18M-Q291V</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q18MQ291V)</td>
</tr>
<tr class="even">
<td style="text-align: left;">DBL</td>
<td style="text-align: left;">Q18M-Q291K</td>
<td style="text-align: left;">O2</td>
<td style="text-align: left;">260</td>
<td style="text-align: left;">MG1655::<span class="math inline"><em>Δ</em></span><em>lacIZYA</em>; <em>galK</em>&lt;&gt;25O2+11-YFP; <em>ybcN</em>&lt;&gt;3*1-RBS1027LacI(Q18MQ291K)</td>
</tr>
</tbody>
</table>
<table style="width:100%;">
<caption>Oligonucleotides used for mutant generation.</caption>
<colgroup>
<col style="width: 30%" />
<col style="width: 23%" />
<col style="width: 23%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;"><strong>Primer Name</strong></th>
<th style="text-align: left;"><strong>Sequence (5' <span class="math inline">→</span> 3')</strong></th>
<th style="text-align: left;"><strong>Description</strong></th>
<th style="text-align: left;"><strong>Method</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">10.1</td>
<td style="text-align: left;"><code>acctctgcg gaggggaag cgtgaacctc tcacaagacg gcatcaaatt acactagca acaccagaac agc</code></td>
<td style="text-align: left;">Integration into <em>ybcN</em> locus</td>
<td style="text-align: left;"><span class="math inline"><em>λ</em></span>-Red Recombineering</td>
</tr>
<tr class="even">
<td style="text-align: center;">10.3</td>
<td style="text-align: left;"><code>ctgtagatgt gtccgttca tgacacgaat aagcggtgta gccattacg ccggctaat gcacccagt aagg</code></td>
<td style="text-align: left;">Integration into <em>ybcN</em> locus</td>
<td style="text-align: left;"><span class="math inline"><em>λ</em></span>-Red Recombineering</td>
</tr>
<tr class="odd">
<td style="text-align: center;">GCMWC-001</td>
<td style="text-align: left;"><code>ccggcatac tctgcgaca</code></td>
<td style="text-align: left;">Amplification of plasmid</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="even">
<td style="text-align: center;">GCMWC-002</td>
<td style="text-align: left;"><code>gtgtctctta tATGaccgt ttcccgc</code></td>
<td style="text-align: left;">Q18M Mutation (<code>CAG</code><span class="math inline">→</span><code>ATG</code>)</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="odd">
<td style="text-align: center;">GCMWC-003</td>
<td style="text-align: left;"><code>tgtctcttat GCGaccgttt cccgc</code></td>
<td style="text-align: left;">Q18A Mutation (<code>CAG</code><span class="math inline">→</span><code>GCG</code>)</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="even">
<td style="text-align: center;">GCMWC-004</td>
<td style="text-align: left;"><code>gttaacggcg ggatataac</code></td>
<td style="text-align: left;">Amplification of plasmid</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="odd">
<td style="text-align: center;">GCMWC-005</td>
<td style="text-align: left;"><code>caccatcaaa GTGgatttt cgcctgc</code></td>
<td style="text-align: left;">Q291V Mutation (<code>CAG</code> <span class="math inline">→</span> <code>GTG</code>)</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="even">
<td style="text-align: center;">GCMWC-006</td>
<td style="text-align: left;"><code>caccatcaaa AAGgattttcg cc</code></td>
<td style="text-align: left;">Q291K Mutation (<code>CAG</code><span class="math inline">→</span><code>AAG</code>)</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="odd">
<td style="text-align: center;">GCMWC-007</td>
<td style="text-align: left;"><code>cagtattatt ACCtcccatga agacgg</code></td>
<td style="text-align: left;">F161T Mutation (<code>TTC</code><span class="math inline">→</span><code>ACC</code>)</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="even">
<td style="text-align: center;">GCMWC-008</td>
<td style="text-align: left;"><code>ttgatgggtg tctggtcag</code></td>
<td style="text-align: left;">Amplification of plasmid</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="odd">
<td style="text-align: center;">GCMWC-009</td>
<td style="text-align: left;"><code>gcatactctg cgacatcgta taa</code></td>
<td style="text-align: left;">Amplification of plasmid</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="even">
<td style="text-align: center;">GCMWC-010</td>
<td style="text-align: left;"><code>cggtgtctct ATTcagaccg tttc</code></td>
<td style="text-align: left;">Y17I Mutation (<code>TAT</code><span class="math inline">→</span><code>ATT</code>)</td>
<td style="text-align: left;">QuickChange Mutagenesis</td>
</tr>
<tr class="odd">
<td style="text-align: center;">GCMWC-017</td>
<td style="text-align: left;"><code>ccatcaaaAG Ggattttcgc ctgctggg gcaaaccag</code></td>
<td style="text-align: left;">Q291R Mutation (<code>CAG</code><span class="math inline">→</span><code>AGG</code>)</td>
<td style="text-align: left;">Gibson Assembly</td>
</tr>
<tr class="even">
<td style="text-align: center;">GCMWC-018</td>
<td style="text-align: left;"><code>ggcgaaaatc CCTtttgatg gtggttaa cggcggg</code></td>
<td style="text-align: left;">Q291R Mutation (<code>CTG</code><span class="math inline">→</span><code>CCT</code>)</td>
<td style="text-align: left;">Gibson Assembly</td>
</tr>
</tbody>
</table>
<h2 class="unnumbered" id="references" class="unnumbered">References</h2>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-bintu2005">
<p>Bintu, Lacramioara, Nicolas E Buchler, Hernan G Garcia, Ulrich Gerland, Terence Hwa, Jané Kondev, and Rob Phillips. 2005. “Transcriptional Regulation by the Numbers: Models.” <em>Current Opinion in Genetics &amp; Development</em> 15 (2): 116–24. <a href="https://doi.org/10.1016/j.gde.2005.02.007">https://doi.org/10.1016/j.gde.2005.02.007</a>.</p>
</div>
<div id="ref-brewster2014">
<p>Brewster, Robert C., Franz M. Weinert, Hernan G. Garcia, Dan Song, Mattias Rydenfelt, and Rob Phillips. 2014. “The Transcription Factor Titration Effect Dictates Level of Gene Expression.” <em>Cell</em> 156 (6): 1312–23. <a href="https://doi.org/10.1016/j.cell.2014.02.022">https://doi.org/10.1016/j.cell.2014.02.022</a>.</p>
</div>
<div id="ref-carpenter2017">
<p>Carpenter, Bob, Andrew Gelman, Matthew D. Hoffman, Daniel Lee, Ben Goodrich, Michael Betancourt, Marcus Brubaker, Jiqiang Guo, Peter Li, and Allen Riddell. 2017. “Stan: A Probabilistic Programming Language.” <em>Journal of Statistical Software</em> 76 (1): 1–32. <a href="https://doi.org/10.18637/jss.v076.i01">https://doi.org/10.18637/jss.v076.i01</a>.</p>
</div>
<div id="ref-daber2009a">
<p>Daber, Robert, Kim Sharp, and Mitchell Lewis. 2009. “One Is Not Enough.” <em>Journal of Molecular Biology</em> 392 (5): 1133–44. <a href="https://doi.org/10.1016/j.jmb.2009.07.050">https://doi.org/10.1016/j.jmb.2009.07.050</a>.</p>
</div>
<div id="ref-daber2011a">
<p>Daber, Robert, Matthew A. Sochor, and Mitchell Lewis. 2011. “Thermodynamic Analysis of Mutant Lac Repressors.” <em>Journal of Molecular Biology</em>, The Operon Model and its Impact on Modern Molecular Biology, 409 (1): 76–87. <a href="https://doi.org/10.1016/j.jmb.2011.03.057">https://doi.org/10.1016/j.jmb.2011.03.057</a>.</p>
</div>
<div id="ref-garcia2011">
<p>Garcia, H. G., and R. Phillips. 2011. “Quantitative Dissection of the Simple Repression Input-Output Function.” <em>Proceedings of the National Academy of Sciences</em> 108 (29): 12173–8. <a href="https://doi.org/10.1073/pnas.1015616108">https://doi.org/10.1073/pnas.1015616108</a>.</p>
</div>
<div id="ref-gilbert1966">
<p>Gilbert, Walter, and Benno Müller-Hill. 1966. “Isolation of the Lac Repressor.” <em>Proceedings of the National Academy of Sciences of the United States of America</em> 56 (6): 1891–8.</p>
</div>
<div id="ref-good1950">
<p>Good, Isadore J. 1950. <em>Probability and the Weighting of Evidence</em>. New York City.</p>
</div>
<div id="ref-milo2010">
<p>Milo, Ron, Paul Jorgensen, Uri Moran, Griffin Weber, and Michael Springer. 2010. “BioNumbersthe Database of Key Numbers in Molecular and Cell Biology.” <em>Nucleic Acids Research</em> 38 (suppl_1): D750–D753. <a href="https://doi.org/10.1093/nar/gkp889">https://doi.org/10.1093/nar/gkp889</a>.</p>
</div>
<div id="ref-ogorman1980">
<p>O’Gorman, R. B., J. M. Rosenberg, O. B. Kallai, R. E. Dickerson, K. Itakura, A. D. Riggs, and K. S. Matthews. 1980. “Equilibrium Binding of Inducer to Lac Repressor Operator DNA Complex.” <em>Journal of Biological Chemistry</em> 255 (21): 10107–14.</p>
</div>
<div id="ref-razo-mejia2018">
<p>Razo-Mejia, Manuel, Stephanie L. Barnes, Nathan M. Belliveau, Griffin Chure, Tal Einav, Mitchell Lewis, and Rob Phillips. 2018. “Tuning Transcriptional Regulation Through Signaling: A Predictive Theory of Allosteric Induction.” <em>Cell Systems</em> 6 (4): 456–469.e10. <a href="https://doi.org/10.1016/j.cels.2018.02.004">https://doi.org/10.1016/j.cels.2018.02.004</a>.</p>
</div>
<div id="ref-talts2018">
<p>Talts, Sean, Michael Betancourt, Daniel Simpson, Aki Vehtari, and Andrew Gelman. 2018. “Validating Bayesian Inference Algorithms with Simulation-Based Calibration.” <em>arXiv:1804.06788 [Stat]</em>, April. <a href="http://arxiv.org/abs/1804.06788">http://arxiv.org/abs/1804.06788</a>.</p>
</div>
<div id="ref-weinert2014">
<p>Weinert, Franz M., Robert C. Brewster, Mattias Rydenfelt, Rob Phillips, and Willem K. Kegel. 2014. “Scaling of Gene Expression with Transcription-Factor Fugacity.” <em>Physical Review Letters</em> 113 (25). <a href="https://doi.org/10.1103/PhysRevLett.113.258101">https://doi.org/10.1103/PhysRevLett.113.258101</a>.</p>
</div>
</div>
						</div><!-- /.content -->
					</div><!-- /.col -->
					<div class="col-md-4 col-md-offset-1">
						<div class="sections-list-wrapper">
							<div class="sections-list js-sections js-affix js-scrollspy hidden-xs hidden-sm"><span style="font-size: 14pt">Table of Contents</span><hr></div><!-- /.sections-list -->
						</div>
					</div><!-- /.col -->
				</div><!-- /.row -->
			</div><!-- /.container -->
		</div><!-- /.section -->
		
			<div class="section section--grey">
				<div class="container">
					<div class="row">
						<div class="col-md-7">
							<div id="disqus_thread"></div>
<script>
	/**
	 *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
	 *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
	 */
	 /*
	var disqus_config = function () {
	this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
		this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
	};
	*/
	(function() {
		var d = document, s = d.createElement('script');

		s.src = 'https://gchure-phd.disqus.com/embed.js';

		s.setAttribute('data-timestamp', +new Date());
		(d.head || d.body).appendChild(s);
	})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>

						</div><!-- /.col -->
					</div><!-- /.row -->
				</div><!-- /.container -->
			</div><!-- /.section -->
		
		<div class="js-footer-area">
			
				<nav class="page-nav">	
					<div class="container">
						<div class="row">
							<div class="col-xs-12">
								
									<a href="https://gchure.github.io/phd/chapter_06" class="page-nav__item page-nav__item--prev">
										<i class="icon icon--arrow-left"></i>
										Chapter VI
									</a><!-- /.page-nav__item -->
								
								
									<a href="https://gchure.github.io/phd/chapter_08" class="page-nav__item page-nav__item--next">
										Chapter VIII
										<i class="icon icon--arrow-right"></i>
									</a><!-- /.page-nav__item -->
								
							</div><!-- /.col -->
						</div><!-- /.row -->
					</div><!-- /.container -->
				</nav><!-- /.page-nav -->
			
			
			
	<footer class="site-footer">
		<div class="container">
			<div class="row">
				<div class="col-sm-6">
					
						<a href="https://gchure.github.io/phd//" class="site-footer__logo">Griffin D. Chure | PhD Thesis</a>
					
					
						<hr>
						<p class="site-footer__copyright">Copyright &copy; 2020. - Griffin D. Chure <br>All rights reserved.</p>
					
				</div><!-- /.col -->
				
					<div class="col-sm-6 align-right">
						<ul class="social-list">
							
								<li>
									<a href="https://twitter.com/gdchure" target="_blank" class="social-list__item social-list__item--twitter">
										<i class="icon icon--twitter"></i>
									</a>
								</li>
							
						</ul><!-- /.social-list -->
					</div><!-- /.col -->
				
			</div><!-- /.row -->
		</div><!-- /.container -->
	</footer><!-- /.site-footer -->


<script src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/affix.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/bootstrap/scrollspy.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/vendor/matchHeight.min.js"></script>
<script type="text/javascript" src="https://gchure.github.io/phd//doks-theme/assets/js/scripts.min.js"></script>




		</div><!-- /.js-footer-area -->
	</body>
</html>
