## Introduction
Thermodynamic treatments of transcriptional regulation have been fruitful in
their ability to generate quantitative predictions of gene expression as a
function of a minimal set of physically meaningful parameters [@ackers1982;
@buchler2003; @vilar2003; @garcia2011; @daber2009a; @brewster2014;
@weinert2014; @rydenfelt2014b; @razo-mejia2014; @razo-mejia2018; @bintu2005;
@bintu2005a; @kuhlman2007]. These models quantitatively describe numerous
properties of input-output functions, such as the leakiness, saturation,
dynamic range, steepness of response, and the $[EC_{50}]$ â€“ the concentration
of inducer at which the response is half maximal. The mathematical forms of
these phenotypic properties are couched in terms of a minimal set of
experimentally accessible variables, such as the inducer concentration,
transcription factor copy number, and the DNA sequence of the binding site
(see Chapter 2 and @razo-mejia2018). While the amino acid sequence of the transcription factor
is another controllable variable, it is seldom implemented in quantitative
terms considering mutations with subtle changes in chemistry frequently yield
unpredictable physiological consequences. In this work, we examine how a
series of mutations in either the DNA binding or inducer binding domains of a
transcriptional repressor influence the values of the biophysical parameters
which govern its regulatory behavior.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We build upon the results presented in Chapter
2 of this thesis and present a theoretical framework for understanding how
mutations in the amino acid sequence of the repressor affect different
parameters and alter the free energy of the system. We find that the
parameters capturing the allosteric nature of the repressor, the repressor
copy number, and the DNA binding specificity contribute independently to the
free energy of the system with different degrees of sensitivity. Furthermore,
changes restricted to one of these three groups of parameters result in
characteristic changes in the free energy relative to the wild-type
repressor, providing falsifiable predictions of how different classes of
mutations should behave.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Next, we test these descriptions experimentally
using the well-characterized transcriptional repressor of the *lac* operon
LacI in
*E. coli* regulating expression of a fluorescent reporter. We introduce
a series of point mutations in either the inducer binding or DNA binding
domain. We then measure the full induction profile of each mutant, determine
the minimal set of parameters that are affected by the mutation, and predict
how each mutation tunes the free energy at different inducer concentrations,
repressor copy numbers, and DNA binding strengths. We find in general that
mutations in the DNA binding domain only influence DNA binding strength, and
that mutations within the inducer binding domain affect only the parameters
which dictate the allosteric response. The degree to which these parameters
are insulated is notable, as the very nature of allostery suggests that all
parameters are intimately connected, thus enabling binding events at
one domain to be "sensed" by another.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;With knowledge of how a collection of DNA
binding and inducer binding single mutants behave, we predict the induction
profiles and the free energy changes of pairwise double mutants with
quantitative accuracy. We find that the energetic effects of each individual
mutation are additive, indicating that epistatic interactions are absent
between the mutations examined here. Our model provides a means for
identifying and quantifying the extent of epistatic interactions in a more
complex set of mutations, and can shed light on how the protein sequence and
general regulatory architecture coevolve.
